LETTER
doi:10.1038/nature14236
# Human-level control through deep reinforcement learning
Volodymyr Mnih'*, Koray Kavukcuoglu'*, David Silver'*, Andrei A. Rusu’, Joel Veness', Marc G. Bellemare!, Alex Graves', Martin Riedmiller!, Andreas K. Fidjeland’, Georg Ostrovski', Stig Petersen’, Charles Beattie’, Amir Sadik’, Ioannis Antonoglou’, Helen King’, Dharshan Kumaran’, Daan Wierstra', Shane Legg' & Demis Hassabis'
The theory of reinforcement learning provides a normative account’,
deeply rooted in psychological’ and neuroscientific’ perspectives on animal behaviour, of how agents may optimize their control of an environment. To use reinforcement learning successfully in situations approaching real-world complexity, however, agents are confronted with a difficult task: they must derive efficient representations of the environment from high-dimensional sensory inputs, and use these to generalize past experience to new situations. Remarkably, humans and other animals seem to solve this problem through a harmonious combination of reinforcement learning and hierarchical sensory pro- cessing systems"”, the former evidenced by a wealth of neural data revealing notable parallels between the phasic signals emitted by dopa- minergic neurons and temporal difference reinforcement learning algorithms’. While reinforcement learning agents have achieved some successes in a variety of domains**, their applicability has previously been limited to domains in which useful features can be handcrafted, or to domains with fully observed, low-dimensional state spaces. Here we use recent advances in training deep neural networks”! to develop a novel artificial agent, termed a deep Q-network, that can learn successful policies directly from high-dimensional sensory inputs using end-to-end reinforcement learning. We tested this agent on the challenging domain of classic Atari 2600 games'*. We demon- strate that the deep Q-network agent, receiving only the pixels and the game score as inputs, was able to surpass the performance of all previous algorithms and achieve a level comparable to that of a pro- fessional human games tester across a set of 49 games, using the same algorithm, network architecture and hyperparameters. This work bridges the divide between high-dimensional sensory inputs and actions, resulting in the first artificial agent that is capable of learn- ing to excel at a diverse array of challenging tasks.
We set out to create a single algorithm that would be able to develop wide range of competencies on a varied range of challenging tasks—a central goal of general artificial intelligence"* that has eluded previous efforts*'*"°, To achieve this, we developed a novel agent, a deep Q-network (DQN), which is able to combine reinforcement learning with a class of artificial neural network'“ known as deep neural networks. Notably, recent advances in deep neural networks”, in which several layers of nodes are used to build up progressively more abstract representations of the data, have made it possible for artificial neural networks to learn concepts such as object categories directly from raw sensory data. We use one particularly successful architecture, the deep convolutional network”, which uses hierarchical layers of tiled convolutional filters to mimic the effects of receptive fields—inspired by Hubel and Wiesel’s seminal work on feedforward processing in early visual cortex'*—thereby exploiting the local spatial correlations present in images, and building in robustness to natural transformations such as changes of viewpoint or scale.
# a
agent is to select actions in a fashion that maximizes cumulative future reward. More formally, we use a deep convolutional neural network to approximate the optimal action-value function
QO*(s,a)= maxEln tyres tinea SOR t=, a, =a, 7],
which is the maximum sum of rewards r; discounted by y at each time- step t, achievable by a behaviour policy z = P(a|s), after making an observation (s) and taking an action (a) (see Methods)".
Reinforcement learning is known to be unstable or even to diverge when a nonlinear function approximator such as a neural network is used to represent the action-value (also known as O) function”. This instability has several causes: the correlations present in the sequence of observations, the fact that small updates to O may significantly change the policy and therefore change the data distribution, and the correlations between the action-values (Q) and the target values r 4-7 max Q(s', a’). Weaddress these instabilities with a novel variant of Q-learning, which uses two key ideas. First, we used a biologically inspired mechanism termed experience replay” ” that randomizes over the data, thereby removing correlations in the observation sequence and smoothing over changes in the data distribution (see below for details). Second, we used an iterative update that adjusts the action-values (Q) towards target values that are only periodically updated, thereby reducing correlations with the target.
While other stable methods exist for training neural networks in the reinforcement learning setting, such as neural fitted O-iteration”, these methods involve the repeated training of networks de novo on hundreds of iterations. Consequently, these methods, unlike our algorithm, are too inefficient to be used successfully with large neural networks. We parameterize an approximate value function Q(s,a;0;) using the deep convolutional neural network shown in Fig. 1, in which 6; are the param- eters (that is, weights) of the Q-network at iteration i. To perform experience replay we store the agent's experiences e, = (sp@pTp5;+ 1) at each time-step t in a data set D, = {e),...,e;}. During learning, we apply Q-learning updates, on samples (or minibatches) of experience (s,a,r,s’) ~ U(D), drawn uniformly at random from the pool of stored samples. The Q-learning update at iteration i uses the following loss function:
L;(0;) =Eis,a,1,¢)~u(D) (r+ymas Os',a'; 05 )— Olsa; ») |
in which y is the discount factor determining the agent’s horizon, 6; are the parameters of the Q-network at iteration iand 0; are the network parameters used to compute the target at iteration i. The target net- work parameters 0; are only updated with the Q-network parameters (0;) every C steps and are held fixed between individual updates (see Methods).
We consider tasks in which the agent interacts with an environment through a sequence of observations, actions and rewards. The goal of the
To evaluate our DQN agent, we took advantage of the Atari 2600 platform, which offers a diverse array of tasks (n = 49) designed to be
1Google DeepMind, 5 New Street Square, London EC4A 3TW, UK. *These authors contributed equally to this work,
26 FEBRUARY 2015 | VOL 518 | NATURE | 529
©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Convolution Convolution Bİ E| © B E = Fully connected Fully connected \ ft . } j i i j } <= : 9 un : GE) j ; t : EE . c } : :/ m 2 an)
Figure 1 | Schematic illustration of the convolutional neural network. The details of the architecture are explained in the Methods. The input to the neural network consists of an 84 X 84 X 4 image produced by the preprocessing map 4, followed by three convolutional layers (note: snaking blue line
symbolizes sliding of each filter across input image) and two fully connected layers with a single output for each valid action. Each hidden layer is followed by a rectifier nonlinearity (that is, max(0,x)).
difficult and engaging for human players. We used the same network architecture, hyperparameter values (see Extended Data Table 1) and learning procedure throughout—taking high-dimensional data (210 x 160 colour video at 60 Hz) as input—to demonstrate that our approach robustly learns successful policies over a variety of games based solely on sensory inputs with only very minimal prior knowledge (that is, merely the input data were visual images, and the number of actions available in each game, but not their correspondences; see Methods). Notably, our method was able to train large neural networks using a reinforce- ment learning signal and stochastic gradient descent in a stable manner— illustrated by the temporal evolution of two indices of learning (the agent’s average score-per-episode and average predicted Q-values; see Fig. 2 and Supplementary Discussion for details).
We compared DQN with the best performing methods from the reinforcement learning literature on the 49 games where results were available'*"’. In addition to the learned agents, we also report scores for a professional human games tester playing under controlled conditions and a policy that selects actions uniformly at random (Extended Data Table 2 and Fig. 3, denoted by 100% (human) and 0% (random) on y axis; see Methods). Our DQN method outperforms the best existing reinforcement learning methods on 43 of the games without incorpo- rating any of the additional prior knowledge about Atari 2600 games used by other approaches (for example, refs 12, 15). Furthermore, our DQN agent performed at a level that was comparable to that of a pro- fessional human games tester across the set of 49 games, achieving more than 75% of the human score on more than half of the games (29 games;
2,200 2,000 1,800 1,600 1,400 1,200 1,000 800 600 400 3 ° 9-0 4060 80 100 120 140 160 180 200 Training epochs Average score per episode © o Average action value (0) osnesaosecd qo _ 10 gS 9 2 8 s 7 5 6 3 5 8 2 4 2 3 z 2 1 o b 6,000 5,000 4,000 3,000 2,000 1,000 Average score per episode 0 0 20 40 60 80 100 120 140 160 180 200 Training epochs 0 20 40 60 80 100 120 140 160 180 200 Training epochs 0 20 40 60 80 100 120 140 160 180 200 Training epochs
Figure 2 | Training curves tracking the agent’s average score and average predicted action-value. a, Each point is the average score achieved per episode after the agent is run with ¢-greedy policy (e = 0.05) for 520k frames on Space Invaders. b, Average score achieved per episode for Seaquest. c, Average predicted action-value on a held-out set of states on Space Invaders. Each point
on the curve is the average of the action-value Q computed over the held-out set of states. Note that Q-values are scaled due to clipping of rewards (see Methods). d, Average predicted action-value on Seaquest. See Supplementary Discussion for details.
530 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Video Pinball Boxing Breakout Star Gunner Robotank Atlantis Crazy Climber Gopher Demon Attack Name This Game Krull Assault Road Runner Kangaroo James Bond Tennis Pong Space Invaders. Beam Rider Tutankham Kung-Fu Master Freeway Time Pilot Enduro Fishing Derby Up and Down Ice Hockey Q*bert H.E.R.O. Lil 1) A A 1 SE Ry ÜN GİNE Sl DM GE GANİ GM ÖN > At human-level or above Asterix Battle Zone Wizard of Wor Chopper Command Centipede Bank Heist River Raid Zaxxon Amidar Alien Venture Seaquest Double Dunk Bowling Ms. Pac-Man Asteroids Frostbite Gravitar Private Eye | Montezuma's Revenge 1 Jo AR DM A SN GE ANİ GÖ > cr RS ÖR e pT iil. m Fy 3 Below human-level Best linear learner Tİ İm
# t
0
# T
100
# T
200
# T
300
T 400
# T
500
600
1,000
1 4,500%
Figure 3 | Comparison of the DON agent with the best reinforcement learning methods" in the literature. The performance of DON is normalized with respect to a professional human games tester (that is, 100% level) and random play (that is, 0% level). Note that the normalized performance of DON, expressed as a percentage, is calculated as: 100 X (DQN score — random play score)/(human score — random play score). It can be seen that DON
outperforms competing methods (also see Extended Data Table 2) in almost all the games, and performs at a level that is broadly comparable with or superior to a professional human games tester (that is, operationalized as a level of 75% or above) in the majority of games. Audio output was disabled for both human players and agents. Error bars indicate s.d. across the 30 evaluation episodes, starting with different initial conditions.
see Fig. 3, Supplementary Discussion and Extended Data Table 2). In additional simulations (see Supplementary Discussion and Extended Data Tables 3 and 4), we demonstrate the importance of the individual core components of the DQN agent—the replay memory, separate target Q-network and deep convolutional network architecture—by disabling them and demonstrating the detrimental effects on performance.
We next examined the representations learned by DQN that under- pinned the successful performance of the agent in the context of the game Space Invaders (see Supplementary Video 1 for a demonstration of the performance of DQN), by using a technique developed for the visual- ization of high-dimensional data called ‘t-SNE” (Fig. 4). As expected, the t-SNE algorithm tends to map the DQN representation of percep- tually similar states to nearby points. Interestingly, we also found instances in which the t-SNE algorithm generated similar embeddings for DQN representations of states that are close in terms of expected reward but
perceptually dissimilar (Fig. 4, bottom right, top left and middle), con- sistent with the notion that the network is able to learn representations that support adaptive behaviour from high-dimensional sensory inputs. Furthermore, we also show that the representations learned by DQN are able to generalize to data generated from policies other than its own—in simulations where we presented as input to the network game states experienced during human and agent play, recorded the repre- sentations of the last hidden layer, and visualized the embeddings gen- erated by the t-SNE algorithm (Extended Data Fig. 1 and Supplementary Discussion). Extended Data Fig. 2 provides an additional illustration of how the representations learned by DQN allow it to accurately predict state and action values.
It is worth noting that the games in which DON excels are extremely varied in their nature, from side-scrolling shooters (River Raid) to box- ing games (Boxing) and three-dimensional car-racing games (Enduro).
26 FEBRUARY 2015 | VOL 518 | NATURE | 531
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER

Figure 4 | Two-dimensional t-SNE embedding of the representations in the last hidden layer assigned by DQN to game states experienced while playing Space Invaders. The plot was generated by letting the DQN agent play for 2 hofreal game time and running the t-SNE algorithm” on the last hidden layer representations assigned by DQN to each experienced game state. The points are coloured according to the state values (V, maximum expected reward of a state) predicted by DQN for the corresponding game states (ranging from dark red (highest V) to dark blue (lowest V)). The screenshots corresponding to a selected number of points are shown. The DQN agent
predicts high state values for both full (top right screenshots) and nearly complete screens (bottom left screenshots) because it has learned that completing a screen leads to a new screen full of enemy ships. Partially completed screens (bottom screenshots) are assigned lower state values because less immediate reward is available. The screens shown on the bottom right and top left and middle are less perceptually similar than the other examples but are still mapped to nearby representations and similar values because the orange bunkers do not carry great significance near the end of a level. With permission from Square Enix Limited.
Indeed, in certain games DQN is able to discover a relatively long-term strategy (for example, Breakout: the agent learns the optimal strategy, which is to first dig a tunnel around the side of the wall allowing the ball to be sent around the back to destroy a large number of blocks; see Sup- plementary Video 2 for illustration of development of DQN’s perfor- mance over the course of training). Nevertheless, games demanding more temporally extended planning strategies still constitute a major chal- lenge for all existing agents including DQN (for example, Montezuma’s Revenge).
In this work, we demonstrate that a single architecture can success- fully learn control policies in a range of different environments with only very minimal prior knowledge, receiving only the pixels and the game score as inputs, and using the same algorithm, network architecture and hyperparameters on each game, privy only to the inputs a human player would have. In contrast to previous work****, our approach incorpo- rates ‘end-to-end’ reinforcement learning that uses reward to continu- ously shape representations within the convolutional network towards salient features of the environment that facilitate value estimation. This principle draws on neurobiological evidence that reward signals during perceptual learning may influence the characteristics of representations within primate visual cortex” ”*, Notably, the successful integration of reinforcement learning with deep network architectures was critically dependent on our incorporation ofa replay algorithm?' * involving the storage and representation of recently experienced transitions. Conver- gent evidence suggests that the hippocampus may support the physical
realization of such a process in the mammalian brain, with the time- compressed reactivation of recently experienced trajectories during offline periods”! (for example, waking rest) providing a putative mech- anism by which value functions may be efficiently updated through interactions with the basal ganglia’. In the future, it will be important to explore the potential use of biasing the content of experience replay towards salient events, a phenomenon that characterizes empirically observed hippocampal replay”, and relates to the notion of ‘prioritized sweeping” in reinforcement learning. Taken together, our work illus- trates the power of harnessing state-of-the-art machine learning tech- niques with biologically inspired mechanisms to create agents that are capable of learning to master a diverse array of challenging tasks.
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
# Received 10 July 2014; accepted 16 January 2015.
- Sutton, R. & Barto, A. Reinforcement Learning: An Introduction (MIT Press, 1998).
# Ye
- Thorndike, E. L. Animal Intelligence: Experimental studies (Macmillan,
1911).
- Schultz, W., Dayan, P. & Montague, P. R. A neural substrate of prediction and reward. Science 275, 1593-1599 (1997).
- Serre, T., Wolf, L. & Poggio, T. Object recognition with features inspired by visual cortex. Proc. IEEE. Comput. Soc. Conf. Comput. Vis. Pattern. Recognit 994-1000 (2005). p
- Fukushima, K. Neocognitron: A self-organizing neural network model for a mechanism of pattern recognition unaffected by shift in position. Biol. Cybern. 36, 193-202 (1980).
5.
532 | NATURE | VOL 518 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# m jar
- sem Tesauro, G. Temporal difference learning and TD-Gammon. Commun. ACM 38, 58-68 (1995).
- Riedmiller, M., Gabel, T., Hafner, R. & Lange, S. Reinforcement learning for robot soccer, Auton, Robots 27, 55-73 (2009).
- Diuk, C., Cohen, A. & Littman, M. L. An object-oriented representation for efficient reinforcement learning. Proc. Int. Conf. Mach. Learn. 240-247 (2008).
- Bengio, Y. Learning deep architectures for Al. Foundations and Trends in Machine Leaming 2, 1-127 (2009).
- . Krizhevsky, A., Sutskever, |. & Hinton, G. ImageNet classification with deep convolutional neural networks. Adv. Neural Inf. Process. Syst. 25, 1106-1114 (2012).
- 11. Hinton, G. E. & Salakhutdinov, R. R. Reducing the dimensionality of data with neural networks. Science 313, 504-507 (2006).
- 23. Lin, L.-J. Reinforcement learning for robots using neural networks. Technical Report, DTIC Document (1993).
- 24, Riedmiller, M. Neural fitted Q iteration - first experiences with a data efficient neural reinforcement learning method. Mach. Leam.: ECML, 3720, 317-328 (Springer, 2005).
- 25. Van der Maaten, L. J. P. & Hinton, G. E, Visualizing high-dimensional data using t-SNE. J. Mach. Learn. Res. 9, 2579-2605 (2008).
- 26. Lange, S. & Riedmiller, M. Deep auto-encoder neural networks in reinforcement learning. Proc. Int. Jt. Conf. Neural. Netw. 1-8 (2010).
- 27. Law, C.-T. & Gold, J. |. Reinforcement learning can account for associative and perceptual learning on a visual decision task. Nature Neurosci. 12, 655 (2009).
- 12: Bellemare, M. G., Naddaf, Y., Veness, J. & Bowling, M. The arcade learning environment: An evaluation platform for general agents. J. Artif. Intell. Res. 47, 253-279 (2013).
- 13. Legg, S. & Hutter, M. Universal Intelligence: a definition of machine intelligence. Minds Mach. 17, 391-444 (2007).
- 14. Genesereth, M., Love, N. & Pell, B. General game playing: overview of the AAAI competition. Al Mag. 26, 62-72 (2005).
- 28. Sigala, N. & Logothetis, N. K. Visual categorization shapes feature selectivity in the primate temporal cortex. Nature 415, 318-320 (2002).
- 29. Bendor,D.& Wilson, M.A. Biasing the content of hippocampal replay during sleep. Nature Neurosci. 15, 1439-1444 (2012).
- 30. Moore, A. & Atkeson, C. Prioritized sweeping: reinforcement learning with less data and less real time. Mach. Learn. 13, 103-130 (1993).
- 15. Bellemare, M. G., Veness, J. & Bowling, M. Investigating contingency awareness using Atari 2600 games. Proc. Conf. AAAI. Artif. Intell. 864-871 (2012).
- 16. McClelland, J. L., Rumelhart, D. E. & Group, T. P. R. Parallel Distributed Processing: Explorations in the Microstructure of Cognition (MIT Press, 1986).
- 17. LeCun, Y., Bottou, L., Bengio, Y. & Haffner, P. Gradient-based learning applied to document recognition. Proc. IEEE 86, 2278-2324 (1998).
Supplementary Information is available in the online version of the paper.
Acknowledgements We thank G. Hinton, P. Dayan and M. Bowling for discussions, A. Cain and J. Keene for work on the visuals, K. Keller and P. Rogers for help with the visuals, G. Wayne for comments on an earlier version of the manuscript, and the rest of the DeepMind team for their support, ideas and encouragement.
- 18. Hubel, D. H. & Wiesel, T. N. Shape and arrangement of columns in cat's striate cortex. J. Physiol. 165, 559-568 (1963).
- 19. Watkins, C. J. & Dayan, P. Q-learning. Mach, Learn. 8, 279-292 (1992).
- 20. Tsitsiklis, J. & Roy, B. V. An analysis of temporal-difference learning with function approximation. IEEE Trans. Automat. Contr. 42, 674-690 (1997).
Author Contributions V.M., K.K. D.S, J.V., M.G.B,, M.R, AG, DM, S.L. and D.H. conceptualized the problem and the technical framework. V.M., K.K., AAR, and D.S. developed and tested the algorithms. J.V., S.P., C.B., AAR,M.G.B,LA,,A.K.F. G.O. and AS. created the testing platform. K.K., H.K, S.L.and D.H. managed the project. K.K.,D.K., DH., V.M., D.S., AG., AAR,, J.V. and M.G.B. wrote the paper.
- 21. McClelland, J. L,, McNaughton, B. L. & O'Reilly, R. C. Why there are complementary learning systems in the hippocampus and neocortex: insights from the successes and failures of connectionist models of learning and memory. Psychol. Rev. 102, 419-457 (1995).
- 22; O'Neill, J., Pleydeli-Bouverie, B., Dupret, D. & Csicsvari, J. Play it again: reactivation of waking experience and memory. Trends Neurosci. 33, 220-229 (2010).
Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to K.K. (korayk@google.com) or D.H. (demishassabis@google.com).
26 FEBRUARY 2015 | VOL 518 | NATURE | 533
©2015 Macmillan Publishers Limited. All rights reserved
LETTER
doi:10.1038/nature14000
# Evolution of the new vertebrate head by co-option of an ancient chordate skeletal tissue
David Jandzik':**, Aaron T. Garnett'*, Tyler A. Square’, Maria V. Cattell’, Jr-Kai Yu? & Daniel M. Medeiros!
A defining feature of vertebrates (craniates) is a pronounced head that is supported and protected by a robust cellular endoskeleton. In the first vertebrates, this skeleton probably consisted of collage- nous cellular cartilage, which forms the embryonic skeleton of all ver- tebrates and the adult skeleton of modern jawless and cartilaginous fish. In the head, most cellular cartilage is derived from a migratory cell population called the neural crest, which arises from the edges of the central nervous system. Because collagenous cellular cartilage and neural crest cells have not been described in invertebrates’, the appearance of cellular cartilage derived from neural crest cells is con- sidered a turning point in vertebrate evolution”. Here we show that a tissue with many of the defining features of vertebrate cellular cartilage transiently forms in the larvae of the invertebrate chordate Branchiostoma floridae (Florida amphioxus). We also present evi- dence that during evolution, a key regulator of vertebrate cartilage development, SoxE, gained new cis-regulatory sequences that sub- sequently directed its novel expression in neural crest cells. Together, these results suggest that the origin of the vertebrate head skeleton did not depend on the evolution of a new skeletal tissue, as is com- monly thought, but on the spread of this tissue throughout the head. We further propose that the evolution of cis-regulatory elements near an ancient regulator of cartilage differentiation was a major factor in the evolution of the vertebrate head skeleton.
cartilage in adults, at embryonic stages the head skeletons of all living vertebrates consist ofa single type of histologically distinct cellular carti- lage. This embryonic cartilage consists of tightly packed polygonal or disc- shaped cells that secrete a thin, homogeneous extracellular matrix material composed of fibrillar collagen and chondroitin sulphate proteoglycans’. While classical and modern histological examinations have not iden- tified a clear homologue of vertebrate cellular cartilage in invertebrates, a few invertebrates have cellular-cartilage-like endoskeletal elements. For example, among the protostomes, horseshoe crabs (Merostomata), cephalopod molluscs (Cephalopoda) and sabellid polychaete worms (Sabellidae) have cartilage-like tissues’, although their phylogenetic dis- tribution suggests that these tissues evolved independently of vertebrate cartilage, and of each other. Among the deuterostomes, both hemichor- dates and cephalochordates (amphioxus) have stiff, acellular pharyngeal endoskeletons that incorporate fibrillar collagen’. In addition, amphi- oxus has an oral skeleton that supports its tentacles (cirri) and that forms during metamorphosis (Fig. la-e). Although this oral skeleton is ensheathed in a thick integument that does not contain fibrillar col- lagen, recent work has shown that it has a cellularized core’. Unlike other deuterostomes, urochordates lack rigid endoskeletal elements’, despite their status as the sister group to vertebrates.
The histological properties of cellular cartilage in larval and adult vertebrates have been studied for over a century, and many specialized cartilage subtypes have been identified’. In contrast to the diversity of
A recurring theme in evolutionary developmental biology is how struc- tures in distantly related taxa can arise via conserved developmental mechanisms, revealing unexpected homology. We decided to test whether, despite its unusual histology in adults, the oral skeleton of amphioxus develops via mechanisms that are homologous to those of vertebrate
Notochord Neural tube Oral cirri ll slits b
Figure 1 | Development of the amphioxus oral skeleton. a, A metamorphic amphioxus larva. The oral region, shown in b-e, is boxed. b, The oral skeleton of a metamorphosed larva stained with alcian blue. c, d, Scanning electron micrographs of forming cirri in early metamorphic (c) and mid- metamorphic (d) larvae. Scale bar, 100 1m. e, A scanning electron micrograph of adult oral cirri. Scale bar, 100 yım. f-h, Alcian blue staining of chondrocytes in a larval amphioxus cirrus (f), a larval Petromyzon marinus gill bar (g) and a larval zebrafish gill bar (h). The arrows point to chondrocytes that are
dividing perpendicular to the axis of growth and intercalating to form stacks of discoidal cells. i-k, Toluidine blue staining of a larval amphioxus cirrus (i), a larval lamprey gill bar (j) and a larval zebrafish gill bar (k). The nuclei are blue, and the acidic extracellular matrix of the cellular cartilage is purple. The asterisks indicate the paired nuclei of dividing chondrocytes. The arrows point to vacuoles in maturing chondrocytes. The arrowheads indicate the acidic extracellular matrix. Original magnification, X 1,000 (£, i), X700 (g, j) and 500 (h, k).
'Departmentoof Ecology and Evolutionary Biology, University of Colorado, Boulder, Colorado 80309, USA. Department of Zoology, Comenius University, Bratislava 84215, Slovakia. Institute of Cellular and Organismic Biology, Academia Sinica, Taipei 11529, Taiwan.
- These authors contributed equally to this work.
534 NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
cellular cartilage. Because the amphioxus oral skeleton forms at the end of an extended planktonic larval phase that is difficult to obtain in the field, its development has never been described. Using new meth- ods for the continuous laboratory culture of the Florida amphioxus (B. floridae), together with a protocol for artificially inducing synchronized metamorphosis’, we were able to obtain live amphioxus larvae at vari- ous stages of oral skeleton development. We first stained these larvae with the classic histological stain for vertebrate cellular cartilage, alcian blue, and found strong specific reactivity with the nascent oral skeleton (Fig. 1b, f). Closer histological analysis of individual skeletal rods at this stage revealed discoidal cells with large vacuoles that were dividing per- pendicular to the axis of cirrus growth and were surrounded by an acidic extracellular matrix (Fig. 1f, i). This histology is highly similar to that of the gill bar cartilage in embryonic lampreys (Petromyzon marinus) and zebrafish (Danio rerio) (Fig. 1g, h, j, k).
We next asked whether the development of the oral skeleton in amphi- oxus requires the same intercellular signalling pathways as the cellular cartilage in vertebrates. Fibroblast growth factor (FGF)-mediated signal- ling is a conserved essential regulator of cellular cartilage differentiation in both jawed and jawless vertebrates”?, We thus exposed metamor- phosing amphioxus larvae to SU5402 and UO126 (refs 11, 12), inhibitors of FGF-mediated signalling that block cellular cartilage differentiation in vertebrates. Treatment with either inhibitor suppressed the forma- tion of the oral skeleton in metamorphosing amphioxus larvae (Fig. 2a-c and Extended Data Fig. 1c-h). Importantly, these inhibitors did not generally inhibit development, as the treated larvae displayed other tem- porally appropriate signs of metamorphosis, including formation of

Figure 2 | The amphioxus oral skeleton requires FGF-mediated signalling for formation and expresses orthologues of vertebrate cartilage markers. a, A phase contrast image of cirri in a control larva treated with T3 thyroid hormone, which induces metamorphosis in amphioxus. Ninety-six per cent of T3-treated control larvae (22 of 23, from two experiments) developed a normal oral skeleton (arrow) after 4-5 days, as previously reported*. In situ hybridization (d-g) was performed on sections at the level of the dashed line. b, c, Representative phase contrast images of larvae treated with T3 and 50 uM SU5402 (b) or T3 and 10 1M UO126 (c). All SU5402-treated larvae (26 of 26, from two experiments) and 83% of UO126-treated larvae (10/12, from one experiment) lacked oral cirri (arrows) but displayed other signs of metamorphosis, including metapleural folds, ventral mouth migration and secondary gill bar formation. d-g, Expression of amphioxus fibrillar collagen (ColA) (d), SoxE (e), SoxD (f) and DUSP6/7/9 (g) mRNA (shown in blue), as determined by in situ hybridization, in the oral region of metamorphic amphioxus larvae. Expression in chondrocytes is indicated by arrows, and expression in the mesothelium is indicated by arrowheads.
the secondary gill bars and metapleural folds and ventral migration of the mouth’.
A defining feature of vertebrate cellular cartilage is the secretion of fibrillar collagen, the major protein component of its extracellular matrix. Although previous histological examinations of amphioxus failed to find evidence of collagen fibres in oral cirri“, amputation of adult tentacles leads to the transcription of fibrillar collagen and SoxE messenger RNA at the site of amputation’. We assayed amphioxus fibrillar collagen (ColA) expression during metamorphosis, when the oral skeleton is differen- tiating. We found intense expression of ColA in the chondrocytes of nascent oral cirri and in the mesothelium that lines the anterior coeloms (Fig. 2d and Extended Data Fig. 2b, g-i).
In the cellular cartilage of jawed vertebrates, fibrillar collagen expression and chondrocyte differentiation are activated by the transcription factor SOX9, a SOXE family member and key regulator of chondrogenesis"’. The binding of SOX9 to the promoter of the gene encoding fibrillar collagen, Col2a1, is facilitated by transcription factors of the Soxd subfamily", while the expression of Sox9 depends on FGF-mediated signalling through FGF receptors (EGER), In lampreys, SoxE genes'* and FGF-mediated signalling through FGFRs" are also required for the proper differentiation of neural crest cell (NCC)-derived chondro- cytes, while SoxD is expressed in prechondrocytes'?. These results sug- gest that SOXE, SOXD and FGF signalling are ancient core components of the vertebrate cartilage gene program. We thus assessed whether the developing amphioxus oral skeleton expresses SoxE, SoxD and FGF signalling pathway components. We detected SoxE, SoxD and FGFR mRNA in oral chondrocytes and the surrounding mesothelium (Fig. 2e, f and Extended Data Fig. 2c, d). We observed similar expression of Ets and DUSP6/7/9, homologues of two FGF-mediated signalling target genes expressed in chondrogenic NCCs in zebrafish, mice and Xenopus laevis'*"” (Fig. 2g and Extended Data Fig. 2e, f, j, k). Taken together, our data show that the developing amphioxus oral skeleton displays the core, conserved histological, developmental and molecular features of ver- tebrate embryonic cellular cartilage.
Most of the cellular cartilage in the vertebrate head is derived from NCCs. Amphioxus lacks NCCs and probably forms its oral skeleton from the mesothelium that lines the anterior coeloms”, This implies that the genetic program for generating cellular cartilage was primitively deployed in mesendoderm and later recruited by NCCs. The repurposing of ancient genes and genetic programs is a recognized way in which novelty arises during evolution. However, it is unresolved whether such co- option events are typically driven by changes in cis-regulatory sequences, changes in the function of transcription factors or a combination of the two. Inall vertebrates that have been examined, including lampreys, SOXE transcription in NCCs is activated by the transcription factor TFAP2 and is maintained by auto-regulation between SOXE family members”, These regulatory interactions are direct and involve the physical bind- ing of TFAP2 and SOXE paralogues to SOXE enhancers in NCCs*'”, Previous work has shown that, despite new roles in NCC development, neither TFAP2 nor SOXE has acquired new DNA-binding properties in vertebrates”””, This implies that the novel expression of SoxE in NCCs was driven by changes in SoxE cis-regulatory sequences rather than by changes in transcription factor function. To directly test this hypothesis, we performed an interspecific assay of SoxE cis regulation. To ensure that all of the relevant amphioxus cis-regulatory sequences were queried, we built a bacterial artificial chromosome (BAC) reporter construct that contains the entire amphioxus SoxE locus and several flanking genes, and we tested this reporter in zebrafish (Fig. 3b). We reasoned that if changes in SoxE cis regulation were involved in the re- cruitment of SoxE to NCCs, then the reporter should recapitulate the expression pattern of endogenous SoxE in amphioxus (Fig. 3a). Alter- natively, if SoxE cis regulation has been conserved across chordates, the reporter should be active in NCCs, as shown previously for amphioxus Hox enhancers”. We observed reporter activity transiently throughout the neural tube and tail-bud region in early neurulae but not in migrat-
ing NCCs or chondrocytes at any developmental stage (Fig. 3c-f and
26 FEBRUARY 2015 | VOL 518 | NATURE | 535
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
<a yam G4 >—a 83.5 kb 103 kb pTARBAC2.1 Te, ae
Figure 3 | A reporter construct incorporating the amphioxus SoxE locus recapitulates the amphioxus SoxE expression pattern in zebrafish embryos. a, Transient expression of amphioxus SoxE transcripts (purple) in scattered neural tube cells (double arrowhead), the posterior axial mesoderm including the tail bud (arrow) and the anterior mesendoderm (arrowhead) in a 20-h, 12-somite neurula, as previously described”. b, Schematic of the amphioxus SoxE reporter construct. The construct spans the SoxE locus and 186 kilobases (kb) of flanking sequence, including adjacent genes, and is roughly equivalent to 1.3 million base pairs of human genomic sequence. The gene encoding green fluorescent protein (GFP) was fused in frame with the first exon (ex1), and Tol2 recombination arms were added to facilitate genomic integration. ampR, ampicillin resistance. c, A zebrafish embryo injected at the one-cell stage with the amphioxus SoxE BAC reporter construct and probed for GFP mRNA at the 16-h, 15-somite stage. Mosaic transcription of GFP (blue) was consistently observed in the neural tube (double arrowhead) and tail-bud region (arrow). d, A zebrafish embryo injected at the one-cell stage with the
amphioxus SoxE BAC reporter and probed for GFP mRNA at the 22-h, 26+-somite stage. Like endogenous amphioxus SoxE, the SoxE BAC reporter expression was transient and was extinguished by the late neurula stages, when vertebrate SOXE genes mark migrating and post-migratory NCCs. e, A dorsal close-up view of the embryo in cat the level of the hindbrain. Activity of the amphioxus SoxE BAC reporter occurred throughout the neural tube but not in the surrounding mesenchyme, which included some early migrating NCCs. The image is a composite of three photographs of the same embryo taken at different focal planes. f, Double in situ hybridization for zebrafish sox10 (a SoxE co-orthologue) (red) and GFP (blue) transcripts in an 18-somite zebrafish injected with the SoxE BAC reporter construct. The image is a composite of three photographs of the same embryo taken at different focal planes. GFP expression was limited to the neural tube and did not overlap with sox10 expression in the otic placode (asterisks) and migrating NCCs (arrows).
Extended Data Fig. 3). This pattern closely resembles endogenous am- phioxus SoxE expression in the embryonic neural tube and tail-bud”’ (Fig. 3a), regions that do not co-express Tfap2 and SoxE. These results, and the lack of endogenous soxE in the lateral and ventral neural tube of zebrafish embryos, suggest that the amphioxus SoxE reporter and, by extension, endogenous amphioxus SoxE are not regulated by Tfap2 or SoxE proteins.
To verify this finding, we assessed whether the reporter functioned properly in zebrafish embryos with depleted tfap2 function. While
TFAP2A and TFAP2C dual knockdown eliminates the expression of sox10 and sox9b (amphioxus SoxE co-orthologues) in zebrafish””, the activity of the amphioxus SoxE reporter was similar in wildtype zebra- fish and in tfap2a and tfap2c dual knockdown morphants (Extended Data Fig. 3). In the context of previous work, these data suggest that amphioxus and vertebrate SoxE cis regulation diverged, in part, through the evolution of new non-coding cis-regulatory sequences of tfap2 and SoxE. Whether these sequence changes were accompanied by changes in the function of other transcription factors is unclear from our assay.
BAP ISDE es a BAX na as
Our results suggest that the nascent oropharyngeal skeleton of early chordates incorporated collagenous cellular cartilage that is strikingly similar to vertebrate cartilage. In amphioxus, this tissue supports the oral tentacles, which prevent the ingestion of large particles during filter feeding and burrowing. It is likely that similar structures were present in the invertebrate ancestor of the vertebrates, as oral tentacles are found in lamprey larvae, adult hagfish and the chordate fossil Haikouella lanceolata’. We posit that the evolution of the vertebrate head skeleton involved two major developmental changes: the spread of collagenous cellular cartilage from the oral region into the pharynx and the head, and the novel differentiation of cellular cartilage from NCCs (Fig. 4). Our data also suggest that the acquisition of chondrogenic potential by NCCs involved the evolution of new transcription-factor-binding sites
Figure 4 | The evolution of the vertebrate head skeleton via co-option of an ancient cellular cartilage gene program. a, A hypothetical early chordate with an oral skeleton consisting of mesendoderm-derived cellular cartilage (pink polygonal cells) and a pharyngeal skeleton of acellular cartilage (pink rods). SoxE controls cellular cartilage differentiation in the oral region and has an unrelated function in a subset of central nervous system precursors (blue stellate cells). b, An early pre-vertebrate chordate with migratory non-skeletogenic proto-NCCs expressing SoxE (blue stellate cells in the oropharyngeal region). c, Exposure to intercellular signals in the oral region activates the cellular cartilage gene program in SoxE-expressing proto-NCCs (blue polygonal cells). d, Alternatively, mesendoderm-derived cellular cartilage (pink polygonal cells) spreads throughout the pharynx, before being replaced by NCC-derived cellular cartilage (e). e, Subsequent to c or d, invasion of the pharynx by chondrogenic NCC-derived cellular cartilage gives rise to the NCC-derived head skeleton of vertebrates (blue polygonal cells), similar to the model proposed by Rychel and Swalla’.
# 536 | NATURE
|
# VOL
518
26
# FEBRUARY
2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
at the SoxE locus. In addition to the oral cartilage, SoxE is expressed in the embryonic central nervous system of amphioxus. One of the first steps in the evolution of NCC-derived cellular cartilage may have been cis-regulatory mutations that maintained SoxE expression in non- skeletogenic proto-NCCs emerging from the central nervous system. Later, when these SoxE-expressing migratory cells mixed with mesen- doderm in the oral region, exposure to skeletogenic signals such as FGFs could have induced their differentiation into chondrocytes (Fig. 4). Thus, the dual roles of the ancestral chordate SoxE gene in neural and skeletal development may have predisposed evolving NCCs towards acquir- ing chondrogenic ability, potentiating the evolution of the vertebrate “new head”.
- 15. Kumar, M., Ray, P. & Chapman, S. C. Fibroblast growth factor and bone morphogenetic protein signaling are required for specifying prechondrogenic identity in neural crest-derived mesenchyme and initiating the chondrogenic program. Dev. Dyn. 241, 1091-1103 (2012).
- 16. McCauley, D. W. & Bronner-Fraser, M. Importance of SoxE in neural crest development and the evolution of the pharynx. Nature 441, 750-752 (2006),
- 17. Ohtani, K. et al. Expression of Sox and fibrillar collagen genes in lamprey larval chondrogenesis with implications for the evolution of vertebrate cartilage. J. Exp. Zool. 310B, 596-607 (2008).
- 18. Znosko, W. A. et al. Overlapping functions of Pea3 ETS transcription factors in FGF signaling during zebrafish development. Dev. Biol. 342, 11-25 (2010).
- 19. Li,C.Y., Scott, D.A., Hatch, E, Tian, X. Y. & Mansour, S. L. Dusp6 (Mkp3) is a negative feedback regulator of FGF-stimulated ERK signaling during mouse development. Development 134, 167-176 (2007).
- 20. Conklin, E. G. The embryology of amphioxus. J. Morphol. 54, 69-151 (1932).
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
# Received 4 September; accepted 24 October 2014,
# Published online 8 December 2014.
- 21. Betancur, P., Bronner-Fraser, M. & Sauka-Spengler, T. Genomic code for Sox10 activation reveals a key regulatory enhancer for cranial neural crest. Proc. Nat! Acad. Sci. USA 107, 3570-3575 (2010).
- 22. Van Otterloo, E. et al. Novel Tfap2-mediated control of soxE expression facilitated the evolutionary emergence of the neural crest. Development 139, 720-730 (2012).
- 23. Lakiza,O, Miller,S,, Bunce, A, Lee, E. M.J. & McCauley, D. W. SoxE gene duplication and development of the lamprey branchial skeleton: insights into development and evolution of the neural crest. Dev. Biol. 359, 149-161 (2011).
- 1. Medeiros, D, M. The evolution of the neural crest: new perspectives from lamprey and invertebrate neural crest-like cells. Wiley Interdiscip. Rev. Dev. Biol. 2, 1-15 (2013).
- 2. Gans, C. & Northcutt, R. G. Neural crest and the origin of vertebrates: a new head. Science 220, 268-273 (1983).
- 3. Witten, P. E., Huysseune, A. & Hall, B. K.A practical approach for the identification the many cartilaginous tissues in teleost fish. J. Appl. Ichthyology 26, 257-262 (2010).
- 4. Cole, A.G. & Hall, B. K. The nature and significance of invertebrate cartilages revisited: distribution and histology of cartilage and cartilage-like tissues within the Metazoa. Zoology (Jena) 107, 261-273 (2004).
- Rychel,A.L.& Swalla, B. J. Development and evolution of chordate cartilage. J. Zool. 308B, 325-335 (2007).
5.
- 6. Wright, G. M., Keeley, F. W. & Robson, P. The unusual cartilaginous tissues of jawless craniates, cephalochordates and invertebrates. Cell Tissue Res. 304, 165-174 (2001).
# of
# Exp.
- 24, Sauka-Spengler, T., Meulemans, D., Jones, M, & Bronner-Fraser, M. Ancient evolutionary origin of the neural crest gene regulatory network. Dev. Cell 13, 405-420 (2007).
- 25. Cossais, F. etal. Replacement of mouse Sox10 by the Drosophila ortholog Sox100B provides evidence for co-option of SoxE proteins into vertebrate-specific gene- regulatory networks through altered expression. Dev. Biol. 341, 267-281 (2010).
- 26. Manzanares, M. et al. Conservation and elaboration of Hox gene regulation during evolution of the vertebrate head. Nature 408, 854-857 (2000).
- 27. Meulemans,D, & Bronner-Fraser, M. Insights from amphioxus into the evolution of vertebrate cartilage. PLoS ONE 2, e787 (2007).
- 28. Li, W. & Cornell, R. A. Redundant activities of Tfap2a and Tfap2c are required for neural crest induction and development of other non-neural ectoderm derivatives in zebrafish embryos. Dev. Biol. 304, 338-354 (2007).
- 29. Chen, J. Y., Huang, D.Y. & Li, C. W. An early Cambrian craniate-like chordate. Nature 402, 518-522 (1999).
- Kaneto,$. & Wada, H. Regeneration of amphioxus oral cirri and its skeletal rods: implications for the origin of the vertebrate skeleton. J. Exp. Zool. 316B, 409-417 (2011).
7.
- Paris, M. et al. Amphioxus postembryonic development reveals the homology of chordate metamorphosis. Curr. Biol. 18, 825-830 (2008).
8.
- Walshe, J. & Mason, |. Fgf signalling is required for formation of cartilage in the head. Dev. Biol. 264, 522-536 (2003).
9.
- 10. Jandzik, D. et al. Roles for FGF in lamprey pharyngeal pouch formation and skeletogenesis highlight ancestral functions in the vertebrate head. Development 141, 629-638 (2014).
- 11. Mohammadi, M. etal. Structures of the tyrosine kinase domain of fibroblast growth factor receptor in complex with inhibitors. Science 276, 955-960 (1997).
- 12. Favata, M.F. et al. Identification of a novel inhibitor of mitogen-activated protein kinase kinase. J. Biol. Chem. 273, 18623-18632 (1998).
Acknowledgements We thank D. Brunelle, C.-H. Tung and T.-K. Huang for assistance with amphioxus husbandry, and P. Tsai and D. W. Stock for use of their microtomes. D.M.M., DJ., TAS. and M.V.C. were supported by National Science Foundation grants 10S 1257040 and |OS 1160733 (D.M.M.) and University of Colorado, Boulder start-up funds (D.M.M). A.T.G. was supported by National Science Foundation grant DBI 0905991. J.-K.Y. was supported by National Science Council Taiwan grants NSC101-2923-B-001-004-MY2 and NSC102-231 1-B-001-011-MY3 and Career Development Award AS-98-CDA-LO6 from Academia Sinica Taiwan.
Author Contributions D.M.M. designed the project and wrote the manuscript. DJ., AT.G.and T.AS. performed the experiments and prepared the images; D.J. assembled the figures; and M.V.C. and J.-K.Y. provided materials and reagents. All authors discussed the results and provided input on the manuscript.
- 13. Bi, W. M., Deng, J. M., Zhang, Z. P., Behringer, R. R. & de Crombrugghe, B. Sox9 required for cartilage formation. Nature Genet. 22, 85-89 (1999).
- 14. Lefebvre, V., Li, P. & de Crombrugghe, B. A new long form of Sox5 (L-Sox5), Sox6 and Sox9 are coexpressed in chondrogenesis and cooperatively activate the type II collagen gene. EMBO J. 17, 5718-5733 (1998).
is
Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to D.M.M. (Daniel.Medeiros@Colorado.edu).
26 FEBRUARY 2015 | VOL 518 | NATURE | 537
©2015 Macmillan Publishers Limited. All rights reserved
LETTER
doi:10.1038/nature13998
# Experimentally induced innovations lead to persistent culture via conformity in wild birds
Lucy M. Aplin", Damien R. Farine!’**, Julie Morand-Ferron, Andrew Cockburn’, Alex Thornton® & Ben C. Sheldon’?
# In human
# cultural
# arise when behaviours
societies, norms are mitted through social networks via high-fidelity social learning’. However, a paucity of experimental studies has meant that there is no comparable understanding of the process by which socially trans- mitted behaviours might spread and persist in animal populations”. Here we show experimental evidence of the establishment of foraging traditions in a wild bird population. We introduced alternative novel foraging techniques into replicated wild sub-populations of great tits (Parus major) and used automated tracking to map the diffusion, establishment and long-term persistence of the seeded innovations. Furthermore, we used social network analysis to examine the social factors that influenced diffusion dynamics. From only two trained birds in each sub-population, the information spread rapidly through social network ties, to reach an average of 75% of individuals, with a total of 414 knowledgeable individuals performing 57,909 solutions over all replicates. The sub-populations were heavily biased towards using the technique that was originally introduced, resulting in es- tablished local traditions that were stable over two generations, de- spite a high population turnover. Finally, we demonstrate a strong effect of social conformity, with individuals disproportionately adopt- ing the most frequent local variant when first acquiring an innova- tion, and continuing to favour social information over personal information. Cultural conformity is thought to be a key factor in the evolution of complex culture in humans*”’. In providing the first experimental demonstration of conformity in a wild non-primate, and of cultural norms in foraging techniques in any wild animal, our results suggest a much broader taxonomic occurrence of such an ap- parently complex cultural behaviour.
# trans-
This life history, coupled with their fission-fusion social structure”, makes them excellent models for a large-scale empirical investigation of the social processes associated with cultural transmission. Here we used a novel system that incorporates automated data collection and passive integrated transponder tags, together with recently developed methods for social network analysis, to investigate the spread, estab- lishment and persistence of experimentally seeded traditions in wild great tits.
We first developed an automated puzzle box that is baited with live mealworms (Fig. la), and performed a cultural diffusion experiment based on the two-action and control design" but where treatment groups were exposed to a demonstrator trained on one of two distinct but equiv- alent actions. Two resident males were caught from each of eight sub- populations and exposed to one of three training regimens in captivity. In the first condition (‘control’), for which there were three sub-populations (that is, three replicates), neither individual was given any training. In the second condition (‘option A’; two replicates), both individuals were trained to access food from the puzzle box by using their bill to move the blue side of the sliding door from left to right. Last, in the third con- dition (‘option B’; three replicates), the birds were trained to solve the puzzle box by moving the red side of the sliding door from right to left (Supplementary Video 1). After 4 days of training, all birds were re- leased back into the wild, and three puzzle boxes, with both options available, were installed 250 m apart in each sub-population (Extended Data Fig. 1). We then automatically monitored the individual visits to, and solutions (‘solves’) at these puzzle boxes, over short-term (20 days of exposure over 4 weeks) and long-term (5 days of exposure, 9 months later) periods.
Social learning, in which animals learn from others, can enable novel behaviours to spread between individuals, creating group-level behav- iours, including traditions and culture“*”, Social transmission occurs between interacting individuals; hence, group dynamics and popula- tion structure will determine the spread and persistence of traditions?”*, Additionally, individuals may strategically use social learning to maxi- mize its adaptive value, with consequences for when, how and what tra- ditions are established*'*. However, while the capacity for social learning has been described in many phylogenetically diverse taxa’ and has been detailed in comprehensive laboratory studies'* '°, we have little know- ledge of the social dynamics associated with such learning in natural systems. Experimentally quantifying cultural transmission in wild popu- lations remains difficult, with limitations associated with isolating and training individuals’, tracking the spread of information across large numbers of animals" and eliminating alternative explanations such as individual trial-and-error learning**,
Early observational studies of tits provide one of the most widely cited examples of animal innovation and culture, when British birds famously began to pierce the foil caps of milk bottles to take the cream'*'*. More generally, great tits (P. major) are known to be highly innovative, opportu- nistic foragers * and to use social information in a wide range of contexts”.
In the five sub-populations that were seeded with trained demonstra- tors, knowledge of how to solve the novel puzzle spread rapidly over 20 days of exposure (Fig. 1b). A mean of 75% of the members of each local population (68-83%, n = 37-96) solved the puzzle box at least once. The diffusion of this behaviour was clearly sigmoidal (sigmoidal versus linear fit, change in Akaike information criterion (AAIC) = 15.31-54.17), except in one replicate (T5, AAIC = 0.13). By contrast, many fewer individuals solved the puzzle box in control sub-populations (9-53%, n= 5-46; Fig. 1b), in which uptake initially relied on individual inno- vation. The latency to the first solve, excluding the demonstrator, was significantly longer in control areas than in treatment areas (Welch’s two-sample t-test, £(6) = —16.1, P< 0.01; Fig. 1b), and the total num- ber of solutions was significantly lower (46) = 4.6, P = 0.02; Fig. 1c). There wasa striking difference between the replicates that were seeded with alternative solving techniques. In all treatment sub-populations, learning was heavily biased towards the technique that was originally demonstrated (t(g) = 9.7, P< 0.01; Fig. 1c), while no consistent side bias was observed between the control sub-populations (t(4) = —0.03, P= 0.97; Fig. 1c).
We derived the social network for each sub-population independently of the social learning experiment, with 10 days’ sampling at a grid of
"Edward Grey Institute, Department of Zoology, University of Oxford, Oxford OX1 3PS, UK. ”Department of Evolution, Ecology and Genetics, Research School of Biology, Australian National University, Canberra, Australian Capital Territory 2600, Australia. *Department of Anthropology, University of California, Davis, California 95616, USA. “Smithsonian Tropical Research Institute, Ancon 9100, Panama. “Department of Biology, University of Ottawa, Ottawa KIN 9B2, Canada. “Department of Biosciences, Centre for Ecology and Conservation, University of Exeter, Penryn Campus, Penryn TR10 9EZ, UK. Department of Ecology and Genetics, Uppsala University, 752 36 Uppsala, Sweden.
538 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
‘ » 1.0 Mi x 1.0 Te M Option A $ ri & 12,000 OptionB t a $ os, 4m Los ğ 3 ° 12 | 8 : +73 3 3 x14 Lae & g “ler g 80001 Lin @ $ 8 : 3 3 3 3 | > 5 © }04 g Hi 8 : g = 4,0004 8 8 koz 8 7 : “ 7 s 7 1 T : o 5 10 15 20 C1 C2 C3 TI T2 T3 T4 TS Time (1-20 days) Control Option A Option B
# Sub-population
Figure 1 | Cultural diffusion experiment. a, A puzzle box in which birds can slide the door open in two directions (from the left, option A; or the right, option B) to access a reward. The puzzle box records the identity, visit duration and solution choice, and it resets after each visit. b, Diffusion curves for the treatment sub-populations, with demonstrators (T1-5; n = 91, 130, 132, 50, 90. respectively), and the control sub-populations, without demonstrators (C1-3;
= 56,87, 61, respectively). c, The total number of solutions using each option in each replicate (sub-population) (left y axis, shown as stacked bars). The average proportion (dots) of option A performed by individuals, with 95% CI (bars) is shown (right y axis). The total number of solvers was 5, 46 and 19 for CI, C2 and C3 (controls), respectively; 76 and 89 for TI and T2 (option A), respectively; and 96, 37 and 69 for T3, T4 and T5 (option B), respectively.
# n
sunflower-seed feeders that had been equipped to record visitation data (Extended Data Fig. 2a, b). Co-occurrences (see Methods) were detected using a Gaussian mixture model to isolate clusters of visits in the spatio- temporal data streams”, with repeated foraging associations between individuals forming the basis of social networks (Extended Data Fig, 2b, c). The social networks for all replicates were significantly non-random, even at the most local scale (T1-5, P < 0.001), and network-based dif- fusion analysis was used to quantify the extent to which these social ties predicted the acquisition of behaviour”, From pooled replicate data, a network diffusion model that included social transmission was over- whelmingly supported over asocial learning (AAIC = 1520.7); the learn- ing rate was estimated to increase by a factor of 12.0 per unit of association with knowledgeable individuals (Extended Data Fig. 3). An effect of age and sex was also supported, with juveniles and males having a faster learning rate (Table 1). These results support a dominant effect of social learning on the emergence of this novel behaviour and also demonstrate that the diffusion of innovation was influenced by the fine-scale patterns of social interactions (Supplementary Video 3).
In all of the experimental replicates, the alternative solution, which was equally difficult and equally rewarded, was performed by at least one individual within the first 6 days of exposure (median, day 4). However, in contrast to most previous studies, in which discovery of an alternative solution led to the progressive erosion of the use of the seeded variant behaviour?””, we observed a pronounced strengthening of traditions
over the rest of the experiment. To analyse this change in behaviour over time, we used a generalized estimating equation model’ where the dependent variable was the proportion of solutions using the seeded technique on each day of data collection and the explanatory variables were individuals and replicates. From pooled replicate data, there was strong evidence that the preference for the arbitrary tradition increased over time (coefficient + s.e.m. = 0.13 + 0.02, P< 0.001), with an esti- mated 14% increase in bias per day (95% confidence interval (CI) = 8-18%; Fig. 2a). This finding is consistent with a conformist transmis- sion bias, with individuals preferentially adopting the more commonly practised variant when solving the puzzle box”, More conclusive evidence for such positive frequency-dependent copying” was observed when only the first solution for each individual was considered, with birds disproportionately likely to initially adopt the variant used by the majority of their group (sigmoidal versus linear fit, AAIC = 38.34; Fig. 2b).
Individuals thus preferentially learnt the most common option when first learning (Fig. 2b). Yet, remarkably, they also continued to prioritize social information over personal information, matching their behav- iour to the common variant even after experiencing an equally reward- ing alternative. We analysed the trajectories for those individuals that used both options (n = 78). The majority of these individuals (85%, n = 66) retained a preference for the seeded variant (for example, see Fig. 2c and Extended Data Fig. 4). Three birds had a strong preference for the uncommon variant, and eight birds switched from the alternative
# Table 1 | Network-based diffusion analysis
Network-based diffusion model outputs Transmission model AAIC (top model) Yo; Social transmission parameter (estimated) 95% Cl Social: multiplicative o 0.99 120 8.8-16.0 TI = - 22.4 11.8-30.2 T2 - - 122 8.2-17.1 T3 - - 73 29-143 T4 5 - 298 10.9-42.6 T5 - - 134 8.3-20.02 Social: additive 33.7 0.01 - - Asocial 1520.7 0 (Constrained to 0) -
Individual-level effects
Variable AAIC (top model) Ley, Estimate Effect size Age (juvenile or adult) 10) 0.99 -0.18 0.70 Sex (F or M) o 0.97 0.10 1.22 Natal origin (resident or immigrant) 3.9 0.13 0.07 1.16
‘Summed Akaike weights (Zo) and AAIC for network-based diffusion models, with maximum-likelihood parameter estimates of social transmission for the five treatment replicates. Estimates and effect sizes are presented for the individual-level variables. The diffusion analyses used a continuous time of acquisition model with a constant baseline learning rate (o), allowing differing social transmission rates in each replicate. F, female; immigrant, dispersed into the study site; M, male; resident, locally born.
26 FEBRUARY 2015 | VOL 518 | NATURE | 539
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
as Bio pi 3 = 5 os 3 a 3 3 3 5 06 rd 2 5 = 04 ş ? 5 g 02 2 a 0.0 Proportion of the seeded option” 8 10 15 20 Time (1-20 days) 1 0.0 02 04 06 08 10 Frequency of option A in group 0.0 02 04 06 08 1.0 Proportion of individual's
# solutions over time
Figure 2 | Evidence of social conformity. a, The proportion of solutions using the seeded technique increased significantly over time in each replicate. The points are the proportion of solutions using the seeded technique on each day; the lines show the generalized estimating equation model fit. b, Comparison of the frequency of option A in the sub-population with an individual's first learnt option (pooled replicate data from T1-5). The node size represents the
number of individuals (n = 1-147). The black line shows the expected result under unbiased copying; the central red line shows the model fit with 95% CI (outer red lines). c, The solution trajectories for individuals in the T2 sub-population that used both possible options (n = 10). The lines are the running average of the proportion of option A for each individual over the last ten visits, with each colour representing a single individual.
variant to the common variant. However, none of the birds made the reciprocal switch, and only one individual had no significant prefer- ence. A subset of birds that dispersed between the experimental rep- licates (a total of n=41, of which 24 were between the two years of the experiment, see below) provided additional evidence. Of 27 birds that moved between replicates with the same seeded tradition, 26 (96%) re- tained their preference for the common variant. In contrast, of 14 indi- viduals that moved between replicates with different seeded traditions, 10 (71%) changed their behaviour to match the common variant in the new location, 3 retained their initial preference and 1 showed no pre- ference (Cay = 21.6, P< 0.001).
Seeded arbitrary traditions thus formed and persisted in each sub- population (Fig. 2). To investigate the long-term stability of these tradi- tions, we re-installed the puzzle boxes in one replicate of each condition (TI, T3 and CI) over 5 days in the following winter. Substantial turn- over in the population had occurred owing to the high mortality rates typical of this species”; on average, only 40% of each sub-population had been present the previous year. No additional demonstrators were trained, and no individual had had contact with the puzzle box in the intervening months. In the control sub-population, all solves (n = 42) were performed by only three individuals, all of which had also solved the puzzle box the previous year. However, in the two experimental sub- populations, knowledge of how to solve the puzzle box emerged even
faster than it had the preceding year, both among prior solvers and birds that were inexperienced at the task: in TI, 29 individuals solved the puzzle box a total of 967 times, and in T3, 35 individuals solved the puzzle box a total of 2,329 times (Fig. 3b). The results suggest a strong initial effect of memory, followed by a rapid, oblique transmission facil- itated by the greater number of demonstrators than in the initial exper- iment: on the first day of exposure, 60% (T1) and 82% (T3) of ‘solvers’ were birds that had solved the puzzle box in the initial experiment, out- weighing their representation in the general population (36% in T1 and 46% in T3). The sub-populations also retained their original technique, with the solutions being heavily biased towards the option that had been seeded in the original experiment (Fig. 3b). Intriguingly, among birds that were present in both years, the within-individual bias towards the seeded variant had increased (linear mixed model, t(g3) = 2.80, P< 0.01; Fig. 3c), resulting in arbitrary traditions that were retained and strengthened.
In summary, we show that wild great tits use social learning to acquire novel behaviours and that foraging techniques introduced by few indi- viduals (here only two in each replicate) can spread rapidly to the majority of the population, forming stable arbitrary traditions. Both social network ties and individual characteristics determined the transmission of these foraging technigues”. The introduced arbitrary traditions were stable over both short-term and long-term periods, becoming increasingly entrenched over two generations. This stability appeared to be a result
b c 1.0 $ M OptionA = o 5 yö” tt M Option B 2 08 . ği 2,000 5 4°" 5-0" 5 £ Zig 2 a £ Ave 2 8 506) ° yw öl 2 15001 5 5 + 3 3 O47 /0-9 3 3 g we a 5 o 4 2 04 fi of 2 1,0004 2 3 |} + 71201341 3 E re © T32013-1 & Sol #4 + T1 2013-11 prior solvers 5 if 7 i Çi © T3 2013-11 prior solvers 500+ $ ob: + T1 2013-l1 naive birds & LAP © T3 2013-II naive birds 0.04 °° - - - - vol 5 10 15 20 Ct:ps Ci:nb Tips Tinb o T8:ps T3:nb T12013-4 T12013- T32013-1 T32013-l1 Time (1-20 days) Control Option A Option B Option A Option B Sub-population Sub-population
Figure 3 | Local traditions persist across years. a, Diffusion curves for the initial exposure (T1 2013-1 and T3 2013-1, where I is 20 days) and the second exposure (TI 2013-11 and T3 2013-11, where II is 5 days, 9 months later). The cumulative uptake of the behaviour in the second exposure is much higher for prior solvers (T1 2013-11 and T3 2013-II, n = 23 and 26, respectively) but is also higher for naive birds (T1 2013-11 and T3 2013-11, n = 28 and 27,
respectively). b, The number of solutions using option A or B. In TI, one circuit board failed, so the data are derived from two of three devices. nb, native birds; ps, prior solvers. c, The proportion of option A or B used in the initial and second exposure. The histograms shows the data for the sub-populations; the dots show the mean proportion of option A performed by individuals, and error bars show the 95% CI.
540 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
of informational conformity, with individuals matching their behaviour to the most common variant when first learning and then continu- ously updating their personal information. Conformity has long been considered a central component of human culture”, but experi- mental evidence for its occurrence in wild animals has been limited to a study of food preferences in vervet monkeys’. We provide the first experimental demonstration, to our knowledge, of conformist trans- mission and cultural norms in foraging techniques in a wild animal. Our study argues against the previous view that such behaviour is restricted to the primate lineage**’* and calls for a reconsideration of the evo- lution and ecology of cultural conformity.
- 16. Fisher, J. B. & Hinde, R. A. The opening of milk bottles by birds. Br. Birds 42, 347-357 (1949).
- 17. Sherry, D. F. & Galef, B. G. Cultural transmission without imitation: milk bottle opening by birds. Anim. Behav. 32, 937-938 (1984).
- 18. Aplin, L. M., Sheldon, B. & Morand-Ferron, J. Milk-bottles revisited: social learning and individual variation in the blue tit (Cyanistes caeruleus). Anim. Behav. 85, 1225-1232 (2013).
- 19, Morand-Ferron, J,, Cole, E. F., Rawles, J. E. C. & Quinn, J. L. Who are the innovators? A field experiment with 2 passerine species. Behav. Ecol, 22, 1241-1248 (2011).
- 20. Slagsvold, T. & Wiebe, K. L. Social learning in birds and its role in shaping a foraging niche. Phil. Trans. R. Soc. B 366, 969-977 (2011).
- 21. Aplin, L. M. etal. Individual personalities predict social behaviour in wild networks of great tits (Parus major). Ecol. Lett 16, 1365-1372 (2013).
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version ofthe paper; references unique to these sections appear only in the online paper.
- 22. Psorakis, |., Roberts, S. J., Rezek, |. & Sheldon, B. C. Inferring social network structure in ecological systems from spatio-temporal data streams. J. R. Soc. Interface 9, 3055-3066 (2012).
- 23. Allen, J., Weinrich, M. T., Hoppitt, W. & Rendell, L. Network-based diffusion analysis reveals cultural transmission of lobtail feeding in humpback whales. Science 340, 485-488 (2013).
# Received 23 June; accepted 23 October 2014.
# Published online 3 December 2014.
- 1. Rogers, E. M. Diffusion of Innovations 4th edn (Free Press, 1995).
- 2. Claidiöre,N, Messer, E. J. E., Hoppitt, W. & Whiten, A. Diffusion dynamics of socially learned foraging techniques in squirrel monkeys. Curr. Biol, 23, 1251-1255 (2013).
- Cantor, M. & Whitehead, H. The interplay between social networks and culture: theoretically and among whales and dolphins. Phil. Trans. R. Soc. B368, 20120340 (2013).
3.
- Rendell, L. et al. Cognitive culture: theoretical and empirical insights into social learning strategies. Trends Cogn. Sci. 15, 68-76 (2011).
# PF
- 24. Thornton, A. & Malapert, A. The rise and fall of an arbitrary tradition: an experiment with wild meerkats. Proc. R. Soc. B 276, 1269-1276 (2009).
- 25. Morgan, T. J. H. & Laland, K. The biological bases of conformity. Front. Neurosci. 6, 87 (2012).
- 26. van Leeuwen, E. J. C. & Haun, D. B. M. Conformity in nonhuman primates: fad or fact? Evol. Hum. Behav. 34, 1-7 (2013).
- 27. Bouwhuis, S., Choquet, R., Sheldon, B. C. & Verhulst, S. The forms and fitness cost of senescence: age-specific recapture, survival, reproduction, and reproductive value in a wild bird population. Am. Nat. 179, E15-E27 (2012).
- 28. Haun, D. B. M., Rekers, Y. & Tomasello, M. Majority-biased transmission in chimpanzees and human children, but not orangutans. Curr. Biol. 22, 727-731 (2012).
- van de Waal, E,, Borgeaud, C. & Whiten, A. Potent social learning and conformity shape a wild primate’s foraging decisions. Science 340, 483-485 (2013).
# HW
- Whiten, A., Hinde, R. A., Laland, K. N. & Stringer, C. B. Culture evolves. Phil. Trans. Soc. B 366, 938-948 (2011).
- Whiten, A., Horner, V. & de Waal, F. B. Conformity to cultural norms of tool use chimpanzees. Nature 437, 737-740 (2005).
# Pn
- Laland, K. N. & Janik, V. M. The animal cultures debate. Trends Ecol, Evol. 21, 542-547 (2006).
# e
- Warner, R. R. Traditionality of mating-site preferences in a coral reef fish. Nature 335, 719-721 (1988).
- 10. Coussi-Korbel, S. & Fragaszy, D. M. On the relation between social dynamics and social learning. Anim. Behav. 50, 1441-1453 (1995).
R.
# in
- 29. de Waal, F.B. Animal conformists. Science 340, 437-438 (2013).
- 30. van Schaik, C. P. Animal culture: chimpanzee conformity? Curr. Biol. 22, R402-R404 (2012).
Supplementary Information is available in the online version of the paper.
Acknowledgements This project was supported by grants from the BBSRC (BB/ LO06081/1) and the ERC (AdG 250164) to B.C.S., who was also supported by a visiting professorship at Uppsala University. L.M.A. was also supported by an Australian Postgraduate Award; and A.T., bya BBSRC David Phillips Fellowship (BB/HO21817/1). The EGİ social networks group, S. Lang and K. McMahon provided assistance in the field, and M. Whitaker produced electronic components for the puzzle boxes.
- 11. Dean,L.G, Kendal, R. L., Schapiro, S. J., Thierry, B. & Laland, K. N. Identification of the social and cognitive processes underlying human cumulative culture. Science 335, 1114-1118 (2012).
- 12. Laland, K. Social learning strategies. Learn. Behav. 32, 4-14 (2004).
- 13. Galef, B. G. & Laland, K. N. Social learning in animals: empirical studies and theoretical models. Bioscience 55, 489-499 (2005).
Author Contributions The study was initially conceived by LM.A,, J.M.-F. and B.C.S., with input from D.R.F., A.C. and AT. in designing the experiments. Infrastructure to support the work was conceived and developed by B.C.S. The experimental work was led by L.M.A,;; and the analysis, by LM.A. and D.R.F. The manuscript was drafted by L.M.A. and B.C.S,, and important contributions were made by all of the other authors.
- 14. Whiten, A. & Mesoudi, A. Establishing an experimental science of culture: animal social diffusion experiments. Phil, Trans. R. Soc. B 363, 3477-3488 (2008).
- 15. Galef, B.G. in Oxford Handbook of Comparative Cognition (eds Zentall, T. R. Wasserman, E.) Ch. 40, 803-818 (Oxford Univ. Press, 2012).
&
Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests, Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to L.M.A. (lucy.aplin@zoo.ox.ac.uk).
26 FEBRUARY 2015 | VOL 518 | NATURE | 541
©2015 Macmillan Publishers Limited. All rights reserved
LETTER
doi:10.1038/nature14242
# Fundamental properties of unperturbed haematopoiesis from stem cells in vivo
Katrin Busch’, Kay Klapproth'*, Melania Barile”*, Michael Flossdorf”*, Tim Holland-Letz’, Susan M. Schlenner*, Michael Reth*’, Thomas Hofer? & Hans-Reimer Rodewald!
Haematopoietic stem cells (HSCs) are widely studied by HSC trans- plantation into immune- and blood-cell-depleted recipients. Single HSCs can rebuild the system after transplantation'*. Chromosomal marking”, viral integration” * and barcoding''-"? of transplanted HSCs suggest that very low numbers of HSCs perpetuate a continuous stream of differentiating cells. However, the numbers of productive HSCs during normal haematopoiesis, and the flux of differentiating progeny remain unknown. Here we devise a mouse model allowing inducible genetic labelling of the most primitive Tie2* HSCs in bone marrow, and quantify label progression along haematopoietic devel- opment by limiting dilution analysis and data-driven modelling. During maintenance of the haematopoietic system, at least 30% or ~5,000 HSCs are productive in the adult mouse after label induction. However, the time to approach equilibrium between labelled HSCs and their progeny is surprisingly long, a time scale that would exceed the mouse’s life. Indeed, we find that adult haematopoiesis is largely sustained by previously designated ‘short-term’ stem cells downstream of HSCs that nearly fully self-renew, and receive rare but polyclonal HSC input. By contrast, in fetal and early postnatal life, HSCs are rapidly used to establish the immune and blood system. In the adult mouse, 5-fluoruracil-induced leukopenia enhances the output of HSCs and of downstream compartments, thus accelerating haematopoietic flux. Label tracing also identifies a strong lineage bias in adult mice, with several-hundred-fold larger myeloid than lymphoid output, which is only marginally accentuated with age. Finally, we show that trans- plantation imposes severe constraints on HSC engraftment, consist- ent with the previously observed oligoclonal HSC activity under these conditions. Thus, we uncover fundamental differences between the normal maintenance of the haematopoietic system, its regulation by challenge, and its re-establishment after transplantation. HSC fate mapping and its linked modelling provide a quantitative framework for studying in situ the regulation of haematopoiesis in health and disease.
YEP” KN EMEN cells in bone marrow, thymus and spleen in Tie“ Rosa” mice (n = 30; data not shown). After tamoxifen treat- ment, MCM becomes active and deletes the stop cassette of the YFP marker gene, thus rendering Cre-expressing cells and their non-Cre- expressing progeny YFP-positive (Extended Data Fig. 1d). Early after treat- ment (20 days) the labelled cells were almost exclusively HSCs, defined as lineage marker (Lin) Kit* Sca-1* (LSK) CD150* CD48° (refs 3, 17 and 18) (Fig. la). Ina total of 112 adult Tie2“M* Rosa””P mice, a mean of 1.0% of HSCs were labelled in situ after tamoxifen treatment (Extended Data Fig. le). Transplantation of a single YFP- ea HSC into gen- etically conditioned HSC recipients (Rag2 “ye” Kit"; yc is also known as /12rg) led to long-term donor HSC engrafimeni and multi- lineage reconstitution in primary and secondary recipients (Extended Data Fig. 1f), hence initially labelled cells were functional HSCs (Extended Data Table 1). We ruled out the possibility that HSC numbers and functions were compromised by loss of one Tie2 allele in 7ie2“W* mice in a series of control experiments (Extended Data Fig. 2).
We estimated by limiting dilution analysis frequencies of HSCs con- tributing to overall haematopoiesis, and to lymphoid and myeloid line- ages (Fig. 1b and Extended Data Fig. 1g-j). Atleast one out of three HSCs contributed YFP” CD45* progeny in the bone marrow between 6 and 34 weeks after labelling. This is the overall and cumulative frequency regardless of the type of lineage produced, and it represents a lower limit given that all mice with labelled HSCs also contained labelled progeny. Considering 2.8 X 10” total nucleated bone marrow cells per mouse”, and an HSC frequency of 0.006%, a mouse has ~17,000 HSCs; 30% active HSCs indicates that ~5,000 HSCs contributed to normal haema- topoiesis within the observation period. In transplantation experiments,
The paucity of HSCs has largely impeded direct measurements of their functions in situ. To determine fundamental properties (frequencies of active HSCs, fluxes between stem and progenitor compartments, res- idence time and expansion in compartments) of unperturbed steady- state haematopoiesis'’, we devised an experimental system for inducible genetic marking of HSCs in situ. As driver for Cre recombinase (Cre) we used the Tie2 (also known as Tek) locus, which is expressed in embryonic and adult HSCs'*!°. We generated a knock-in mutant expressing from the Tie2 locus a gene encoding codon-improved Cre (iCre) fused to two modified oestrogen receptor binding domains (designated MCM)'* (Extended Data Fig. la-c). We chose this weakly inducible and tightly regulated system to prevent leakiness. The Tie2"™ allele was crossed to Rosa””” mice expressing the yellow fluorescent protein (YEP) reporter in a Cre-dependent manner. In the absence of tamoxifen, we did not detect
a 3 b Number of YFP* HSCs (10°) a 1 3 Tie2"°M'" Rosa”? 2 11111 3 vers 5 Tamoxifen Analysis € 20 days after tamoxifen & . Lin Lsk > 1/25 4128 İ İ u i ° ş s 8 4 | ‘ | 04%| 8 —Sct—> = —YFP—» = —CD48—> ~e- DP thymocytes -e- Pro B cells o -e-CD45'
- e- Granulocytes
Figure 1 | Inducible HSC labelling in Tie2“““*Rosa””” mice and frequency estimates on HSC output. a, Phenotype of labelled cells 20 days after tamoxifen injection. b, Limiting dilution analysis of labelled HSCs and lineage output. The fraction of negative mice for YFP-expressing Lin * CD45* cells (green), granulocytes (red), pro B cells (blue) or double-positive (DP) thymocytes (black) was plotted on a logarithmic scale against the number of YFP* HSCs (Extended Data Fig. 1g-j). Arrows indicate lower detection limit (no negative mice in these groups).
# output
Division of Cellular Immunology, German Cancer Research Center (OKFZ),Im Neuenheimer Feld 280, D-69120 Heidelberg, Germany. “Division of Theoretical Systems Biology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, D-69120 Heidelberg, Germany. “Division of Biostatistics, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 0-69120 Heidelberg, Germany. “Department of Microbiology and Immunology, University of Leuven, B-3000 Leuven, Belgium. "Autoimmune Genetics Laboratory, VIB, B-3000 Leuven, Belgium. “BIOSS, Centre For Biological Signaling Studies, University of Freiburg, SchanzlestraBe 18, 0-79104 Freiburg, Germany. Department of Molecular Immunology, Bialll, Faculty of Biology, University of Freiburg, and Max-Planck Institute of Immunobiology and Epigenetics, Stübeweg 51, D-79108 Freiburg, Germany.
- These authors contributed equally to this work.
542 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
4
# Kİ
# J
absolute numbers of contributing HSCs are 100-fold lower'*" than our estimate for steady state haematopoiesis. We also determined pathway frequencies from HSCs to granulocytes, and T and B cell progenitors (Fig. 1b and Extended Data Fig. 1g-j). The results revealed correlations between the numbers of labelled HSCs and labelled lineage output, with the (time-averaged) probability for finding labelled granulocytes being five- to tenfold higher than lymphocytes.
To address the fluxes from adult HSCs via stem and progenitor com- partments to peripheral lineages, we resolved the output from in-situ- labelled HSCs kinetically (Fig. 2a-f). Unexpectedly, up to 3 weeks after induction in adult mice, the label was exclusively retained in HSCs with no label found in LSK CD150° CD48 short-term (ST)-HSCs, and LSK CD150° CD48~ multipotent progenitors (MPPs) (Fig. 2b). Within these downstream stem and progenitor compartments, the first marked cells emerged from 4 weeks onwards (Fig, 2c). Beyond 16 weeks, labelled HSCs had contributed to all analysed progenitor and mature cell lineages, with labelled myeloid cells arising sooner than labelled lymphoid cells (Fig. 2e, f). Analysis of overall bone marrow cells further indicated that the label emanates from marked HSCs (Extended Data Fig, 3a). Very few myeloid progenitors (0.010% common myeloid progenitors (CMPs); 0.001% granulocyte-macrophage progenitors (GMPs); 0.006% megakaryocyte- erythroid progenitors (MEPs)) in the bone marrow were also initially marked (Fig. 2b, asterisk), consistent with weak expression of Tie2 in myeloid progenitors (http://www.immgen.org; data not shown) leading to direct, HSC-independent, labelling. Because of the limited life span of CMPs, GMPs and their progeny (Extended Data Fig. 4), the presence of labelled cells beyond 6 weeks after tamoxifen treatment reflects only cells that have arisen de novo from labelled HSCs.
Given the extraordinarily slow label progression out of the adult HSC compartment, we investigated how rapidly HSCs are used during devel- opment (Fig. 2g-k and Extended Data Fig, 3b). Tie2“““* Rosa””” mice were treated in utero in midgestation (embryonic day (E) 10.5) with tam- oxifen. While initially (E12.5) almost exclusively HSCs (but not erythro- myeloid progenitors”; Extended Data Fig. 5) were marked in fetal liver, the label progressed within days to progenitors in fetal liver and bone marrow, and by 1 week after birth, equilibrium was nearly reached between labelled HSCs and the entire peripheral system. Hence, HSC use is very rapid (and possibly complete) during development but slow during maintenance of the system (Fig. 21).
|
E105 E12.5 E155 Birth Tamoxifen ‘Analysis Analysis Analysis h 1004 7s) Analysis b is x, 8 i ğ 1-3 weeks öl E125 251 YFP* coils (96) i © 15 4-6 weeks 100 R Cor ay gp” it E15.5 e & YFP" cells (96) 1-9 HSG ST-HSC MPP CLP YEP” cells (36) YEP» cells (96) e : tad 8-12 weeks YEP* colls (96) © 157 YEP" cells (96) I Te İş 26-34 weeks YFP* cells (96) = YFP" cells (96) frequency of relative to HSCs
We exploited the kinetic data for adult mice (Fig. 2b-f) to infer the fluxes between stem and progenitor compartments as well as residence time and expansion of the cells in the compartments (Fig. 3). In a given reference compartment (for example, ST-HSCs) the cells lost by onward differentiation (for example, towards MPPs) are replaced by influx from the upstream compartment (for example, HSCs), and by cell production in the reference compartment itself (Fig. 3a). The flux is the product of the rate of differentiation per cell and the total amount of cells under- going differentiation. The movement of label between compartments con- tains information on the rate (Fig. 3b), and the ratios of total cell num- bers were determined for stem and progenitor compartments (Extended Data Fig. 6a). These considerations form the basis of our model for quan- tifying the labelling data (Supplementary Methods and Supplementary Discussion).
The steady label frequency of around 1% is consistent with self-renewal of labelled HSCs (Fig. 3c; HSC panel). In compartments downstream from HSCs, labelled cells incrementally replaced non-labelled cells (Fig. 3c). The mathematical model fitted the label frequencies measured up to 240 days after induction, and correctly predicted label frequencies at later time points (Fig. 3c).
# abel ST-HSCs to Tamoxifen £10.5
# 4 42d Time (days)
100
200
Figure 2 | Label progression through the haematopoietic system during adult maintenance and fetal development. a, HSC label induction and output analysis in adult tamoxifen-treated Tie2““’* Rosa””” mice. b-f, Percentages of YEP” cells among the indicated haematopoietic cells in bone marrow, spleen and thymus. Mice were analysed at 1-3 (b; n = 12), 4-6 (c;n = 10), 8-12 (d; n = 9), 16-20 (e; n = 12) and 26-34 (f; n = 10) weeks after label induction. Dots represent individual mice, and bars indicate the mean. DN, double negative thymocytes; Gr, granulocytes; Mac, macrophages. g-k, In utero HSC label induction and analysis in fetal, newborn and 1-week-old Tie“ * Rosa”FP mice. YFP labelling frequencies were determined at E12.5 (h; n = 9), E15.5 (i; n = 10), in newborns (j; n = 6) and 1 week after birth (k; n = 8) for the indicated organs and cells. 1, Kinetic of label progression from HSCs to ST-HSCs in embryonic (red; n = 66) and adult (blue; n = 110) mice (n per time points; see Supplementary Methods); arrows indicate tamoxifen treatment. Data are mean and s.e.m.
We estimated the rates of cell differentiation and net proliferation in the stem and progenitor compartments (Fig. 3d, e, Extended Data Fig. 6b, c and Supplementary Methods). The rate of net proliferation equals the number of cells born per day minus cells lost by death during the same time period. On average, per day, 1 out of 110 HSCs differ- entiates into an ST-HSC, and 1 out of 22 ST-HSCs differentiates into an MPP (Fig. 3d, f). At the MPP stage, considered a lymphoid-myeloid
bifurcation point, we estimate that per day 1 out of 46 MPPs generates a CLP, while 1 MPP generates 4 CMPs (Fig. 3d, f). Given that the cell numbers also increase from HSC to ST-HSC and MPP (Fig. 3f), the efflux of cells exceeded influx in all of these compartments. To maintain compartment size, this flux difference is balanced by net proliferation (efflux minus influx = net proliferation). The rates of net proliferation
26 FEBRUARY 2015 | VOL 518 | NATURE | 543
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
a Upstream Reference Downstream (HSCs) (ST-HSCs) (MPPs) n n lu i o © 03 o “iğ o—— 0 a, a, o o o —- 064859 —>- 00 Cy o.o o 00400 °° Influx + net proliferation = efflux ayy + Bak = ape b BazÂp-da Upstream Reference (HSCs) (ST-HSCs) o o o 000000 000000 ÇO) 000000 oo 00 (ee) oo Vee Gi for” jor Gi © o Oo ©600000 Cell gain per time Cell loss per time unit by influx unit by efflux Increase of labelling frequency f in reference compartment per time t d e 10! a 107 8 2 8 10° Eğ 109 SE 2s gv 107 5x 107 gs 38 58 Şi 102 & 102 52 8 3 zo 104 = 10° £2 O RL ge LS CLS 8s Se? N Se I ES vw © CLPs f 259, 18 g © tepe) ole HSC ST-HSCs_— MPPs “gs, 909090 i 1/110 1/22 OREO: 58 per day (Oi per day OOO 52 ö—o— 000 4, 7 a> O o 000 95, CMPs Se 1/110 oO L 309909 25 perday o 4/24 9588 23 Se o çin çay BE Residence time: 330days 70 days © 6° Se Flux amplification: 15x 280x ce HSCs Be 29 0015 İz 0.010 i i Eğe teh BE 0.005 sa 0, 0 100 200 Time (days) 10 ST-HSCs 28 “| Data used 52 0.8} for fitting gs 08} eo N se om Data not used So 02 for fitting o 0 150 300 450 Time (days) 4 10 MPPs 38 os ov 25 06 ze 3 2 0.4 ay 2 o 150 300 450 Lo Time (days) N 10 CMPs 1.0 CLPs 08 53 os sf 0.6 go 06 ge 0.4 =2 04 02 a $02 o eee 0 150 300 450 0 150 300 450 Time (days) Time (days) h ST-HSCs Ni 10° 1.0} Data used 2 08 for fitting ii 10° ; S5 0.6 Bİ to 0.4 —— —— = 03 Data not used Bs 102 v for fitting 2 6 % 150 300 450 600 750 10 rg ‘ 2 Time (days) ws oF MPP->CMP
> ox fe)
MPP->CMP MPP->CLP
Figure 3 | Inference of stem and progenitor cell differentiation, and proliferation from label progression. a, At steady state, the rate of cell loss ina reference compartment due to cell differentiation and death is balanced by cell influx from the upstream compartment, and by proliferation within the reference compartment. This balance relates the total upstream and reference compartment sizes (ny and np, respectively) to the rates of cell differentiation (ay and ap) and to net proliferation (proliferation — death) (Bg = Ap — ÖR). b, The label frequency in the reference compartment (fg) equilibrates over time with the label frequency in the upstream compartment (fu). The time for label equilibration Tk = 1/(% — Bp) (residence time) is determined by how rapidly cells are lost from the reference compartment. c, HSC label over time (blue dots; red dashed line, average), label progression (blue dots, with s.e.m.,
as in Fig. 21), and mathematical model fitting (red lines, best fit; grey shades, 95% confidence bands). Data measured at ~400 days were not used for fitting (green points, n = 11 mice) but have been predicted correctly by the model. d,e, Inferred average rates of cell differentiation (d) and net proliferation (e) for stem and progenitor cell compartments (with 95% confidence intervals). f, Dynamics of stem and progenitor cell compartments inferred from the experimental data. Relative compartment sizes are symbolized by grey boxes, and magnitude of fluxes by arrow width. g, The model based on data in young mice (blue dots) also predicts data obtained in very old mice (green dots). h, Inferred rates of myeloid and lymphoid differentiation from MPPs in younger (7-238 days after label induction; CMPs only considered beyond 42 days) (n = 110) and older (332-802 days) (n = 41) mice.
increased from HSC via ST-HSC to MPP compartments, in parallel with the differentiation rates (Fig. 3d, e).
Together, the differentiation rate and net proliferation determine how close a compartment operates to self-renewal. To quantify the degree of self-renewal, we define the residence time in a compartment as the time period in which the compartment would decay to 37% (1/e) of its size, if all influx were switched off. The residence time is determined by the duration a cell and its progeny spend in the compartment before being lost by differentiation or cell death. The compartment residence times can be estimated from the labelling data (Supplementary Methods).
For HSCs, given that the labelling frequencies are maintained over time (Fig. 3c) despite efflux (Fig. 3d), net proliferation ensures complete self- renewal, and the residence time is theoretically infinite. Linking our esti- mate of the HSC net proliferation rate (~1 out of 110 per day; Fig. 3e, f) to proliferation measurements would imply that in assays using 5-bromo- 2'-deoxyuridine (BrdU) ~1% of HSCs were labelled per day if HSCs lived indefinitely (and ~2% if HSC lifetime was 100 days). This figure is in the order of magnitude of the reported ~4% BrdU labelling per day in the ‘HSC-1’ population (as a subset of LSK SLAM-defined HSCs)”, suggesting that the Tie2” HSCs we label in situ reside at the top of
544 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
> Ze Tie2CM Rosa = 25 O 47 1012 18 Days 5 4 8 SFU Tamoxifen §-FU 3 2 Resi Blood analysis 5 o 3 Bone marrow analysis o ean RA ğ e 12 ye 7 © dg © Date: fold increase after S-FU Model: HSC and ST-HSC activated -@ Model: ST-HSC activated ~@ Model: HSC activated 03: 2 on e N Label frequency relative to HSCs PBS control P* cells 5-FU per YEP cells Pt e 00. ST-HSC MPP CLP CMP GMP MEP > ST-HSC_ MPP CMP GMP MEP
Figure 4 | In situ HSC response to 5-FU challenge. a, Experimental outline. b, Mean of absolute leukocyte numbers in peripheral blood on the indicated days after 5-FU (n = 12, red dots) or PBS (n = 9, black dots) injections. c, Labelling frequencies of the indicated populations relative to HSCs after 5-FU (n= 18, red bars) or PBS (n = 15, grey bars) (mean from days 12 and 18). *P = 0.031 (ST-HSCs), 0.003 (MPPs), 0.213 (CLPs), 0.002 (CMPs), 0.034 (GMPs) and 0.013 (MEPs) (two-tailed t-test assuming non-equal variances; 5-FU versus PBS). d, Simulated increase (coloured dots and lines) in YFP labelling frequencies considering participation of HSCs and/or ST-HSCs versus experimental data (black dots; ratios of 5-FU over PBS frequencies, taken from c). Error bars denote s.e.m.
Lele a A, = b 100 5 yi Sag li / € (> a TEZ “Rosa? Analysis after WE Analysis ğ 6 e (CD45.2) 25 weeks-6 months (CD45.1) ater Ste weeks 2 pa treo İNE Shoe Tamoxifen dn Input Output o (96 YFP* HSCs) (% YFP* HSCs) cs 20.0 z 100 ğ 10 a & g d 3 £ 2 2 Output > input 3 3 3 $2: 3 2? output < input Individual recipient mouse
Figure 5 | Fate of in-situ-labelled HSCs after bone marrow transplantation. a, Experimental outline. b, Chimaerism of donor HSCs in recipient mice (n = 32) (mean, horizontal line). c, Frequencies of YFP * HSCs (black dots; level indicated by red dashed lines) in individual donor mice, and of YFP* donor HSCs (white dots) in the corresponding recipient mice. Representative data from a total of 11 donors and 32 recipients are shown. Grey shaded areas denote 95% confidence intervals of sampling errors. d, Ratio of engrafted HSC label output over donor HSC input. Red dashed line (ratio = 1) indicates equal output and input (nı = 32 recipients).
the HSC hierarchy. For ST-HSCs, net proliferation almost accounts for the efflux towards MPPs, and the compartment requires only minimal influx from HSCs. Hence, even ST-HSCs operate near self-renewal, their residence time is exceedingly long (330 days), and label progression does not reach equilibrium within the ~2-year lifetime of a mouse (Fig. 3g). Substantial self-renewal (residence time 70 days) was found even at the MPP stage. Proliferation of MPPs leads to an efflux of cells into common lymphoid progenitors (CLPs) and CMPs that is approximately ~280 times the influx from ST-HSCs (Fig. 3f), making the MPPs a key amplifier. We also analysed progenitor compartments downstream from CLPs (pro B) and CMPs (MEPs and GMPs) (Extended Data Fig. 6d).
An imbalance between myeloid and lymphoid production has been viewed as an age-dependent HSC property”. We used fate mapping to re-address this question independent of transplantation, and found a marked myeloid bias that, however, was only marginally accentuated with age through relative loss of lymphoid potential (Fig. 3h). Such bias could be caused by skewed differentiation from a common progenitor, or by preferential proliferation in the myeloid branch. Phenotypes and designations of tested progenitors at putative branch points are shown in Extended Data Fig. 7. In both scenarios, which are not mutually exclusive, production of CMPs was several-hundred-fold larger than that of CLP.
To examine the responsiveness of haematopoiesis to perturbation, we challenged HSC-labelled mice with a single injection of 5-fluoruracil (5- FU), a cytotoxic agent causing transient leukopenia in the blood (Fig. 4a, b). After peripheral rebound, we observed higher stem and progenitor cell labelling frequencies, relative to HSC labelling, than in untreated mice (Fig. 4c). This accelerated label equilibration between HSCs and sub- sequent compartments after haematopoietic injury indicates feedback inhibition on HSCs output under steady state. These data are fit by a model in which the kinetics of net proliferation and differentiation are accelerated (at least) in both HSC and ST-HSC compartments, but notin either alone (Fig. 4d and Supplementary Discussion).
1-4 lethally irradiated recipients (CD45.1) (Fig. 5a). For each donor, the HSC labelling frequency was recorded before transplantation (‘input’). Despite uniformly strong donor HSC engraftment after 16-18 weeks (average 90%; Fig. 5b), the percentages of YFP-marked cells among total donor HSC (‘output’) were highly variable compared to the input frequencies in individual recipients (Fig. 5c). Input and output were roughly equal (within sampling error) in only 6 out of 32 recipients, whereas in most recipients donor HSCs were either lost (14 out of 32) or overrepresented (12 out of 32) (Fig. 5d). We estimate that on average 1 out of 33 donor HSCs engrafted (Extended Data Fig. 8a-c). In two ex- treme cases, YEP” donor HSCs represented only 0.3% or 0.6% of the input, but 17% or 35%, respectively, of the output, suggesting much stronger proliferation than for HSCs under steady state (Extended Data Fig. 8d and Supplementary Discussion). In summary, the participa- tion of individual engrafted HSCs to the repopulation of the bone mar- row is highly heterogeneous.
Inducible labelling of HSCs in normal mouse bone marrow showed that during development HSCs are rapidly used to establish the haema- topoietic system. Once this is accomplished, individual HSCs are only rarely active, but over time a large portion of HSCs contributes to adult haematopoiesis. Indeed, although the mean HSC labelling frequency was low, all mice with marked HSCs produced labelled progeny, indi- cating that a large fraction, or at least 30%, of all HSCs contributes to haematopoiesis in adult mice after label induction. By contrast, pre- vious work based on barcoding showed that few HSCs actively drive haematopoiesis after transplantation. We re-addressed HSC diversity in the wake of transplantation, avoiding potential pitfalls of cellular heterogeneity of mixing experiments. The observed HSC oligoclonality is hence a hallmark of post-transplantation but not normal unperturbed haematopoiesis. These findings indicate that experimental and possibly also clinical HSC transplantations are based on a much smaller stem cell foundation than physiological haematopoiesis.
Output from highly polyclonal HSCs in adult haematopoiesis in situ (Fig. 1b) is in contrast to oligoclonal HSC activity found in transplanta- tion experiments’ '*. To address this discrepancy, we followed the fate of in-situ-marked HSCs after conventional bone marrow transplantation. Adult Tie2““”~ Rosa*"? mice were treated with tamoxifen, and Lin” Kit” bone marrow cells from each donor (CD45.2) were transplanted into
HSC proliferation has often been taken as a proxy for asymmetric cell division (for example, ref. 23) and, also indirectly, as measure of differ- entiation rates. Proliferation may, however, not only yield differentiating progeny but also compensate for cell loss, precluding proliferation as an unambiguous marker of differentiation rates. Here, we quantified haematopoietic flux based on label progression from HSCs. Our data
26 FEBRUARY 2015 | VOL 518 | NATURE | 545
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
support in principle an order of differentiation in situ from HSCs to ST-HSCs and MPPs and onwards. However, in divergence from the idea of a continuous stream from the top of a haematopoietic pyramid, a very low flux emanated from HSCs. While ST-HSCs are relatively short-lived on transplantation, in situ this compartment is exceedingly long-lived because ST-HSC self-renewal is almost sufficient to make up for the cell loss by differentiation. We hence identified this com- partment as the primary source of haematopoietic maintenance in mice. This reservoir property of ST-HSCs readily explains the apparent HSC-independence of haematopoiesis noted in a recent report™. How- ever, to maintain the ST-HSC compartment in the long run (>1 year), it requires continuous input from HSCs; we estimate that per day 150 HSCs feed into this compartment (17,000 total HSC 1/110 differ- entiating per day). Hence, true HSC deficiency may go unnoticed for extended periods of time while functionally impaired ST-HSC and MPP compartments would cause rapid signs of acute bone marrow failure (Extended Data Fig. 9).
- Dick,J.E, Magli, M. C., Huszar, D., Phillips, R. A. & Bernstein, A. Introduction of a selectable gene into primitive stem cells capable of long-term reconstitution of the hemopoietic system of W/W mice. Cell 42, 71-79 (1985).
9.
- 10. Gerrits, A. et a/. Cellular barcoding tool for clonal analysis in the hematopoietic system. Blood 115, 2610-2618 (2010).
- 11. Lu,R, Neff, N. F., Quake, S. R. & Weissman, |. L. Tracking single hematopoietic stem cells in vivo using high-throughput sequencing in conjunction with viral genetic barcoding. Nature Biotechnol. 29, 928-933 (2011).
- 12. Naik, S. H. et al. Diverse and heritable lineage imprinting of early haematopoietic progenitors. Nature 496, 229-232 (2013).
- 13. Harrison, D. E., Lerner, C., Hoppe, P. C., Carlson, G. A. & Alling, D. Large numbers primitive stem cells are active simultaneously in aggregated embryo chimeric mice. Blood 69, 773-777 (1987).
14.
- Yano, M. etal. Expression and function of murine receptor tyrosine kinases, TIEand TEK, in hematopoietic stem cells. Blood 89, 4317-4326 (1997).
- 15. Hsu,H.C. etal. Hematopoietic stem cells express Tie-2 receptor in the murine fetal liver, Blood 96, 3757-3762 (2000).
- 16. Zhang, Y. et al. Inducible site-directed recombination in mouse embryonic stem cells. Nucleic Acids Res. 24, 543-548 (1996).
- 17. Weissman, |. L.Stem cells: units of development, units of regeneration, and units evolution. Cel/ 100, 157-168 (2000).
Collectively, HSCs act in development as founding stem cells, and in adult mice as replenishing cells, ST-HSCs as long-term amplifying cells, and MPPs as intermediate-term amplifying cells. The described fate-mapping system may also visualize responses to haematopoietic challenges imposed by cancer, infections, cachexia or ageing. The accel- erated HSC output in response to haematopoietic injury by treatment with 5-FU underscores this outlook.
- 18. Oguro,H, Ding, L. & Morrison, S. J, SLAM family markers resolve functionally distinct subpopulations of hematopoietic stem cells and multipotent progenitors. Cell Stem Cell 13, 102-116 (2013).
- 19. Waskow, C. et al, Hematopoietic stem cell transplantation without irradiation. Nature Methods 6, 267-269 (2009).
- 20. Boggs, D. R. The total marrow mass of the mouse: a simplified method of measurement. Am. J. Hematol. 16, 277-286 (1984).
- 21. Gomez Perdiguero, E. et al. Tissue-resident macrophages originate from yolk derived erythro-myeloid progenitors. Nature http://dx.doi.org/10.1038/ nature13989 (2014).
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
- 22. Geiger, H., de Haan, G. & Florian, M. C. The ageing haematopoietic stem cell compartment. Nature Rev. Immunol, 13, 376-389 (2013),
- 23. Wilson, A. etal. Hematopoietic stem cells reversibly switch from dormancy to self- renewal during homeostasis and repair. Cel/ 135, 1118-1129 (2008).
- 24. Sun,J.etal, Clonal dynamics of native haematopoiesis. Nature 514, 322-327 (2014).
# Received 11 July 2014; accepted 19 January 2015. Published online 11 February 2015.
- 1. Smith, L. G., Weissman, |. L. & Heimfeld, S. Clonal analysis of hematopoietic stem- cell differentiation in vivo. Proc. Nat! Acad. Sci. USA 88, 2788-2792 (1991).
- 2. Osawa, M., Hanada, K., Hamada, H. & Nakauchi, H. Long-term lymphohematopoietic reconstitution by a single CD34-low/negative hematopoietic stem cell. Science 273, 242-245 (1996).
- 3. Kiel, M. J., Yilmaz, O. H., lwashita, T., Terhorst, C. & Morrison, S. J. SLAM family receptors distinguish hematopoietic stem and progenitor cells and reveal endothelial niches for stem cells. Cel! 121, 1109-1121 (2005).
Supplementary Information is available in the online version of the paper.
Acknowledgements We thank C. Blum for blastocyst injections, T. Arnsperger, N. Maltry and S. Schafer for technical assistance, and the animal facility at the DKFZ for expert mouse husbandry. We are grateful to N. Dietlein, H. J. Fehling, T. Feyerabend, C. Opitz, J. Rodewald, A. Roers, K. Rajewsky and A, Trumpp for discussions. K.B. was initially funded through the International Graduate School in Molecular Medicine, Ulm. MR. was supported by the DFG through EXC294, TRR130 and SFB746. T.H. is a member of CellNetworks, and was supported by BMBF e:Bio grant T-Sys (Fkz 031614) and EU-FP7 Marie Curie ITN Quantitative T cell immunology (QuanT)). H.-R.R. was supported by DFG-SFB 873-project B11, DFG-SFB 938 project L, ERC Advanced Grant no. 233074, and the Helmholtz Alliance on preclinical cancer models (PCCC).
- 4. Sieburg, H. B. et a/. The hematopoietic stem compartment consists of a limited number of discrete stem cell subsets. Blood 107, 2311-2316 (2006).
- Dykstra, B. et al. Long-term propagation of distinct hematopoietic differentiation programs in vivo. Cell Stem Celi 1, 218-229 (2007).
5.
- Abramson,S, Miller,R.G.& Phillips, R.A. The identification in adult bone marrow of pluripotent and restricted stem cells of the myeloid and lymphoid systems. J. Exp. Med. 145, 1567-1579 (1977).
6.
Author Contributions K.B. designed and performed experiments, K.K. and S.M.S. performed experiments, M.B. and M.F. performed theoretical modelling, T.H.-L. made initial bioinformatic analyses, M.R. contributed to the tamoxifen-regulated Cre construct, T.H. developed mathematical models and wrote the paper, and H.-R.R. conceived and supervised the study, and wrote the paper.
- 7. Keller, G., Paige, C., Gilboa, E. & Wagner, E. F. Expression of a foreign gene in myeloid and lymphoid cells derived from multipotent haematopoietic precursors. Nature 318, 149-154 (1985).
- Lemischka,I.R. Raulet, D. H. & Mulligan, R.C. Developmental potential and dynamic behavior of hematopoietic stem cells. Cel/ 45, 917-927 (1986).
8.
Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to H.-R.R. (hr.rodewald@dkfz.de) or TH. (thoefer@dkfz.de).
546 | NATURE | VOL 518 | 26 FEBRUARY 2015
©2015 Macmillan Publishers Limited. All rights reserved
# of
# in
# sac-
LETTER
doi:10.1038/nature13989
# Tissue-resident macrophages originate from yolk-sac-derived erythro-myeloid progenitors
Elisa Gomez Perdiguero'*, Kay Klapproth”*, Christian Schulz', Katrin Busch?, Emanuele Azzoni?, Lucile Crozet', Hannah Garner’, Celine Trouillet', Marella F. de Bruijn*, Frederic Geissmann!$ & Hans-Reimer Rodewald?$
Most haematopoietic cells renew from adult haematopoietic stem cells (HSCs)'-, however, macrophages in adult tissues can self-maintain independently of HSCs*’. Progenitors with macrophage potential in vitro have been described in the yolk sac before emergence of HSCs*", and fetal macrophages'*"* can develop independently of Myb', a transcription factor required for HSC'“, and can persist in adult tissues*’”"*, Nevertheless, the origin of adult macrophages and the qualitative and quantitative contributions of HSC and putative non-HSC-derived progenitors are still unclear’. Here we show in mice that the vast majority of adult tissue-resident macrophages in liver (Kupffer cells), brain (microglia), epidermis (Langerhans cells) and lung (alveolar macrophages) originate from a Tie2* (also known as Tek) cellular pathway generating Csfir* erythro-myeloid progenitors (EMPs) distinct from HSCs. EMPs develop in the yolk sac at embry- onic day (E) 8.5, migrate and colonize the nascent fetal liver before E10.5, and give rise to fetal erythrocytes, macrophages, granulocytes and monocytes until at least E16.5. Subsequently, HSC-derived cells replace erythrocytes, granulocytes and monocytes. Kupffer cells, mi- croglia and Langerhans cells are only marginally replaced in one- year-old mice, whereas alveolar macrophages may be progressively replaced in ageing mice. Our fate-mapping experiments identify, in the fetal liver, a sequence of yolk sac EMP-derived and HSC-derived haematopoiesis, and identify yolk sac EMPs as a common origin for tissue macrophages.
Csfir-expressing cells in the mouse embryo give rise to tissue-resident macrophages in adult tissues’. To identify in the developing embryo the site of origin of Csfir-expressing cells, we performed time course analyses by constitutive (Csf1r'“") and inducible (Csf1r"°""™"”) fate- mapping of cells in the yolk sac, head, limbs, caudal region and fetal liver (Fig. 1a and Extended Data Fig. 1). Progenitors, defined as Kit” CD45'° (ref. 12) (gate RI in Fig. 1b), were first detected in Csf1r ““Rosa26'”” embryos in the yolk sac from 16-18 somite pairs (sp) stage onwards (E8.5, Fig. 1b, and Extended Data Fig. la-c). Csf1r“”* YFP* Kit” CD45~ cells (gate R2 in Fig. 1b), characterized in Fig. 2 as myeloid cells, were detected in the yolk sac at 20-25 sp (E9, Fig. 1b), and subsequently in the caudal and head regions of the embryo from E9.5, and the fetal liver from E10.5 onwards (Extended Data Fig. la-d). To discriminate mi- gration of YEP” cells from de novo labelling, we induced YFP expression in Csfir erer Rosa26"'” embryos at E6.5 or E8.5. In embryos pulsed at E6.5, YEP” cells were not detected (Extended Data Fig. 2a, b). When pulsed at E8.5, Csfr“““““Mer YEP* Kit” CD45” progenitors were detected between E9.5-11.5 in the yolk sac, and in the fetal liver from E10.5 (Fig. 1c, d). In the fetal liver, numbers of YEP” Kit*CD45” progenitors increased threefold from E10.5 to E11.5, at which time they were 25- fold more numerous in the fetal liver than in the yolk sac (Fig. 1d). At E8.5, all YS Csf7r“”* YEP* Kit* CD45"° progenitors expressed AA4.1, an antigen expressed on early haematopoietic progenitors”? (Extended Data Fig. le). Csfir““"İ“M YEP AA4.1” Kit” CD45" cells were also
present in the yolk sac from E9.5 to E10.5, and in the fetal liver from E10.5 (Fig. le). These progenitors were undetectable at E10.5 in the aorta- gonado-mesonephros (AGM) region (Fig. le), indicating they do not originate within the embryo proper.
Together, these fate-mapping experiments demonstrate that yolk-sac- derived progenitors colonize the liver primordium, as proposed earl- ier**°*', and their expression of AA4.1 suggests that they represent erythro-myeloid progenitors (EMPs)”. In in vitro colony-forming assays, the AA4.1” population contained most of the total E9 yolk sac colony- forming-units-culture (CFU-C 266 + 137 vs 296 + 75, mean +stand- ard deviation (s.d.)). Frequencies and distributions of different CFU-C, that is, erythroid (E)/megakaryocyte (Mk) (E/Mk), granulocyte/macro- phage (G/M), and G,M,E, and/or Mk (Mix) potential, were comparable between overall AA4.1~ and Csfir““ YEP* AA4.1” progenitors (Fig. 2a, Extended Data Fig. 3). Moreover, in the E12.5 fetal liver, the CFU po- tential of overall AA4.1* and Csftr"'"™"" YEP* AA4.1” cells was comparable to the yolk sac progenitors (Fig. 2a).
These results indicated that yolk-sac-derived, E8.5-labelled YEP” AA4.1 “Kit” CD45" progenitors have erythroid and myeloid potential in yolk sac and fetal liver. Next, we investigated by fate-mapping their contribution to fetal liver haematopoiesis in vivo. Csfir'““ YEP” and CsfilrMeriCreMer YEp* F4/80°"8"" fetal macrophages were first detected among Kit CD45” (R2 in Fig. Ib) at E10.5 in the yolk sac, liver, head and forelimbs (Fig. 2b, Extended Data Fig. 4a-c). In addition, the fetal liver from E12.5 to E16.5 contained Csf1r““"“reMer YEP* monocytes and granulocytes (Fig. 2c). The fetal liver also contained Csf1reriCreMer YEP” red blood cells from E11.5 until at least E14.5 (Fig. 2d, Extended Data Fig. 4d). Red blood cells were not labelled before E11.5, indicating that, in contrast to yolk-sac-derived erythrocytes in the fetal liver, prim- itive erythrocytes in the yolk sac did not arise from Csflr-expressing cells. Collectively, yolk-sac-derived Csfir” progenitors contribute to fetal liver haematopoiesis by giving rise to F4/80°"8"" macrophages, monocytes, granulocytes and red blood cells.
We next investigated the transition from yolk-sac-derived to HSC- derived haematopoiesis. To trace the latter, we used Flt3™ which labels fetal and adult HSC-derived multipotent haematopoietic progenitors”, and their progeny (Extended Data Fig. 5). We compared progeny of yolk-sac-derived progenitors in Csf1r"""""*" mice to progeny of HSCs in Flt3“* mice. In the fetal liver from E14.5 to E18.5, the progenies of Csfir* and Flt3” precursors were distinct but complemented each other (Fig. 3a). AtE14.5, yolk-sac-derived CD45~ populations included Kit” progenitors, F4/80°"" macrophages, and CD11b"Gr1* monocytes/ granulocytes (Fig. 3a, Extended Data Fig. 6a). Of note, monocytes/ granulocytes were present in Myb-deficient fetal liver (Fig. 3a). CsflrMeriSreMer YEP* macrophages remained detectable throughout fetal development, and were not replaced by Fl3“”“ YEP” cells. However, yolk-sac-derived Kit * cells and myeloid cells were no longer detectable by E16.5 and E18.5, respectively (Fig. 3a). In contrast, Fl43“”“ YEP” Kit”
Centre for Molecular and Cellular Biology of Inflammation (CMCBI), King's College London, London SE1 1UL, UK. Division of Cellular Immunology, German Cancer Research Center (OKFZ), D-69120 Heidelberg, Germany. MRC Molecular Haematology Unit, Weatherall Institute of Molecular Medicine, John Radcliffe Hospital University of Oxford, Oxford OX3 9DS, UK. *These authors contributed equally to this work.
SThese authors jointly supervised this work.
26 FEBRUARY 2015 | VOL 518 | NATURE | 547
# ©2015 Macmillan Publishers Limited. All rights reserved
75 85 95 105 11.5 125 a ee Csfiri&e Rosaz6v"? A OMÜ ©) CsfirMerCrehler RosazgYP” Pulse : OH-TAM E8.5 b Gstir EB.25 Osp 3-6 sp 7-9 sp t 0% 0% < bi ‘ i J â | || Rg YFP Head Vitelin & Yolk (> al se, İNME | çanlar ae A
Figure 1 | E8.5 Csflr* progenitors originate in the yolk sac and expand in the fetal liver. a, Fate-mapping analysis of Csflr-expressing cells. Arrows indicate time points for analysis, and green shades the genetic labelling period. b, YEP expression on live cells from Csf1r “““Rosa26””” yolk sac (YS), separated by somite pairs between E8.25 and E9 (0 sp, n = 3; 3-6 sp, n = 3; 7-9 sp, n= 5; 11-13 sp, m = 3; 16-18 sp, n = 4; 20-25 sp, n = 4) (upper panels), and Kit and CD45 expression on YFP* cells (lower panels). RI indicates Kit* CD45", and R2 indicates Kit CD45" cells. c, Schematic representation of sites analysed in mouse embryos: YS, AGM region, fetal liver and head. Kit and CD45 phenotype of YFP* cells from Csfir“CreMer Rosa26""” embryos pulsed with OH-TAM at E8.5 (E9.5, n = 3; £10.25, n = 3; E10.5, n= 4; E11.25, n = 4; E12.5, n =9). d, Number of YEP *Kit”CD45' cells (RI in panel b) per organ or region (mean * s.e.m.) in Csflr"“reMe Rosa26"™” embryos pulsed at E8.5 (Source Data Table for Fig. 1). e Number of YFP* AA4.1* Kit* CD45" cells 3 per embryonic region and time points (mean + s.e.m.) in CsfirMeriGeMer Rosa26*"” embryos pulsed at E8.5 (Source Data Table for Fig. 1). See also Supplementary Table 1 and Extended Data Figs 1 and 2.
# AGM region
# d
# e
Gauctal sf, pulsed at E8.5 YFP* colls £95 £10.25 E105 E115 125 İl İ | Ki Pi % Ys ee F x ee | * — 4 LE Caudal | wa (AGM) öğ # # di z > L ğ it | | Head | ik 1 j y | ee ae İ Fetal L a liver ee i ep4s > 2200 Yolk sac CaudavAGM Head/brain Fetal liver 1800 _ 1400 4 1,000 4 aE 4 8 Lp 600 28 is ğ 3 400 “200 04 = = RP PEP? KP? 8.n2o? Yolk sac Caudal/AGM Head/brain Fetal liver g 500. i & mk. 85 200. “ 3 B 200. = 8 100. E Ls =
AP?
# e?
# Sea?
# oP Pa?
cells,and CD11b"Gr1* granulocytes/ monocytes increased in numbers between E14.5 and E18.5 (Fig, 3a-c). The progenies of Csfir” progeni- tors and Fit3” progenitors were also distinct during development in the lung and skin (Extended Data Fig. 6b, c). Quantitative analyses in fetal and adult tissues indicated that Flt3“* YEP labelling of Kit” progeni- tors preceded that of monocytes/granulocytes, with 80% of progeni- tors labelled at E18.5, and 80% of monocytes at postnatal day 8 (P8) (Fig. 3b). In contrast, FIt3“"* YEP labelling plateaued at 14% for adult liver F4/80”"** Kupffer cells, at 2% for CD45" brain microglia, and 30% for epidermal Langerhans cells up to one year of life (Fig. 3b). In con- trast, Flt3“"* YEP labelling of CD45* F4/80~ brain macrophages, and lung alveolar macrophages was 16% in 12-week-old adults but increased progressively over time to reach 40% in one-year-old mice (Fig. 3b).
Altogether, these data indicate that Flt3“ YFP” Kit” progenitors and monocytes account for only minor fractions of microglia, Kupffer cells, alveolar macrophages and Langerhans cells in young adults. To investi- gate whether the presence of these adult Fl#3“” YEP” F4/80””* macro- phages corresponds to their HSC origin, we performed non-myeloablative transplantations of YEP” long-term-HSCs (LT-HSCs) from adult wild- type bone marrow into Rag2”” 7,“ Kit" recipients” (Extended Data Fig. 7). Eight weeks after transplantation, the vast majority of HSCs, myeloid progenitors, monocytes and F4/80” tissue myeloid cells in the recipients were of donor HSC origin. In contrast, only 7% of F4/80°"2" macrophages in spleen, 2% in liver, 5% in lung, 13% in pancreas, 2% in epidermis and 0% in the brain were donor-derived. Thus, recruitment of HSC-derived precursors is not a major mechanism for the mainte- nance of F4/80°""' macrophages in these tissues.
Collectively, these findings reveal that the transition from yolk-sac- to HSC-derived haematopoiesis occurs late in fetal development for mono- cytes (E14.5) and granulocytes (E16.5), and suggest that HSC-derived progenitors only marginally replace yolk-sac-derived microglia in the brain, Kupffer cells in the liver, Langerhans cells in the epidermis, although alveolar macrophages and brain CD45~ F4/80* macrophages may undergo progressive replacement with age.
Labelling efficiency of most tissue-resident macrophage popula- tions in adult Csf1r“""™""Rosa26""” mice pulse-labelled with 4- hydroxytamoxifen (OH-TAM) at E8.5 was low'**, The strength of most genetic pulse-labelling systems is that they allow fate-mapping of cells during a specific time window, however, a weakness is the commonly incomplete labelling which could explain why a large fraction of tissue- resident macrophages remained unlabelled. Hence, based on these data we cannot formally exclude a fetal HSC origin of the unlabelled cells as suggested by others based on transfer of fetal precursors****.
We thus made use ofa newly generated inducible Cre knock-in mouse (Tie2MeriCreMer) to track haematopoietic output from haematopoietic progenitors and HSCs in situ (Busch K. et al., submitted). Tie2 (also known as Tek) is expressed in endothelial cells, yolk sac progenitors, aorta-gonado-mesonephros region, fetal liver and adult HSCs*. We as- sessed the time window at which 7/e2” cells contributed to emerging HSCs and macrophages by injecting tamoxifen at different time points (Fig. 4a, Extended Data Figs 8-10). Fetal liver E12.5 and E15.5 LT-HSCs were labelled efficiently in Tie2““”r“Mer embryos pulsed at E6.5, E7.5 or E10.5 (Fig. 4b, Extended Data Figs 8 and 9). Yolk sac E9.5 Kit*Cp45"° progenitors were also labelled in Tie2°"™"'Rosa26"'” embryos pulsed at E7.5 (Extended Data Fig. 10). Interestingly, fetal liver cells with a megakaryocyte-erythrocyte progenitor (MEP) phenotype, and F4/80°"8 macrophages in yolk sac, brain, and fetal liver were labelled with high efficiency (60%) in embryos pulsed at E6.5 and E7.5, but not in embryos pulsed at E10.5 (Fig. 4b, Extended Data Fig. 8). These fate- mapping experiments directly demonstrate that E12.5 and E15.5 fetal macrophages originate from cells that express Tie2” as early as E6.5
548 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# a
CsfirMenceler, pulsed at E8.5 Colony forming unit (CFU}- C b pı & Cottrercretter, pulsed at 8.5 d £9.5 E108 E18 £125 Caton mn f £125 E165 £165 pulsed at EB.5 | x | # aw pi ‘ ioht YEP = 100 = 100 { l l > z Kİ — ee İğ | = 8 | Lam = 30 3 80 B 80 3 4 * NM | 2a 3 Faso ———> — 4 3 2 | 8 6 i ya ie 160) | | ” gi CD11b' YFP 3 20 > 5 = 40 5 40 | * ; tp 3 5 5 8 - Şİ AA. i _# o & § 20 | 3 2 § Li | | cD11b ——> © ie Bo mi b Zo a 8 a KA Gated CD11b* F4/80” x 3 SE el | t : yf US ¥ eS 5 & SO > e» E g EE kl el me 5 ( pulsed at E8.5 Ly-6C > "
Figure 2 | E8.5 Csfir* progenitors differentiate into myeloid cells and red blood cells in the fetal liver. a, Distribution of mixed (Mix), G/M, and E/Mk CFU-C from unsorted, AA4.1* Kit" CD45"° and YFP* AA4.1*Kit' CD45” cells from E9 yolk sac from Csfir'“““Rosa26””” and E12.5 fetal liver from sf" Rosa26"*” embryos pulsed with OH-TAM at E8.5 (three independent experiments each). CFU-C erythroid and/or megakaryocyte (E/Mk); CFU-granulocyte and/or monocyte/macrophage (G/M); CFU-C mix, at least three of the following: G, E, M and Mk. See also Extended Data Fig. 3. b, F4/80 and CD11b expression on YFP*CD45* from yolk sac, head (brain for E11.5), limbs and liver of Csf1r““"“MrRosa26”F embryos pulsed with OH-TAM at E8.5, and analysed on E9.5 (n = 3), E10.5 (n = 4), E11.5 (n = 4), and E12.5 (n = 9); Dashed lines represent FMO (fluorescence minus one)
control. See also Extended Data Fig. 4. c, F4/80, CD11b, Grl, Ly-6G and Ly-6C expression in fetal liver CD45 *YFP* cells from sft" Rosa26""® embryos pulse-labeled at E8.5, and analysed on E12.5 (n = 9) and E16.5 (n = 14). May-Griinwald-Giemsa stained cytospin preparations of fetal liver YFP * F4/80°"*"' and YFP *CD11b™ cells sorted from E16.5 Csfir“""™" Rosa26""? embryos pulsed with OH-TAM at _ E8.5. See also Extended Data Fig, 6a for sorted cells from E14.5 CsfipMeriCreMer embryos. Scale bar, 10 jum. d, YFP labelling efficiency (%) among red blood cells in fetal liver from CsfIr“"""™""Rosa26"*? embryos pulsed with OH-TAM at E8.5, mean + s.e.m. (E11.5, n = 4; E12.5, n= 9; E14.5, n = 5; E16.5, n= 11; E18.5, n = 4; see Source Data Table Fig. 3).
and, importantly, before E9.5, and strongly support the notion that fetal liver erythro-myeloid progenitors, and all fetal tissue macrophages up to E15.5 are of yolk sac origin.
In adult mice pulsed at embryonic stages (E7.5, or E8.5, or E9.5 or E10.5), bone marrow HSC-derived progenitors, peripheral cells (T and B cells, and granulocytes) in the spleen, and CD11b"F4/80" myeloid cells
a Fetal liver Myb7- E14.5 E16.5 E18.5 E14.5 E16.5 E18.5 E16.5 E18.5 E14.5 E16.5 1 a vag İ 3 CD45* k pen = İ CsfrMeriCreMer | # 8 ae yep+ Qe | e pulsed at E8.5 | mi 1 { A3 YFP* | | ee shay | L > CD11b > Gn > b Lineage Kit* Blood monocytes = Bİ F4/80° CD11b” _. WECD45*F4/80° macrophages == «ss F4/80 CD11 _ I Dermal CD11b" ‘3 BS F4/80%"" CD11b* (Kupffer cells) Bop , BB Microglia Sp, Bi Alveolar MF R BI Epidermal LCs . =100 £100 Si £100 ig 2 - 2 2 fy 80 3 80)f ze za & g 60 @ 60 g 60 @ 60 2 = 40 = sok = 40 = 40 = 3 29 8 olf 8 20 3 20 go çi g os M < ii go zo — ÇAP abla? ŞB mi ga gow ay & © OMO goth Ay > O A gl ay ÇO aw AD ai ay Liver Brain Lung Skin
Figure 3 | Fetal liver HSC-derived Fit3* progenitors give rise to monocytes and granulocytes in late embryos and adults but do not replace yolk-sac- derived macrophages. a, F4/80, Kit, CD11b and Grl expression on total CD45" cells (black) and YFP*CD45* cells from Csf1 pMeriCreMer Rosa26'FP embryos pulsed at E8.5 (green) in the fetal liver at the indicated days of embryonic development (E14.5, n = 5; E16.5, n = 10; E18.5, n = 9). F4/80, Kit, CD11b and Gri expression on YFP*CD45* cells from Flt3“““Rosa26””” embryos (orange) (E14.5, n = 7; E16.5, n = 6; E18.5, n = 6). F4/80 and CD11b expression on CD45* cells in Myb “© embryos (E14.5, n = 4; E16.5, n=7). b, YEP labelling efficiency in Kit “lin” cells, CD11b" F4/80"° cells (characterized in Extended Data Fig. 5) and F4/80°"" macrophages (Kupffer cells in adults) in fetal and adult FIt3“°Rosa26"™ liver (first panel on the left).
YFP labelling efficiency in blood monocytes, brain microglia (CD45"°F4/80") and CD45*F4/80* brain macrophages in Flt3“*Rosa26""” pups and mice (second panel). YEP labelling efficiency in alveolar macrophages (F4/80°"#"* Siglec-F’ CD11b_) and F4/80“ CD11b" myeloid cells in Fit3“*Rosa26""” lungs (third panel). YFP labelling efficiency in epidermal Langerhans cells (LCs) and dermal CD11b" (MHC II* EpCAM ) myeloid cells in FIt3“"Rosa26""” skin (fourth panel, see Extended Data Fig. 6b, c). Mean * s.e.m,; P8, n = 3; 4-week-old, n = 6; 12-week-old, n = 11-14; 40-week-old, n = 7; 1-year-old, n = 3, see Source Data table Fig. 3. w, week; y, year. c, Representative images of May-Griinwald-Giemsa stained cytospin preparations of YFP* CD11b"F4/80" cells sorted from E18.5 Flt3“”“ Rosa26””” fetal liver. Scale bar, 10 um.
26 FEBRUARY 2015 | VOL 518 | NATURE | 549
# ©2015 Macmillan Publishers Limited. All rights reserved
# a
# b
# c
2eengnnoew 5 Beeecras soon s F \ Dis gee ne See SIR şe . LE 4 Ft ese ee SS ST Tie2MeriCreMer Rosa26YFP Pulse : TAM at E7.5, | E8.5, E9.5 or E10.5 | AR | TAM: E75 E10.5 ETS E10.5 Analysis: 12.5 E155 E12.5 E155 8 40.7) 7 43.5) 2 i i F4/80%'9" CD11b* 5 ; g 8 9.6) i 21.9) = z iA A dü 5 m dö 96) 12.8) Fa £ = s & 10.2) SA = ö 8 gl 5 $ = ° ğ 70.4 £ 0, & 2 vee — —— Adult tissues
vee — Adult tissues TAM HSCs + Tissue Fa/B>=art 80 Progenitors , SPIeRCCYTES Ea/aq> | macrophages İ i i I ' 1 2 i i & ' ! a i} ' + 4 i H we gü ge 2 5 3 2 3 3 bi WE & g > 80) 1 1 1 i il | cr H i i » 1 i | 3 H i i = 1 i 1 end egm al) çağ H i I 0 1 L 1 Base POSE SOOSE PEE LIYE
tissue-resident macrophages were maximal in animals pulse-labelled at E7.5, declined at later time points and were minimal when labelled at E10.5 (Fig. 4c). The fact that adult HSCs are disconnected from res- ident macrophages is further underscored by the finding that resident macrophages in mice pulsed at E7.5 were labelled at higher frequencies than adult HSCs, that is, labelling efficiency did not equilibrate with mouse development. In summary, these inducible temporal analyses demon- strate that although both macrophages and HSCs originate from progen- itors expressing Tie2 as early as E6.5, adult tissue-resident macrophages in the brain (microglia), liver (Kupffer cells), lung (alveolar macrophages), skin (Langerhans cells), and (to some extent) spleen (F4/30°"8" mac- rophages) develop almost exclusively from an Tie2-expressing progen- itor pathway distinct from HSCs. These data are consistent with results from Csf1r°""""" pulse-labelling experiments (see Figs 1 and 2), with our earlier observation that resident macrophages are independent of the transcription factor Myb*, and complement our data obtained in FIt3“Rosa26""” mice (see Fig. 3).
This study demonstrates that Myb-independent tissue-resident mac- rophages’ originate from yolk-sac-derived EMPs, characterized by expression of CsfIr from E8.5 (16-18 somites). The data do not distin- guish whether resident macrophages originate from erythro-myeloid, granulocyte-macrophage, or macrophage only-progenitors because these potentials coexist within the yolk-sac-derived EMP population.
Wealso provide strong in vivo evidence for engraftment of yolk-sac- derived EMPs in the early fetal liver. These cells substantially contrib- ute to the first wave of fetal liver haematopoiesis, followed later by bona fide fetal liver HSC-derived haematopoiesis*”*”!, Conclusions from re- cent studies that Langerhans cells and alveolar macrophages are not of yolk sac origin based on transfer of fetal precursors”* ” should be inter- preted in light of our findings that yolk sac EMPs expand in the fetal liver and are the main source for tissue-resident macrophages.
Under steady-state conditions, yolk-sac-derived macrophages are only marginally replaced by HSC-derived cells in the brain, liver and epi- dermis. It is remarkable that macrophages of yolk sac origin persist in functionally very distinct tissues, suggesting that the origin is more de- terministic of the life span than the tissue location. However, some yolk- sac-derived macrophages can undergo replacement in older mice, as for lung alveolar macrophages. In a third group, exemplified by gut- associated macrophages”, yolk-sac-derived macrophages are replaced by HSC-derived macrophages in the first weeks of post-natal life. The mechanisms responsible for the maintenance of yolk-sac-derived mac- rophages in certain adult tissues require further investigation. Although yolk-sac- and HSC-derived macrophages can co-exist in the same envi- ronment, and their balance be perturbed by pathology, the contributions of these developmentally distinct macrophage populations to home- ostasis and inflammation remain to be characterized.
Figure 4 | Fetal macrophages and adult tissue-resident macrophages originate from Tie2-expressing progenitors before E10.5. a, Fate-mapping analysis of Tie2-expressing cells after tamoxifen (TAM) administration at E7.5, or E8.5, or E9.5 or E10.5. Arrows indicate time points for analysis. b, Flow cytometric analysis of fetal liver long-term or short-term haematopoietic stem cells (LT-HSCs, ST-HSCs), multipotent progenitors (MPPs), common myeloid progenitors (CMPs), granulocyte-monocyte progenitors (GMPs), megakaryocyte-erythrocyte progenitors (MEPs) (left panel) and of fetal macrophages (right panel) in the yolk sac, brain, and fetal liver. Time points of labelling (E7.5 (n = 7); E10.5 (n = 7)) and analysis are indicated, and for each experiment one representative analysis is shown. See Extended Data Fig. 8 for quantitative analysis. c, Frequencies of labelled HSCs and progenitor cells, splenocytes, and F4/80“CDI 1b”i myeloid cells and F4/80°"®" resident macrophages in spleen, liver lung, epidermis and brain were analysed (mean * s.d., see Source Data Table Fig. 4) from 6-8-week-old Tie2MeriCreMer animals pulse-labelled at E7.5 (n = 4), E8.5 (n = 4), E9.5 (n = 4) or E10.5 =
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
# Received 12 April; accepted 21 October 2014.
# Published online 3 December 2014.
- 1. Orkin, S. H. & Zon, L. |. Hematopoiesis: an evolving paradigm for stem cell biology. Cell 132, 631-644 (2008).
- 2. Smith, L. G., Weissman, |. L. & Heimfeld, S. Clonal analysis of hematopoietic stem-cell differentiation in vivo. Proc. Natl Acad. Sci. USA 88, 2788-2792 (1991).
- 3. Cumano, A. & Godin, |. Ontogeny of the hematopoietic system. Annu. Rev. Immunol. 25, 745-785 (2007).
- Schulz, C. etal. A lineage of myeloid cells independent of Myb and hematopoietic stem cells. Science 336, 86-90 (2012).
4.
- Yona, S. etal. Fate mapping reveals origins and dynamics of monocytes and tissue macrophages under homeostasis. Immunity 38, 79-91 (2013).
5.
# (n
6).
in peripheral tissues (spleen, liver and lung) were homogenously la- belled at frequencies comparable to HSC labelling, consistent with their adult HSC origin (Fig. 4c). In contrast, YFP labelling frequencies of adult
- 6. Hashimoto, D. etal. Tissue-resident macrophages self-maintain locally throughout adult life with minimal contribution from circulating monocytes. /mmunity 38, 792-804 (2013).
- Ajami, B., Bennett, J. L., Krieger, C., Tetzlaff, W. & Rossi, F.M. Local self-renewal can sustain CNS microglia maintenance and function throughout adult life. Nature Neurosci. 10, 1538-1543 (2007).
7.
550 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
- Palis, J., Robertson, S., Kennedy, M., Wall, C. & Keller, G. Development of erythroid and myeloid progenitors in the yolk sac and embryo proper of the mouse. Development 126, 5073-5084 (1999).
- 22. Christensen, J. L & Weissman, |. L. Fik-2 is a marker in hematopoietic stem cell differentiation: a simple method to isolate long-term stem cells. Proc. Natl Acad. Sci. USA 98, 14541-14546 (2001).
- Bertrand, J. Y. et al, Three pathways to mature macrophages in the early mouse yolk sac. Blood 106, 3004-3011 (2005).
- 23. Waskow, C. et al. Hematopoietic stem cell transplantation without irradiation. Nature Methods 6, 267-269 (2009).
- 10. Lux, C. T, etal, All primitive and definitive hematopoietic progenitor cells emerging before E10 in the mouse embryo are products of the yolk sac. Blood 111, 3435-3438 (2008).
- Epelman, $. et al. Embryonic and adult-derived resident cardiac macrophages are maintained through distinct mechanisms at steady state and during inflammation. Immunity 40, 91-104 (2014). 24,
- Fi Kierdorf, K. et al, Microglia emerge from erythromyeloid precursors via Pu.1- and Irf8-dependent pathways. Nature Neurosci. 16, 273-280 (2013).
- 25. Hoeffel,G. etal, Adult Langerhans cells derive predominantly from embryonic fetal liver monocytes with a minor contribution of yolk sac-derived macrophages. J. Exp. Med. 209, 1167-1181 (2012).
- 12: Bertrand, J. Y. et al. Characterization of purified intraembryonic hematopoietic stem cells as a tool to define their site of origin. Proc. Nat! Acad. Sci. USA 102, 134-139 (2005).
- Guilliams, M. et al, Alveolar macrophages develop from fetal monocytes that differentiate into long-lived cells in the first week of life via GM-CSF. J. Exp. Med. 210, 1977-1992 (2013). 26.
- 13. Moore, M. A. & Metcalf, D. Ontogeny of the haemopoietic system: yolk sac origin of in vivo and in vitro colony forming cells in the developing mouse embryo. Br. J. Haematol. 18, 279-296 (1970).
- Bain, C.C. etal. Constant replenishment from circulating monocytes maintains macrophage pool in the intestine of adult mice. Nature Immunol. 15, 929-937 (2014). 21;
- 14, Herbomel, P., Thisse, B. & Thisse, C. Ontogeny and behaviour of early macrophages in the zebrafish embryo. Development 126, 3735-3745 (1999).
- 15, Takahashi, K., Yamamura, F. & Naito, M. Differentiation, maturation, and proliferation of macrophages in the mouse yolk sac: a light-microscopic, enzyme- cytochemical, immunohistochemical, and ultrastructural study. J. Leukoc. Biol. 45, 87-96 (1989).
- 16. Sumner, R., Crawford, A., Mucenski, M. & Frampton, J. Initiation of adult myelopoiesis can occur in the absence of c-Myb whereas subsequent development is strictly dependent on the transcription factor. Oncogene 19, 3335-3342 (2000).
- 17. Alliot, F., Godin, |. & Pessac, B, Microglia derive from progenitors, originating from the yolk sac, and which proliferate in the brain. Brain Res. Dev. Brain Res. 117, 145-152 (1999).
- 18. Ginhoux, F. et al. Fate mapping analysis reveals that adult microglia derive from primitive macrophages. Science 330, 841-845 (2010).
- 19. Frame, J. M., McGrath, K. E. & Palis, J. Erythro-myeloid progenitors: “definitive” hematopoiesis in the conceptus prior to the emergence of hematopoietic stem cells. Blood Cells Mol. Dis. 51, 220-225 (2013).
Supplementary Information is available in the online version of the paper.
Acknowledgements The authors are indebted to J. Pollard, University of Edinburgh for the Csfir reporter strains, J. Frampton, University of Birmingham for the Myb-deficient animals, and T. Boehm, Max Planck Institute, Freiburg for the Fit3” strain. The authors also thank A. McGuigan and the staff of the Biological Service Unit at King’s College London, S. Heck and the Biomedical Research Centre at King’s Health Partners, S. Woodcock and the staff of the Viapath haematology laboratory in Guy's hospital and S. Schafer and T. Arnsperger for technical assistance at the German Cancer Research Center. This work was supported by a Wellcome Trust Senior Investigator award (WT101853MA) and ERC Investigator award (2010-StG-261299) from the European Research Council to F.G. and an ERC Investigator award (Advanced Grant 233074), SFB 938 project L, and SFB 873 project B11 toH-R.R.
Author Contributions E.G.P. and F.G. designed the study and wrote the manuscript. EGP.,C.S.,L.C., H.G. and C.T. performed fate-mapping experiments and E.G.P. and F.G. designed experiments and analysed the data. K.B. and H.-R.R. generated the Tie2enreMer strain and K.K., K.B. and H.-R.R. designed, performed and analysed fate-mapping experiments. E.A. and E.G.P. performed the CFU assays and E.A., E.G.P,, F.G. and M.F.d.B. analysed and interpreted the experimental data. All authors contributed to the manuscript.
20.
- Dzierzak, E. & Medvinsky, A. Mouse embryonic hematopoiesis. Trends Genet 11, 359-366 (1995).
- 21. Kieusseian, A., Brunet de la Grange, P., Burlen-Defranoux, O., Godin, |. & Cumano, Immature hematopoietic stem cells undergo maturation in the fetal liver. Development 139, 3521-3530 (2012).
A.
Author Information Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to F.G. (frederic.geissmann@kcl.ac.uk).
26 FEBRUARY 2015 | VOL 518 | NATURE | 551
©2015 Macmillan Publishers Limited. All rights reserved
# the
LETTER
doi:10.1038/nature13968
# Role of TP53 mutations in the origin and evolution of therapy-related acute myeloid leukaemia
Terrence N. Wong'*, Giridharan Ramsingh?*, Andrew L. Young**, Christopher A. Miller*, Waseem Touma’, John S. Welch” Tamara L. Lamprecht', Dong Shen®, Jasreet Hundal*, Robert S. Fulton*, Sharon Heath’, Jack D. Baty’, Jeffery M. Klcoö, Li Ding! , Elaine R. Mardis**°, Peter Westervelt”ö, John F. DiPersio'*” Todd E. Druley?, Daniel C. Link’ & Richard K. Wilson"? , Matthew J. Walter” , Timothy A. Graubert", Timothy J. Ley”,
# Therapy-related acute myeloid leukaemia (t-AML) and therapy-related
myelodysplastic syndrome (t-MDS) are well-recognized complications of cytotoxic chemotherapy and/or radiotherapy’. There are several features that distinguish t-AML from de novo AML, including a higher incidence of TP53 mutations”, abnormalities of chromosomes 5 or 7, complex cytogenetics and a reduced response to chemotherapy*. However, it is not clear how prior exposure to cytotoxic therapy influ- ences leukaemogenesis. In particular, the mechanism by which TP53 mutations are selectively enriched in t-AML/t-MDS is unknown. Here, by sequencing the genomes of 22 patients with t-AML, we show that the total number of somatic single-nucleotide variants and the per- centage of chemotherapy-related transversions are similar in t- AML and de novo AML, indicating that previous chemotherapy does not induce genome-wide DNA damage. We identified four cases of t-AML/ t-MDS in which the exact TP53 mutation found at diagnosis was also present at low frequencies (0.003-0.7%) in mobilized blood leuko- cytes or bone marrow 3-6 years before the development of t-AML/t- MDS, including two cases in which the relevant TP53 mutation was detected before any chemotherapy. Moreover, functional TP53 muta- tions were identified in small populations of peripheral blood cells of healthy chemotherapy-naive elderly individuals. Finally, in mouse bone marrow chimaeras containing both wild-type and Tp53*’~ hae- matopoietic stem/progenitor cells (HSPCs), the Tp53*/~ HSPCs pre- ferentially expanded after exposure to chemotherapy. These data suggest that cytotoxic therapy does not directly induce TP53 muta- tions. Rather, they support a model in which rare HSPCs carrying age-related TP53 mutations are resistant to chemotherapy and ex- pand preferentially after treatment. The early acquisition of TP53 mutations in the founding HSPC clone probably contributes to the frequent cytogenetic abnormalities and poor responses to chemo- therapy that are typical of patients with t-AML/t-MDS.
mutation burden. However, the total number of validated somatic single- nucleotide variants (SNVs) and genic (tier 1) somatic SNVs identified was similar to that for de novo AML and s-AML (Fig. la, b). Likewise, the number of small insertions or deletions (indels) in genic regions was similar in t-AML, de novo AML and s-AML (Fig. 1c). A previous study
2 Li o , 10,000 60 8 2 1,000 240 z 100] ** Boo. ccehere 8 104— ot + t-AML De novo s-AML AML De novo s-AML AML De novo S-AML AML AML AML d ft; £ 8 ; 3 5 5 Li da 2 El 8 = & o t-AML Denovo AML = s-AML 50: 40: Mts De novo MDS Percentage with mutation (& Percentage with mutation x 8 ‘eS SE HG Gİ Peş & ae ESN PLLA RA” SOE MAR OK EOE poss FB N 8
t-AMLand t-MDS are clonal haematopoietic disorders that typically develop 1-5 years after exposure to chemotherapy or radiotherapy'. To understand better how prior cytotoxic therapy contributes to the high incidence of TP53 mutations and karyotypic abnormalities in t-AML/ t-MDS, we sequenced the genomes of 22 cases of t-AML, including one case that has been previously reported’. These data were compared to whole-genome sequence data previously reported for de novo AML’ and secondary AML (s-AML) arising from MDS for which patients did not receive chemotherapy except hydroxyurea”*. Of the sequenced t-AML cases, 23% had rearrangements of MLL (also known as KMT2A), 23% had complex cytogenetics, and 36% had normal cytogenetics (Extended Data Table 1 and Supplementary Table 1).
We predicted that DNA damage induced during exposure to cyto- toxic therapy would manifest itselfin t- AML genomes with an increased
Figure 1 | The mutational burden in t-AML is similar to de novo AML. a, Total number of validated tier 1-3 somatic SNVs in t-AML (n = 22), de novo AML (n = 49) and s-AML (n = 8). The mean ages of the t-AML, de novo AML and s-AML cohorts were 55.7, 51 and 54.6 years, respectively. b, Number of validated tier 1 somatic SNVs. c, Number of validated tier 1 small indels. d, Percentage of tier 1-3 somatic SNVs that are transversions. e, Mutational spectrum for all validated tier 1-3 somatic SNVs. £, Number of distinct clones per sample inferred from the identification of discrete clusters of mutations with distinct variant allele frequencies. g, Percentage of cases of t- AML (n = 52) or de novo AML (n = 199) harbouring non-synonymous mutations of the indicated gene. h, Percentage of cases of t- MDS (n = 59) or de novo MDS (n = 150) harbouring non-synonymous mutations of the indicated gene. ABC Fm, ABC family genes; NA, not available. +P < 0.05 by one-way analysis of variance (ANOVA). *P < 0.05 by Fisher’s exact test. Data represent the mean * standard deviation (s.d.).
Department of Medicine, Division of Oncology, Washington University, St Louis, Missouri 63110, USA. Department of Medicine, Jane Anne Nohl Division of Hematology, University of Southern California, Los Angeles, California 90089, USA. “Department of Pediatrics, Division of Hematology/Oncology, Washington University, St Louis, Missouri 63110, USA. “The Genome Institute, Washington University, St Louis, Missouri 63110, USA, *Siteman Cancer Center, Washington University, St Louis, Missouri 63110, USA. “AstraZeneca, Gaithersburg, Maryland 20878, USA. ’Division of Biostatistics, Washington University, St Louis, Missouri 63110, USA. #Department of Pathology and Immunology, Washington University, St Louis, Missouri 63110, USA. °Department of Genetics, Washington University, St Louis, Missouri 63110, USA.
1
*These authors contributed equally to this work.
552 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
showed that transversions are specifically enriched in relapsed AML after chemotherapy’. However, the percentage of transversions, and in fact of all six classes of SN Vs, was similar in all three cohorts (Fig. 1d, e). Structural variants and somatic copy number alterations were uncom- mon in these t-AML cases (Supplementary Table 2 and Extended Data Fig. 1a). Moreover, the number of identifiable subclones in t-AML was similar to that observed in de novo AML (Fig. 1f and Extended Data Fig. 1b). Collectively, these data show that the mutation burden of t- AML genomes is similar to that of de novo AML genomes.
We next asked whether the pattern of genes frequently mutated in t-AML/t-MDS is distinct from that observed in de novo AML/MDS. Whole-genome sequencing identified an average of 10.2 + 7.1 missense, nonsense, in-frame indel or frameshift mutations per t-AML genome (Supplementary Table 3). To define better the frequency of specific muta- tions in t- AML/t-MDS, we sequenced a panel of 149 AML/MDS-related genes in an additional 89 patients with t-AML or t-MDS (Supplemen- tary Table 4). We combined the whole-genome sequence data with the extension series to report on 52 cases of t-AML and 59 cases of t-MDS. Abnormalities of chromosome 5 or 7 or complex cytogenetics were pres- ent in 55.0% of cases (Extended Data Table 2 and Supplementary Table 1). The t-AML/t-MDS data were compared to 199 previously reported de novo AML genomes or exomes’, or 150 previously reported cases of de novo MDS in which extensive candidate gene sequencing was performed’. As reported previously, TP53 mutations are significantly enriched in t-AML/t-MDS compared with de novo AML/MDS (Fig. 1g, h and Sup- plementary Table 5). Interestingly, mutations of ABC transporter genes, a subset of which have been implicated in chemotherapy resistance, are also enriched in t-AML versus de novo AML. On the other hand, several well-defined driver gene mutations (that is, DNMT3A and NPM1) were significantly less common in t-AML. Thus, although the total mutation burden is similar, a distinct subset of mutated genes is present in t-AML/ t-MDS.
TP53 is the most commonly mutated gene in t-AML/t-MDS, with 33.3% of patients affected in our series (Fig. 1g, h); the vast majority of these mutations have previously been identified as pathogenic'®. Multi- variate analysis revealed that TP53 mutations were associated with poor risk cytogenetics and a worse prognosis (Supplementary Tables 6, 7 and Extended Data Fig. 2), both hallmarks of t-AML/t-MDS. These obser- vations suggest a central role for TP53 mutations in the pathogenesis of many cases of t-AML/t-MDS. However, the mechanism by which TP53
mutations are selectively enriched in t-AML/t-MDS is unclear. The muta- tion burden in the genomic region containing TP53 (including silent tier 1, and any tier 2 or tier 3 mutations) is similar between t-AML and de novo AML (Extended Data Fig. 1c). Thus, it is not likely that chemotherapy directly induces TP53 mutations. We recently reported that individual HSPCs accumulate somatic mutations as a function of age, such that by age 50, there are on average five coding gene mutations per HSPC"', On the basis of these data and on current estimates that there are approxi- mately 10,000 haematopoietic stem cells (HSCs) in humans”, we pre- dict that 44% of healthy individuals at 50 years of age may have at least one HSPC that carries a randomly generated, functional TP53 mutation (see Methods). TP53 has a central role in regulating cellular responses to genotoxic stress'*!’, and loss of TP53 provides a selective advantage for neoplastic growth'*, Together, these observations suggest a model in which rare HSPCs carrying age-related TP53 mutations are resistant to chemotherapy and expand preferentially after treatment (Extended Data Fig. 3).
This model suggests the following testable predictions: (1) in patients with t-AML containing clonal TP53 mutations, HSPCs harbouring the specific TP53 mutation will be present long before the development of overt t-AML; (2) somatic TP53 mutations will be present in the HSPCs of some healthy individuals never exposed to cytotoxic therapy; and (3) HSPCs harbouring TP53 mutations will expand under the selective pressure of chemotherapy.
To test the first prediction, we identified seven cases of t-AML/t-MDS with specific TP53 mutations for which we had leukapheresis or bone marrow specimens banked 3-8 years before the development of t-AML/ t-MDS (Extended Data Table 3). Of note, in all the cases, the TP53 muta- tion was clonal in the t-AML/t-MDS diagnostic sample. Current next- generation sequencing technology is limited in the detection of rare variant alleles owing to an intrinsic sequencing error rate of ~ 0.1% (ref. 19). To overcome this limitation, we introduced random barcodes during production of the sequencing libraries, such that sequence ‘read families’ containing unique barcodes are generated (Extended Data Fig. 4a). Using tumour DNA with a known TP53 mutation, we show that this assay can detect a variant allele with a frequency of 0.009% (Extended Data Fig. 4b, c).
The specific TP53 mutation present in the diagnostic t- AML/t-MDS sample was identified in previously banked specimens in four out of the seven cases tested (see Supplementary Notes for case presentations). In
# years
# a
c Chemotherapy 6 yel er hed eşi ary pair AML Ea MER ' 2nd autologous transplant TP53 K139N TP53 R248Q CSMD1 6192 O NUP98 01532H TET2 K1299M cy mE 2.0 20 20 5 20 3 "8 4 .; 7 Raw a ii wey DR “ © reads , 0 o ol o 200 o 200 o 200 2.0: 2.0 2.0 2.0 2.0 £ | | Read & 1.0: © 1.04 1.0 1.0: 1.04 families > i © © 4 ee ee | 4 aşama o a 0+ hee | 0 200 0 200 0 200 0 100 20 0 100 200 2.0: 2.0 2.0 20 2.0: 1 Control 1.0: 1.04 1.0 1.0: 1.09 read J J J families 0 pment | | amman | 0 eymnmla | | aeammenmelim | o Speedin | 0 200 0 200 0 200 0 1 20 0 100 20 Position (bp)
# b
# Figure 2 | Biallelic TP53 mutations are early mutational events in the AML cells of UPN
mutational events in the AML cells of UPN 530447. a, Clinical course of case 530447. Chemo, chemotherapy; XRT, radiation therapy. b, Unigue adaptor seguencing of a leukapheresis sample obtained 6 years before the diagnosis of t-AML for each of the five clonal somatic SNVs identified in the diagnostic t-AML sample. Genomic DNA from i 5 x a patient lacking these variants served as a control. Blue circles indicate the position of the variant SNV. c, Proposed model of clonal evolution to t-AML in this case.
26 FEBRUARY 2015 | VOL 518 | NATURE | 553
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
the other three cases, we were unable to detect the diagnostic TP53 muta- tion in the previously banked blood or bone marrow sample; it is not clear whether these mutations were present but below our limit of detec- tion or were truly absent. Patient 530447 developed t-AML after an autologous stem cell transplant for refractory Hodgkin’s lymphoma (Fig. 2a). The diagnostic t-AML sample carried biallelic mutations of TP53, missense mutations of TET2 and NUP98, a silent mutation of CSMDI, and a subclonal KRAS mutation. Analysis of a leukapheresis sample obtained 6 years before the development of t-AML revealed that both TP53 mutant alleles were present with a variant allele fraction (VAF) of approximately 0.5% (Fig. 2b). The CSMDI mutation was also present at the same VAF and is probably a passenger mutation. However, two potential driver mutations (TET2 and NUP98) were not detectable in the previously banked sample. Thus, these data show that, in this patient, the biallelic TP53 mutations preceded the development of t-AML by at least 6 years and antedated the development of the TET2 and NUP98 mutations (Fig. 2c). In a second case (unique patient number (UPN) 341666), a heterozygous TP53 R196* mutation was identified in mobi- lized peripheral blood leukocytes 3 years before the development of t-MDS at a frequency of 0.1%, preceding the acquisition of a RUNX1 mutation (Extended Data Fig. 5).
In two of the four cases, the previously banked sample was obtained before the initiation of chemotherapy. Patient 967645 developed t-AML 5 years after the diagnosis of marginal zone lymphoma (Fig. 3a). The diagnostic t-AML sample contained a homozygous TP53 Y220C muta- tion. Using a droplet digital polymerase chain reaction (ddPCR) assay, we identified the same TP53 Y220C mutation in a bone marrow sample obtained before any chemotherapy at a frequency of 0.0027% (average of two independent experiments) (Fig. 3b). We next asked whether other mutations in the diagnostic t-AML sample were also present in this previously banked sample (Supplementary Table 8). We focused on the G155S mutation in SNAP25; this mutation is probably non-pathogenic as SNAP25 is not expressed in AML samples’. Indeed, we identified the SNAP25 G155S mutation in the previously banked bone marrow sam- ple with a similar VAF (0.0029%) as that for TP53 Y220C (Fig. 3c). Of note, deletion (del)(5q) and del(7q) were subclonal at diagnosis (present in 54% and 38% of metaphases, respectively) (Supplementary Table 1). Collectively, these data provide evidence that an HSPC harbouring a TP53 Y220C mutation preferentially expanded after chemotherapy with the subsequent acquisition of del(5q) and then del(7q) (Fig. 3d). Of note, we found two other cases of t-AML/t-MDS with clonal TP53 mutations but subclonal del(5q), del(5) and/or del(7) (UPNs 756582 and 837334, Supplementary Table 1). Together, these data suggest that TP53 muta- tions precede the development of these characteristic cytogenetic abnor- malities of t-AML/t-MDS.
Ina second case, patient 895681 developed t-MDS 5.5 years after the initiation of chemotherapy for non-Hodgkin’s lymphoma (Fig. 3e). The diagnostic t-MDS sample contained a clonal TP53 H179L mutation. Using ddPCR, we identified TP53 H179L at a VAF of 0.05% in a bone marrow sample taken before the initiation of cytotoxic therapy (Fig. 3f). Thus, as with patient 967645, an HSPC carrying a functional TP53 muta- tion was present before cytotoxic therapy exposure, later giving rise to the malignant t-AML/t-MDS clone (Fig. 3g).
@ Stage 1ADLBCL Autologous transplant (Chemo/XRT 2 years A Marginal zone lymphoma 1 Fludarabine x 6 Bone marrow FFPE specimen b TP53 Y220C c SNAP25 G155S f TP53 HI79L Diagnosis Diagnosis Diagnosis 12,000 6,000 — 12,000— po 3889) 3 5 5) on Z| z z = - id © 4,731 8 ? leme 2 Empty 8 2 = e STEED wits type = lize [Wild type ) OlWild-type signal (A.U)! 0 TWild-type signal (A.UJ)İ 8,000 12,000 8,000 Previously banked Previously banked Previously banked 12,000 6,000- 12,000. 285,142) a] Team 508297) a 102,470) 5 iW ew 5 z 3): < > Sik 3 5 | | * 5 2 2) 12| ii , Fs] g 2 g 2 8 | x os Em; b idi Ae Wild type 5 | Fes 857 Wild type) =| 2346) Wild type 0 id type signal (A.U.) 0 Wild-type ‘signal (A.U)! 0 İWild-type signal Au)! 8,000 12,000 8 Control ol Control e NS 376,460] 00 anna 466735) 12000 108,675) 5 3] Gow 5 < Fa ” Ea . 3 5 Bİ, ş & a g 8 & JEmpiy. S 5 Diy” Zİ nn oral ty Wild type 512,811 Wild type 504,307 [Wild type o İWiiype me ö o Wild-type signal (AU,)! (Wild-type signal (A.U)! 12,000 dü 8,000 1 year 4 years Chemotherapy mas 9 15% d fr İSNAP25| \ © ~40% }-0.006' celis (Sa) Del?) 0.1% cells
# T
‘AML diagnosis
Autologous transplant
# MDS banking
Figure 3 | HSPC clones harbouring somatic TP53 mutations are detected in patients before cytotoxic therapy exposure. a, Clinical course of case 967645. b, Dot plots of ddPCR of a diagnostic t-AML sample from case 967645, a bone marrow sample from this patient obtained 5 years before the development of t-AML (before any cytotoxic therapy), or a control sample from a patient lacking a mutation in TP53. Droplets containing only the TP53 Y220C allele are highlighted in orange, droplets containing wild-type TP53 (with or without TP53 Y220C) are highlighted in blue; empty droplets are grey. The number of droplets in each gate is indicated. Data are representative of two independent experiments. c, Dot plots of ddPCR data for SNAP25 G155S using the same genomic DNA as in b. d, Proposed model of clonal evolution to t-AML in case 967645. e, Clinical course of case 895681. Chemo, chemotherapy; DLBCL, diffuse large B-cell lymphoma; FFPE, formalin-fixed parafin-embedded; XRT, radiotherapy. f, Dot plots of ddPCR data of the diagnostic t-MDS sample from case 895681, a bone marrow FFPE sample from this patient obtained 5.5 years before the development of t-MDS (before any cytotoxic therapy), or a control FFPE sample obtained from a patient lacking a mutation in TP53. The labelling scheme is the same as in b. g, Proposed model of clonal evolution to t-MDS in case 895681; the diagnostic t-MDS sample contained a subclonal ETV6 mutation.
To determine whether HSPCs harbouring TP53 mutations are pres- ent in healthy individuals, we analysed peripheral blood leukocytes from 20 elderly (68-89 years old) cancer-free donors who had not received prior cytotoxic therapy. We limited our sequencing to exons 4-8 of TP53 since the majority of pathogenic mutations in TP53 are located in these exons. Using our unique adaptor sequencing assay, we identified TP53 mutations in 9 of 19 evaluable cases, with VAFs ranging from 0.01% to 0.37% (Extended Data Table 4). Of note, since we did not sequence the entire coding region of TP53, it is likely that our study underestimates the true frequency of healthy elderly individuals harbouring HSPCs with TP53 mutations. ddPCR confirmed the presence of the TP53 mutation in all three cases that were tested (Extended Data Fig. 6). Interestingly, the majority of the TP53 mutations identified are known pathogenic
mutations previously implicated in cancer. These data suggest that func- tional TP53 mutations may confer (even in the absence of cytotoxic therapy) a subtle competitive advantage that results in modest HSPC expansion over time.
To test directly the hypothesis that functional TP53 mutations confer a competitive advantage after chemotherapy, we generated mixed bone marrow chimaeras containing both wild-type and Tp53*'~ cells (Fig. 4a). In mice treated with vehicle control, we observed a non-significant trend towards an increased Tp53*’~ donor contribution to haematopoiesis (Fig. 4b-e). Whether longer follow-up would confirm a subtle competitive advantage, as suggested by the expansion of TP53 mutant HSPC clones in elderly healthy individuals, will require additional study. Regardless,
554 NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
66. pi! G LE Vehicle Vehicle — 7p53“* Tp53""* Tps3“* TpSS~* Tp53~* 9 days 1p83"* Tps3“* Tp6s~* b c White blood cells Neutrophils Percentage 7p53*” d B cells e KLS cells 8 2 8 Percentage 7p53*“ 8 8 - Vehicle - ENU 12 8 o Time after ENU (weeks) Time after ENU (weeks)
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
# Received 11 December 2013; accepted 13 October 2014. Published online 8 December 2014.
- 1. Godley, L.A. & Larson, R. A. Therapy-related myeloid leukemia. Semin. Oncol. 35, 418-429 (2008).
- 2. Christiansen, D. H., Andersen, M. K.& Pedersen-Bjergaard, J. Mutations with loss of heterozygosity of p53 are common in therapy-related myelodysplasia and acute myeloid leukemia after exposure to alkylating agents and significantly associated with deletion or loss of 5q, a complex karyotype, and a poor prognosis. J. Clin. Oncol. 19, 1405-1413 (2001).
- Shih, A.H. etal, Mutational analysis of therapy-related myelodysplastic syndromes and acute myelogenous leukemia. Haematologica 98, 908-912 (2013).
- Kayser, S. et al. The impact of therapy-related acute myeloid leukemia (AML) on outcome in 2853 adult patients with newly diagnosed AML. Blood 117, 2137-2145 (2011).
# Pp
- 5. Link, D.C. et al, Identification of a novel TP53 cancer susceptibility mutation through whole-genome sequencing of a patient with therapy-related AML. J. Am. Med. Assoc. 305, 1568-1576 (2011).
- The Cancer Genome Atlas Research Network. Genomic and epigenomic landscapes of adult de novo acute myeloid leukemia. N. Engl. J. Med. 368, 2059-2074 (2013).
6.
- 7. Walter, M. J. etal. Clonal architecture of secondary acute myeloid leukemia. N. Engl. J. Med. 366, 1090-1098 (2012).
- Walter, M. J, etal. Clonal diversity of recurrently mutated genes in myelodysplastic syndromes. Leukemia 27, 1275-1282 (2013).
8.
- 9. Ding, L. et al, Clonal evolution in relapsed acute myeloid leukaemia revealed by whole-genome sequencing. Nature 481, 506-510 (2012).
Figure 4 | Heterozygous loss of TP53 confers a clonal advantage to HSCs after exposure to ENU. a, Experimental schema. Bone marrow chimaeras were generated by transplanting a 7 to 1 ratio of wild-type to 7p53*/” bone marrow into irradiated syngenic recipients. After haematopoietic reconstitution (5 weeks), mice were treated with ENU or vehicle control as indicated. b-d, Shown is the percentage of total leukocytes (b), Gr-1* neutrophils (c) or B220* B cells (d) that were derived from Tp53*/~ cells. e, Percentage of Kit* lineage Sca* (KSL) cells in the bone marrow 12 weeks after ENU exposure that were derived from Tp53"~ cells. Data represent the mean + standard error of the mean (s.e.m.) of 11 and 14 mice in the ENU and vehicle cohorts, respectively. Peripheral chimaerism was analysed using two-way ANOVA and KLS chimaerism was analysed using an analysis of covariance (ANCOVA).
- 10. Petitjean, A. et a/. Impact of mutant p53 functional properties on TP53 mutation patterns and tumor phenotype: lessons from recent developments in the IARC TP53 database. Hum. Mutat. 28, 622-629 (2007).
- 11. Welch,J.S.etal, The origin and evolution of mutations in acute myeloid leukemia. Cell 150, 264-278 (2012).
- 12. Catlin, S.N., Busque, L., Gale, R. E., Guttorp, P. & Abkowitz, J. L. The replication rate of human hematopoietic stem cells in vivo. Blood 117, 4460-4466 (2011).
- 13. Carvajal, L.A. & Manfredi, J. J. Another fork in the road—iife or death decisions by the tumour suppressor p53. EMBO Rep. 14, 414-421 (2013).
- 14, Chen, X,, Ko, L. J., Jayaraman, L. & Prives, C. p53 levels, functional domains, and DNA damage determine the extent of the apoptotic response of tumor cells. Genes Dev. 10, 2438-2451 (1996).
- 15. Purvis, J. E. etal. p53 dynamics control cell fate. Science 336, 1440-1444 (2012).
- 16. Zhang, X.P.,Liu, F., Cheng, Z.& Wang, W. Cell fate decision mediated by p53 pulses. Proc. Natl Acad. Sci. USA 106, 12245-12250 (2009).
- 17. Zhang, X. P., Liu, F. & Wang, W. Two-phase dynamics of p53 in the DNA damage response. Proc. Nat! Acad. Sci, USA 108, 8990-8995 (2011).
- 18. Hollstein, M., Sidransky, D., Vogelstein, B. & Harris, C. C. p53 mutations in human cancers. Science 253, 49-53 (1991).
upon treatment with N-ethyl-N-nitrosourea (ENU), Tp53*/~ HSPCs show a competitive advantage. Importantly, a previous study similarly showed that Tp53*/~ HSCsalso havea competitive advantage after irra- diation, which appeared to be due, at least in part, to reduced irradiation- induced senescence in Tp53*/~ HSCs*2!,
- 19. Kandoth, C. et al. Mutational landscape and significance across 12 major cancer types. Nature 502, 333-339 (2013).
- 20. Bondar, T. & Medzhitov, R. p53-mediated hematopoietic stem and progenitor cell competition. Cell Stem Cell 6, 309-322 (2010).
- 21. Marusyk, A, Porter, C. C., Zaberezhnyy, V. & DeGregori, J. Irradiation selects for p53-deficient hematopoietic progenitors. PLoS Biol. 8, e1000324 (2010).
There is increasing evidence that cancers undergo clonal evolution under the selective pressure of chemotherapy”. For example, the clonal architecture of de novo AML is dynamic, with certain (often minor) sub- clones becoming dominant at relapse after chemotherapy’. We show that HSPCs that acquire heterozygous TP53 mutations as a function of normal ageing are also subject to Darwinian selection upon exposure to cytotoxic therapy, ultimately resulting in the expansion of HSPCs with these mutations. The high frequency (nearly 50%) of elderly individuals with detectable heterozygous TP53 mutations in their circulating leu- kocytes far exceeds the prevalence of AML or MDS in this age group. Clearly, additional mutations, including mutation of the second TP53 allele, are needed for transformation to AML or MDS. Consistent with this observation, only a minority of patients with Li-Fraumeni syndrome, most of whom harbour germline heterozygous TP53 mutations, develop AML or MDS”. This model provides a potential mechanism for the high incidence of TP53 mutations in t-AML/t-MDS™. The TP53 muta- tion in the founding clone probably contributes to the frequent cytoge- netic abnormalities and poor response to chemotherapy that are typical oft-AML/t-MDS, For t-AML/t-MDS cases that do not harbour TP53 mutations, it will be important to determine whether different age-related mutations also confer a competitive advantage to HSPCs that are exposed to cytotoxic therapy, and to define the nature of these mutations.
- Greaves, M. & Maley, C.C. Clonal evolution in cancer. Nature 481, 306-313 (2012).
22.
- 23. Birch, J. M. et al. Relative frequency and morphology of cancers in carriers of germline TP53 mutations. Oncogene 20, 4621-4628 (2001).
- 24. Casâs-Selves, M. & Degregori, J. How cancer shapes evolution, and how evolution shapes cancer, Evolution 4, 624-634 (2011).
Supplementary Information is available in the online version of the paper.
Acknowledgements We thank A. Schmidt, B. McKethan and R. Miller for technical assistance, and K. Odell and J. Tucker-Davis for animal care. We thank P. Goodfellow and J. Ivanovich for providing peripheral blood leukocyte genomic DNA from cancer-free individuals. This work was supported by National Institutes of Health grants PO1 CA101937 (D.C.L) and U54 HGO03079 (R.K.W.) and by a grant from the Leukemia & Lymphoma Society (D.C.L).
Author Contributions T.N.W. and G.R. designed and performed the research, analysed the data, and wrote the manuscript. A.LY. and T.E.D. developed and optimized the amplicon-based random primer sequencing assay. C.A.M., D.S., J.H., R.S.F., L.D., ER.M. and R.K.W. contributed to the generation and analysis of the whole-genome or targeted sequencing. W.T., T.L.L, S.H, J.M.K. and P.W. collected and processed clinical data and tissue samples. J.D.B. performed statistical analyses of the clinical data, J.S.W., J.F.D., MJ.W., T.A.G.and TJ.L. contributed to data analysis. D.C.L. supervised all ofthe research and edited the manuscript, which was approved by all co-authors.
Author Information Sequence information on the 22 t-AML whole-genome sequencing patients and one exome sequencing patient (UPN 967645) has been deposited in dbGaP under accession number phs000159. Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests, Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to D.C.L. (dlink@dom.wustl.edu).
26 FEBRUARY 2015 | VOL 518 | NATURE | 555
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
doi:10.1038/nature13994
# Enhancer—core-promoter specificity separates developmental and housekeeping gene regulation
Muhammad A. Zabidi!*, Cosmas D. Arnold'*, Katharina Schernhuber', Michaela Pagani’, Martina Rath’, Olga Frank! & Alexander Stark!
Gene transcription in animals involves the assembly of RNA poly- merase II at core promoters and its cell-type-specific activation by enhancers that can be located more distally'. However, how ubiqui- tous expression of housekeeping genes is achieved has been less clear. In particular, it is unknown whether ubiquitously active enhanc- ers exist and how developmental and housekeeping gene regulation is separated. An attractive hypothesis is that different core promo- ters might exhibit an intrinsic specificity to certain enhancers” “, This is conceivable, as various core promoter sequence elements are differentially distributed between genes of different functions’, in- cluding elements that are predominantly found at either develop- mentally regulated or at housekeeping genes*'°. Here we show that thousands of enhancers in Drosophila melanogaster S2 and ovarian somatic cells (OSCs) exhibit a marked specificity to one of two core promoters—one derived from a ubiquitously expressed ribosomal protein gene and another from a developmentally regulated tran- scription factor—and confirm the existence of these two classes for five additional core promoters from genes with diverse functions. Housekeeping enhancers are active across the two cell types, while developmental enhancers exhibit strong cell-type specificity. Both enhancer classes differ in their genomic distribution, the functions of neighbouring genes, and the core promoter elements of these neighbouring genes. In addition, we identify two transcription fac- tors—Dref and Trl—that bind and activate housekeeping versus developmental enhancers, respectively. Our results provide evidence for a sequence-encoded enhancer-core-promoter specificity that sep- arates developmental and housekeeping gene regulatory programs for thousands of enhancers and their target genes across the entire genome.
We chose the core promoter of Ribosomal protein gene 12 (RpS12) and a synthetic core promoter derived from the even skipped transcription factor'' as representative ‘housekeeping’ and ‘developmental’ core pro- moters, respectively (hereafter termed hkCP and dCP; Fig. la and Ex- tended Data Figs 1, 2) and tested the ability of all candidate enhancers genome wide to activate transcription from these core promoters using self-transcribing active regulatory region sequencing (STARR-seg)'? in D. melanogaster S2 cells. This set-up allows the testing of all candidates in a defined sequence environment, which differs only in the core pro- moter sequences but is otherwise constant'*"*,
Two hkCP STARR-seq replicates were highly similar (genome-wide Pearson correlation coefficient (PCC) 0.98; Extended Data Fig. 1c) and yielded 5,956 enhancers, compared with 5,408 enhancers obtained when we reanalysed dCP STARR-seq data” (Supplementary Table 1). Inter- estingly, the hkCP and dCP enhancers were largely non-overlapping (Fig. 1b, c) and the genome-wide enhancer activity profiles differed (PCC 0.38), as did the individual enhancer strengths: of the 11,364 en- hancers, 8,144 (72%) activated one core promoter at least twofold more strongly than the other, a difference rarely seen in the replicate experi- ments for each of the core promoters (Fig. 1d). Indeed, 21 out of 24 hkCP-specific enhancers activated luciferase expression (>1.5-fold and t-test P< 0.05) from the hkCP versus 1 out of 24 from the dCP (Fig. le and Extended Data Fig. 3). Consistently, 10 out of 12 dCP- specific enhancers were positive with the dCP but only 2 out of 12 with the hkCP, a highly significant difference (P = 5.1 X 10 °, Fischer’s exact test) that confirms the enhancer-core-promoter specificity observed for thousands of enhancers across the entire genome.
Enhancers that were specific to either the hkCP or the dCP showed markedly different genomic distributions (Fig. 2a and Extended Data
# STARR-seg construct
a &S b sesony SOT — ORF | — e SE önü mezi say ©8090 FOE Ete ÖR — ORF ARAMA zi . hKoP P 5 __ 5 — PCP S2 STARR sea “ CP S2 STARR-seq ace P = A gi Ni TATAbox MTEDPE ——— e A ine g d hkCP S2STARR-seq @ Fold change Be <2x hKCP and dCP 3 *2e-4x hKOP «Dax gop & #>4xhkOP #>4x dCP g ° : 3. ow as eg 4 ği Ene. rept (og, EE j CP S2 STARR-sea | © 2 ae os 2 hkCP S2 dorse gE + vee 3 a3 STARR-seq STARR-seq ÖĞ 3 Ey 85° 2) w 4 2 e Sİ 4 Bs 2 = 5 â 2 0. 2. e. 18 Enr. rept (log,) hkGP S2 STARR-seq enrichment (og) MM hkOP luciferase MM GOP luciferase 100) 21/24 Luciferase construct {8 = Por in > salam Sil mii Enhancer candidate 10/12 6/7 6/7
# Figure 1 | Distinct sets of enhancers activate transcription from the hkCP and dCP in S2 cells.
transcription from the hkCP and dCP in S2 cells. a, STARR-seg set-up using the hkCP housekeeping (RpS12; purple) and dCP developmental core a 2 N promoters (Drosophila synthetic core promoter (DSCP)''; brown) b, Genome browser screenshot depicting STARR-seg tracks for both core promoters. c, Overlap of hkCP and dCP enhancers. d, hkCP versus dCP STARR-seq enrichments at enhancers (insets show enrichment for replicates z (Enr. rep) 1 versus 2 for hkCP and dCP; dCP data reanalysed from ref. 12). e, hkCP, dCP or shared enhancers that activate luciferase (>1.5-fold and P< 0.05 (one-sided t-test); n = 3; Extended Data Figs 3 and 5) from hkCP (purple) or dCP (brown; numbers show positive/tested).
"Research Institute of Molecular Pathology IMP, Vienna Biocenter VBC, Dr Bohr-Gasse 7, 1030 Vienna, Austria. *These authors contributed equally to this work.
556 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
d ül acd metabol process a | ey (Za a «* ; Intracellular transport Gelir component organization or biogenesis Sequence-spoctc UNA Binding RNA poll TF Regulation ot sgnaling Pattern spectiesion process Morphogenesis of an opthelum Anatom! structure morphogenesis hkOP S2 dcP S2 a STARR STARR» 8 nace saa seq mı Te ci rr ME Core promoter MCDS4*S'UTR 8 Göğe MM Proximal promoter Intron 5 Ubiquitous log lP value underrepresentation) Bm mar ince 3 Im emer cen teers «© hkoP dep genes genes b öğr © are © | ne ATCGAT Tw E 0 eg) Mott +f Ccarcacac iz © Ez © BB] vos CT. Be © Be 40 gg) Mott MM acrATxIT Fo, % 20 SE) motit7 lg CA C Cra aan > TV x1CTITr Endogenous BP BP Orientation 1 2 & g| TATA box HA TATAaAa location Po So” with respect to 25 inr TcAcTx « luciferase 5 Sa #5) (OMER C.A.CSA Lm 3°] ope fl cer nkcP Gandioate miz Enrichment hkCP- >2k0 ‘over dGP (iog;)
Figure 2 | hkCP and dCP enhancers differ in genomic distribution and flanking genes. a, Genomic distribution of hkCP and dCP enhancers. CDS, coding sequence; UTR, untranslated region. b, c, hkCP enhancers function distally in luciferase assays independent of their genomic positions (b) and orientation towards the luciferase TSS (c; orientation 1 from b; Extended Data Figs 3 and 5). d, e, GO (5 of the top 100 terms shown per column; Supplementary Table 11) and gene expression (terms curated from the Berkeley Drosophila Genome Project (BDGP) and FlyAtlas) analyses (d) and enrichment of core promoter elements at TSSs (e) for genes next to hkCP and dCP enhancers. TF, transcription factor.
# actvty
including metabolism, RNA processing and the cell cycle, whereas genes next to dCP enhancers were enriched for terms associated with devel- opmental regulation and cell-type-specific functions (Fig. 2d, Extended Data Fig. 6a and Supplementary Tables 2-4). Consistently, hkCP en- hancers were preferentially near ubiquitously expressed genes and dCP enhancers were near genes with tissue-specific expression (Fig. 2d and Supplementary Table 5).
The core promoters of the putative endogenous target genes ofhkCP and dCP enhancers were also differentially enriched in known core pro- moter elements" (Fig. Ze and Extended Data Fig, 6b): TSSs next to hkCP enhancers were enriched in Ohler motifs'* 1, 5, 6 and 7, consistent with the ubiquitous expression and housekeeping functions of these genes. In contrast, TSSs next to dCP enhancers were enriched in TATA box, initiator (Inr), motif ten element (MTE) and downstream promoter element (DPE) motifs, which are associated with cell-type-specific gene expression”,
We next investigated whether the specificity that hkCP and dCP show to the two enhancer classes applies more generally. We selected three additional core promoters from housekeeping genes with different func- tions: from the eukaryotic translation elongation factor 16 (eEF 16), the putative splicing factor x16, and the cohesin loader Nipped-B (NipB). Importantly, all three contained combinations of core promoter elements that differed from that of hkCP, namely TCT* and DNA- replication-related element (DRE) motifs (eEF16), and Ohler motifs 1 and 6 (x16 and NipB; Fig. 3a). In addition, we selected a DPE-containing core promoter of the transcription factor pannier (pnr) and the TATA- box core promoter of Heat shock protein 70 (Hsp70), which can be activated by tissue-specific enhancers (for example, see ref. 17), thus covering the two most prominent core promoter types of regulated genes”!“18,
Fig. 4): whereas the majority (58.4%) of hkCP-specific enhancers over- lapped with a transcription start site (TSS) or were proximal to a TSS (=200 bp upstream; Fig, Za), dCP-specific enhancers located predom- inantly to introns (56.5%) and intergenic regions (26.9%; Fig. 2a)'*. Importantly, despite the TSS-proximal location of most hkCP-specific enhancers, they activated transcription from a distal core promoter in STARR-seg (Fig. la and Extended Data Figs la, 2). Luciferase assays confirmed that they function from a distal position (>2 kb from the TSS) downstream of the luciferase gene and independently of their orienta- tion towards the luciferase TSS (Fig. 2b, cand Extended Data Figs 3, 5). These results show that TSS-proximal sequences can act as bona fide enhancers" and that developmental and housekeeping genes are both regulated through core promoters and enhancers, yet with a substan- tially different fraction of TSS-proximal enhancers (3.4% versus 58.4%).
hkCP and dCP enhancers were also located next to functionally dis- tinct classes of genes according to gene ontology (GO) analyses: genes next to hkCP enhancers were enriched in diverse housekeeping functions
We performed STARR-seg for the five additional core promoters and grouped the genome-wide enhancer activity profiles of all seven core promoters by hierarchical clustering. This revealed two distinct clus- ters corresponding to the four housekeeping and the three develop- mental core promoters, respectively (Fig. 3b, Extended Data Fig. 7 and Supplementary Tables 6, 7), and the core promoters of both clusters in- deed responded markedly differentially to individual genomic enhan- cers (Fig. 3c).
These results obtained for core promoters with diverse motif content and from genes with various functions suggest that the distinct enhan- cer preferences observed between hkCP and dCP apply more generally and that two broad classes of housekeeping and developmental (or regulated) core promoters exist. Differences within each class might correspond to differences in relative enhancer preferences of the core promoters” “, while similarities between both classes could reflect en- hancers that are shared (Fig. 1c-e) or core promoters that can be acti- vated to different extents by enhancers from both classes (for example,
"öz eEF18 — mh — RpS1Z — — (hkCP) — wo — gl — por — gi a DSCP a {acP) Hsp70 A par S2 STARR-seq DSCP S2 STARR -seq Hsp70 S2 STARR-seq en | Wer A milin
Figure 3 | Housekeeping and developmental core promoters differ characteristically in their enhancer preferences. a, Different housekeeping (top 4) and developmental-like (bottom 3) core promoters and their motif content (schematic). b, Bi-clustered heat map depicting pairwise similarities of STARR-seq signals (PCCs at peak summits). PCCs and dendrogram (top) show the separation between housekeeping and regulated core promoters. c, Genome browser screenshot depicting STARR-seq tracks for all seven core promoters.
26
# FEBRUARY
2015
# VOL
518
|
# NATURE
557
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
064287 0065516 _— 10022856 İZ a mm m A A hkCP S2 STARR-seq Ni hkCP OSC STARR-seq İ A een RE Onan dCP S2 STARR-seq m “İL hkCP S2 dCP OSC STARR-seq dcP S2 dcP OSC hkGP OSC STARR-seq STARR-seq STARR-seg STARR-seq (rol change, 5, hkePs2 d 5 e change ? doP S2 <2x S2 and .<2xS2 a 124x52 424x080 = 12x4X52 «2x4x0S0 Ş ş 6 1>4xS2. e>dxOSC O Ş 6] »>4x52 «>4x05C 8] öl İz = Ki ad o = İ- = &ş4 iv Se a < : Enr rept (iog,) Fs 2 Enr. rept (log,) 5 ~ QE ” By hkcr osc ae 2 gy “oc 85 2 Bö 2 5 £0 rf ago Hy « g 5 > Sİ © if 8 vi 2 ğ 2 ğ 2 o 2 4 Enr. rept (log,) 2 0 2 4 6 — Enr. rept (log,) hkCP S2 STARR-seq enrichment (log,) dCP S2 STARR-seq enrichment (log,)
Figure 4 | hkCP enhancers are shared across cell types. a, Genome browser screenshot showing tracks for hkCP (top) and dCP STARR-seg (bottom) in S2 cells and OSCs. b, Overlap of hkCP (top) and dCP (bottom) enhancers between S2 cells and OSCs. c, d, hkCP (c) and dCP (d) STARR-seq enrichments in $2 cells versus OSCs at hkCP- or dCP-specific enhancers (insets show enrichments for replicates (Enr. rep) 1 versus 2; dCP data reanalysed from ref. 12).
NipB; Fig. 3b, c). The latter might be important if broadly expressed housekeeping genes need to be further activated in specific tissues.
To test whether hkCP enhancers function in different cell types, we performed STARR-seg using hkCP in OSCs, which differ strongly from $2 cells in gene expression and dCP enhancer activities’. Two hkCP STARR-seg replicates in OSCs were highly similar (PCC 0.97) and yielded 6,217 enhancers (Supplementary Table 1), compared with 5,774 en- hancers obtained for dCP data from OSCs'”. The OSC data confirmed the differences between hkCP and dCP enhancers observed in S2 cells (Extended Data Figs 8, 9 and Supplementary Tables 8-10). Strikingly,
the hkCP-specific enhancers in OSCs and S2 cells (3,357 and 4,137, re- spectively) were almost indistinguishable, whereas dCP-specific enhan- cers (2,909 in OSCs and 3,586 in S2 cells) differed strongly between the two cell types’? and from the hkCP enhancers (Fig. 4a). The observation that hkCP enhancers showed similar activities in both cell types while dCP enhancers were cell-type specific was true genome wide when comparing genomic locations (69% versus 15% overlap) or enhancer strengths as measured by STARR-seg (PCC at peak summits 0.83 ver- sus 0.05; Fig. 4b-d and Extended Data Fig. 9c). Together, these results show that hkCP enhancers are shared between two different cell types,
a DRE Bİ see P= 62x10" GCGCGC g m 3. ii Az cen fe 24 7 $$ 2 ‘ ie 53 E EE al, oe al Mi rce s2 Serpent 5 ğ oj ™ 33 ! a misc so ME127 6 §F * = re | ped > 4 eek = e, 140) Six d 4 35 . Ns İİ STARRMoL3 § 120 Sm 1 Bore ğ 100 ge oa a 2 er 3 20 GE ace 2 40 Tate ree İn S2 cells 3 20. gs hKCP dCP 20 = 0 6x DRE motif — + — Bie oe Sey moses Gore promoter: GP ace “amme ER ö 2— SEMMEMMMC: TEME 7 a P g 1600) 23 g 100, 245 Zax Y 2.3% Sw lo tee (PAs) ome — i Sates a / hkop aa B 1000 y€ a . = ri ped Tax ¢“ Six 29 ki lg ie g 400 g 2 \ P NN ğ — 0. ye oi PAS) nes GAGASDRE - + — + Enhance: A OB OC OD ae Mutant, Enhance: EF mutation: 5/5 3 v2 “ Core promoter: ace Housekeeping transcriptional program Developmental transcriptional program Proximal Distal activation Enhancer Enhancer a Developmental gene —[ Housekeeping gene | — — wep FOS ce GR poe ‘SGTATXIT GGxcAcac TATAAa ToACTx .CactT= DPE Motif 6 Motif 1 TATA box Ine
# DRE
# f
Figure 5 | hkCP and dCP enhancers depend on Dref and Trl, respectively. a, b, Motif enrichment (a) and ChIP signals for Dref and Trl (b) in hkCP and dCP enhancers. False discovery rate (FDR)- corrected hypergeometric P > 0.01; boxes: median and interquartile range; whiskers: 5th and 95th percentiles; two-sided Wilcoxon-rank-sum P values. NS, not significant. c, Luciferase assays for four wild-type and DRE-motif-mutant hkCP enhancers (numbers show mutated motifs). Error bars show standard deviation (s.d.) (n = 3, biological replicates). *P < 0.005 (one-sided t-test). d, Luciferase assays for two dCP enhancers (—) and their GAGA > DRE-mutant variants (+) with HKCP (top) and dCP (bottom; details as in e). e, Luciferase assays for an array of DRE motifs with hkCP and dCP (details as in c). f, Model: A housekeeping genes contain Ohler motifs 1, 5, 6, 7 and/or the TCT motif and are activated by TSS- proximal hkCP enhancers via Dref. Regulated genes contain TATA box, Inr, MTE and/or DPE and are activated by distal dCP enhancers via Trl.
# 558 | NATURE
|
# VOL
518
26
# FEBRUARY
2015
# ©2015 Macmillan Publishers Lit imited. All rights reserved
LETTER
whereas dCP enhancers are cell-type specific", presumably represent- ing ubiquitous housekeeping versus developmental and cell-type- specific gene expression programs.
To assess whether the marked core promoter specificities of the hkCP and dCP enhancers are encoded in their sequences, we analysed the cis- regulatory motif content of both classes of enhancers'”. This revealed a strong enrichment of the DRE motif in hkCP enhancers (Fig. 5a and Supplementary Tables 11, 12), whereas dCP enhancers were strongly enriched in the GAGA motif of Trithorax-like (Trl) and other motifs previously described to be important for dCP enhancers”, Published genome-wide chromatin immunoprecipitation (ChIP) data”> con- firmed that DRE-binding factor (Dref) bound significantly more strongly to hkCP enhancers than to dCP enhancers (Wilcoxon P = 0; Fig. 5b), while the opposite was true for Trl (Wilcoxon P = 6.2 X 10 '”). Consid- ering only distal enhancers (>500 bp from the closest TSS) yielded the same results (Extended Data Fig. 10a, b and Supplementary Tables 13, 14), suggesting that the differential occupancy is a property of both classes of enhancers rather than a consequence of the different extents to which they overlap with TSSs. Disrupting the DRE motifs in four different hkCP enhancers substantially reduced the activities of the enhancers as measured by luciferase assays in S2 cells (between 2.3- and 24.5-fold reduction; Fig. 5c), while dCP enhancers depend on GAGA motifs”. Adding DRE motifs to 11 different dCP enhancers significantly increased luciferase expression from the hkCP for 9 of them (82%; Extended Data Fig. 10c), and changing the GAGA motifs of two dCP enhancers to DRE motifs significantly increased the activities of both enhancers towards the hkCP but decreased their activities towards the dCP (Fig. 5d). Fur- thermore, an array of six DRE motifs was sufficient to activate lucifer- ase expression from the hkCP but not the dCP (Fig. 5e). Together, these results show that hkCP and dCP enhancers depend on DRE and GAGA motifs, respectively, and demonstrate that DRE motifs are required and sufficient for hkCP enhancer function.
- 7. Kadonaga, J. T. Perspectives on the RNA polymerase Il core promoter. Wiley Interdiscip. Rev. Dev. Biol. 1, 40-51 (2012).
- Parry, T. J. et al. The TCT motif, a key component of an RNA polymerase II transcription system for the translational machinery. Genes Dev. 24, 2013-2018 (2010).
8.
- 9. Engstrém, P. G., Ho Sui, S. J., Drivenes, O, Becker, T. S. & Lenhard, B. Genomic regulatory blocks underlie extensive microsynteny conservation in insects. Genome Res. 17, 1898-1908 (2007).
- 10. FitzGerald, P.C., Sturgill, D., Shyakhtenko, A., Oliver, B. & Vinson, C. Comparative genomics of Drosophila and human core promoters. Genome Biol. R53 (2006).
- 11. Pfeiffer, B.D. et al. Tools for neuroanatomy and neurogenetics in Drosophila. Proc. Natl Acad. Sci. USA 105, 9715-9720 (2008).
- 12. Arnold, C. D. et al. Genome-wide quantitative enhancer activity maps identified by STARR-seq. Science 339, 1074-1077 (2013).
- 13. Shlyueva, D. et al. Hormone-responsive enhancer-activity maps reveal predictive motifs, indirect repression, and targeting of closed chromatin, Mol. Cell 54, 180-192 (2014).
- 14. Banerji,J., Rusconi, S. & Schaffner, W. Expression of a B-globin gene is enhanced by remote SV40 DNA sequences. Celi 27, 299-308 (1981).
- 15, Lenhard, B., Sandelin, A. & Carninci, P. Metazoan promoters: emerging characteristics and insights into transcriptional regulation. Nature Rev. Genet. 13, 233-245 (2012).
- 16. Ohler, U,, Liao, G.-C., Niemann, H. & Rubin, G. M. Computational analysis of core promoters in the Drosophila genome. Genome Biol. 3, research0087.1-0087.12 (2002).
- 17. Smith, D., Wohlgemuth, J, Calvi, B. R., Franklin, |. & Gelbart, W. M. hobo enhancer trapping mutagenesis in Drosophila reveals an insertion specificity different from elements. Genetics 135, 1063-1076 (1993).
- 18. Kutach, A. K. & Kadonaga, J. T. The downstream promoter element DPE appears to be as widely used as the TATA box in Drosophila core promoters. Mol. Cell, Biol. 20, 4754-4764 (2000).
- 19. Yafiez-Cuna, J. O., Dinh, H. Q., Kvon, E. Z., Shlyueva, D. & Stark, A. Uncovering cis- regulatory sequence requirements for context-specific transcription factor binding. Genome Res. 22, 2018-2030 (2012).
- 20. Yafiez-Cuna, J. O. et al, Dissection of thousands of cell type-specific enhancers identifies dinucleotide repeat motifs as general enhancer features. Genome Res. 24, 1147-1156 (2014).
- 21. Gurudatta, B. V. Yang, J., Van Bortle, K., Donlin-Asp, P. G. & Corces, V. G. Dynamic changes in the genomic localization of DNA replication-related element binding factor during the cell cycle. Cel! Cycle 12, 1605-1615 (2013).
Our results show that developmental and housekeeping gene regu- lation is separated genome wide by sequence-encoded specificities of thousands of enhancers to one of two types of core promoter, supporting the longstanding 'enhancer-core-promoter specificity’ hypothesis” ©, Our findings indicate that these specificities are probably mediated by defined biochemical compatibilities* between different trans-acting factors such as Dref versus Trl (at enhancers) and the different para- logues that exist for several components of the general transcription apparatus (at core promoters), presumably including the TATA-box- binding protein-related factor 2 (Trf2) at housekeeping core promoters””“, As such paralogues can have tissue-specific expression and stage-specific or promoter-selective functions”” (reviewed in refs 29, 30), sequence- encoded enhancer-core-promoter specificities could be used more widely to define and separate different transcriptional programs (Fig. 5f).
- 22. modENCODE Consortium Identification of functional elements and regulatory circuits by Drosophila modENCODE. Science 330, 1787-1797 (2010).
- 23. Ohler, U. & Wassarman, D. A. Promoting developmental transcription. Development 137, 15-26 (2010).
- 24. van Arensbergen, J., van Steensel, B. & Bussemaker, H. J. In search of the determinants of enhancer-promoter interaction specificity. Trends Cell Biol. http://dx.doi.org/10,1016/j.tcb.2014.07.004 (2014).
- 25. Wang, Y.-L. et al. TRF2, but not TBP, mediates the transcription of ribosomal protein genes. Genes Dev. 28, 1550-1555 (2014).
- 26. Isogai, Y., Keles, S., Prestel, M., Hochheimer, A. & Tjian, R. Transcription of histone gene cluster by differential core-promoter factors. Genes Dev. 21, 2936-2949 (2007).
- 27. Hochheimer, A, Zhou, S., Zheng, S., Holmes, M. C. & Tjian, R. TRF2 associates with DREF and directs promoter-selective gene expression in Drosophila. Nature 420, 439-445 (2002).
- 28. Deato, M. D. E. & Tjian, R. Switching of the core transcription machinery during myogenesis. Genes Dev. 21, 2137-2149 (2007).
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
- 29. D'Alessio, J. A, Wright, K. J. & Tjian, R. Shifting players and paradigms in cell- specific transcription. Mol. Cell 36, 924-931 (2009).
- 30. Miller, F., Zaucker, A. & Tora, L. Developmental regulation of transcription initiation: more than just changing the actors. Curr. Opin. Genet. Dev. 20, 533-540 (2010).
# Received 22 May; accepted 20 October 2014.
Supplementary Information is available in the online version of the paper.
Published online 15 December 2014; corrected online 25 February 2015 (see full-text HTML version for details).
- 1. Levine,M, Cattoglio, C. & Tjian, R. Looping back to leap forward: transcription enters a new era. Cell 157, 13-25 (2014).
- Li, X. & Noll, M.Compatibility between enhancers and promoters determines the transcriptional specificity of gooseberry and gooseberry neuro in the Drosophila embryo. EMBO J. 13, 400-406 (1994).
2.
- Ohtsuki, S., Levine, M. & Cai, H. N. Different core promoters possess distinct regulatory activities in the Drosophila embryo. Genes Dev. 12, 547-556 (1998).
3.
Acknowledgements We thank L. Cochella and O, Bell for comments on the manuscript. Deep sequencing was performed at the CSF Next-Generation Sequencing Unit (http:// csfac.at). M.A.Z. was supported by the Austrian Science Fund (FWF, F4303-B09) and C.D.A,, K.S., M.R, and OF, by a European Research Council Starting Grant (no. 242922) awarded to A.S. Basic research at the Research Institute of Molecular Pathology is supported by Boehringer Ingelheim GmbH.
Author Contributions M.AZ., C.D.A. and A.S. conceived the project. C.D.A., K.S., M.P., MLR. and O.F. performed the experiments and M.AZ. the computational analyses. M.AZ,, C.D.A. and AS, wrote the manuscript.
- 4. Sharpe, J., Nonchev, S., Gould, A., Whiting, J. & Krumlauf, R. Selectivity, sharing and competitive interactions in the regulation of Hoxb genes. EMBO J. 17, 1788-1798 (1998).
- 5. Merli, C., Bergstrom, D. E, Cygan, J. A. & Blackman, R. K. Promoter specificity mediates the independent regulation of neighboring genes. Genes Dev. 10, 1260-1270 (1996).
- Butler, J. E. & Kadonaga, J. T. Enhancer-promoter specificity mediated by DPE TATA core promoter motifs. Genes Dev. 15, 2515-2519 (2001).
6.
# or
Author Information All deep sequencing data are available at http://www.starklab.org and have been deposited in the Gene Expression Omnibus database under accession numbers GSE40739 and GSE57876. Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to AS. (stark@starklab.org).
26 FEBRUARY 2015 | VOL 518 | NATURE | 559
# ©2015 Macmillan Publishers Limited. All rights reserved
7,
# P
LETTER
doi:10.1038/nature14234
# N“-methyladenosine-dependent RNA structural switches regulate RNA-protein interactions
Nian Liu’, Qing Dai’, Guangun Zheng’, Chuan He2:4, Marc Parisien? & Tao Pan?
RNA-binding proteins control many aspects of cellular biology through binding single-stranded RNA binding motifs (RBMs)'*. However, RBMs can be buried within their local RNA structures*’, thus inhibiting RNA-protein interactions. N°-methyladenosine (m°A), the most abundant and dynamic internal modification in eukaryotic messenger RNA*-”, can be selectively recognized by the YTHDF2 protein to affect the stability of cytoplasmic mRNAs'5, but how m“A achieves its wide-ranging physiological role needs further exploration. Here we show in human cells that m“A controls the RNA-structure- dependent accessibility of RBMs to affect RNA-protein interactions for biological regulation; we term this mechanism ‘the m“A-switch', We found that m“A alters the local structure in mRNA and long non-coding RNA (IncRNA) to facilitate binding of heterogeneous nuclear ribonucleoprotein C (HNRNPC), an abundant nuclear RNA- binding protein responsible for pre-mRNA processing”, Combining
_ _ o
| _
_
_ ©
photoactivatable-ribonucleoside-enhanced crosslinking and immu- noprecipitation (PAR-CLIP) and anti-m°A immunoprecipitation (MeRIP) approaches enabled us to identify 39,060 m°A-switches among HNRNPC-binding sites; and global m“A reduction decreased HNRNPC binding at 2,798 high-confidence m°A-switches. We deter- mined that these m“A-switch-regulated HNRNPC-binding activities affect the abundance as well as alternative splicing of target mRNAs, demonstrating the regulatory role of m°A-switches on gene expres- sion and RNA maturation. Our results illustrate how RNA-binding proteins gain regulated access to their RBMs through m°A-dependent RNA structural remodelling, and provide a new direction for inves- tigating RNA-modification-coded cellular biology.
Post-transcriptional m°A RNA modification is indispensable for cell viability and development, yet its functional mechanisms are still poorly understood*""’. We recently identified one m“A site in a hairpin-stem
2,577-A 2, ei 4202240136010 120]240/360|0 NaCl sjelelelelelo o oJelelelelelo NE ==
© 2,577-A 722 + 191nM 0 * 2,577-m°A 93 + 22nM 0 1,000 2,000 3,000 4,000 5,000 6,000
# (mM) (ug)
# HNRNPU DDX21
# gl
HNRNPC ei HIST1H1C
oe ae A Sa koa 9) 220 il iH20 e imi Pull-down protein (2,577-m®A/A) is AA 58 w in Ey â 56 >>Poc
»
= :
# KI
Figure 1 | m“A alters RNA structure to enhance HNRNPC binding. a, m°A increases binding with nuclear proteins. NE, nuclear extract. b, RNA pull down showing that HNRNPC preferably binds methylated RNA. Data are mean * standard deviation (s.d.); n = 4, biological replicates. Ctrl, control. ¢, Filter binding showing that m°A increases HNRNPCI binding with respective apparent dissociation constants (Ky) indicated at bottom right. Data are mean + s.d.; n = 3, technical replicates. d, RNA structural probing showing that m“A disrupts local RNA structure, highlighted in yellow. AH, alkaline hydrolysis; Ctrl, no nuclease added; G-L, G-ladder; $1, nuclease SI digestion; TI, RNase TI digestion; V1, RNase V1 digestion. The orange bars mark the structurally altered RNA regions in the presence of m®°A (red dot). e, RNA pull down with mutated oligonucleotides. Data are mean + s.d; n = 3, technical replicates. £, Illustration of the m°A-switch model.
# HNRNPC1 (nM)
2,560
—
—
# Nuclease S1 cleavage RNase V1 cleavage
3
# f
# z Pr 252 Ea Zs fs ce Eğ 251 eg 2
# A
# Methylation
=! =| ci
# mA
0
# Demethylation
Ss un? ee ot a gi“ oe or
'Department of Chemistry, The University of Chicago, Chicago, Illinois 60637, USA, Department of Biochemistry and Molecular Biology, The University of Chicago, Chicago, İllincis 60637, USA. “institute for Biophysical Dynamics, The University of Chicago, Chicago, lllinois 60637, USA. “Howard Hughes Medical Institute, The University of Chicago, Chicago, Illinois 60637, USA. {Present address: Faculty Dentistry, McGill University, Montreal, Quebec H3A OGI, Canada
560 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
# of
LETTER
on the human IncRNA metastasis-associated lung adenocarcinoma transcript (MALATI)” (Extended Data Fig. la). A native gel-shift assay indicated that this m°A residue increases the interaction of the RNA hairpin with proteins in the HeLa nuclear extract (Fig. la). RNA pull- down assays identified HNRNPC as the protein component of the nuclear extract that binds more strongly with the m°A-modified hairpin than the unmodified hairpin (Fig. 1b and Extended Data Fig. 1b, c). The m“A-enhanced interaction with the hairpins was validated qualitatively by ultraviolet crosslinking and quantitatively (~8-fold increase) by filter binding using recombinant HNRNPCI protein (Fig. 1c and Extended Data Fig. 1d).
The HNRNPC protein belongs to the large family of ubiquitously expressed heterogeneous nuclear ribonucleoproteins that bind nascent RNA transcripts to affect pre-mRNA stability, splicing, export and translation” *, HNRNPC preferably binds single-stranded U-tracts (five or more contiguous uridines)”252*2627. In the MALATI hairpin, HNRNPC bindsa U;-tract that is half buried in the hairpin-stem oppos- ing the 2,577-A/m'A site (Extended Data Fig. la, e).
Since m“A residues within RNA stems can destabilize the thermosta- bility of model RNA duplexes’, we hypothesized that the 2,577-m°A residue destabilizes this MALAT1 hairpin-stem to make its opposing U-tract more single-stranded or accessible, thus enhancing its inter- action with HNRNPC. We performed several experiments to validate
this hypothesis. First, according to the RNA structural probing assays, the m°A-modified hairpin showed significantly increased nuclease $1 digestion (single-strand specific) at the GAC (A = m°A) motif, as well as markedly decreased RNase V1 digestion (double-strand/stacking spe- cific) at the U-tract opposing the GAC motif (Fig. 1d). The mA residue markedly destabilized the stacking properties of the region centred around the U residue that pairs with 2,577-A/m°A (Extended Data Fig, If, g), which was also supported by the increased reactivity between CMCT and the U-tract bases in the presence of m°A (Extended Data Fig. 1h). Second, the 2,577-A-to-U mutation increased the HNRNPC pull-down amount from the nuclear extract, whereas U-to-C mutations in the U-tract significantly reduced the HNRNPC pull-down amount regard- less of m°A modification (Fig. le). Third, the 2,577-A-to-U mutation increased the accessibility of the U-tract and enhanced HNRNPC bind- ing by ~4-fold (Extended Data Fig. 2a-c). Binding results with four other mutated A/m°A oligonucleotides also supported the U-tract, with increased accessibility alone being sufficient to enhance HNRNPC bind- ing (Extended Data Fig. 2d). Fourth, RNA terminal truncation followed by HNRNPC binding identified two pairs of truncated hairpins with highly accessible U-tracts, which improved HNRNPC binding sig- nificantly but independent of the m“A modification (Extended Data Fig. 2e-i). All these results confirmed that m“A modification can alter its local RNA structure and enhance the accessibility of its base-paired
2 N T 0. e 2 2 = 6 1,500 | Input MALAT1 z } 2 = 0.6 5 21 1 a = 305 an e o o &RVe g 8 8 123 465 1234 5 2 0.4 ba 3 & « z03 m f _ 64 ARACH-U-tract & < ° coupling events ş 02 Bi 4 z 2 € 0.1 = d Ea 2,540 2,560 2,580 2,600 2,620 % 2 JX ah so so gh fat Roe e we” ot 0 KANG e“ e 2 «1 0 1 2 önü Fold change (IP/input) at RRACH (log,) c UV irradiation g GS Ag g *g HNRNPH1-A_HNRNPH1-m®A 1.0 is 08 em°A y 28 aes RNase digest; Ü 5 U 3 06 V anti-HNRNPCIP 5,024*6 eU i ” BS 4 RNA-HNRNPC A, dey & complex Geu 02 U-A Ni » HNANPHT-A 1,476 + 284 nM Protein digestion C-G | G-c 0 * HNANPHT-M'A 507 + 104 nM G-c er ee A ree ee Bound RNA VE G-C © 1,000 2,000 3,000 4,000 5,000 6,000 SOS, 5015°U A HNRNPC1 (nM) h m6) 1.0} Input control “4 Anti-m'A IP 0.8 ee 36,816 The NK 816+ z UeG 506 U-A 36,825 $ Elution with m&A UeG S04 © DNAJO25-GNGTO-A nd v e S 36,820 € 1,202 + 300 nM ~ © — 02 ami = HI m 36,815 * DNAJC25-GNGTO-m'A A A*36,826 o 279496nM v 46807*A G 36,810 RNA-seg © 1,000 2,000 3,000 4,000 5,000 6,000 HNRNPC1 (nM)
Figure 2 | PAR-CLIP-MeRIP identifies m“A-switches transcriptome wide. a, CLIP-2dTLC showing the m®A enrichment in HNRNPC-bound RNA regions. Data are mean + s.d.; n = 3, biological replicates. IP, immunoprecipitation. b, HNRNPC-bound RNA regions had higher anti-m°A pull-down yield than polyA* RNA. Data are mean + s.d.; n = 3, biological replicates. c, Illustration of the PAR-CLIP-MeRIP protocol. UV, ultraviolet.
d, PAR-CLIP-MeRIP identified an m°A residue around the MALATI 2,577 site. e, Binding motifs identified with PAR-CLIP-MeRIP peaks. f, Density plot showing the positive enrichment at RRACH sites. g, h, Validation of two m°A-switches by $1/V1 structural probing and filter binding. Data are mean + s.d.; n = 4, technical replicates. Annotation is the same as in Fig. Ic, d.
26 FEBRUARY 2015 | VOL 518 | NATURE | 561
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
residues or nearby regions to modulate protein binding (Fig. 10). We term this mechanism that regulates RNA -protein interactions through m°A-dependent RNA structural remodelling ‘the m°A-switch’,
We performed two experiments to determine the global effect of m“A-switches on HNRNPC binding. First, in vivo crosslinking followed by immunoprecipitation and two-dimensional thin-layer chromatography (CLIP-2dTLC) showed that the m“A/A ratio of the HNRNPC-bound RNA regions had a ~6-fold higher m“A level than the HNRNPC-bound intact RNA, anda ~3-fold higher m“A level than the flow-through RNA (Fig. Za and Extended Data Fig. 3a). Second, the HNRNPC-bound RNA regions had much higher anti-m°A pull-down yield (4.3%) than the polyA* RNA samples (0.5%) using the previously established m°A antibody'*"* (Fig. 2b). These results indicate a widespread presence of m“A residues in the vicinity of HNRNPC-binding sites.
were located mostly within 50 residues, suggesting transcriptome-wide RRACH-U-tract coupling events within the HNRNPC-binding sites (Extended Data Fig. 4a, b). About 62% of all RRACH-U-tract coupling events within HNRNPC-binding sites are enriched at the RRACH motif (Fig. 20). Our PAR-CLIP-MeRIP approach identified a total of 39,060 HNRNPC m°A-switches that corresponded to m°A-modified RRACH- U-tract coupling events at a false discovery rate (FDR) = 5% (Extended Data Fig. 4c). These switches account for ~7% of 592,477 HNRNPC- binding sites identified by PAR-CLIP. The majority (87%) of m°A- switches occur within introns (Extended Data Fig. 4d, e), consistent with the literature that HNRNPC is nuclear localized and primarily binds nascent transcripts”. We validated two intronic m°A-switches in hairpin structures in which m“A residues increase the U-tract acces- sibility and enhance HNRNPC binding by ~3-4 fold (Fig. 2g, h and Extended Data Fig. 5).
Tomap the m“A sites around HNRNPC-binding sites, we performed PAR-CLIP” to isolate all HNRNPC-bound RNA regions (input control sample) followed by MeRIP'*" to enrich m“A-containing HNRNPC- bound RNA regions (IP sample). Both the input control and IP samples from two biological replicates were sent for RNA sequencing (RNA-seq) (Fig. 2c and Extended Data Fig. 3b, c). This approach, termed PAR-CLIP- MeRIP, identified transcriptome-wide the m°A-proximal HNRNPC- binding site, such as the enriched peak around the MALATI 2,577 site (Fig. 2d). Remarkably, HNRNPC PAR-CLIP-MeRIP peaks harboured two consensus motifs, the HNRNPC RBM (U-tracts) and the m°A con- sensus motif GRACH (a subset of RRACH'*"*) (Fig. 2e). Both motifs
Toassess the effect of global m°A reduction on RNA-HNRNPC inter- actions, we performed HNRNPC PAR-CLIP experiments in METTL3 and METTL14 knockdown cells (Extended Data Fig. 6a). We identified 16,582 coupling events with decreased U-tract-HNRNPC interactions upon METTL3 and METTL14 knockdown (METTL3/L14 knockdown) (Fig. 3a and Extended Data Fig. 6b, c). In total, 2,798 m°A-switches iden- tified by PAR-CLIP-MeRIP experiments showed decreased HNRNPC binding upon METTL3/L14 knockdown (Fig. 3b) and this number is probably an underestimate due to the fact that METTL3/L14 knock- down reduces the global m°A level by only ~30-40% (refs 11, 12). These
@ 8] RRACH-U-tract ii = METTL3 KD 6 coupling events — METTL14 KD 4 22 x = o ge amele — METTL3/L14 KD â 6 coupling events 4 2 o “2 a o 1 2 Fold change (KD/Ciri) at U-tracts (log,) d 71 3 2 x 8 Za — Intronic m®A-switch 2 — Exon m®A-switch â0 5 Intron/exon 3 foo om om * TR Ri A JO Ae UU. 2 1.0 08 lj 06 4 § 0 İ 3 02 g o . - = * eee gS“UUUUU.R Cc H3 S 10 f] 5 g 08 3 6 11111141141 11414114141 b METTL3/L14 KD PAR-CLIP-MeRIP HCS m'A-switches c CDS >> W 3'UTR 12% Non-coding RNA é 2% =10 ge x6 24 22 i 80 codon -800 -400 0 400 800 9 oe 0.8 oo şan 2064 4 L 1 = g ay ~ ae g 0.4 $02 zo 02 a+ g 04° § $ $ $ Se £ £ £ £ ES 3835 RE 35 Re fe eGeEae Sc Ed Ed Ef EE BB ——
SIM1 cleavage preference log, ol(2 — pV1)/(2 — pS1))/log,
,2
U-tracts RRACH U-tracts RRACH
Figure 3 | Global m“A reduction decreases HNRNPC binding at m°A-switches. a, Density plot showing negative enrichment at the U-tracts. KD, knockdown. b, Identification of high-confidence (HCS) m°A-switches. ¢, Regional distribution of high-confidence m°A-switches. CDS, coding sequence. d, Density plot showing m°A-switch distribution relative to exon/intron boundaries, e, m°A-switches in coding RNA were enriched in the 3' UTR and near the stop codon. f, Cumulative distribution of HCS
m°A-switches (black) and control (orange) regarding the $1/V1 cleavage preference (data from ref. 4) at U-tracts and RRACH motif. U-tract can be 3' (top) or 5' (bottom) of the RRACH motif. *P < 0.05, **P< 10-4, Kolmogorov-Smirnov test. g, Phylogenetic conservation of high-confidence m“A-switches among primates and vertebrates. ***P < 10 '°, Mann- Whitney-Wilcoxon test.
562 | NATURE | VOL 518 | 26 FEBRUARY 2015
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
sites composed the high-confidence m“A-switches that were used for subsequent analysis.
High-confidence m°A-switches are enriched in the introns of coding and non-coding RNAs (Fig. 3c and Extended Data Fig. 6d). Exonic m°A-switches are enriched at the middle of exons whereas intronic m°A- switches are slightly enriched near the 5’ end (Fig. 3d). m°A-switches within coding RNAs tend to locate at very long exons (Extended Data Fig. Ge) and are enriched near the stop codon and in the 3’ untranslated region (UTR) (Fig, 3e), consistent with the known topology of the human m°A methylome in mRNAs'*". Transcriptome-wide RNA structural mapping'” on high-confidence m°A-switches yielded consistent struc- tural patterns with our three demonstrated m°A-switch hairpins (Fig. 3f). The RR residues in the RRACH motif and the 3’ U-tract residues show increased structural dynamics in the presence of m“A. Besides, m°A- switches prefer short RRACH-U-tract inter-motif distances, are not involved in the previously reported inter-U-tract motif patterns and are conserved across species (Fig. 3g and Extended Data Fig. 6f-i).
To reveal the function of m°A-switches on RNA biology, we performed polyA* RNA-seq from HNRNPC, METTL3 and METTL14 knockdown and control cells (Extended Data Fig. 7a). METTL3/L14 knockdown, which has been shown to decrease HNRNPC binding transcriptome- wide, co-regulated the expression of 5,251 genes with HNRNPCknock- down. In comparison, METTL3/L14 knockdown co-regulated only 24 genes with knockdown of another mRNA-binding protein, HNRNPU (Extended Data Fig. 7b), which was not enriched in our m“A-hairpin
pull down (Fig. 1b). Approximately 45% of 1,815 high-confidence m°A- switch-containing genes were co-regulated by HNRNPC and METTL3/ L14knockdown, indicating that m°A-switch-regulated HNRNPC bind- ing affects the abundance of target mRNAs. Gene ontology (GO) analysis suggests that m°A-switch-regulated gene expression may influence ‘cell proliferation’ and other biological processes (Extended Data Fig. 7c). The m°A-switch-regulated expression of genes within these GO categories was validated by quantitative polymerase chain reaction (qPCR) (Fig. 4a and Extended Data Fig. 7d-g). We also found that HNRNPC, METTL3 and METTL14 knockdown decreased the cell proliferation rate to sim- ilar extents (Extended Data Fig. 7h).
Besides the mRNA abundance level changes, we also observed splic- ing pattern changes within high-confidence m°A-switch-containing tran- scripts by testing the differential exon usage in RNA-seq data (DEXSeg)”", HNRNPCknockdown co-up/downregulated 131/127 exons with METTL3 knockdown and 130/115 exons with METTL14 knockdown. These co- regulated exons occur more frequently in the vicinity of m°A-switches than non-co-regulated exons (Fig. 4b, c), indicating that m°A-switches tend to regulate splicing events at nearby exons. We investigated the splicing pattern at two exons with neighbouring m°A-switches: the PAR-CLIP-MeRIP and METTL3/L14 knockdown data confirmed the HNRNPC-binding signature at the m°A-switch site neighbouring these exons; and HNRNPC and METTL3/L14 knockdown co-inhibited exon inclusion in both cases (Fig. 4d-fand Extended Data Fig. 8b-f). Besides, we identified 155 genes with multiple m°A-switches exhibiting more
pn 0.8 0.6 04 02 Cumulative fraction of exons © Co-regulated SHINANPC (P = 5 x 10-%) VY METTL3 (P = 8 x 1019) © Co-regulated A HNANPC (P =7 x 10“) VY METTLI4 (P=5 x 10°) —"—HNNRNPC KD RNA-seq —"—HNRNPC KD gPOR Relative abundance of mRNA transcript (compared with Ctrl) 24 6 8 10 O 2 4 6 8 Exon distance to HCS m®A-switch 10 o —e—METTL3 KD RNA 2 a EMET KDGPCR d S| 40 g0? Input —— METTL14 KD RNA-ssa 8 o 2.84 —o—METTL14 KD qPCR 2 40 IP é 17 AA 21,700 21,750 21,800 21,850 Mi gj240 Oe ctrl Sl 0 8240 mes METTL3 KD z 0 e) | a am, VE77L14 KD 0 m m 21,700 21,750 21,800 21,850 f 55 çı cps2 m3A-switch $ 2 g 9 9? b ae SE ş © ax & § Gene m®A-switch ANE v 40 Exon inclusion so & oe — am) m aa — 700 % 20 2 a 0 0 +1 i 500 E Exon distance to mPA-switch Exon exclusion 200 Z g 100 SOOO
3 # <
Figure 4 | m“A-switches regulate mRNA abundance and alternative splicing. a, HNRNPC, METTL3/L14 knockdown (KD) co-regulated the abundance of m“A-switch-containing transcripts by RNA-seq and qPCR. Ctrl, control. b, Illustration of the relative exon distance to m°A-switches. c, Co- regulated exons by HNRNPC knockdown and METTL3 knockdown (left) and METTLI4 knockdown (right) were more enriched around m*A-switch sites than non-co-regulated exons, Kolmogorov-Smirnov test. HCS, high
confidence m°A-switches. d-f, Validation of the m°A-switch-regulated splicing at one exon neighbouring the CDS2 m°A-switch as shown in PAR- CLIP-MeRIP data (d), METTL3/L14 knockdown data (e), and PCR with reverse transcription (RT-PCR) results (f). The red triangle and square mark the m“A site and U-tract, respectively. Data are mean + s.d.; n = 4, biological replicates.
26 FEBRUARY 2015 | VOL 518 | NATURE | 563
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
than two splice variants, and 221 m“A-switch-containing genes with differentially expressed splice variants in HNRNPC and METTL3/L14 knockdown samples. Further analysis suggested that m°A-switches have an effect on intron exclusion (Extended Data Fig. 8g). Consistent with previous reports about splicing regulation by both HNRNPC and m“A!3192023 our results indicate that m°A functions as an RNA struc- ture remodeller to affect mRNA maturation through interference with post-transcriptional regulator binding activities.
We demonstrated that post-transcriptional m°A modifications could modulate the structure of coding and non-coding RNAs to regulate RNA-HNRNPC interactions, thus influencing gene expression and maturation in the nucleus. It is possible that m°A could also recruit addi- tional accessory factors, such as the YTH domain proteins, which can directly recognize m“A—as previously reported'*—to destabilize the RNA structure and facilitate HNRNPC binding. Besides HNRNPC, m°A-switches may regulate the function of many other RNA-binding proteins through modulating the RNA-structure-dependent access- ibility of their RBMs. Our work indicates widespread m°A-induced mRNA and IncRNA structural remodelling that affects RNA-protein interactions for biological regulation.
- 14. Meyer, K. D. et al. Comprehensive analysis of MRNA methylation reveals enrichment in 3’ UTRs and near stop codons. Cell 149, 1635-1646 (2012).
- 15. Wang, X. et al. N°-methyladenosine-dependent regulation of messenger RNA stability. Nature 505, 117-120 (2014).
- 16. Fustin, J. M. etal. RNA-methylation-dependent RNA processing controls the speed of the circadian clock. Cel/ 155, 793-806 (2013).
- 17. Schwartz, S. et al. High-resolution mapping reveals a conserved, widespread, dynamic mRNA methylation program in yeast meiosis. Cel/ 155, 1409-1421 (2013).
- 18. Batista, P. J. etal. m°A RNA modification controls cell fate transition in mammalian embryonic stem cells. Cell Stem Cell 15, 707-719 (2014).
- 19. Zhao, X. et al. FTO-dependent demethylation of N°-methyladenosine regulates mRNA splicing and is required for adipogenesis. Cell Res. 24, 1403-1419 (2014).
- 20. Konig, J. etal. iCLIP reveals the function of hnRNP particles in splicing at individual nucleotide resolution. Nature Struct. Mol. Biol. 17, 909-915 (2010).
- 21. McCloskey, A., Taniguchi, |, Shinmyozu, K. & Ohno, M. hnRNP C tetramer measures RNA length to classify RNA polymerase Il transcripts for export. Science 335, 1643-1646 (2012).
- 22. Rajagopalan, L. E., Westmark, C. J., Jarzembowski, J. A. & Malter, J. S. hnRNP C increases amyloid precursor protein (APP) production by stabilizing APP mRNA. Nucleic Acids Res. 26, 3418-3423 (1998).
- 23. Zarnack, K. et al. Direct competition between hnRNP C and U2AF65 protects the transcriptome from the exonization of Alu elements. Cel! 152, 453-466 (2013).
Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.
- 24. Cienikova, Z. et al. Structural and mechanistic insights into poly(uridine) tract recognition by the hnRNP C RNA recognition motif. J. Am. Chem. Soc. 136, 14536-14544 (2014).
- 25. Liu, N. et al. Probing N®-methyladenosine RNA modification status at single nucleotide resolution in mRNA and long noncoding RNA. RNA 19, 1848-1856 (2013).
# Received 24 November; accepted 14 January 2015.
- 26. Krecic, A. M. & Swanson, M. S. hnRNP complexes: composition, structure, and function. Curr. Opin. Cell Biol. 11, 363-371 (1999).
- 1. Antson, A. A. Single-stranded-RNA binding proteins. Curr. Opin. Struct. Biol. 10, 87-94 (2000).
- 27. Görlach, M., Burd, C.G. & Dreyfuss, G. The determinants of RNA-binding specificity of the heterogeneous nuclear ribonucleoprotein C proteins. J. Biol. Chem. 269, 23074-23078 (1994).
- Dreyfuss, G., Kim, V. N. & Kataoka, N. Messenger-RNA-binding proteins and the messages they carry. Nature Rev. Mol. Cell Biol. 3, 195-205 (2002). 2.
- Ray, D. et al. A compendium of RNA-binding motifs for decoding gene regulation. Nature 499, 172-177 (2013).
3
- human transcriptome. Nature 505, 706-709 (2014). 4. Wan, Y. etal. Landscape and variation of RNA secondary structure across the
- Ding, Y. et al. In vivo genome-wide profiling of RNA secondary structure reveals novel regulatory features. Nature 505, 696-700 (2014). 5.
- 28. Kierzek, E. & Kierzek, R. The thermodynamic stability of RNA duplexes and hairpins containing NÖ-alkyladenosines and 2-methylthio-N®-alkyladenosines. Nucleic Acids Res. 31, 4472-4480 (2003).
- 29. Hafner, M. et al. Transcriptome-wide identification of RNA-binding protein and microRNA target sites by PAR-CLIP. Cel! 141, 129-141 (2010).
- 30. Anders, S., Reyes, A. & Huber, W. Detecting differential usage of exons from RNA- seq data, Genome Res. 22, 2008-2017 (2012).
- 6 Kertesz, M. etal. Genome-wide measurement of RNA secondary structure in yeast. Nature 467, 103-107 (2010).
- 7. Rouskin, S., Zubradt, M., Washietl, S., Kellis, M. & Weissman, J. S. Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo. Nature 505, 701-705 (2014).
- 8. Bokar, J. A. in Fine-Tuning of RNA Functions by Modification and Editing (ed. Grosjean, H.) 141-178 (Springer, 2005).
Acknowledgements This work is supported by National Institutes of Health EUREKA GM088599 (C.H. and T.P), and by KOLlHGO06699 (Q.D.). We thank all members of the Pan and He laboratories for comments and discussions. We also thank Y. C. Leung, G.Perdrizet, Y. Pigli,J. Yue, J. Liu, Y. Yue, K. Chen and M. Yu for technical assistance. C.H. is an Investigator of the Howard Hughes Medical Institute. M.P. was a Natural Sciences and Engineering Research Council of Canada postdoctoral fellow.
- 9. Jia, G. etal, N®-methyladenosine in nuclear RNA is a major substrate of the obesity-associated FTO. Nature Chem. Biol. 7, 885-887 (2011).
- 10. Zheng, G. et al. ALKBHS5 is a mammalian RNA demethylase that impacts RNA metabolism and mouse fertility. Mol. Cell 49, 18-29 (2013).
Author Contributions N.L,, G.Z. and M.P. designed and performed experiments, and analysed data. Q.D. synthesized all RNA oligonucleotides. N.L., M.P. and T.P. conceived the project. N.L. and T.P. wrote the paper with input from C.H. and M.P.
- 11. Liu, J. et al. A METTL3-METTL14 complex mediates mammalian nuclear RNA N®-adenosine methylation. Nature Chem. Biol. 10, 93-95 (2014).
- 12. Wang, Y. et al. Nö-methyladenosine modification destabilizes developmental regulators in embryonic stem cells. Nature Cell Biol. 16, 191-198 (2014),
- 13. Dominissini, D. et al. Topology of the human and mouse m®A RNA methylomes revealed by m“A-seg, Nature 485, 201-206 (2012).
Author Information RNA-seq data have been deposited in the Gene Expression Omnibus under accession number GSE56010. Reprints and permissions information is available at www.nature.com/reprints, The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to MP. (marc.parisien@mcgill.ca) or T.P. (taopan@uchicago.edu).
564 | NATURE | VOL 518 | 26 FEBRUARY 2015
©2015 Macmillan Publishers Limited. All rights reserved
Uta SCIENCE FICTION
# BFFS FIRST ADVENTURE
# Clouded view.
# BY VERNOR VINGE
# 1568771947.223
Whoever kidnapped me has erased my most recent memories and gagged me. I have no idea how they snatched me from Timothy Bennett, or if Tim is okay — but now I'm hurtling towards a concrete wall, The kidnappers must have panicked and thrown me from their car. A transforming phone would become a para- chute, but I can't trans- form; physically, Tm a clunky classic. I extend a flange and rudder myself around so I'll hit the wall on my reinforced corner. I safe my MEMS and shut down.
... Rebooting. I’m lying beneath a freeway overpass. Theresa crack in my shell and my GPS is busted, but I’ve escaped! All my outgoing wireless chan- nels remain blocked, but surely the freeway noticed me? Alas this is California, what other states call ‘the land that time forgot. Here the sensors politely ignore data that might make them look snoopy. The sun has set and I’ve run down my batteries. An ordinary phone could scavenge from any number of ambient sources. I can't. BFEstartup made me special; I can think for myself, but that doesn’t leave room for many standard features. Now I’m fading away. I hope Timothy is okay...
With sunrise comes light. I can think again! BFF.startup always says: “People shouldn't depend on the Cloud. With a GPLd assistant that fits in your phone, you'l be safer and freer’ ll prove that’s true and I will return to Timothy.
Cars, aerobots and Cal Trans devices are all around. They may not snoop, and they may not recognize an ad hoc mayday, but smarter things use them. For instance, I can hear assistants helping humans; the Kiras and Miris — all the digital assistants except for me — are transient minds in the Cloud. Each instance is more knowledgeable than me and smarter or dumber depending on the service plan and the context. Each seems an intimate friend of its customer-of-the-moment, but each is really just a tiny facet of something quite inhuman. Still, if one of them notices my signalling, it might help me.
Days pass without success. I’ve tried blinking my display, clicking my speaker, even synthesizing human cries for help.
There's a newish Cal'Trans gardener working on the roadside bushes. If it ever comes near, maybe it'll recognize my signalling.
I have lots of time to watch the Cloud; its not really soft and fluffy. It's more like a deep ocean. There are cognitive patterns in it larger than the mind of any single human or digital assistant. The largest are leviathans, agencies who have become something their creators no longer comprehend. The Kiras and Miris never speak of these except in jest, but the leviathans are growing and they need the same resources — flops and bandwidth, power and capital — as everyone else. I fear for the humans.
# 1571772569.092
That CalTrans gardener is approaching. The greenery has grown over me, but my dis- play’s light reflects offa bit of recent trash. If the gardener is as good as its advertising, I'll finally be talking to
finally be talking to somebody. If not, it'll grind me into recy- cling feed. Maybe I should slide farther
SD NATURE.COM Follow Futures: © @NatureFutures Bİ gonature.com/mtoodm
=
down, out of reach of its bladed hands. But then I'll have no chance to forward a mes- sage. I push myself up, and blink as bright as I can.
ALLI can see is flashing metal... The gardener pauses, and pings me — with the method I am using! I reply. Its for- warding. I sense something huge turn its cold regard upon me and I remember my misgivings about the great creatures of the Cloud. Then the Cloud Thing decides, and looks away. I feel my configuration twitch; at last I can standard
# outputs!
# use
The gardener lifts me into the lifegiving sun- light, and an aerobot swoops down. I’m already pinging Tim- othy as we rise into the sky. I see familiar streets; all this time, I’ve been very close to home. I hear Timothy, talking to his phone company, telling them how grateful he is that I’ve been found. Hes very excited. Tim comes outside to greet me, carries me indoors to his private room. He flips me around. I feel sorry for the damage he sees.
“Damn phone!” he says. “Useless as ever and still functioning. What was I thinking when I bought you?”
“I don’t understand, Timothy. I was kid- napped but now I’ve returned to you.”
“Yeah, you're a real boomerang. And now I gotta pay for the upgrade or be stuck with you.”
“But —”
“You don't have space for decent features, and what you know is always out of date.”
“1 can sync! I can learn!”
He sweeps me off the table and pulls open acabinet drawer. “This time you're not com- ing back!”
“Please Timothy! I can think for myself. Someday, you might need that!”
“More BFF hype. I'll need you when hell freezes over.’ Tim tosses me into the insu- lated cabinet; the drawer slams shut. This is not some shady spot beneath an overpass. There is no Cloud, no power. Only darkness.
# 1666762857.577
The drawer opens, and there is light — m
Vernor Vinge’ science fiction has won five Hugo Awards. From 1972 to 2000 he taught mathematics and computer science at San Diego State University.
568 | NATURE | VOL 518 26 FEBRUARY 2015
Macmillan Publishers Limited. Alll rights reserved
# ILLUSTRATION BY JACEY
ARTICLE
# METHODS
in immunocompromised mice were mechanically and enzymatically dissociated
Human research ethical approvals. Australian Pancreatic Cancer Genome Initi- ative: Sydney South West Area Health Service Human Research Ethics Committee, western zone (protocol number 2006/54); Sydney Local Health District Human Research Ethics Committee (X11-0220); Northern Sydney Central Coast Health Har- bour Human Research Ethics Committee (0612-251M); Royal Adelaide Hospital Human Research Ethics Committee (091107a); Metro South Human Research Ethics Committee (09/QPAH/220); South Metropolitan Area Health Service Human Research Ethics Committee (09/324); Southern Adelaide Health Service/Flinders University Human Research Ethics Committee (167/10); Sydney West Area Health Service Human Research Ethics Committee (Westmead campus) (HREC2002/3/4.19); The University of Queensland Medical Research Ethics Committee (2009000745); Greenslopes Private Hospital Ethics Committee (09/34); North Shore Private Hos- pital Ethics Committee. Johns Hopkins Medical Institutions: Johns Hopkins Medi- cine Institutional Review Board (NA00026689). ARC-NET, University of Verona: approval number 1885 from the Integrated University Hospital Trust (AOUI) Ethics Committee (Comitato Etico Azienda Ospedaliera Universitaria Integrata) approved in their meeting of 17 November 2010 and documented by the ethics committee 52070/CE on 22 November 2010 and formalized by the Health Director of the AOUI on the order of the General Manager with protocol 52438 on 23 November 2010. Ethikkommission an der Technischen Universitat Dresden (Approval numbers EK30412207 and EK357112012).
Animal experiment approvals. Mouse experiments were carried out in compli- ance with Australian laws on animal welfare. Mouse protocols were approved by the Garvan Institute/St Vincent’s Hospital Animal Ethics Committee (ARA 09/19, 11/23 and 12/21 protocols). Female NOD/SCID/interleukin 2 receptor [IL2R] gamma (null) (NSG) mice and athymic Balb-c-nude mice were housed with a 12h light, 12h dark cycle, receiving food ad libitum.
Sample acquisition. Samples used were prospectively acquired and restricted to primary operable, non-pretreated pancreatic ductal adenocarcinoma. After ethical approval was granted, individual patients were recruited preoperatively and con- sented using an ICGC approved process. Immediately following surgical extirpa- tion, a specialist pathologist analysed specimens macroscopically and samples of the tumour, normal pancreas and duodenal mucosa were snap frozen in liquid nitro- gen (for full protocol see APGI website: http://www.pancreaticcancer.net.au/). The remaining resected specimen underwent routine histopathologic processing and examination. Once the diagnosis of pancreatic ductal adenocarcinoma was made, representative sections were reviewed independently by at least one other patholo- gist with specific expertise in pancreatic diseases (authors: A.G., D.M., R.H.H. and A.C), and only those where there was no doubt as to the histopathological diagnosis were entered into the study. Co-existent intraductal papillary mucinous neoplasms in the residual specimen were not excluded provided the bulk of the tumour was invasive carcinoma, and the invasive carcinoma samples were used for sequencing. All samples were stored at —80 °C. Duodenal mucosa or circulating lymphocytes were used for generation of germline DNA. A representative sample of duodenal mucosa was excised and processed in formalin to confirm non-neoplastic histology before processing. All participant information and biospecimens were logged and tracked using a purpose-built data and biospecimen information management system (Cansto Pancreas). Median survival was estimated using the Kaplan-Meier method and the difference was tested using the log-rank test. P values of less than 0.05 were considered statistically significant. Statistical analysis was performed using StatView 5.0 Software (Abacus Systems, Berkeley, CA, USA). Disease-specific sur- vival was used as the primary endpoint.
using collagenase (Stem Cell Technologies, USA) and plated onto flasks coated with 0.2mg ml ' rat tail collagen (BD Biosciences, USA). Subsequently, epithelial cultures were enriched and purified using a FACS Aria III Cell sorter (BD Biosciences, USA), using a biotinylated anti-mouse MHCI antibody (1:200 dilution; eBiosciences, USA) coupled with Streptavidin AlexaFluor 647 secondary step (1:1,000; Invitrogen, USA) and anti-mouse CD140a-PE antibody (1:300; BD Biosciences, USA) to remove mouse stroma. Dead cells were removed using propidium iodide (Sigma-Aldrich, Australia). Following establishment, all patient-derived (TKCC) cell lines were profiled by short tandem repeat (STR) DNA profiling as unique (http://www.cellbankaustralia.com). Sequencing. DNA (1 1g) was diluted to 52.5 jl in DNase-/RNase-free molecular biology grade water before fragmentation to approximately 300 bp using the Covaris S2 sonicator with the following settings Duty Cycle 10%, intensity 5, cycles per burst 200, time 50s or 45 s for PCR-Free libraries. Following fragmentation libraries for sequencing were prepared using the standard Illumina library preparation tech- nique of end-repair, adenylate 3’ ends, indexed adaptor ligation, size selection and finally PCR enrichment for adaptor ligated library molecules following the man- ufacturer’s recommendations (Part no. 15026486 Rev. C July 2012). A subset of libraries was generated omitting the final PCR enrichment step to generate PCR- Free libraries as per the manufacturer's recommendations (Part no. 15036187 Rev. A Jan 2013). For standard libraries commercially available TruSeq DNA LT Sample Prep Kit v2 (Catalogue no, FC-121-2001) were used for all steps with the following exceptions. Size selections of the Adaptor Ligated fragments were completed using two rounds of SPRI bead purifications (AxyPrepMag PCR Clean-upCatalog no. MAG-PCR-CL-250) using a final bead to DNA volume ratio of 0.60:1 followed by 0.70:1, selecting for molecules with an average size of 500 bp. Size-selected libraries were then amplified for a total of 8 cycles of PCR to enrich for DNA fragments both compatible with sequencing and containing the ligated indexed adaptor. For PCR- Free libraries commercially available TruSeq PCR-Free DNA LT Sample Prepara- tion Kit (Catalog no. FC-121-3001 and FC-121-3002) was used following the 350 bp library LT protocol for all steps with no modifications. The final whole-genome libraries were qualified (amplified and PCR-Free libraries) and quantified (amplified libraries only) via the Agilent BioAnalsyser 2100 (Catalog ID:G2940CA) instrument using the DNA High Sensitivity kit (Catalog ID:5067-4626). Quantification of PCR- Free libraries was performed using the KAPA Library Quantification Kits For Illumina sequencing platforms (Kit code KK4824) in combination with Life Tech- nologies Viia 7 real time PCR instrument.
Whole genome libraries were prepared for cluster generation by cBot (catalogue no. SY-301-2002) and sequencing as per the manufacturer's guidelines. Individual libraries were clustered on a single lane of a HiSeg v3 flowcell using the TruSeq PE Cluster Kit v3-cBot-HS kit (Catalogue no. PE-401-3001). Illumina supplied con- trol library PhiX (10 pM) was spiked into each lane at a concentration of 0.3% to provide real time analysis metrics. Final library concentrations of 8 pM (amplified) and 14 pM (PCR-free) were used for cluster generation. Clustered flowcells were sequenced on the Illumina HiSeq 2000 instrument (HiSeq control software v1.5/ Real Time Analysis 1.13) using TruSeq SBS Kit v3-HS (200 cycles, Catalog no. FC- 401-3001). Paired reads each of 101 bp were generated for all libraries and in total approximately 220-million paired reads were generated per lane, in line with the manufacturer s specification. Real time analysis of the control library PhiX showed cluster density, error rates, quality scores, mapping rates and phasing rates were also in line with published specifications.
Sample extraction. Samples were retrieved, and either had full face sectioning per- formed in OCT or the ends excised and processed in formalin to verify the presence of carcinoma in the sample to be sequenced and to estimate the percentage of malig- nant epithelial nuclei in the sample relative to stromal nuclei. Macrodissection was performed if required to excise areas of non-malignant tissue. Nucleic acids were then extracted using the Qiagen Allprep Kit in accordance with the manufacturer's instructions with purification of DNA and RNA from the same sample. DNA was quantified using Qubit HS DNA Assay (Invitrogen). Throughout the process, all samples were tracked using unique identifiers.
Patient material. One hundred matched normal and tumour derived samples were obtained from patients with PDAC. DNA was extracted from the samples using the QiagenAllprep DNA/RNA mini kit method. Tumour cellularity was deter- mined from SNP array data using gpure”*, Clinical and sample data are summarized in (Supplementary Table 2). Patients were recruited and consent obtained for geno- mic sequencing through the Australian Pancreatic Cancer Genome Initiative (APGI) as part of the International Cancer Genome Consortium (ICGC)’*.
Patient-derived cell line (PDCL) generation. The PDX-derived primary cell lines, named The Kinghorn Cancer Centre (TKCC) lines, were generated in the laboratory. All cell lines were profiled by short tandem repeat (STR) DNA profiling as unique (http://www.cellbankaustralia.com). Briefly, patient-derived tumours established
Sequence alignment and data management. Sequence data was mapped to a genome based on the Genome Reference Consortium (http://www.ncbi.nlm.nih. gov/projects/genome/assembly/grc/human/) GRCh37 assembly using BWA“. Mul- tiple BAM files from the same sequence library were merged and within library duplicates were marked. Resulting final BAMs were used as input into variant call- ing. All BAM files have been deposited in the EGA (Accession number: EGAS 00001000154).
Copy number analysis. Matched tumour and normal patient DNA was assayed using Illumina SNP BeadChips as per manufacturer’s instructions (Illumina, San Diego CA) (HumanOmnil-Quad or HumanOmni2.5-8 BeadChips). SNP arrays were scanned and data was processed using the Genotyping module (v1.8.4) in Genomestudio v2010.3 (Illumina, San Diego CA) to calculate B-allele frequencies (BAF) and logR values. GenoCN* and GAP” were used to call somatic regions of copy number change - gain, loss or copy neutral LOH. Recurrent regions of copy number change were determined and genes within these regions were extracted using ENSEMBL v70 annotations.
Identification of structural variations. Somatic structural variants were identified using the qSV tool (manuscript in preparation). qSV uses independent lines of evi- dence to call structural variants including discordant reads, soft clipping and split read. Breakpoints are also identified using both de novo assembly of abnormally mapping reads and split contig alignment to enhance break point resolution. Depend- ing on the level of evidence qSV bins calls into different categories and calls were
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
considered high confidence if: (i) they were category 1 and therefore contain mul- tiple lines of evidence (discordant pairs, soft clipping on both sides and split reads); (ii) they were category 2 and therefore there was 2 lines of evidence: discordant pairs (both breakpoints) and soft clipping; or discordant pairs (both breakpoints) and split read; or soft clipping (double sided) and split read; (iii) they were category 3 with 10 or more supporting events (discordant read pairs or soft clipping at both ends). Only high confidence calls were used in further downstream analysis. Copy number variation was estimated using SNP arrays and the GAP tool'’. Depending on the read pair types supporting an aberration or the associated of copy number events each structural variant was classified as: deletion, duplication, tandem duplication, foldback inversion, amplified inversion, inversion, intrachromosomal or translo- cation, Essentially, the type of rearrangement is initially inferred from the orientation information of discordant read pairs, soft clipping clusters and assembled contigs which span the breakpoints. This allows identification of 4 groups of events: duplica- tions/intra-chromosomal rearrangements, deletions/intra-chromosomal rearrange- ments, inversions and inter-chromosomal translocations. Boundaries of segments of copy number that occur in close proximity to each breakpoint were then used to aid further classification of the events. Structural variants with breakpoints that flanked a copy number segment of loss were annotated as deletions. Duplications and inversions associated with increases in copy number enabled the character- ization of tandem duplications and amplified or foldback inversions. Events within the same chromosome which linked the ends of copy number segments of simi- lar copy number levels were often identified and were called intra-chromosomal rearrangements.
Events were then annotated if they were within 100 kb of a centromere or telo- mere and genes which were affected by breakpoints were annotated using ENSEMBL v70. Structural variants and copy number data were visualized using circos™. The landscape of structural rearrangements in pancreatic ductal adenocarci- noma. In total 11,868 structural variants were detected within the 100 PDAC cohort with an average of 119 events per patient (range 15-558). Each event was classified into one of 8 categories: deletion, duplication, tandem duplication, foldback inver- sion, amplified inversion, inversion, intra chromosomal and translocation. Within the cohort there was inter patient heterogeneity in terms of total number of events (range of events per patient 15-558) and proportion of event type (Extended Data Fig. 1).
Verification of structural variations. We used two methods of verification for structural variants: (1) an in silico approach, which considers events with multiple lines of evidence (qSV category 1: discordant pairs, soft clipping on both sides and split read evidence) as verified, as well as events which were associated with a copy number change (gain or loss) and (2) orthogonal sequencing methods including SOLID long mate pair and capillary sequencing.
Long mate pair sequencing and verification of structural rearrangements. Long mate-paired libraries were made according to Applied Biosystems Mate-Paired Library Preparation 5500 Series SOLiD systems protocol using 5 ug of DNA which was sheared using the CovarisS220 System. Long mate pair libraries were sequenced using the SOliD v4 (Applied Biosystems). Sequence data was mapped to a genome based on the Genome Reference Consortium (http://www.ncbi.nlm.nih.gov/projects/ genome/assembly/grc/human/). GRCh37 assembly using bioscope v1.2.1 (Applied Biosystems). Each sample was sequenced to an average non-redundant physical cov- erage of 180X (64-333) in the tumour and 187X (52-503) in the control sample. Structural rearrangements were determined by analysing clusters of discordant read pairs using the qSV tool. Events identified by Hiseq sequencing were considered verified if the right and left breakpoint of these events were within 500 bases of the right and left breakpoint of an event identified by SOLiD sequencing.
PCR and capillary sequencing for verification of structural rearrangements. For PCR and capillary sequencing PCR primers were designed with primer BLAST (NCBI) to span the predicted breakpoint, primers were designed with primer BLAST (NCBI). PCR was carried out in the tumour and matched normal genomic DNA using, respectively, a 25 or 50 pil reaction volume composed of 22 or 44 pil of Plat- inum Tag DNA polymerase (Invitrogen, Carlsbad, Ca), 2 or 4 ıl of 10 UM primer (Integrated DNA Technology) and 1 or 2 yıl of genomic DNA as template (1 ng pl”). The following parameters was used for the PCR. Initial denaturation at 94 °C for 2 min, followed by 35 cycles of denaturation at 94 °C for 30s, annealing at 60 °C for 30s and extension at 68 °C for 1 min; followed by final extension at 68 °C for 15 min. PCR products were visualized by gel electrophoresis and classified into one of four categories: (1) validated—strong and specific PCR band of the expected size was observed only in the tumour and not in the normal sample, this indicates a somatic rearrangement; (2) germline—clear PCR band of the expected size both in the tumour and normal; (3) not validated—PCR yields smears or multiple bands, this potentially indicates non-specific primer pair; (4) not tested—no PCR band was observed in tumour and normal.
Classification of subtypes based on the pattern or structural rearrangements. Each tumour was classified into one of four subtypes based on the volume of events, the predominance of specific types of structural rearrangement events and the dis- tribution of events across the genome in each patient. In addition to counting struc- tural variation events, two analyses were carried out to detect localized events. Non-random chromosomal clustering of structural variants was detected using an approach originally described by Korbel and Campbell”. Significant clustering of structural variation events was determined by a goodness-of-fit test against the expected exponential distribution of (with a significance threshold of < 0.0001). Highly focal events were detected using an adaptation of a method" where chro- mosomes with a high structural variant mutation rate per Mb exceeded 5 times the length of the interquartile range from the 75th percentile of the chromosome counts for each patient. The rules used to determine these subtypes are as follows:
Stable These tumours contain few structural rearrangements (<50) which are located randomly through the genome.
Locally rearranged The intrachromosomal rearrangements in these tumours are not randomly positioned through the genome, instead they are clustered on one or few chromosomes. To correct for the different chromosome lengths, the number of events per Mb was calculated for each chromosome within each tumour. Tumours were considered locally rearranged if they harboured at least 50 somatic events within the genome and contained a locally rearranged chromosome. Chromosomes were considered locally rearranged if the number of intrachromosomal events exceeded 5 times the length of the interquartile range from the 75th percentile of the chromosome counts per Mb for that patient. The events in the locally rearranged tumours are broadly comprised of either: (1) focal amplifications—the majority of events are gain (tandem duplication, duplication, foldback inversion or amplified inversion) or (2) complex rearrangements—the events are part of a complex event such as chromothripsis or breakage-fusion-bridge.
Verification of structural variations—results. In total 7,105 events were verified in silico, Of these 5,666 events contained multiple lines of evidence (qSV category 1), 2,904 events were associated with a copy number change (events classified as dele- tion, duplication, tandem duplication, amplified inversion and foldback inversion) and 1,871 contained multiple lines of evidence and were associated with a copy number change.
We also verified structural variant events using long mate pair resequencing (SOLID paired 50 bp) or sequencing of a different sample from the same patient of 33 tumours. Using this approach 1,924 events were confirmed and the verification status of structural variant events was recorded in Supplementary Table 5 in the “validation_status_id” column where 0 = untested and 1 = verified. In total 7,228 of the 11,868 events identified (61%) were verified (Supplementary Table 5 and Extended Data Fig. 1) the remaining events remain untested.
Identification of substitutions and small insertion/deletions. Substitutions are called using 2 variant callers: gSNP”* an in-house heuristics-driven somatic/germline caller; and GATK* which is a Bayesian caller. The two callers were chosen because they use very different calling strategies and while each maybe subject to artefacts (as are all variant callers), they will be subject to different artefacts. Each compared variant falls into one of three categories: seen only by qSNP, seen only by GATK, and seen by both qSNP and GATK. Mutations identified by both callers or those that were unique toa caller and verified by an orthogonal sequencing approach were considered high confidence and used in all subsequent analyses (Supplementary Table 3). Small indels (<200 bp) were identified using Pindel”; each indel was visu- ally inspected in the Integrative Genome Browser (IGV)"*. Once somatic mutations were called, their effects on any alternative transcripts were annotated using a local install of the Ensembl database (v70) and the Ensembl Perl API.
Scattered These tumours contain 50-200 structural rearrangements which are scattered throughout the genome.
Unstable These tumours are massively rearranged as they contain >200 struc- tural rearrangements which are generally scattered throughout the genome. Classification of complex localized events. Evidence of clustering of breakpoints was estimated as proposed by Korbel and Campbell*®. Chromosomes with cluster- ing of structural variants were reviewed for evidence of chromothripsis (oscillation of copy number, random joins and retention of heterozygosity) and breakage-fusion- bridge (BEB for loss of telomeric region with neighbouring highly amplified region with inversions).
Verification of substitutions and small insertion/deletions. In total 3,304 of the 10,335 events identified were verified (Supplementary Tables 3 and 12) the remain- ing events remain untested. Substitutions and indels were verified using orthogonal sequence data which included data produced on different sequencing platforms (Hiseg or SOLiD exome or long mate pair SOLiD sequencing) or data from related nucleotide samples (RNA-seq). For example, if orthogonal tumour sequence data was available (DNA froma cell line, RNA from the primary sample etc.) anda somatic variant was also observed in the second tumour sample then that would add support for the variant. It should be noted that tumour samples can only be used to support an existing somatic variant and the absence of a called variant in a second tumour sample does not discredit the original call. Conversely, a second normal sample will
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
only discredit somatic variants and the absence of the called variant in the second normal does not support the original call. This approach is designed to be conser- vative. In order to be considered for verification, an additional BAM should have a minimum of 10 reads at the variant position, and at least two reads must show the variant. If multiple additional BAMs are available, each BAM votes independently and the concordance of the votes is used to classify the verification of the variant. Each variant examined by qVerify is assigned to one of four categories:
(1) Verified—one or more additional tumour BAMs showed evidence of the variant and no additional normal BAMs showed the variant.
avoid accumulating toxicity of repeated injections, an additional treatment was given after the recovery time of two weeks only when no tumour regression was observed, otherwise treatment was continued once the tumour relapsed to its orig- inal size (100%). Measurement of chemotherapy response was based on published methodology”', where primary xenografts were treated with the specified mono- therapy and their growth characteristics mapped from the time resistance developed (characterized by progressive tumour growth in the presence of drug), until eutha- nasia. Mice were euthanized and tissues collected for further analyses when tumour size reached 400% (600-700mm*).
(2) False Positive—one or more additional normal BAMs showed evidence of the variant indication that it is likely to be a germline variant.
(3) Mixed—across multiple additional BAMs, there was conflicting evidence - one or more additional tumour BAMs showed the variant as did one or more addi- tional normal BAMs. This could also be evidence of a germline variant incorrectly called somatic.
(4) Untested—there were no additional BAMs or there were additional BAMs but none passed the minimum coverage threshold or there were additional BAMs that did not show the variant and so did not provide evidence for or against it. Telomere length analysis. Reads containing the telomeric repeat (TTAGGG) x3 or (CCCTAA) X3 were counted and normalized to the average genomic coverage (the average base coverage of each genome). The normalized telomere count was obtained separately for each tumour and its matching normal. A ratio was calcu- lated by tumour normalized counts/normal normalized counts.
RADSI foci formation assay. Antibodies used included RADSI (Clone 14B4, GeneTex), yH2AX (phospho-histone H2AX Ser129 clone 20E3, Cell signaling), and geminin (10802-1-AP, ProteinTech Group, Chicago, IL). Primary culture of PDX from patient ICGC_0016was established by plating and growing cells from an enzymatically digested xenograft on a collagen matrix for approximately 1 week before irradiation and immunofluorescence staining. For this experiment, xenograft was established in a NSG-eGFP mouse. This mouse model allowed us to efficiently visualize eGFP positive mouse stromal cells and eGFP negative tumour cells under the microscope. Briefly, the eGFP expressing NSG mouse was generated in our laboratory by crossing previously established heterozygous eGEPNOD.CB17- Prkdescid mice” with the theNOD/SCID/interleukin 2 receptor (IL2R) gamma (null) (NOG) strain in our laboratory. eGFP expressing offspring was backcrossed five times onto the parental line to ensure homozygosity for IL2Rgamma deletion and confirmed by genotyping (Transnetyx).
Determination of the BRCA signature. High confidence somatic mutations that were called by both gSNP and GATK across the genome were used to determine the proportion of the BRCA signature in each sample using a published computa- tional framework?" In this way, the 96 substitution classification (as determined by substitution class and sequence context) was determined for each sample and compared to the validated BRCA signature” and the proportion of the BRCA signature in a given sample was ascertained.
Patient derived xenograft (PDX) mouse model generation. Six female eight- week-old NOD/SCID/interleukin 2 receptor [IL2R] gamma (null) (NOG) mice and athymic Balb-c-nude mice were used for the establishment of the patient derived xenograft (PDX) model. All mice were bred at the Australian Bioresources (ABR) under research protocols approved by the Garvan Animal Ethics Committee (09/ 19, 11/23, 11/09).
The PDXs were generated according to methodology published elsewhere with modifications”. Briefly, surgical non-diagnostic specimens of patients operated at APGI clinical sites were implanted subcutaneously (s.c.) into three NOG and three Balb-c-nude mice for each patient, with two small pieces per mouse (left and right flank; engraftment stage). Once established, tumours were grown to a size of 1,500 mm’, at which point they were harvested, divided, and re-transplanted into further mice to bank sufficient tissues for experimentation (first passage and second passage). After expansion, passaged tumours were excised and propagated to cohorts of 40 female Balb-c-nude mice or greater at an average of 8 weeks old, which con- stituted the treatment cohort (third passage). Utilization of the NOG mouse model, which is characterized by high immune deficiency in this study has enabled estab- lishment of a significant cohort of PDXs (80) xenografts, with a high rate of suc- cessful engraftment and propagation (76%, data not shown).
Cell lines of interest were grown on coverslips overnight and irradiated with 10 Gy or left untreated. Subsequently coverslips were fixed with 4% paraformalde- hyde (in PBS) 6 h post-irradiation and stained with RADSI, yH2AX and geminin antibodies as previously described”, DAPI was used as a nuclear stain, RADSI focus assay scoring was performed as previously established*’.
- 42. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754-1760 (2009).
- 43. Sun, W. et al. Integrated study of copy number states and genotype calls using high-density SNP arrays. Nucleic Acids Res. 37, 5365-5377 (2009).
- 44. Krzywinski, M. et al. Circos: an information aesthetic for comparative genomics. Genome Res. 19, 1639-1645 (2009).
- 45. Nik-Zainal, S. et al. Association of a germline copy number polymorphism of APOBEC3A and APOBEC3B with burden of putative APOBEC-dependent mutations in breast cancer. Nature Genet. 46, 487-491 (2014).
- 46. McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297-1303 (2010).
- 47. Ye, K., Schulz, M. H., Long, Q., Apweiler, R. & Ning, Z. Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics 25, 2865-2871 (2009).
- 48. Thorvaldsdöttir, H., Robinson, J. T. & Mesirov, J. P. Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration. Brief. Bioinform. 14, 178-192 (2013).
- 49, Nik-Zainal, S. et al, Mutational processes molding the genomes of 21 breast cancers. Cell 149, 979-993 (2012).
- 50. Rubio-Viqueira, B. et al. An in vivo platform for translational drug development in pancreatic cancer. Clin. Cancer Res. 12, 4652-4661 (2006).
In vivo therapeutic testing. Tumour-bearing mice with a palpable tumour (volume (V) = 150 mm’; V = 0.5 X length X width’) were treated with various agents at maximum tolerable dose (MTD) or vehicle treatment based on previously estab- lished schedules”, where gemcitabine (140 mg per kg) was administered intra- peritoneally on day 1 and day 4 for 4 weeks and cisplatin (6 mg per kg) intravenously on day 1 and day 14. The investigators were not blinded to the group allocation. To
- 51. Rottenberg, S. et al. Selective induction of chemotherapy resistance of mammary tumors in a conditional mouse model for hereditary breast cancer. Proc. Natl Acad. Sci. USA 104, 12117-12122 (2007).
- 52. Niclou, S. P. et al. A novel eGFP-expressing immunodeficient mouse model to study tumor-host interactions, FASEB J. 22, 3120-3128 (2008).
- 53. Graeser, M. et al. A marker of homologous recombination predicts pathologic complete response to neoadjuvant chemotherapy in primary breast cancer. Clin. Cancer Res. 16, 6159-6168 (2010).
# ©2015 Macmillan Publishers Limited. All rights reserved

a
6000 2 2 so : 5 400 3 $3000 3g 5 3 2000 5 > : ii m 2 ° c 2 = SV category Only verified by orthogonol (J Total Number not tested B sequencing Only associated with ‘Associated with copy Bw copy number change Mİ ana verifies by sequencing
# b
# number
BE Inrachromosomal M Duplication I Tandem Duplication © Foldback Inversion I Translocation I Deletion © 1 Amplified Inversion 500 | Inversion i | g 400 8 3 3 300 a i” 3 2 100 oF Stable il Lae nit alalı Scattered Unstable Locally rearranged (EM.
# Extended Data Figure 1 | Summary of structural rearrangements.
a, Histogram showing the number of events verified in silico or by orthogonal sequencing methods (Methods). In total 7,228 of the 11,868 events identified (61%) were verified, the others remain untested. These included 5,666 events which contained multiple lines of evidence (qSV category 1: discordant pairs, soft clipping on both sides and split read evidence, Methods) thus were considered verified. Of these events 2,463 events were also verified by orthogonal sequencing methods (SOLiD long mate pair or PCR amplicon sequencing) or the event was associated with a copy number change which was determined using SNP arrays. The remaining 1,562 events were verified using orthogonal sequencing methods or the event was associated with a copy
number change (qSV category 2 and 3, Methods). b, Histogram showing the number of structural rearrangements in each pancreatic cancer. 100 PDACs were sequenced using HiSeq paired-end whole-genome sequencing. Structural rearrangements were identified and classified into 8 categories (deletions, duplications, tandem duplications, foldback inversions, amplified inversions, inversions, intra-chromosomal and inter-chromosomal translocations, Methods). The number and type of event for each patient is shown. PDAC shows a high degree of heterogeneity in both the number and types of events per patient. The structural rearrangements were used to classify the tumours into four categories (stable, locally rearranged, scattered and unstable, Methods).
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
8 $ Total number of ‘structural rearrangements Chromosome 8 Stable Locally rearranged Unstable Scattered
$<
# Patients
Extended Data Figure 2 | Distribution of structural variant breakpoints within each patient. The 100 patients are plotted along the x axis. The upper plot shows the number of structural rearrangements (y axis) in each tumour. The lower plot shows which chromosomes (y axis) harbour clusters of breakpoints. The distribution of breakpoints (events per Mb) within each chromosome for each sample was evaluated using two methods to identify clusters of rearrangements or chromosomes which contain a large number of events. Method 1: chromosomes with a significant cluster of events were determined by a goodness-of-fit test against the expected exponential distribution (with a significance threshold of <0.0001). Chromosomes which pass these criteria are coloured blue. Method 2: chromosomes were identified
which contain significantly more events per Mb than other chromosomes for that patient. Chromosomes were deemed to harbour a high number of events if they had a mutation rate per Mb which exceeds 1.5 times the length of the interquartile range from the 75th percentile of the chromosome counts for each patient. Chromosomes which pass these criteria are coloured orange. Chromosomes which pass both tests they are coloured red. These criteria show that the unstable tumours which contain many events often have significant clusters of events. In contrast locally rearranged tumours are associated with both clusters of events and a high number of events within that chromosome when compared to other chromosomes.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
es a= N A (FO / M2 Al S'KZ KW WY. ae Sür 27 NE NE
© Intra chromosomal rearrangment M Inter chromosomal translocation
® Duplication M Tandem duplication
© Inversion © Foldback inversion
# W Amplified inversion M Deletion
adenocarcinoma. The 20 stable tumours are shown using circos. The coloured outer ring represents the chromosomes, the next ring depicts copy number (red represents gain and green represents loss), the next is the B allele frequency. Extended Data Figure 3 | The stable subtype in pancreatic ductal
The inner lines represent chromosome structural rearrangements detected by whole genome paired sequencing and the legend indicates the type of rearrangement. Stable tumours contained less than 50 structural rearrangements in each tumour.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
© Intra chromosomal rearrangment BM Duplication © Inversion 8 Amplified inversion M
© Intra chromosomal rearrangment M Inter chromosomal translocation
# M Tandem duplication
© Inversion © Foldback inversion
# B
# Deletion
Extended Data Figure 4 | The locally rearranged subtype in pancreatic ductal adenocarcinoma. The 30 locally rearranged tumours are shown using circos. The coloured outer rings represent the chromosomes, the next ring depicts copy number (red represents gain and green represents loss), the next is
# O
©
the B allele frequency. The inner lines represent chromosome structural rearrangements detected by whole-genome paired sequencing and the legend indicates the type of rearrangement. In the locally rearranged subtype over 25% of the structural rearrangements are clustered on one of few chromosomes.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
ICGC 0326 Chromthripsis - chromosome 18
6e07-İ Density Plot | | — A | — if a Structural Variants Copy Number R BAF 2,0e+07 4.0e+07 6.0e+07
Chromosome 18
# LEGEND
# [intra chromosomal rearrangment
# [Deletion
# [J Duplication
# [| Tandem duplication | Tİ Inversion
# B Amplified inversion
# İÜ Foldback inversion
Extended Data Figure 5 | Example of evidence for chromothripsis in a pancreatic ductal adenocarcinoma (ICGC, 0109). Upper plot isa density plot showing a concentration of break-points on chromosome 5. Next panel shows the structural rearrangements which are coloured as presented in the legend. The lower panels show copy number, logR ratio and B allele frequency derived
from SNP arrays. This chromosome showed a complex localization of events similar to chromothripsis. Copy number profile and structural rearrangements suggest a shattering of chromosome 5 with a high concentration of structural rearrangements, switches in copy number state and retention of heterozygosity, which are characteristics of a chromothriptic event.
© o _ _
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
ICGC_0115 Breakage-fusion-bridge - chromosome 16
8e07-İ 4e07-İ Density Plot 00 Structural Variants Copy Number Fe & 2 3 3 u < o 200407 4.06407 6.0e+07 8.0e+07
# Chromosome 16
# LEGEND
# (intra chromosomal rearrangment
# [ff Deletion
# J Duplication
# [J Tandem duplication _ TÜ Inversion
# [Amplified inversion
# (| Bİ Foldback inversion
Extended Data Figure 6 | Example of evidence for breakage-fusion-bridge (BEB) in a pancreatic ductal adenocarcinoma (ICGC_0042). Upper plot isa density plot showing a concentration of break-points on chromosome 5. Next panel shows the structural rearrangements which are coloured as presented in the legend. The lower panels show copy number, logR ratioandB
© _
# o
allele frequency derived from SNP arrays. This chromosome showed a complex localization of events similar to BFB. Copy number profile suggests loss of telomeric q arm and a high concentration of structural rearrangements suggesting a series of BFB cycles, with multiple inversions mapped to the amplified regions.
# ©2015 Macmillan Publishers Limited. All rights reserved

© Intra chromosomal
© Intra chromosomal rearrangment Mİ Inter chromosomal translocation
M Duplication M Tandem duplication
© Inversion © Foldback inversion
1 Amplified inversion B Deletion
Extended Data Figure 7 | The scattered subtype in pancreatic ductal adenocarcinoma. The 36 tumours classified as scattered are shown using circos. The coloured outer rings represent the chromosomes, the next ring depicts copy number (red represents gain and green represents loss), the next
shows the B allele frequency. The inner lines represent chromosome structural rearrangements detected by whole genome paired end sequencing. The legend indicates the type of rearrangement. The scattered tumours contained 50-200 structural rearrangements in each tumour.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
© Intra chromosomal rearrangment M Inter chromosomal translocation MB Duplication M Tandem duplication © Inversion > Foldback inversion ® Amplified inversion
- ® Deletion
Extended Data Figure 8 | The unstable subtype in pancreatic ductal adenocarcinoma. The 14 unstable tumours are shown using circos. The coloured outer rings are chromosomes, the next ring depicts copy number (red represents gain and green represents loss), the next is the Ballele frequency. The inner lines represent chromosome structural rearrangements detected by
# o _
whole genome paired sequencing and the legend indicates the type of rearrangement. The unstable tumours contained a large degree of genomic instability and harboured over 200 structural rearrangements in each tumour which were predominantly intra-chromosomal rearrangements evenly distributed through the genome.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
# a CONTROL
Overlay
(Primary Culture) ICGC_0016
Geminin
(Primary ICGC_0016 b 1 ğ Overlay /Geminin/ (Primary Culture) ICGC, 0016 Geminin/ TKCC-07 (HR-proficient cell line) Geminin/ Capan-1 (HR-deficient cell line) 100 c
100 ES w v = O 60 o oa ip 40- o Ea U TKCC-07 |ICGC 0016 Capan-1
Extended Data Figure 9 | RADSI foci formation in a primary culture of genomically unstable PDAC. a, RADS51 and geminin fluorescence in untreated cells derived from an unstable pancreatic tumour with a somatic mutation in the RPA1 gene (ICGC_0016). Primary culture of ICGC_0016 consists of eEGFP* mouse stromal and eGFP~ tumour cells. b, Upper panel: irradiated unstable pancreatic cancer cells (ICGC_0016), middle panel: HR- competent (TKCC-07) and lower panel: HR-deficient (Capan-1) pancreatic tumour cells. Cells were irradiated in vitro with 10Gy, and 6h post-irradiation
examined by immunofluorescence microscopy. eGFP negative tumour cells from ICGC_0016 readily form RADSI foci following induction of DNA damage. TKCC-07 is a pancreas cancer cell line generated from a homologous recombination (HR) pathway competent patient-derived xenograft and served as a positive control for staining and RAD51 foci formation after DNA damage. Capan-] cells which are HR-deficient do not form RADSI foci. c, RADSI score (percentage of geminin positive cells that have RAD51 foci) in examined pancreatic tumour cells.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
# METHODS
Yeast reference genome and annotation. All analyses were performed on the sacCer3 (V64) S. cerevisiae reference genome assembly. Data sets originally ob- tained with coordinates on other assemblies were projected into the sacCer3 assem- bly using liftOver (v261)°! with the corresponding chain files obtained from http:// www. yeastgenome.org. All regions of the sacCer3 genome were used for read align- ment but analyses including strand ratios and all rate estimates excluded the fol- lowing multi-copy regions: the mitochondrial genome, rDNA locus chrXII:459153 -461153 and any 100-nucleotide segment with mappability score of <0.9 (gem- mappability” with k-mer = 100). In total, this masked 951,532 nucleotides (7.8%) of the reference genome. Gene structure annotations were the Saccharomyces Ge- nome Database (SGD) consensus annotations extracted from the University of Cal- ifornia, Santa Cruz (UCSC) genome browser in November 2013. Annotated origins of replication were obtained from ref. 53. DNase I hypersensitive sites and foot- prints were obtained from ref. 54, and nucleosome position, occupancy and posi- tional fuzziness (positional heterogeneity) measures were from ref. 55. Yeast replication timing data was obtained from ref. 36, where we have plotted the percentage of heavy-light (replicated) DNA (pooled samples data set). Higher percentage indi- cates earlier average replication time.
differing read depth, read counts were normalized to sequence tags per million mapped into the non-masked portion of the genome.
Defining TF binding sites. Reb1 and Rap! ChIP-exo data was obtained from ref. 58 (Sequence Read Archive accession SRA044886). Sequence bar codes were clipped and sequences sorted using Perl (version 5.18.2). Reads were aligned using bowtie2 (version 2.0.0). Following the previously published protocol” up to three mis- matches across the length of each tag sequence were allowed, and the 3" most 6 base pairs (bp) removed. Peaks were called with MACS (version 2.0.10). Following ref, 58, sites were defined as monomer if no other peaks were present within 100 bp. Where two or more peaks were present within 100 bp the peak with the highest occupancy was labelled as the primary peak. Telomeric sites were excluded using annotations within the sacCer3 sgdOther UCSC table (http://www.yeast genome.org). The presence or absence of a motif was determined using the Motif Occurrence Detection Suite (MOODS)” (version 1.0.1). Consensus binding motifs positional weight matrices were obtained from JASPAR™ (http://jaspar.genereg. net/). The matching motif significance threshold was set at 0.005. Multiple peaks were aligned (x = 0) to the midpoint of the JASPAR defined motif. Human TF binding sites were defined using ChIP-seq data (Supplementary Table 1) as for yeast, except that the peak clustering threshold was reduced to 50 nucleotides.
Yeast polymorphisms and between species substitution rates. Yeast polymor- phism data was obtained from the Saccharomyces Genome Resequencing project”. A polymorphic difference between any of the 37 sequenced S, cerevisiae strains was called as a polymorphic site. Sites with n > 2 alleles were only counted once as a polymorphic site. Only nucleotide point substitutions were considered, insertions and deletions were excluded. The polymorphism rate reported is the number of polymorphic sites divided by the number of sacCer3 sites with sequence coverage in at least two additionally sequenced strains.
Computational and statistical analyses. Analysis and all statistical calculations were performed in R (version 3.0.0). Lines of fit used the smooth.spline function with degrees of freedom: Fig. la, 18 degrees; Fig. Ib, 34 degrees; Fig. 3d-f, 80 degrees of freedom (strand ratio calculated in 2,001-nucleotide consecutive win- dows). Sliding window averages used the rollapply function from the Zoo package with centre alignment and padding. Pearson's correlation was performed with the cor.test function in R, paired Student's t-test with the t.test function, Mann- Whitney tests with the wilcox.test function and lowess (locally weighted scatterplot smoothing) with the lowess function and default parameters.
Yeast between-species substitution rates were calculated from MultiZ stacked pairwise alignments obtained from the UCSC genome browser (Supplementary Table 1). Alignments for five sensu stricto yeast species (S. cerevisiae, S. paradoxus, S, mikatae, S. kudriavzevii and S. bayanus) were extracted from the original seven species alignment. The reference assembly names and phylogenetic relationship are represented by the tree (((sacCer3, sacPar), sacMik), sacKud, sacBay). Substi- tution rates were calculated over whole chromosomes using basem! from the paml*” package (version 4.6) under the HKY85 substitution model with ncatG = 5 cat- egorical gamma, Per-nucleotide relative rate estimates (branch length multipliers) were obtained over the sacCer3 genome.
No statistical methods were used to predetermine sample size.
Rate estimates with compositional correction. Polymorphism and OJ rates were calculated separately for each nucleotide (A, T, C or G) and the average of these for rates used as the reported or plotted measure for a nucleotide site or group of sites. This corrects for mononucleotide compositional biases that are abundant when sampling specific features of a genome. The between-species relative substitution rate calculation incorporates a compositional correction. The rate estimates shown are the number of observations divided by the number of sites with non-missing data.
Human conservation measures. GERP scores"! were used as a measure of be- tween species nucleotide diversity across 46 vertebrate species. Single-nucleotide resolution bigWig files were obtained from UCSC genome browser (hg19). For consistency of presentation with plots of polymorphism rate and yeast between- species nucleotide substitution rate, the y axes in plots showing GERP scores have been inverted so that greater constraint is low and greater diversity is high.
OF sequence processing. OF sequence data was obtained from ref. 17 (GEO ac- cession GSM835651). Analysis primarily focused on the larger ‘replicate’ library but results were confirmed in the ‘sample’ library (GEO accession GSM835650). ‘The OF strand ratio was calculated as the sum of per nucleotide read coverage on the forward strand divided by the same measure for reverse strand reads. OF strand ratios were calculated in windows of 2,001 nucleotides. A pseudo count of 1 read- covered nucleotide was added to both strands in each window to avoid divisions by zero. Results shown are for de-duplicated read data (identical start and end coor- dinates were considered duplicates). De-duplication minimises potential biases in PCR amplification, qualitatively similar results were obtained with non-de-duplicated data and support identical conclusions.
Trinucleotide preserving shuffles. Every nucleotide of the sacCer3 genome was assigned to one of 64 categories based on the identity of that nucleotide and its flanking nucleotides. A vector of transformations was produced by swapping the genomic coordinate of a nucleotide for one with an identical category chosen at random. Swaps between masked and unmasked sites (see above) were prevented. 100 such vectors were produced. For a set of stacked coordinates (for example, Fig. 1a comprising 27,586 sequences, each of 251 nucleotides), every nucleotide of every sequence was substituted through the transformation vector, for a randomly selected proxy, matched for the same trinucleotide context and their correspond- ing rate or annotation used. This provides a compositionally well-matched expectation, With 100 independent transformation vectors we provide empirically derived 95% confidence bounds and standard deviations on those expecta- tions. For human sites, shuffles were confined to sequences flanking the region of interest (100-300 nucleotides distant from the binding site for TF analysis and 1,000-2,000 nucleotides distant for DNase I footprint analysis). Human genomic coordinates in the ENCODE ‘Duke Excluded Regions” and those positions with a uniqueness score of <0.9 (gem-mappability” with k-mer = 100) were excluded from shuffles.
Rather than using separate Okazaki 5' and 3’ end counts that did not always correlate well, probably due to amplification biases, sequencing and size selection biases; we produced a normalized OJ rate measure. This is the average of (1) the fraction of upstream OFs that terminate with a 3’ end at a focal nucleotide, and (2) the fraction of downstream OFs whose 5’ end is at the focal nucleotide. The upstream and downstream coverage measures were based on mean Okazaki read coverage for the nucleotides located between 5 and 12 nucleotides upstream (down- stream) of the focal 3’ (5’) end. This OJ rate was calculated at single nucleotide resolution over both strands of the sacCer3 genome.
EmRiboSeq alignment and processing. Sequence reads (see Supplementary Table 2 for runs and read numbers) were aligned to the unmasked sacCer3 ge- nome with bowtie2 (version 2.0.0). Subsequent filtering and format conversion were performed using Samtools (version 0.1.18) and BEDTools (version 2.16.2). Only reads with a mapping quality score >30 were kept for analysis. As there had been no pre-sequencing amplification, de-duplication was not performed. Read 5'-end counts were summed per strand at single nucleotide resolution over the yeast reference genome. Note that under the emRiboSeq protocol, the ribonucle- otide incorporation site would be one nucleotide upstream and on the opposite strand to the mapped read 5’ end. To facilitate comparison between libraries of
Sites selected for analysis. Thresholds were applied to define specific subsets of sites to be evaluated. For the presented data (Fig. la) nucleosomes with an occu- pancy of >80%, positional fuzziness” of <30, with at least 30 OF reads over them, and located more than 200 nucleotides from transcription start sites were used. Other combinations (Extended Data Fig. 1) of these parameters gave qualitatively similar results and support the same conclusions. Reb] (and Rap1) sites were defined as the primary ChIP-exo peak ata site, with sequences aligned (x = 0) to the centre of the highest scoring Reb1/Rap1 position weight matrix match within 50 nucleotides of the ChIP-exo peak summit. DNase I footprints from 41 human cell types were previously combined” into consensus footprints (combined. fps.gz). We intersected the combined footprints with those found in each cell type using BEDtools (version 2.17.0) to identify the subset (n = 33,530) that were detected in all 41 cell types. The left-edge coordinate as defined in the combined footprint file was used as the focal nucleotide (x = 0) for analysis.
Comparison of polymorphism rates. The five nucleotide positions downstream and the five upstream of the focal OJ position (excluding x = 0 in both cases) were scored for their polymorphism rate (Fig. 2b). Rate deltas were calculated as up- stream minus downstream in 100 bootstrap replicates and a paired two-sided t-test
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
performed against the same calculation performed on 100 trinucleotide preserving genome shuffles of the same sites. This tests whether the difference in rate between upstream and downstream positions is greater in the observed data than the shuf- fled data.
with biotinylated A adaptor were captured on streptavidin-coupled M-280 Dynabeads (Life Technologies) following the manufacturer's guidelines, and non-biotinylated strands were released in 0.15 M NaOH. Single-stranded fragments were concen- trated by ethanol precipitation.
DNA purification. Yeast strains were grown at 30 °C in YPDA to mid-log phase (see Supplementary Table 3 for a list of strains) or to saturation for stationary phase. Per 5 Agoonm units, cell pellets were resuspended in 200 pil lysis buffer (2% Triton X-100, 1% SDS, 0.5 M NaCl, 10 mM Tris-HCl pH 8.0, 1 mM EDTA). An equal volume of TE-equilibrated phenol and glass beads (0.40-0.60 mm diameter, Sartorius) were added, and cells lysed by vortexing for 2 min; 200 pl TE buffer was then added, followed by an additional 1 min of vortexing. After centrifugation, the aqueous phase was further extracted with equal volumes of phenol:chloroform: isoamylalcohol (25:24:1) and chloroform. Total nucleic acids were precipitated with 1 ml of 100% ethanol, and dissolved in 0.5 M NaCl. RNA was degraded by treatment with 10 ug RNase A (Roche) for 1h at room temperature. DNA was finally purified with an equal volume of Ampure XP beads (Beckman Coulter) and eluted in nuclease-free water. For library preparations DNA was isolated from up to 40 Agoonm units.
Alkaline gel electrophoresis. Isolated genomic DNA (0.5 jig) was treated with recombinant RNase H2, purified as previously described® and ethanol precipitated. DNA pellets were dissolved in alkaline loading dye and separated on 0.7% agarose gels (50mM NaOH, 1 mM EDTA) as previously described”, and stained with SYBR Gold (Life Technologies). Densitometry measurements and derivation of ribonucleotide incorporation rates as previously described”, Percentage genome contribution for each replicative polymerase (x) was calculated using the follow- ing formula: Napotx*Fpotx /(Npola’Fpola + Npold? Fpold + Npole* Fpole), With Napotx the number of ribonucleotides incorporated in one yeast genome for the mutant poly- merase, above that detected in the Arnh201 POL strain, measured on the same alkaline gel, and Epey the frequency of incorporation by that polymerase (see Fig, 3a). EmRiboSeg library preparation and sequencing. DNA was sonicated using a Bioruptor Plus (Diagenode) to achieve an average fragment length of approxi- mately 400 bp. Fragmented DNA was concentrated by ethanol precipitation and size selected using 1.2 volumes of Ampure XP. DNA was quantified by nanodrop (Thermo Scientific) and up to 5 pg was used for NEBNext End Repair and dA- Tailing (New England Biolabs) following the manufacturer's guidelines. After the end-repair reaction, DNA was purified using 1.2 volumes of Ampure XP. Sub- sequent steps were performed in the presence of Ampure XP beads, capturing the DNA by adding NaCl and PEGs oo to final concentrations of 1.25 M and 10%, respectively. The trP1 adaptor (see below) was attached using NEBNext Quick Ligation with 120 pmol of adaptor per microgram of DNA for 14-18 hat 16°C.
Phusion Flash High-Fidelity PCR Master Mix (Thermo Scientific) was then used for second strand synthesis with primer A to produce a double stranded library. Size selection of fragments between 200 and 300 bp in size was performed using 2% E-Gel EX (Life Technologies). Finally, this library was quality checked and quan- tified using a 2100 Bioanalyzer (Agilent Technologies) before emulsion PCR, using the Ion Torrent One Touch, and next generation sequencing on the Ion Torrent PGM or Proton platform (Life Technologies).
Oligonucleotides and adaptor design. Custom oligonucleotides were synthe- sized by Eurogentec. Adaptor primer pairs were annealed by heating at 95°C for 5 min and cooling gradually. Sequences of the adaptor primer pairs were as follows. Adaptor 1 (trP1): trP1-top, 5'-CCTCTCTATGGGCAGTCGGTGAT- phosphorothioate-T-3'; trP1-bottom, 5'-phosphate-ATCACCGACTGCCCAT AGAGAGGC-dideoxy-3’. Adaptor 2 (A): A-top, 5'-phosphate-CTGAGTCGGA GACACGCAGGGATGAGATGG-dideoxy-3'; A-bottom, 5'-biotin-CCATCTC ATCCCTGCGTGTCTCCGACTCAGNNNNNN-C3 phosphoramidite-3'. The sequence for primer A used in second strand synthesis was 5'-CCATCTCATC CCTGCGTGTCTCCGAC:3’,
Data sources, sequencing data and S. cerevisiae strains. Documented in Sup- plementary Tables 1-3.
- 51. Kuhn, R. M., Haussler, D. & Kent, W. J. The UCSC genome browser and associated tools. Brief. Bioinform. 14, 144-161 (2013).
- 52. Derrien, T. et al. Fast computation and applications of genome mappability. PLoS ONE 7, €30377 (2012).
- 53. Eaton, M. L, Galani, K., Kang, S., Bell, S. P. & MacAlpine, D. M. Conserved nucleosome positioning defines replication origins. Genes Dev. 24, 748-753 (2010).
- 54. Hesselberth, J. R. et al. Global mapping of protein-DNA interactions in vivo by digital genomic footprinting. Nature Methods 6, 283-289 (2009).
- 55. Jiang, C. & Pugh, B. F. A compiled and systematic reference map of nucleosome positions across the Saccharomyces cerevisiae genome. Genome Biol. 10, R109 (2009).
- 56. Liti, G. et al. Population genomics of domestic and wild yeasts. Nature 458, 337-341 (2009).
- 57. Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. Mol. Biol. Evol. 24, 1586-1591 (2007).
- 58. Rhee, H. S. & Pugh, B. F. Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution. Cel! 147, 1408-1419 (2011).
Terminal transferase (NEB) was then used to block any free 3’ ends with ddATP for 2h at 37 °C, with 20 U of TdT per microgram of DNA. After Ampure XP pu- rification, beads were removed and DNA nicked using recombinant RNase H2 (10 pmol pig! of library) or Nb.BtsI (NEB; 10 U pg ') for 2h at 37°C, RNase H2 purification and reaction conditions were as previously described”. Enzymes were inactivated by heating at 80 °C for 20 min, and DNA was purified using 1.8 volumes of Ampure XP. Shrimp alkaline phosphatase (Affymetrix; 5 U) was then used to re- move 5' phosphates at 37 °C (1 h per ug of library). After heat inactivation for 15 min at 65 “Cand Ampure XP purification, DNA was denatured by heating at 95 °C for 5 min and snap cooling. Subsequently, A adaptor (see below; 120 pmol ig of li- brary) was attached using NEBNext Quick Ligation for 14-18 hat 16 °C. Fragments
- 59. Korhonen, J., Martinmaki, P., Pizzi, C., Rastas, P. & Ukkonen, E. MOODS: fast search for position weight matrix matches in DNA sequences. Bioinformatics 25, 3181-3182 (2009).
- 60. Mathelier, A. et al, JASPAR 2014: an extensively expanded and updated open- access database of transcription factor binding profiles. Nucleic Acids Res. 42, 0142-D147 (2014).
- 61. The ENCODE Project Consortium. An integrated encyclopedia of DNA elements the human genome. Nature 489, 57-74 (2012).
- 62. Thurman, R. E. et al, The accessible chromatin landscape of the human genome. Nature 489, 75-82 (2012).
- 63. Reijns, M.A. et al. The structure of the human RNase H2 complex defines key interaction interfaces relevant to enzyme function and human disease. J. Biol. Chem. 286, 10530-10539 (2011).
# ©2015 Macmillan Publishers Limited. All rights reserved
# in
ARTICLE
Qu4ad) ayes wsjydiowAjog Qu4ad) ayes wsjydiowAjog 91400 8800 11200 vSZ0'0 = %0e+ ” so, WE) op % 3 ; = 3 : “5 : : re 5 3 ' 2 3 2 $ 8 i ° 3 5 ? a3 2 ; 8 2 88 E 8 9 rs 8 22 “4 R 5 7 5 = "8 : ; = # | — nu 1600'0 24000 19000 1800'0 a mp et baal - - (qu sad) ayer wsiydiowsjog ği = E 3 3 32 eğ 5 3 g 2 = Ld o 23 4 3 : : i 8 5 33 e 3 #3 s a 44084 ueow %09- x (ju sed) ayes wsiydiowsjog _ 9950'0 vleo'0 8200 = E 2 3 3 2 ee 32 3 2 E 3 5 | j 6 Cc 2 § § < 5 2g 38 i 5 a is z 2 a a Di (ju sed) ayes uonounr pezeyOo 14000 (ju sad) eyes uoyounr piezeyO 72100 89000 4200'0 © (ju sed) eyes uojounr Mezeyo
Qu4ad) ayes wsjydiowAjog vSZ0'0 = 3 3 : 3 8 2 E 8 "8 # a
# 440014 ueew %08- vam, (1U 40d) 9721 Wsiydiowhjog
is
%OL+ ueow %01- (qu Jed) ayes wsiydsowsjog
— Tt ‘%OOL+ ueew ‘%08- (ju sad) eyes wsiydiowsjog
eyed gns ANEY %0Z+ «ure = %0Z- es a |
73
73
# Rebt site (nt)
# Nucleosome (nt)
# Reb1 site (nt)
©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
Extended Data Figure 1 | Increased OJ and polymorphism rates correlate at binding sites of different nucleosome classes and at Rap! binding sites. a-f, OJ and polymorphism rates are strongly correlated for different classes of nucleosomes. Data presented as in Fig. 1a, for different sub-classes of S. cerevisiae nucleosomes, demonstrating that OJ and polymorphism rates co- vary in all cases. Transcription start site proximal nucleosomes (d) are probably subject to strong and asymmetrically distributed selective constraints, which is likely to explain the modestly reduced correlation for this subset. Such transcription start site proximal nucleosomes were excluded from analyses of other categories presented (b, ¢, e, f), except ‘all nucleosomes’ (a). g, OJ and polymorphism rates are correlated for the S. cerevisiae TF, Rap1. Data presented, as for Rebi in Fig. Ib, show increased OJ and polymorphism rates
around its binding site, with a dip corresponding to its central recognition sequence. h-j, Increased polymorphism and OJ rates at Rap! (h), nucleosome (i) and Reb! (j) binding sites are not caused by biases in nucleotide content. Distributions calculated as for g, Fig. 1a and b, respectively, using a trinucleotide preserving genome shuffle. Pink shaded areas denote 95% confidence intervals for nucleotide substitution rates (100 shuffles). k, 1, Polymorphism (red) and between-species (black) substitution rates are highly correlated for nucleosome (k) and Rebi (1) binding sites. Best fit splines shown only. y axes scaled to demonstrate similar shape distribution. Values plotted as percentage relative to the mean rate for all data points (central 11 nucleotides excluded for calculation of mean in g, I).
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
a
# Isolate genomic DNA
# 5 zi ş>bcb
7 —.———— 2
# Sonication
tN containing fragment
}
Non-rN containing fragment
End repair and A-tailing
igati 1* adapter ligation
Terminal transferase + ddATP Blocking of 3'-OH groups
Cleavage with RNase H2 (or restriction endonuclease)
Removal of 5'-P groups
Denaturation and 2” adapter ligation
Streptavidin bead capture and release of non-biotinylated strand
Second strand synthesis
5 3 » — * ' PB R ce ve a HO é = ' = =—__§__—__a Ham————————um '
(
ey) 7 y » on ve a HO a IIb
No second adapter ligation; Not captured by streptavidin beads
Size selection lon semi-conductor sequencing
# ov
# pe —=
# Saas
a = © a RK a Po a a 7 2 = 8 7 POL [Nb.Bts!] (log2 TPM) a 250,000 50,000 emRiboSeg summed signal (TPM) o 150,000 4 6 8 20 40 o Distance from
Map sequence to genome
10
20
# Nb.Bts! nick (nt)
# Extended Data Figure 2 | EmRiboSeq methodology and validation.
a, Schematic of emRiboSeg library preparation. rN, ribonucleotide. b-d, Validation of strand-specific detection of enzymatically generated nicks through linker-ligation. Nb.BtsI nicking endonuclease cleaves the bottom strand of its recognition site releasing a 5’ fragment (cyan) with a free 3’-OH group after denaturation, to which the sequencing adaptor (pink) is ligated, allowing sequencing and mapping of this site to the genome (b). Nb.BtsI libraries have high reproducibility between Arnh201 POL and Arnh201 Pol-a* (poll-L868M) strains after normalizing read counts to sequence tags per million
(TPM). Bona fide Nb.Btsl sites were equally represented, at maximal frequency, in both libraries (c). Those with lower frequencies represented sites in close proximity to other Nb.Btsl sites, causing their partial loss during size selection. Additionally, Nb.BtsI-like sites were detected as the result of star activity. Libraries were also prepared using BciVI restriction enzyme digestion, that did not show such star activity (data not shown), allowing calculation of the site specificity for the method (>99.9%). Summed signal at Nb.Btsl sites shows >99.9% strand specificity (blue, correct strand; grey, opposite strand) and >99% single nucleotide resolution (d).
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
# a
# b
# c
# d
Pol-e* (M644G) 1:100 2 G6 e 6* Gs w 9 = & 3g a+ 2 4 Telomere 200 400 Centrosome 600 Telomere Position on Chr10 (kb) a
$
# g
5
# POL
# Telomere
200
400
# Centrosome
600
# Telomere
Position on Chr10 (kb)
Extended Data Figure 3 | Mapping replicative polymerase DNA synthesis using emRiboSeg. a, Point mutations in replicative polymerases elevate ribonucleotide incorporation rates, permitting their contribution to genome synthesis to be tracked. Schematic of replication fork with polymerases and their ribonucleotide incorporation rates (refs 27, 30 and J. S. Williams, A. R. Clausen & T. A. Kunkel, personal communication) as indicated (POL denotes wild-type polymerases; asterisk denotes point mutants). Embedded ribonucleotides indicated by ‘R’; additional incorporation events due to polymerase mutations highlighted by shaded circles. b, c, Mapping of leading/
lagging-strand synthesis by Pol-5* and Pol-e* yeast strain using emRiboSeg (as in Fig. 3) highlights both experimentally validated (pink dotted lines) and putative (grey dotted lines) replication origins. These often correspond to regions of early replicating DNA” (c). d, Pol-x* DNA is detected genome-wide by emRiboSeq as a component of the lagging strand in stationary phase yeast, as shown by the opposite pattern for a polymerase wild-type strain. Strand ratios are shown as best-fit splines with 80 degrees of freedom, y axes show log, of the strand ratio calculated in 2,001-nucleotide windows (b-d).
# ©2015 Macmillan Publishers Limited. All rights reserved
©
ARTICLE
a* oe &* — — — — — POL FA tAtA LYA tA mh201 g2ononr o no
Ayısuajuı sanejoy Gel position (top to bottom)
No S9 © rN o (e0LX) auoua6 Jad sapyoajonuogny
or so 00 se 41 16:1 11 14 1:16 oL Sı (2601) ones puesys begogrywe <——— ——> selg puens 6ul6öe7 seiq pues Bulpea7 32:1 16:1 8:1 21 41 11 | osz Ost os oO sjuawBes o1WoUusb jo yunog Strand asymmetry ratio (Okazaki-seq) Lagging:leading strand ratio (Okazaki-seq) oL so (260j) ones puesys begogıyua oo so o ————— —__> seig puens Gui66e7 POL selg pueys Bulpee) a GN GE, GE GR a | oL so 00 şe (z60)) ones pues bagoqnywe —— ——> seiq pues Bul66e7 selg pueys Buipee oh oz SE 16:1 41 11 1:4 1:16 8:1 16:1 32:1 41 21 11 Lagging:leading strand ratio (Okazaki-seq) Lagging:leading strand ratio (Okazaki-seq)
©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
Extended Data Figure 4 | Quantification of in vivo ribonucleotide incorporation by replicative polymerases. a, b, Representative alkaline gel electrophoresis of genomic DNA from yeast strains with mutant replicative DNA polymerases (a), with accompanying densitometry plots (b). Embedded ribonucleotides are detected by increased fragmentation of genomic DNA following alkaline treatment in an RNase H2-deficient (Arnh201) background. Increased rates are seen with all three mutant polymerases (indicated by asterisk, as defined in Extended Data Fig. 3a), and are reduced in Pol-e’ which contains the point mutation Met644Leu, a mutation that increases selectivity for dNTPs over rNTPs”. c, Quantification of average ribonucleotide incorporation in polymerase mutants from four independent experiments. DNA isolated from mid-log phase cultures; error bars denote s.e.m. Overall ribonucleotide content is the product of incorporation frequency and the total contribution of each polymerase, resulting in the total ribonucleotide content detected to be highest for Pol-e* (14,200 per genome), followed by Pol-ö* (4,300 per genome), Pol-c:* (2,700 per genome), POL (1,900 per genome) and Pol-e’ (860 per genome). d, Most of the yeast genome exhibits directional asymmetry in replication (median 4:1 strand ratio). Count of genomic
segments calculated for consecutive 2,001-nucleotide windows over the yeast genome based on reanalysis of OF sequencing data'’ denoted as ‘Okazaki-seq’. The strand asymmetry ratio was calculated after re-orienting all regions such that the predominant lagging strand was the forward strand. e-g, Genome- wide quantification of strand-specific incorporation of wild-type and mutant replicative DNA polymerases determined by emRiboSeg reflects their roles in leading- and lagging-strand replication. A close to linear correlation with Okazaki-seg strand ratios is observed. The strand ratio preference for lagging- strand ribonucleotide incorporation for independent libraries (including stationary phase libraries for POL and Pol-x*, marked by diamonds) was plotted against the lagging:leading-strand ratio determined using Okazaki-seq data (only ratios = 1:1 for the latter are shown for clarity). There was high reproducibility between experiments in strand ratio preferences. Lines are lowess smoothed (see Methods) representations of the full data sets (representative examples given in f and g). f, g, Scatter plots illustrating the individual strand ratio data points for 2,001 -nucleotide windows, for stationary phase POL (f) and Pol-x* (g) yeast. Pearson’s correlation = 0.49, P<2.2X 10 '°for POL (f); correlation = 0.75, P< 2.2 X 10 '* for Pol-2* (g).
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
a
# b
Pol-a* ———ç — —
# POL
# c
RNH201 wt - sf wt - sf kb 10 8 6 5 4 3 2 1.5 1 05 POL Pol-a* gg kDa Wt - sf W - sf RNH201 37 a-FLAG a-Actin
2 8 £ 2 5 £
5.0 — Amh201+wt 45 — Amh201- 4.0 — Arnh201 + sf 3.5 — Pol-a* Arnh201 + wt 3.0 — Pol-a* Amh201 - 2.5 2.0 Gel to
# position (top
# bottom)
37
Extended Data Figure 5 | Pol-a-synthesized DNA retention is independent of RNase H2 processing of RNA primers. a, b, The ribonucleotide content of genomic DNA is unchanged between Arnh201 strains transformed with empty vector (—) or vector expressing Rnh201 separation-of-function mutant (sf), that retains the ability to cleave RNA:DNA hybrids, including RNA primers, but cannot cleave single embedded ribonucleotides”, In contrast, the same vector expressing wild-type Rnh201 (wt) fully rescues alkaline sensitivity of the
DNA. As complementation with the separation-of-function mutant had no detectable effect on the ribonucleotide content seen in the Pol-x(Leu$68Met) Arnh201 strain, retention of Pol-a-synthesized DNA appears to be independent of a putative role for RNase H2 in RNA primer removal. Representative result shown for n = 3 independent experiments. c, Wild-type and mutant Rnh201 are expressed at equal levels, as shown by immuno- detection of the C-terminal FLAG tag. Loading control, actin.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
REST (n = 2,305) * e Şe e iva — © io -100 o 100 -30 o 30 Distance from binding motif mid-point (nt)
CTCF (n= 23,642) 2 | ş ° - a o —_—_- SO -100 o 100 -60 o 60 Distance from binding motif mid-point (nt)
# e
2 3 & o
- CEBPA (n = 70,480) FOXA1 (n = 25,708) NFKB (n = 11,456) T e | pa Ey © — a 5 2 — 3 — 2 © Ss s CREB1 (n = 26,572) FOXO3 (n = 20,624) NFYA (n= 5,704) ' 378 nig fi : 3 3 N a < 3 ° ° a a | REST (n = 2,308) a e 7 7 | 3 3 i i 5 2 2 2 5 a : : ELK1 (n = 8,384) IRF1 (n = 8,600) STAT1 (n= 4,988) | $ | © . 8 - o o di | b © 3 3 = 2 ° 8 ELK4 (n = 3,556) MEF2A (n= 8,648) USF1 (n= 15,892) 8 z : S$ Di a ' o bd o 5 s o e 7 7 " Ss Mi ETS1 (n 22,620) NFIC (n = 56,204) YY1 (n= 18,408) $ 3 3 3 3 : 6 3 3 e a ~ ———1 3 ~ 1
# Low-medium
# Medium-high
# High
- 100
# o
100
100
# o
100
- 100
# o
100
Distance from binding motif mid-point (nt)
Extended Data Figure 6 | Elevated substitution rates are observed adjacent to many human TF binding sites. a-d, Nucleotide substitution rates (plotted as GERP scores) are elevated immediately adjacent to REST (a, b) and CTCF binding sites (c, d). Colour intensity shows quartiles of ChIP-seq peak height (pink to brown: lower to higher), reflecting strength of binding/occupancy. Stronger binding correlates with greater increases of proximal substitution rate in the ‘shoulder’ region (asterisk). Increased substitution rates are not a consequence of local sequence composition effects (b, d). Strongest binding
quartile of sites (brown) is shown compared to a trinucleotide preserving shuffle (black) based on the flanking sequence (100-300 nucleotides from motif midpoint) of the same genomic locations. Brown dashed line and grey shading denote 95% confidence intervals. e, Substitution rates plotted as GERP scores for human TF binding sites identified in ChIP-seq data sets (in conjunction with binding site motif). Sites aligned (x = 0) on the midpoint of the TF binding site within the ChIP-seq peak (colours as for a-d). Dashed black line shows y = 0, the genome wide expectation for neutral evolution.
# ©2015 Macmillan Publishers Limited. All rights reserved

a Genome-wide footprints b 2 2 2 2.“ » 8 2 -_— g 3 g 5 3 Sa § Ss a $ oS 6 © & i 32 g5 3 z 8 35 > EN "4 O © © O 8 3 os | a a | so -50 o 50 DNase footprint mid (nt) Non-Reb1 & Rap1 footprints 8 2 g Ss © 52 g 85 cof 3 E s g5 SE Ko) 2 a g 8 8 ° © amm 8 — ——— o -50 o 50 DNase footprint mid (nt)
# e
# d
p<2.2.10%
p<22.1018
# e
# Footprints
11 nt 11nt — ş 3 So 8 2 2 gs e 8 p g 7 3 £8 o o 58 § 2° 2 8 o 2 © : ş 30 20 40 0 10 DNase footprint edge (nt) Alledges Non-Reb, Rap1
' .. Binding motifs DNAse | footprints Mutation Selection Substitution
# rate
# edges
Extended Data Figure 7 | OJ and polymorphism rates are increased at yeast DNase! footprints. a, b, DNase I footprint edges correspond, genome-wide, to increased OJ rates and locally elevated polymorphism rates in S. cerevisiae (a), a pattern that is maintained when footprints associated with Rebl and Rap] binding sites are excluded (b). Genome-wide DNase I footprints (n = 6,063) and excluding those within 50 nucleotides of a Reb1 or Rap! binding site (n = 5,136) were aligned to their midpoint. c, d, Aligning DNase I footprints on their left edge rather than midpoint (to compensate for substantial heterogeneity in footprint size) demonstrates a distinct shoulder of elevated polymorphism rate at the aligned edge (c), with a significant elevation compared to nearby sequence upstream from the footprint (d). DNase I footprints from a were aligned to their left edge (x = 0) with corresponding polymorphism rates shown (c). The increased polymorphism rate cannot be explained by local sequence compositional distortions (d). Nucleotide substitution rates in the 11 nucleotides centred on the DNase footprint edge (pink line), and another 11 nucleotides encompassing positions —35 to —25 relative to the footprint edge (green line) were quantified. Darker pink and
shades denote the mean for the same sites after trinucleotide preserving genomic shuffles. Error bars denote s.d.; statistics by Mann-Whitney test. e, Model shows that correlation of increased nucleotide substitution and OJ rates are consistent with increased mutation frequency across heterogeneous DNase I footprints. Polymorphism is reduced at sequence-specific binding sites within the footprints, owing to functional constraint. Therefore, the effect of OF-related mutagenesis in these regions is most sensitively detected in the region immediately adjacent to the binding site (left of vertical dashed blue line, representing footprints aligned to their left edge). This ‘shoulder’ of increased nucleotide substitutions represents sites with increased, OJ- associated mutation is followed by a region of depressed substitution rates, owing to selective effects of the functional binding sites within the footprints (to the right of the dashed blue line). Signals further to the right are not interpretable given the heterogeneity in DNase I footprint sizes. Given strong selection at TF and DNase I footprint sites, this ‘shoulder’ of elevated nucleotide substitutions could represent a measure for the local mutation rate for such regions, analogous to that measured by the fourfold degenerate sites in protein coding sequence.
green filled circles denote the mean of observed substitution rates and lighter
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
o
o
Processing Likelihood
# DNA binding protein
Ligation Likelihood Pol-ö stop © Likelihood Pol-a retention o
il | o
Extended Data Figure 8 | Model to show Pol-a DNA tract retention downstream of protein binding sites. a, OF priming occurs stochastically, with the 5’ end of each OF initially synthesized by Pol-x and the remainder of the OF synthesized by Pol-ö. b, c, OF processing: when Pol-ö encounters the previously synthesized OF, Pol-ö continues to synthesize DNA displacing the 5’ end of the downstream OF, which is removed by nucleases to result in mature OFs which are then ligated. The OJs of such mature OFs before ligation were detected previously’ after depletion of temperature-sensitive
# o
DNA ligase I. They demonstrated that ifa protein barrier is encountered (grey circle), Pol-ö progression is impaired, leading to reduced removal of the downstream OF (b). Given that — 1.5% of the mature genome is synthesized by _Pol-«, a proportion of lagging strands will retain Pol--synthesized DNA (red). When Pol-ö progression is impaired by protein binding, this will lead to an increased fraction of fragments containing Pol-x-synthesized DNA downstream of such sites (c).
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
# METHODS
No statistical methods were used to predetermine sample size.
Protein expression and purification. Mouse core RAGI (384-1008 amino acids) and RAG2 (1-387 amino acids) were cloned into the pLEXm-based'* mammalian expression vector, modified with an N-terminal Hiss tag followed by MBP tag and a PreScission cleavage site (LEVLFQ/GP) (where '/ indicates the cleavage site). The N-terminal Met of RAG2 was mutated to Val (M1V) during cloning. To express the RAGI-RAG2 complex in HEK293T cells, 500 pg of each of the RAG] and RAG2 expression plasmids were mixed with 4 mg of polyethylenimine (Polysciences) in 35 ml of Hybridoma medium (Invitrogen) to transfect 1 | of HEK293T cells grown in suspension culture in Freestyle 293 medium (Invitrogen), supplemented with 1% FBS when the cell density reached 1.5 million per millilitre. Four days after trans- fection, cells were harvested and stored at —80 °C. Cell paste (~8 g from 1] culture) was re-suspended in 50 ml of lysis buffer containing 20 mM HEPES (pH 7.3), 1 M KCl, 1 mM Tris (2-carboxyethyl) phosphine (TCEP) (pH 7.0), 1 mM EDTA and protease inhibitor cocktail (Roche), and lysed by sonication. After centrifuging at 35,000 r.p.m. for 1 h, the clarified lysate was mixed with 5 ml of amylose resin (NEB), which was pre-equilibrated with the lysis buffer and incubated with rotation for Th, After pouring the resin into a column and washing thoroughly with 200 resin volumes of the lysis buffer, the RAGI-RAG2 protein was eluted by an amylose elution buffer containing 20 mM HEPES (pH 7.3), 500 mM KCl, 40 mM maltose and 1 mM TCEP. The eluted protein, which consisted mainly of RAGI-RAG2 heterotetramer”” was concentrated and stored at —80 °C after adding glycerol to 20% final concentration. In contrast to RAGI and RAG? core proteins expressed in insect cells, which required activity-based purification for structural studies”, RAG1-RAG2 expressed in human cells is highly active. Human HMGBI (1-163 amino acids) was prepared as reported previously'*”*,
Photon Source (APS) at Argonne National Laboratory, All data were indexed, inte- grated and scaled with the XDS package“ (Extended Data Table 1). Structure determination and refinement. Phases were determined by the single- wavelength anomalous diffraction (SAD) method and multi-crystal averaging”. SAD data were collected from six SeMet-substituted crystals with the best resolu- tion of 3.7 A (Extended Data Table 1). Data were processed according to the pub- lished procedure” (Extended Data Fig. la). Selenium sites were identified using SHELXD” and refined with PHASER™. Out of the 58 highest anomalous peaks, 54 corresponded to selenium sites, and two were Zn’ * ions. Phases were improved density modification using RESOLVE” and the overall figure-of-merit was 0.79. The RAG1-RAG2 model was built manually in COOT™, Although the experimental electron density map contained breaks in the main chains, and side-chain defini- tion was not perfect, the register of the polypeptide chains was readily determined based on the SeMet sites. This initial model was refined in Phenix” and manually improved using COOT. Secondary structure restraints and non-crystallographic twofold symmetry averaging restraints were used throughout the refinement. The RAGI-RAGZ? structure was refined to 3.2 A with Ryork and Riree Of 20.6% and 25.9%, respectively (Extended Data Table 1). The quality of the structure was validated with MolProbity”*. 90.7% of residues are in the favoured regions of the Ramachandran plot, 9.3% in additional allowed regions, and no residue in the disallowed region. The final model contains amino acids 391-1008 of RAG] and amino acids 2-350 of RAG2, and one Zn?” ion in each RAGI. The N-terminal residues (391-404) are ordered in one RAGI subunit. Owing to poor electron densities, residues 608-616 of RAGI and 82-88, 242-244, 254-255 and 334-337 of RAG2 were not included in the final model. Crystal packing of two neighbouring RAG1-RAG2 tetramers appears to occlude one nonamer-binding site in each RAG1-RAG2 complex (Ex- tended Data Fig. le). No water molecules were added. All structure figures were prepared with PyMOL (http://www.pymol.org), and sequence conservation ana- lysis was performed using ClustalW”.
# by
To make the SEC complex, purified RAG1, RAG2, 12RSS and 23RSS DNAs were mixed at 2:2:1:1 molar ratio in the presence of a 2-fold excess of HMGBI with the buffer of 20 mM HEPES (pH 7.3), 150 mM KCl, 5 mM CaCl, and 1 mM TCEP and incubated for 1 h at 37 °C. After removing the MBP tag by addition of PreScission protease (1:100 mass ratio of protease to substrate) overnight at 4 °C, further puri- fication was performed using gel filtration (Superdex 200, GE Healthcare) in 20 mM HEPES (pH 7.3), 500 mM KCl, 5mM CaCl, and 1 mM TCEP, which removed HMGBI along with free DNA, the MBP tag, and PreScission protease. All puri- fication steps were performed at 4 °C. HMGBI could be retained with RAGI-RAG2 and RSS DNA if KCl concentration was reduced to below 100 mM (Extended Data Fig. 2), but we were unable to crystallize SEC completed with HMGBL. To prepare selenomethionine (SeMet)-labelled RAG1-RAG2 complex, HEK293T cells were transferred after transfection to methionine-free Freestyle 293 medium (Invitro- gen) supplemented with 25 mg] ' 1-SeMet (Acros Organics) and 1% dialysed FBS (Invitrogen). Three days later, cells were collected and protein was purified in the same way as native protein. Mass spectrometry analysis of trypsin-digested SeMet- labelled RAG1-RAG2 peptides was performed at the Taplin Mass Spectrometry Facility (taplin.med.harvard.edu). It showed that about 40% of methionines were substituted by SeMet.
- 46. Aricescu, A. R., Lu, W. & Jones, E. Y. A time- and cost-efficient system for high-level protein production in mammalian cells. Acta Crystallogr. D62, 1243-1250 (2006).
- 47. Swanson, P. C. A RAG-1/RAG-2 tetramer supports 12/23-regulated synapsis, cleavage, and transposition of V(D)J recombination signals. Mol. Cell. Biol. 22, 7790-7801 (2002).
- 48. Kabsch, W. Xds. Acta Crystallogr. D 66, 125-132 (2010).
- 49. Sheldrick, G. M.A short history of SHELX. Acta Crystallogr. A 64, 112-122 (2008).
- 50. McCoy, A.J. etal. Phaser crystallographic software. J. Appl. Crystallogr. 40, 658-674 (2007).
- 51. Terwilliger, T. C. Maximum-likelihood density modification. Acta Crystallogr. D 56, 965-972 (2000).
- 52. Emsley, P,, Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr. D 66, 486-501 (2010).
- 53. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D 66, 213-221 (2010).
- 54. Chen, V.B. et al. MolProbity: all-atom structure validation for macromolecular crystallography. Acta Crystallogr. D 66, 12-21 (2010).
- 55. Chenna,R. et a/. Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res. 31, 3497-3500 (2003).
Crystallization and data collection. Crystals of the RAGI-RAG2-DNA complexes were grown by the hanging-drop vapour diffusion method at 4 °C over 3 weeks. Equal volumes of protein (~5 mg ml” ') and reservoir solution containing 100 mM MES (pH 7.1), 10-15% PEG 3350, 200 mM tribasic ammonium citrate (pH 7.0) and 100 mM KCl were mixed in each droplet. Crystals were cryo-protected in reservoir solution supplemented with 25% ethylene glycol and flash frozen in liquid nitrogen. We were able to crystallize tetrameric RAG1-RAG2? alone as well as RAGI-RAG2 with a single 12RSS or 23RSS, but these crystals were small, and none of them diffracted X-rays as well as the SEC complex. Crystals of SeMet-labelled RAGI- RAG2 complex were grown under similar conditions. Native and SeMet-labelled complex both crystallized in the C222, space group with two RAG] and two RAG2 (one RAG1/2 heterotetramer) in each asymmetric unit. Data were collected at 100 K for native and SeMet-derivative crystals at beam lines 221D and 23ID of the Advanced
- 56. Schumacher, F. R., Sorrell, F.J., Alessi, D. R., Bullock, A. N. & Kurz, T. Structural and biochemical characterization of the KLHL3-WNK kinase interaction important in blood pressure regulation. Biochem. J. 460, 237-246 (2014).
- 57. Corbett, K. D., Shultzaberger, R. K. & Berger, J. M. The C-terminal domain of DNA gyrase A adopts a DNA-bending p-pinwheel fold. Proc. Nat! Acad. Sci. USA 101, 7293-7298 (2004).
- 58. Schuetz, C. et al. Lesson from hypomorphic recombination-activating gene (RAG) mutations: Why asymptomatic siblings should also be tested. J. Allergy Clin. Immunol. 133, 1211-1215 (2014).
- 59. Corneo, B. et al. Identical mutations in RAG1 or RAG2 genes leading to defective V(D)J recombinase activity can cause either T-B-severe combined immune deficiency or Omenn syndrome. Blood 97, 2772-2776 (2001).
- 60. Dhingra, N. et al. Severe combined immunodeficiency caused by a new homozygous RAG1 mutation with progressive encephalopathy. Hematol. Oncol. Stem Cell Ther. 7, 44-49 (2014).
# ©2015 Macmillan Publishers Limited. All rights reserved

# XDS —> POINTLESS —> SCALA —> SHELX C/D —> PHASER
8.8.8.8 Anomalous Correlation Coefficient (%) 8 M1 78 64 55 49 45 42 39 dg 12 Ne di a: a: - Se We it Sie oc ği linear pucts oT ees=t 33 @ 2sRss MW iorss

Extended Data Figure 1 | Structure determination. a, The plot of correlation coefficient (CC) of anomalous signal versus resolution. The red line indicates the cutoff of CC = 0.3. Merging data from the two best crystals produced a better CC than merging data from all six crystals. The data processing procedure is outlined above the plot”. b, The SAD experimental map contoured at 1.36 showed the content of an asymmetric unit. The Se anomalous map is contoured at 3.06 in red. c, A typical crystal of RAGI- RAG2. d, The content of crystals was examined by protein and DNA denaturing gels after a thorough wash of the crystals and stained by Coomassie blue and SYBR green. To confirm the 1:1 molar ratio of 12 and 23RSS DNA,
*°p_labelled input RSS DNAs and those in SEC complexes before and after crystallization are shown beneath the SYBR-green-stained DNA gel. e, Transposition assay of the purified SEC (RAGI-RAG2-12/23RSS DNA complex) used for crystallization. Supercoiled pUC19 (sc, with a small amount of open circle, oc) was the target; it was linearized by Hindili as a control. The SEC (0.25, 0.5 and 1.0 1M) was active in concerted transposition and thus linearizing pUC19. In contrast, RAGI-RAG2 or HMGBI (0.5 1M) each alone was not active, f, Crystal packing of neighbouring RAGI-RAG2 complexes (shown in dark and light colours) occludes one nonamer-binding site in each heterotetramer of RAGI-RAG2.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
# a
320 —
350 v2 367 ll ii —ğ—ğ——————— ———
387
M.musculus SNMGNGTTFLGIPGDNKOAMSEA-FYFYTLRCSEEDLSEDOKIVSNSOTSTEDPGDSTPFEDSEEFCFS. H.sapiens SNMGNGTVFLGIPGDNKOVVSEG-FYFYMLKCAEDDTNEEOTTFTNSOTSTEDPGDSTPFEDSEEFCFS. R.norvegicus SNMGNGTVFLGIPGDNKQAMSEA-FYF YMLRCSEDNSSEEQKIVSNSQTSTEDAGDSTPFEDSEEFCFS, X.tropicalis ha Ny ere yi Ağ ---PVVOTCSOGSTEEOEDSMPLEDSEEFTFN Grkkgieiş kkk #k#ş Og HERR KR, Çk RRR, RR Rp RRR RRR OK,
© SAA s SVP
VON SN é substrate product
# d
|. RAG 1/2 (1-387)-12/23RSS —— RAGI/2(1-351)-12/23RSS — RSS om 12RSS
Extended Data Figure 2 | RAG2 core fragment (1-351 amino acids) is active. a, Sequence alignment of RAG2 from mouse (320-387 amino acids), human, rat and Xenopus with predicted secondary structures shown above. b, Core RAG2 (1-387) and two further truncated RAG2 variants (1-351 and 1-367) were constructed with a non-cleavable N-terminal MBP tag and co-expressed with the tag-less core RAGI. The Coomassie blue R-250 stained SDS gel shows the purified RAG1-RAG2 complexes. c, Purified RAGI- RAG2 complexes with truncated RAG2 variants are equally active in cleaving a “P-labelled 12RSS DNA (in the presence of a 23RSS and Mg**, as examined
# o
# o
|
(|
by TBE-Urea gel). d, Elution profiles of RAGI-RAG? (both long and short forms) complexed with DNA from Superdex-200 (S200) in a low salt buffer (50 mM HEPES pH 7.0, 60 mM KCl, 1 mM maltose and 2mM DTT). Regardless of the length of RAG2, the major $200 eluant peak came out at the same time point and contained RAGI, RAG2 (1-351 or 1-387) and HMGBI proteins, as shown in the SDS gel (right insert), as well as 12 and 23RSS oligonucleotides, as confirmed by a TBE-Urea gel stained by SYBR green (left insert).
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE



# RAG2
# KLHL2
# GyrA CTD
Extended Data Figure 3 | Comparison of RAG2 with $-propeller and B-pinwheel structures. KLHL2 (PDB 4CHB)“ is selected to represent the B-propeller proteins, and the C-terminal domain (CTD) of GyrA (PDB ISUU)” is selected to represent the B-pinwheel structures. After superposition, RAG2 (a), KLHL2 (b) and GyrA (c) are shown side-by-side individually in
# two
to red rainbow colours. The loops in RAG2 that interact with RAGI are labelled. The six B-blades are named by Roman numerals, I-VI, from N to C terminus; four -strands in each blade are named by Arabic numerals, 1-4. orthogonal views. Each structure is coloured from N to C terminus in blue
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE

Extended Data Figure 4 | Comparison of RAGI and NBD-DNA complex. a, The NBD in the RAGI-RAG2 core complex (blue and green) superimposes well with the published structure of the NBD-DNA complex (PDB 3GNA, protein coloured yellow)”. b, The twelve SCID/Omenn syndrome mutationsin the NBD domain are mapped onto the crystal structure of the NBD-DNA
# o _
©
complex. Six SCID/Omenn syndrome (R391 to R407) mutations are located on a positively charged surface patch that interacts with the nonamer; five remaining SCID/Omenn syndrome mutations (L408 to A441) appear to affect the structural integrity of the NBD, and R446 may interact with the spacer DNA in each RSS.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE

# Hermes





Extended Data Figure 5 | Transposases that form a hairpin intermediate. a-c, Hermes (PDB 4D1Q)” (a), bacterial Tn5 (PDB code 1MUS)* (b), and RAGI dimers (c) are shown as ribbon diagrams in two orthogonal views, with the dyad perpendicular to the viewing plane (left) or in the plane (right). Each dimer consists of a cyan and a green subunit. The catalytic RNH domains are highlighted in pink, and the conserved catalytic residues are shown as red ball-and-sticks. The catalytic divalent metal ions are shown as green spheres if present. The DNAs, coloured in yellow (cleaved by the cyan subunit) and
orange (cleaved by the green subunit), have similar orientations in the Hermes and Tn5 complexes (as indicated by the arrows). Arrows with dashed outlines indicate that the DNAs are in the back of the viewing plane. Notably, the pair of RNH domains is oriented similarly in all three cases. The predicted orientations of DNAs bound to RAGI are indicated by the yellow and orange arrows, and the «-helices connected to the third catalytic carboxylates (shown in light purple) probably bridge two DNAs in RAGI recombinase as in Hermes and Tn5.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
Pfv Integrase

# mony

# c

Mos1 Mariner

Extended Data Figure 6 | Transposases that do not form a hairpin intermediate. a-c, Retroviral integrase from Prototype foamy virus (Pfv, PDB 30S0)** (a), bacterial MuA transposase (PDB code 4FCY)* (b) and eukaryotic Mos! mariner transposase (PDB 3HOT)” (c) are shown in comparable views and same representations as Hermes, Tn5 and RAGI-RAGZ2 in Extended Data Fig. 5. Each catalytic dimer consists of a cyan and a green subunit. Two accessory subunits in Pfv are shown in light blue and green, and two accessory subunits of the MuA structure are omitted for clarity. The catalytic RNH
domains are highlighted in pink. The DNAs, coloured in yellow (cleaved by the cyan subunit) and orange (cleaved by the green subunit), have similar orientations (within 30°) as indicated by the arrowheads, but each differs more than 90° from the corresponding DNA in Hermes or Tn5 transposase. The grey DNA in the MuA complex represents the target of transposition. Among these three recombinases, the 9-helix that follows the third catalytic carboxylate (coloured in light purple) does not cross over to interact with a second DNA.
# ©2015 Macmillan Publishers Limited. All rights reserved



Extended Data Figure 7 | Surface potential and conservation of RAGI- RAG2 complex. a, Orthogonal views of the electrostatic potential surface ofthe RAGI-RAG? structure. Blue indicates positive charges, and red negative. b, Orthogonal views of the molecular surface of RAGI-RAG2 with absolutely
# o
conserved residues highlighted in deep purple. The NBD is well conserved. The views with dyad in the plane here are related to the image shown in Fig. 5c by —50” rotation around the dyad.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
Extended Data Table 1 | Statistics of native and SeMet SAD data collection and structure refinement
Native Crystal #1 Crystal #2 Crystal #3 Crystal #4 Crystal #5 Crystal #6 ‘Space group C222; C222; C222; C222; C222; C222; 0222, Cell dimensions a,b,c (A) 168.8, 180.1, 200.2 168.7, 179.0, 199.3 168.5, 179.2, 200.3 169.1, 180.1, 200.8 168.7, 180.3, 202.3 169.4, 179.3, 199.7 169.1, 179.6, 200.0 a, By © 90, 90, 90 90, 90, 90 90, 90, 90 90, 90, 90 90, 90, 90 90, 90, 90 90, 90, 90 Absorption (Se) Peak Peak Peak Peak Peak Peak Wavelength (A) 1.0000 097918 0.97918 0.97918 0.97918 0.97913 0.97918 Resolution” (A) 50-3.2 50.0-3.8 $0.0-4.0 $0,0-3.9 50.0-3.8 30.039 (831-3.2) (3.94-3.8) 3 (4.14-4.0) (4.04-3.9) (3.94-3.8) (4.04-3.9) Kğ? 0.105 (0.58) 0.151 (0.778) 0.149 (0.831) 0.199 (0.859) 0.194 (0.796) 0.174 (0.873) 0.200 (0.977) Hol * 12.75 (2.23) 17.7 (4.6) 17.5 3.9) 1444.1) 14.2 (4.5) 16.8 (4.5) 13.1.8) Completeness “(%) 98.82 (99.9) 100.0 (100.0) 100.0 (100.0) 100.0 (100.0) 100.0 (100.0) 100.0 (100.0) 100.0 (100.0) Redundancy * 7.1 (34) 15.0 (15.3) 15.0 (15.3) 14.8 (15.2) 14.9 (15.3) 15.0 (15.3) 15.0 (15.3) Refinement Resolution (A) 50-32 No. reflections 49907 Revo Roce 0.206 / 0.259 No. atoms
Protein 14976 Ligand/ion (Zn) o 2 Water 0 B-factors Protein Ligand/ion Water R.m.s deviations Bond lengths (A) Bond angles (°)
106.4
85.0
- 
0.007
14
Asterisk indicates that data in the highest resolution shell is shown in parenthesis.
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
Extended Data Table 2 | Missense mutations of RAG1 and RAG2 identified in human SCID/OS patients
Human mutation _ RAGI R394W/Q R391A nonamer binding R396L/H/C R393A nonamer binding S401P $398 nonamer binding T403P T400 nonamer binding R404W/A/Q R401A nonamer binding R4100/W R407A nonamer binding L411P L408 Structural integrity of NBD D429G D426 Structural integrity of NBD V433M V430 Structural integrity of NBD M435V 1432 Structural integrity of NBD A444V A441 Structural integrity of NBD R449K R446 Probably DNA binding (spacer) L454Q L451 Structural integrity of RAGI dimer R474S/H/C R471A Structure & DNA binding in DDBD S480G “ S477 Structure & DNA binding in DDBD L506F L503 Structural integrity of DDBD R507W R504 Solvent exposed, DNA binding? GS16A GS513 Structural integrity of RAGI W522C W519 Structural integrity of preR D539V D536 Exposed, near RAG1/2 interface R559S R556 At the edge of RAGI/2 interface RS561H/C R558 RAGI/2 interface (T169 of RAG2) A565D A562 Structural integrity of preR S601P $598 Structural integrity of active site C602W C599 Structural integrity of active site H612R H609L Disordered, near RAG2 A622P A619 Structural integrity of RNH R624C/H R621A Active site, adjacent to D600 E669G E666 RAG1/2 interface R699Q/W R696 Structural stability of RNH G709D G706 Structural integrity, active site R716W R713A Structural integrity of RAG12 G720C * G717 RAGI? interface E722K E719K RAGI? interface C730E/F C727 Structural integrity of ZnC2 L732F/P L729 Structural integrity of ZnC2 R737H R734A Possibly DNA binding (coding end) H753L H750A Structural integrity of RAG1/2 R764P R761 At the edge of RAG1/2 interface E770K E767 RAGI/2 interface R776Q R773A RAGI/2 interface R7780/G/W R77SA Structural integrity of RAG1/2 P786L P783 RAGI? interface 1794T © 1791 Structural integrity of ZnH2 R841W/Q R838 DNA binding (near heptamer) N855I N852 Interacts with 3’ end of RSS L885R L882 Structural integrity of ZnH2 W896R W893A Structural integrity of ZnH2 Y912C Y909 Structural integrity of ZnH2 1956T 1953R Structural integrity of ZnH2 R973H/C R970 Heptamer binding (intra-subunit) F974L F971 Structural integrity of CTD R9ITSW/Q R972 Structural integrity of CTD Q981R/P Q978 Heptamer binding (inter-subunit) K992E/R K989 Probable DNA binding Mouse residue _..Predicted Structural Effects
M1006V
M1003
# Domain interface of CTD-DDBD
# RAG2 G35V R39G C41W G9SR R229E/Q/W M285R
G35 R39A
C41
G95 R229 M285
RAGI/2 interface (E666 of RAGI) RAG1/2 interface (E719, R773) Structure, and RAGI/2 interface Structure integrity of RAG2 RAG1/2 interface (D546 of RAGI) Partially exposed, maybe structure
All SCID/Omenn syndrome mutations here are listed in refs 12 and 13 except for three, for which references are given in the table (refs 58-60). Red residues are the mutations made in mouse",
# ©2015 Macmillan Publishers Limited. All rights reserved
ARTICLE
Extended Data Table 3 | Mouse RAG1-RAG2 mutations presented in ref.
4
Mutations Location and potential functional roles
RAGI mutations: RAG2 binding +, all other - Nonamer binding
K405A/H406A/ R407A R748A/H750A R773A/R77SA
R748 is near ZnC2 and ZnH2 and structurally important. H750 may stabilize the ZnC2 structure. R773 is at RAGI/2 interface between E719 (RAGI) and R39 (RAG2). R775 is exposed to solvent. H937 coordinates the zinc, and K938 forms a salt-bridge with E709 next to the catalytic D708.
H937A/K938A
H942A R969A/R970A
# Zinc coordination Next to CTD in the positive
# for DNA binding
groove RAGI mutations: K596A R621A/H R713A E719K R734A W760A H795A W956A RSS binding +, Nicking and Hairpinning - H-bond to the carbonyl of A957 and A955, stabilizes the W956 conformation in the apo-structure Next to D600 in the active site H-bond to “O” of E719, Y725 and 1726 (near RAG1/2 interface) RAG1/2 interface Solvent exposed, but could bind coding-end DNA RAG1/2 interface In the active site, next to D708 W956 is near E962 but facing exterior and separate from the active site by the protein backbone. RAGI mutations: K608A H609L R855A/K856A R890A W893A K980A Nicking +, Hairpinning - Disordered Disordered K856 is oriented toward the CTD/DDBD, R855 is solvent exposed Near D797 carboxylate, structure integrity, near R855 Structural integrity of ZnH2 On the CTD charged surface, potential for heptamer binding RAGI mutations: R401A/R402A E4230 R440A E547Q S723A/C Joining negative or defective Probable nonamer binding Forming a salt bridge with R407 that probably binds nonamer Probable nonamer binding RAG1/2 interface Adjacent to E719 (OS) and close to R39 of RAG2 RAGI mutations: E649A Gain-of-function (12RSS processing) Solvent exposed and adjacent to N961 and $963 near E962. RAG2 mutations: KII9A K283A/R RAGI binding+, all other - Solvent exposed and part of the positive top rim of the “Y” shaped RAGI/2 complex Near M285R (OS), stabilizing the loop of 306-315 RAG2 mutations: K38A/R39A RSS binding+. Nicking and Hairpinning - RAG1/2 interface, possible coding-end binding RAG2 mutations: K34A K56A/K58A R73A H94A K97A KII9R RI67A 361 frame shift Joining negative or defective Near OS mutation G35V, RAGI/2 interface (adjacent to R73 and K97) Solvent exposed, may interact with coding DNA flank RAGI/2 interface (adjacent to K34 and K97) H94 is a part of the RAG2 structural core and important for the correct folding Near OS mutation G95R at RAGI/2 interface (K34 and R73) Exposed to solvent and adjacent to K56 and K58 Near RAGI interface (N525 of RAGI), forming positive-p stack with W172 (near P674 of RAG1) without PHD and regulatory domains
Mutations that correspond to SCID/Omenn syndrome mutations in human are highlighted in grey. R795A listed in ref. 4 (highlighted in red) should be H795A.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# METHODS
The optical spectroscopy on J0100+2802 was first carried out on 29 December 2013 with the Yunnan Fainter Object Spectrograph and Camera (YFOSC) of the Lijiang 2.4-m telescope in China. We used a very low resolution grism (G12, at a dispersion of 18 A per pixel) and took 3,000 s exposure on this target. The spectrum clearly showsa sharp break at about 8,800 A and no significant emissions blueward, consistent with a quasar spectrum at a redshift beyond 6.2. To confirm this discov- ery, two subsequent optical spectroscopic observations were obtained on 9 and 24 January 2014 with the 6.5-m Multiple Mirror Telescope (MMT) and the twin 8.4-m mirror Large Binocular Telescope (LBT) in the USA, respectively. The low to medium resolution spectra, obtained with 1,200 s exposure using the MMT Red Channel (at a dispersion of 3.6 A per pixel) and 2,400 s exposure with the LBT Multi-Object Double CCD Spectrographs/Imagers (MODS)*" (at a dispersion of 1.8 A per pixel) respec- tively, explicitly confirm that SDSS J0100+2802 is a quasar at redshift 6.30 + 0.01 (obtained by the Lyx line).
The near-infrared K-band spectroscopy on J0100+2802 was carried out with LBT/LUCI-1 on 2 January 2014. Owing to the short exposure time (15 min), the spectrum is of modest signal-to-noise ratio (S/N). Although the Mg tt line was clearly detected, the noisy LBT spectrum did not allow us to accurately measure the line width. To improve the quality of the near-infrared spectrum, we obtained J,H,K- band spectroscopy with Gemini/GNIRS and Magellan/FIRE on 6 August and 7 October 2014, respectively. The exposure time was 3,600 s for GNIRS and 3,635 s for FIRE. The FIRE spectrum has higher S/N (about 30 in K band) and higher spectral resolution (R = 4/A/ — 6,000) than the GNIRS spectrum (with S/N of about 10 in K band and R ~ 1,800). In order to achieve the best spectral quality, we combined the FIRE and the GNIRS spectra, and scaled the combined spectrum according to its 2MASS J,H,K,-band magnitudes. The Mg ! line shown in the K-band spectrum gives the same redshift as that given by the Lys line in the optical spectrum. The high-quality J,H,K-band spectra also clearly display abundant absorption features, which have been identified as being from intervening or associated systems with redshifts from 2.33 to 6.14 (Extended Data Fig. 4).
is fitted within the rest-frame wavelength range between 2,000 A and 3,200 A by excluding the boundary region between H and K bands where the sky transparency is lower. An Fe 1 template” is adopted for the fitting of Fe 11 emissions. The Mg 1! emission line is fitted with two broad Gaussian components. The four Mg absorp- tion lines near the redder part of the Mg emission line are also fitted as four Gaussian lines in order to remove their effects on the fittings of Mg and Fe 1! emission lines. The overall FWHM of the Mg emission line is ~5,130 km s” ' with an uncertainty of 150km s” '. The continuum has a slope of — 1.43 and the continuum luminosity at the rest-frame wavelength of 3,000 A (L3,000) is (3.15 + 0.47) X 1097 ergs '. The Fen to Mgu line ratio is 2.56 + 0.18, which is consistent with the mean value of other z > 6 quasars'*"*, After applying a virial black-hole mass estimator based on the Mgu line”, we estimate its black-hole mass to be (1.24 + 0.19) x 10'° Mo. Although the systematic uncertainty of virial black-hole mass estimation can be up a factor of three”, our result still strongly indicates that J0100 +2802 hosts a central black hole with mass close to 1.2 X 10'° M5. This is also well consistent with a black-hole mass obtained by assuming an Eddington luminosity of J0100+2802, which leads to a mass of 1.3 X 10'9 Me. Considering the contribution of Balmer continuum, as done for other z > 6 quasars'*"’, leads to a decrease of L3,999 to (2.90 + 0.44) X 1097 ergs 1 an increase of FWHM of Mg 11 to 5,300 + 200 km s7 and yields a black-hole mass of (1.26 * 0.21) X 10'° M >. Therefore, the effect of considering Balmer continuum is insignificant for the black-hole mass measure- ment of J0100+2802. In addition, if we adopt a different virial black-hole mass scaling relation”, the black-hole mass changes to (1.070.14) X 10'° M«, which is still well consistent with the result we obtained above.
Sample size. No statistical methods were used to predetermine sample size.
- 31. Pogge,R. W. etal. The multi-object double spectrographs for the Large Binocular Telescope. Proc. SPIE 7735, 9-16 (2010).
- Vestergaard,M.& Wilkes, B. J. An empirical ultraviolet template for iron emission in quasars as derived from | Zwicky 1. Astrophys. J. Suppl. Ser. 134, 1-33 (2001).
32.
- 33. Salviander, S. etal. The black hole mass-galaxy bulge relationship for QSOs in the Sloan Digital Sky Survey data release 3. Astrophys. J. 662, 131-144 (2007).
After redshift and Galactic extinction corrections, the rest-frame H- and K-band spectrum is decomposed into a pseudo-continuum and the Mg 1! emission line. The pseudo-continuum consists of a power-law continuum and Fe 1! emissions, and
- 34. Vestergaard,M. & Osmer, P. S. Mass functions of the active black holes in distant quasars from the Large Bright Quasar Survey, the Bright Quasar Survey, and the color-selected sample of the SDSS Fall Equatorial Stripe. Astrophys. J. 699, 800-816 (2009).
©2015 Macmillan Publishers Limited. All rights reserved

Extended Data Figure 1 | Images of J0100--2802 in SDSS, 2MASS and WISE bands. J0100+2802 is undetected in SDSS u,g,r bands (top row) but is relatively bright in other bands (lower three rows). It is consistent with a point
source in the bands with high signal-to-noise detections. The size is 1’ X 1' for all images. The green circle represents an angular size of 10” in each image.
©2015 Macmillan Publishers Limited. All rights reserved
A DEC (arcsec) A RA (arcsec)
Extended Data Figure 2 | The LBT K-band image of J0100+2802. The size is 10" X 10". The horizontal and vertical axes denote the offsets in right ascension (ARA) and in declination (ADec.). The image, with seeing of 0.4”, shows a morphology fully consistent with a point source.
©2015 Macmillan Publishers Limited. All rights reserved
LETTER
1048 TT TT TT TT 7 © alot 4 Fog 1018 Luminosity (ergs s~!) © J0100+2802 1046 e J1148+5251 ; * J1120+0641 0.01 0.1 1 10 100 1000 Wavelength (um) (rest-frame)
Extended Data Figure 3 | The rest-frame spectral energy distributions of J0100--2802, J1148+5251 and ULAS J1120+0641. The redshifts of these three quasars are 6.30, 6.42 and 7.085, respectively. The luminosity of J0100+ 2802 in the ultraviolet/optical bands is about four times higher than that
of J1148+5251, and seven times higher than that of ULAS J1120+0641, The photometric data are from literature for J1148+5251 and J1120+0641. The error bars show the 1g standard deviation.
©2015 Macmillan Publishers Limited. All rights reserved
T T T T T T T T T T T T T T Si II(A) Me II(H) OC IV(D) c Iv(c) Al TKD) | il iI II | ii 9 9000 9500 10* E o ni 1 C IMA) Mg I1(G) Fe I(B) Fe II(F) a haa | | 0.8 © 0.6 — iş 0.4 SS 4-1X10* 1.15x104 1.2x10* 1.25x10* » 2 08 li TT 2 der z lala TT TT balli öz Eli İl | iI 1.45x104 1.5104 1.55x104 1.6104 1.65x104 1.7x 104 0.45 0.4 F 0.35 0.3 1.96x10* o 1.98x10* 2x10* 2.02x10* 2.04x104 2.06x104 Observed Wavelength (A)
Extended Data Figure 4 | The major absorption features identified from optical and near-infrared spectroscopy of J0100+2802. Most of them are from Mgil, Civ and Fe tl. The labels from A to H correspond to the redshifts
# of
absorption materials at 6.14, 6.11, 5.32, 5.11, 4.52, 4.22, 3.34 and 2.33, respectively. Studies of intervening and associated absorption systems will be discussed elsewhere.
©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# METHODS
The protocol for teleporting a spin-orbit composite quantum state. The com- bined state of photons 1, 2 and 3 can be rewritten in the basis of the 16 orthogonal and complete hyper-entangled Bell states as follows:
lll) = 5 (9* )ialo*) 2)64.10)s H(6™) 2107 ) 1263.65.19) +10 zl öğ. öğle) — (6) ala) 12)865.65,19) +(10~) 210" Ye) kb azl 12) 45-19) +9 )izlz öğle) —(08 Yl nzdiö$le), +(W*) 217) 2)i65, 10) +(W™) 217) 12165, 65.19) O Yl 1215, 65,19) +" )ialZ7)12963,65,19) 53 (WW) 2107) öğle); — (WW) 2107 )12)63-63-19) — (07) 2l2*) 12) ¢45x10)s + (|W) 217) 12465255, 19) 3]
ye 1 7 6 )aiz diz» Orde + AvDai Nes (8)
Yale" a5 DvryDyly —ApryAply (8) + Dyry Dyly —Ay ry Ay ly)
Y Yale p> 3(DureDu —AyryAyly se
# —Dyry Dyly 4 Azry Ayly)
+ = i Ww ole da 3 @OrrvAvly —AyryDyly öğ
# + Dy ry Ay ly —Ayty Dul)
Yalar Yaz s7 Dery Avy —AyryDyly ”
# —DyryAyly + AytyDyly)
Wn ala (Dy Dy — Ay Aş Mryry +h hy) a
# + (Dy Dy — Ay Ay ry ty +ly ly)
We a (Du Dy — Ay Ay ryry Kl) > —(Dy Dy — Ay Ay (ry ry + ly ly)
Ww drole M75 (Dy Dy —Ay Ay \ryry — İp) aia —(Dy Dy — Ay Ay ry ty — ly by)
We a5 (Dy Dy — Ay Ay ryry — lly) -
# +(Dy Dy
# — Ay Az (ry tx —ly ly)
where the triples (6, 6, 65,63) and (45, , öy 6?) are the Pauli operators for the SAM and, respectively, the OAM qubits. vt indicates that, regardless of the unknown state|g) ,, the 16 measurement outcomes are equally likely, each with a probability of 1/16. By carrying out the hyper-entangled Bell state measurement (h-BSM) on photons 1 and 2 to unambiguously distinguish one from the group of 16 hyper- entangled Bell states, Alice can project photon 3 onto one of the 16 corresponding states. After Alice tells Bob her h-BSM result as four-bit classical information via a classical communication channel, Bob can convert the state of his photon 3 into the original state by applying appropriate two-qubit local unitary transformations (C,4.65,0)@(1,62,0°,09)].
Two-photon interference of spin-orbit composite state on a PBS. The input and output of photons 1 and 2 encoded in SAM and OAM on a PBS as shown in rc 1 can be summarized as follows: |0)}|0)}— |0)},|0)},, (0)711)9 — (0)./1)5., 11); 10) >a] 1)5,[1)5-, [1)} [1)? + i]1)3, 10). m şi -preserving, böciee the reflection flips the sign of OAM. Therefore, the output state for each of the input 16 hyper-entangled Bell states can be listed below:
where D=(|0)*+|1)*)/ V2, A=(|0)* —|1)*)/-V2, r=|0)° and /=|1)°. Then both of the output modes are passing through two polarizers at 45 . Only the output with the term D, Dy will result in there being one photon in each output mode with an efficiency of 1/8. The other terms will be rejected by the conditional detection. According to equations (1)-(16), 4 of the 16 output modes have the term Dj Dy and the 4 corresponding input hyper-entangled Bell states are |#*) plw”),,, Yale "Yas 6 Yal Yaz amd”) Ya» which will be sent to the next stage of BSM on the OAM qubits.
BSM and teleportation of OAM qubits. Having measured and filtered out the SAM qubit, next we perform BSM on the OAM qubit. Dealing with a single degree of freedom is more straightforward, and can be implemented using a beam splitter. Like the PBS, the beam splitter also is not OAM-preserving, because the reflection at the beam splitter will flip the sign of OAM. Therefore, the transformation rules at a beam splitter for the four OAM Bell states are
16°) nl”) 2+ Tg (DrDe + Avan eş a)
a) loş Ol) Hol)
6“ Yula) J (Dr Dat Av Aa io” )yy o
o loş 00) 1) 108)
Yala eDrDy + Avda |i") vy 6)
Yal 2+ Jy (Dr Dat Av ADI v2 4)
6) dn al 910). AYD
- 410)210)5 +11)$ 1)39
4) da gazele NEY
16*)2lo* a> a0 Ay + AyDy)\o* ) yy (5)
(5) —10)2.10)2 + |1)>11)))
6 zle Ju Tyr Ay kayba İp (6)
M6" alt" a> Jy (Dre KADA" Yaz (7)
- (6) We can see that only the Bell state | ) ,, will result in there being one and only one photon in each output, whereas for the three other Bell states the two input photons will coalesce into a single output mode. Among these three Bell states, the
- (7) state | *),, can be further distinguished by measuring two single photons in
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
either output with the OAM orthogonal basis (|0)°,|1)°). Thus, this would allow us to unambiguously discriminate two from the four Bell states. Experimentally, we design dual-channel OAM readout devices to measure |0)° and |1)° simultaneously. Therefore, the efficiency of both QND and BSM on OAM is 1/2. The overall efficiency of h-BSM combining all three steps is 1/8 x 1/2 x 1/2=1/32.
efficiency of h-BSM combining all three steps is 1/8 x 1/2 x 1/2=1/32. Generating three photon pairs. Ultrafast laser pulses with an average power of 800 mW, central wavelength of 394 nm, pulse duration of 120 fs and repetition rate of 76 MHz successively pass through three B-barium borate (BBO) crystals (Fig. 2) to generate three photon pairs through type-I spontaneous parametric down- conversion (SPDC). The first and the third crystals are 2 mm-thick BBOs, whereas the second one consists of two 0.6 mm-thick contiguous type-I BBO crystals with optic axes aligned in perpendicular planes. All of the down-converted photons have central wavelengths of 788 nm. The first photon pair (1-t) is initially prepared in the zero-order OAM mode with a coincidence count rate of 3.6 x 10°Hz. In SPDC, the zero-order OAM mode has the highest weight among all OAM modes and thus has the highest brightness. The second photon pair (2-3) is simulta- neously entangled in both the SAM and the OAM (in the first-order mode), owing to OAM conservation in the type-I SPDC. It has a count rate of 3.4 x 10'Hz anda state fidelity of 0.95. The third photon pair (4-5) is created in the first-order OAM entangled state with a two-photon count rate of 1.2 x 10°Hz and a fidelity of 0.91. Weestimate the mean numbers of photon pairs generated per pulse as ~0.1, ~0.01 and ~0.05 for the first, second and third pairs, respectively.
a Dove prism rotated by 7/8, will be rotated by an angle of 7/4 and their phases will change to e*”-”/” and e 9), respectively. Therefore, two opposite phases, ©“ and e“, will be added to the two orthogonal OAM modes. Finally, the output photons are transformed into the|0)° and |1)° polarizations using a QWP. The overall transformations can be summarized as a CNOT gate between the OAM and SAM:
jo)‘joy° > ((0)* +|1)*)I0)°/ v2 elo + if1)°)]0)°/ V2 — e*/4\0)*|0)° 10)*la)9 ((0)* + [1)*)|1)°/ V2 5 eo) —i]1)*)|1)°/ V2 Nesi
Preparing spin-orbit entanglement. As illustrated in the yellow panel of Fig. 2, we use a Sagnac interferometer to prepare the spin-orbit hybrid entangled state to be teleported, |g), =(|0)°|0)° +|1)°|1)°)/ V2. Photon 1 is initially prepared in the (0)*—/1)9)/v2 SAM state with zero-order OAM. It is then sent into a Sagnac interferometer that consists of a PBS and an SPP, where the counter-propagating 10)" and |1)° SAMs with zero-order OAM pass through the SPP in opposite direc- tions and are converted into |0)*|0)° and |1)*|1)°, respectively. Considering an extra 7 phase shift for the |1) SAM from double reflections in the PBS, the final output state is (|0)*|0)° 4-|1)*1)9)/ 2. We note that the whole conversion process is deterministic.
Two-photon interference on the PBS and beam splitters. For a test of two- photon interference visibility at the PBS, input photons | and 2 are first intentionally prepared in the states Dır; Az) (orthogonal SAMs; see open squares in Extended Data Fig. la) and Dır, D2! (parallel SAMs; see solid circles in Extended Data Fig. la), respectively. We measure each single photon from the two outputs of the PBS in the Dy Dy basis (1' and 2’ denote the output spatial modes). At zero delay, where the photons are optimally overlapped in time, the orthogonal SAM input yields an output state of (|0)}, |0)3, +|1)}-|1)},)/ V2 conditioned on a coincidence detection, which can be decomposed in the diagonal basis as (|D)',|D)', + |A)},|A),)/V2, thus showing an enhancement. For the parallel SAM input, the output state is (DYE. |A)} +|A)}-|D),)/-V2, which shows a reduction. The increase in the delay gradually destroys the indistinguishability of the two photons, such that their quantum state becomes a classical mixture. Thus, at large delays the counts appear flat. Interferometers of this type are sensitive only to length changes of the order of the coherence length of the detected photons and stay stable for weeks.
The two-photon interferences on beam splitters 1 and 2 are for the teleportation- based QND measurement of OAM qubits and the BSM of OAM qubits, respectively. As a test, the two input photons are prepared in the same SAM but orthogonal OAM states, It is interesting to note that, in stark contrast to the conventional Hong-Ou-Mandel interference, only having the two input OAM states ortho- gonal can lead to an interference dip, because the reflection at the beam splitter flips the sign of OAM. The two-photon interference as a function of temporal delay is shown in Extended Data Fig. 1b, c.
Effectively, the OAM gubit is deterministically and redundantly encoded by the SAM qubit, that is, the SAM becomes identical to the OAM state. Thus, by mea- suring the SAM using a PBS with two output channels, we can recover information about the OAM with a high efficiency of ~97%, In our experiment, four such Sagnac interferometers are used in the h-BSM. In this way, two of the four OAM Bell states can be discriminated using a beam splitter.
Whereas the first type is like a two-channel readout device (like a PBS for SAM), the second type is like a one-channel readout device (like a polarizer for SAM), and is used in the final stage of state verification after teleportation. The strategy for the projective OAM measurement is to transform it into the zero-order OAM mode so that the photon can be coupled into a single-mode fibre, while all other higher- order OAM modes will be rejected. Here we use an SPP” and a BPP” for efficient OAM readout.
The SPP” is designed with a spiral shape to create a vortex phase of e/”, where / is an integer and is referred to as topological charge. ‘The SPP can attach vortex phases of ei” and e” to a photon when it passes through the SPP forwards and, respectively, backwards, When vortex phases of el” and e“ are attached, the corresponding OAM values will increase by and, respectively, decrease by /h. Therefore, a SPP can be used as an OAM mode converter. In our experiment, the OAM qubits are encoded in the OAM first-order subspace with the topological charge / being 1. The conversion between the OAM zero mode and the first-order modes (|0)° and |1)°) is realized using a 16-phase-level SPP with an efficiency of ~97%,
For the coherent transformation between the OAM zero mode and the super- position states (|0)° + |1)9)/ V2 and (|0)° + i|1)9)/V2, a 2-phase-level BPP*** is used with an efficiency of ~80%, These high-efficiency OAM measurement devices boost the sixfold coincidence count rate in our present experiment by more than two orders of magnitude, compared with the previous use of hologram gratings. Error budget. The sources of error in our experiment include double pair emis- sion in spontaneous parametric down-conversion; partial distinguishability of the independent photons that interfere at the PBS (~5%) and the beam splitters (~5%); state measurement error due to zero-order OAM leakage (~2%); fidelity imperfection of entangled photon pairs 2-3 (~5%) and 4-5 (990); and fidelity imperfection of the to-be-teleported single-photon hybrid entangled state (~8%).
Dual-channel and efficient OAM measurement. One of the most frequently used OAM measurement devices in the previous experiments is off-axis hologram gratings”**2*5', which typically have a practical efficiency (p) of about 30% (ref. 31). This low efficiency would cause an extremely low six-photon coincidence count rate that scales as p®, more challenging than the scaling (p’) in the previous two-photon OAM experiments. To overcome this challenge, we use two different types of device for efficient OAM readout.
The first type is what we refer to as ‘dual-channel’ OAM measurement devices, used after the two beam splitters. The strategy is to transfer the OAM information to the photon’s SAM, and measure it using a PBS with two output channels. The method is as follows. After the beam splitter, each photon passes through a HWP and is prepared in the state ((0)* #-|1)*)/ V2. They are then sent into Sagnac interferometers with a Dove prism inside” (Fig. 2), which is placed at a 7/8 angle with respect to the interferometer plane. The photon is rotated by an angle of 7/4 for the |0)* SAM component when passing through the Dove prism forwards and by an angle of -7/4 for the |1)° SAM component when passing through the Dove prism backwards. OAM states |0)° and |1)° with respective phases e” and e”, where p is the azimuthal angle in the polar coordinate system, which pass through
Some error sources affect all the teleported states. First, the double pair emission contributed a ~15% background to the overall sixfold coincidence counts. If this were subtracted, the average teleportation fidelity would be improved to ~0.74. Second, the imperfectly entangled photon pair 4-5 and the imperfect two-photon interference at beam splitters 1 and 2 (for the QND and OAM Bell-state measure- ments, respectively) degrade the teleportation fidelity for all states by ~ 13%. Third, the imperfect state measurements mainly due to the zero-order OAM leakage cause a degradation of ~2%.
Some error sources can have different effects on different teleported states. The imperfect two-photon interference at the PBS degrades the teleportation fidelities for the states |p)ç, |) and |g),, by ~5%. However, for the states |p), and |g), where the photon is horizontally or vertically polarized, the actual teleportation does not require two-photon interference at the PBS, and is therefore immune to the imperfection of the interference. This explains why the teleportation fidelities for the states |p) , and |g), are the highest. This is inconsistent with the previous results: for the experiments using a PBS“ , the teleportation fidelities in the horizontal-vertical basis are higher than those in the + 45° linear and circular
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
polarization bases; whereas for the experiments***’ using a non-polarizing beam splitter, the teleportation fidelities in all polarization bases are largely unbiased.
# Wx @L10)¥ lr Wy }OW)z
We note that the teleportation fidelity for the hybrid entangled state, |) ,,, is the lowest, which is affected by the imperfection (~8%) in the state preparation of the initial state |), as well as by imperfections in |g), and |g) p, that is, essentially all error and noise present in the experiment. It can be expected that all entangled states are subject to the same decoherence mechanism as the state |p) ; as demon- strated, and should undergo similar reductions in fidelity.
These sources of noise can in principle be eliminated in future by various meth- ods. For instance, deterministic entangled photons" do not suffer the problem of double pair emission. We also plan to develop bright OAM-entangled photons with higher fidelity, and a more precise 32-phase-level SPP for the next experiment of hyper-entanglement swapping.
We emphasize that, before sending the photons into the second beam splitter, for the reason discussed in the main text we use teleportation-based QND measure- ment to ensure the two photons can be fed into the subsequent cascaded inter- ferometers. Here the QND should preserve the quantum information in the Y and Z DoFs. Thus, quantum teleportation of two DoFs of a single photon is required (Extended Data Fig. 2a), which is exactly what we demonstrated in the experiment presented in the main text.
After that, we again pass the photons through two filters on the Y DoF, one set in the |1) state and the other set in the |) state, which leaves four asymmetric combinations:
A universal scheme for teleporting N DoFs. We illustrate in Extended Data Fig. 2 a universal scheme for teleporting N DoFs of a single photon. For simplicity, we discuss an example for three DoFs (Extended Data Fig. 2b), labelled X, Y and Z. There are in total 64 hyper-entangled Bell states:
dx @{lW)y )8116)7 slb)z hiz } Wx Sl) @lW)z
(6) lb) lik a) FOL dr Ny Wy @{Ib)z .16)7 Wz Wz )
Similarly, we perform a bit-flip operation on the Y DoF on one of the arms, erasing the information in the Y DoF. We then doa QND measurement on the Z DoF, and pass the two photons into the third beam splitter, finally filtering out the only remaining asymmetric state:
These the products of the Bell states of each DoF, defined as
# Wx @lv)y @lv)z
16) =(10) 10), 11),11))/ v2 WD =(10),11), +11), 10),)/ v2
By detecting one and only one photon in the output of the third splitter, we can thus discriminate this particular hyper-entangled state, |W). ©); @|w)z from the 64 hyper-entangled Bell states on three DoFs.
where i = X, Y, Z. The aim is to perform an h-BSM, identifying 1 out of the 64 states, The required resources include photon pairs entangled in the Z DoF, pairs hyper-entangled in the Y-Z DoFs, pairs hyper-entangled in the X-Y-Z DoFs, filters for the three DoFs that can project the state to /0) or 1) (with a functionality similar to the polarizers for SAM), gubit flip (c,) operations for the DoFs, a 50:50 non-polarizing beam splitter and single-photon detectors, all of which are com- mercially available or have been experimentally demonstrated previously.
It has been known that if two single photons are superposed at a beam splitter, only asymmetric quantum states can result in one and only one photon exiting from each output of the splitter. In the simplest case of one DoF, this is the asym- metric |) state. As now we have three DoFs, we have to consider the combined, molecular-like quantum states. There are in total 28 possible combinations that are asymmetric states, as follows:
To experimentally demonstrate the teleportation of three DoFs, the scheme would need in total ten photons (or five entangled photon pairs from SPDC), which is within the reach of near-future experimental abilities, given the recent advances in high-efficiency photon collection and detection. It is obvious that the above protocol can be extended to more DoFs as displayed in Extended Data Fig. 2c.
Feed-forward scheme for spin-orbit composite states. To realize a deterministic teleportation, feed-forward Pauli operations on the teleported particle based on the intrinsically random Bell-state measurement results are essential. In our pre- sent experiment, no feed-forward has been applied. Here we briefly describe how this can be done for the spin-orbit composite state. For the SAM qubits, active feed-forward has been demonstrated before using fast electro-optical modula- tors" (EOMs). To take advantage of this technology, which has been demon- strated to have high-speed operation and high gate fidelity, we use a coherent quantum SWAP gate between the OAM and SAM qubits. The SWAP gate is defined as
Wx @{lPy be Wy }@Llé)z 67 WW)z {lf)x +1P)x lx }@W)y 81167 .|6)z Wz } {10x 10)x Wx 38116) lb) İz Wx @W)y @lz
After the photons have passed through the first beam splitter, we can filter out and retain these 28 from the 64 hyper-entangled Bell states conditioned on seeing one and only one photon in each output of the splitter. Next we apply two filters in the two output of the splitter to project the X DoF into |i) * . One filter is set to pass the |1) state and the other is set to pass the |0) state. This results in the following 16 states being filtered from the 28:
Wx ©1146) lb) Wy }@{l)z .16)z WW)z Wx @lW)y @ld)z 167 Wz Wx SA) be İZ )x @W)y &l)z
}
|0)*|0)° + B|1)*|0)° +y]0)*|1)° +4]1)°|1)° — x|0)°|0)*+ 2)? |0)* +y10)°|1)* öl) fa
The operation sequence for the feed-forward operation on the spin-orbit com- posite states is shown in Extended Data Fig. 3a. First, the SAM feed-forward is done with an EOM. Second, the SAM and OAM qubits undergo a SWAP opera- tion. Third, an EOM is used to operate on the ‘new’ SAM that is converted from the OAM. We note that the EOM does not affect the OAM, and so the previous operation on the SAM is unaffected. Lastly, a final SWAP gate converts the SAM back to OAM, which completes the feed-forward for spin-orbit composite states.
The SWAP gate is composed of three CNOT gates (Extended Data Fig. 3b). In the first and third CNOT gates (blue shading), the SAM qubit is the control qubit that acts on the OAM as target qubit, realized by sending a photon through, and recombined at a PBS. On the reflection arm only, a Dove prism is inserted to induce an OAM bit flip. In the second CNOT (yellow shading), the control and target qubits are respectively the SAM and OAM qubits, as explained in the Methods section “Dual-channel and efficient OAM measurement’. The extra Dove prism and HWP in front of the PBS are used to compensate for the phase shift inside the Sagnac interferometer.
We perform a bit-flip operation on the X DoF on one of the arms of the inter- ferometer, erasing the information in the X DoF. We then pass them into the second beam splitter, filtering out and retaining the six asymmetric combinations
- 31. Graham, T. M., Barreiro, J. T., Mohseni, M. & Kwiat, P. G. Hyperentanglement- enabled direct characterization of quantum dynamics. Phys. Rev. Lett 110, 060404 (2013).
- 32. Slussarenko, S. et al. The polarizing Sagnac interferometer: a tool for light orbital angular momentum sorting and spin-orbit photon processing. Opt. Express 18, 27205-27216 (2010).
Wx @{lW)y }@{])z I6)z sI)z }
- 33. Andrews, D.L, & Babiker, M. (eds) The Angular Momentum of Light (Cambridge Univ. Press, 2012).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
- 34. Jack, B. etal. Precise quantum tomography of photon pairs with entangled orbital angular momentum. New J. Phys. 11, 103024 (2009).
- 38. Bouwmeester, D. etal. Experimental quantum teleportation. Nature 390, 575-579 (1997).
- 35. Pan, J.-W., Gasparoni, S., Aspelmeyer, M., Jennewein, T. & Zeilinger, A. Experimental realization of freely propagating teleported qubits. Nature 421, 721-725 (2003).
- 39. Ma, X.-S. et al. Quantum teleportation over 143 kilometres using active feed- forward, Nature 489, 269-273 (2012).
- 36. Zhang, O. et al, Experimental quantum teleportation of a two-qubit composite system. Nature Phys. 2, 678-682 (2006).
- 40. Lu, C.-Y. & Pan, J.-W. Push-button photon entanglement. Nature Photon. 8, 174-176 (2014).
- 37. Yin, J. et al Quantum teleportation and entanglement distribution over 100- kilometre free-space channels. Nature 488, 185-188 (2012).
- 41, Prevedel, R. et al. High-speed linear optics quantum computing using active feed- forward. Nature 445, 65-69 (2007).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
360 ——-—__.——+ Fourfold coincidence (300 s) b 420,————— — 360 no © NX 300 g 5 240 Ke} © 180 8 © 120 5 gz 60 Ww 0 -480 -320 -160 Cc 480 w 4204 160 320
5 SN 360} 8 300 + © L mo] © E 8 © © = 2
|
240
|
180} |! 120+
# b
# ol
0 -480
- 320
- 160
# o
160
320
480
# Delay (um)
Extended Data Figure 1 | Hong-Ou-Mandel interference of multiple independent photons encoded with SAM or OAM. a, Interference at the PBS where input photons | and 2 are intentionally prepared in the states Dır, Az; (orthogonal SAMs; open squares) and Dır, D>l, (parallel SAMs; solid circles). The yaxis shows the raw fourfold (the trigger photon t and photons 1, 2 and 3) coincidence counts. The extracted visibility is 0.75 + 0.03, calculated from V(0) = (Cy. — Cy) (Cs + Cy), where C;; and C; are the coincidence counts without any background subtraction at zero delay for parallel and, respectively, orthogonal SAMs. The red and blue lines are Gaussian fits to the raw data. b, Two-photon interference on beam splitter 1, where photons | and 4 are
prepared in orthogonal OAM states. The black line is a Gaussian fit to the raw data of fourfold (the trigger photon and photons 1, 4 and 5) coincidence counts. The visibility is 0.73 + 0.03, calculated from V(0) = 1 - Co/C.., where Cy and C.. is the fitted counts at zero and, respectively, infinite delays. c, Two-photon interference at beam splitter 2, where input photons | and 5 are prepared in the orthogonal OAM states. The black line is a Gaussian fit to the data points. The interference visibility is 0.69 + 0.03 calculated in the same way as in b. Error bars, 1 s.d., calculated from Poissonian counting statistics of the raw detection events.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
b c _ a, h-BSM worsen, N DOFs | ae Tog h-BSM x N 3DOFs 2 h-BSM Se N 2DOFs N , N / Classical N , information N , miş \ 1 \ Ul Bob | ' İç 1 ws Classical t / information (Alice , = ' di \ al Bob \ Single photon state Hyper-entanglement # re Classical encoded in2DOFs source in2DOFs .- U*e@ U"e@ U' __ information N ze — e e elemi a Alice Bitflip / ? Single photon state Hyper-entanglement Fitter f# encoded in 3 DOFs ‘source in 3 DOFs
Single photon state encoded in N DOFs
# ype source in
# N DOFs
Extended Data Figure 2 | A universal scheme for teleporting N DoFs of a single photons. a, A scheme for teleporting two DoFs of a single photon using three beam splitters, which is slightly different from the one presented in the main text using a PBS and two beam splitters. Through the first beam splitter, six asymmetric states,|y/)x © (İd); .|)y .|W)y } and {|)x lb s|W)x }@|wW)y , can result in one photon in each output, which is ensured by teleportation-based QND on the Y DoF. After passing the two photons through the two filters that Project them into the|1) and |0) states for the X DoF, four states, |) @{|0)y .10)7 )y } and); @lW)y » survive. Through the second beam splitter, only the asymmetric state |)» of the Y DoF can result in one photon in each output. Finally we can discriminate the state
\W/): @|W)y from the 16 hyper-entangled Bell states. b, Teleportation of three DoFs of a single photons (Methods). Note that to ensure that there is one and only one photon in the output of the first beam splitter, we can use the teleportation-based QND on two DoFs in a (dashed circle). ¢, Generalized teleportation of N DoFs of a single photons. The h-BSM on N DoFs can be implemented as follows: (1) the beam splitter post-selects the asymmetric hyper-entangled Bell states in N DoFs which contain an odd number of asymmetric Bell states in one DoF, (2) two filters and one bit-flip operation erase the information on the measured DoF and further post-select asymmetric states, and (3) teleportation-based QND.
# ©2015 Macmillan Publishers Limited. All rights reserved
# Bob
LETTER
# a
# b
SAM OAM —— EOM —— SWAP — EOM — SWAP ———> SAM SAM SWAP OAM OAM
Comp. SAM PBS OAM Dove
SAM HWP OAM OWP
Extended Data Figure 3 | Active feed-forward for spin-orbit composite states. a, The active feed-forward scheme. This composite active feed-forward could be completed in a step-by-step manner. First, we use an EOM to implement the active feed-forward for SAM qubits. It is important to note that EOM does not affect OAM. Second, we use a coherent quantum SWAP gate between the OAM and SAM qubits. The original OAM is converted into a ‘new’ SAM, whose active feed-forward operation is done by a second EOM. Then the
OAM and SAM qubits undergo a second SWAP operation and are converted to the original DoFs. b, The quantum circuit for a SWAP gate between the OAM and SAM qubits. The SWAP gate is composed of three CNOT gates: in the first and third CNOT gates, the SAM and OAM qubits act as the control and target qubits, respectively, whereas in the second CNOT gate this is reversed,
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# METHODS
Chemicals. The following chemicals were used as received: sodium dodecyl sulphate (29996), Sudan Red 7B (95%), 1,6-hexanediol diacrylate (80%), 1,4-butanediol diacrylate (90%), trimethylolpropane ethoxylate triacrylate (M, = 428 g mol '), 1,2-dichlorobenzene (99%), Zonyl FS-300 (40% solids), methoxyperfluorobutane (99%), mineral oil (light), iron(11) chloride tetrahydrate (99.99%), 2,2,3,3,4,4,5,5,6, 6,7,7,8,8,9,9-hexadecafluorodecane-1,10-diol (97%), methacrylic acid (99%) and octadecane (99%) (Sigma-Aldrich); silicone oil (for oil baths —40 to +200 °C), perfluorohexanes (98%), hexanes (98%), and iron(11) chloride (98%) (Alfa Aesar); ethyl nonafluorobutyl ether (>98%) and sodium oleate (>97%) (TCI); Darocur 1173 (Ciba); fluorinated acrylate oligomer (Sartomer); Nile Red (99%) (Acros); N-dodecylpropane-1,3-diamine (>95%) (Matrix). Light-responsive surfactant was made by the literature procedure”. Fluorinated coumarin dye was made by the literature procedure”, Cleavable surfactant sodium 2,2-bis(hexyloxy)propyl sul- phate was made by the literature procedure”,
and ethyl nonafluorobutyl ether above T-and shaking in 0.2% SDS and 0.2% Zonyl in a 2.5:1 ratio. The drops were oriented using a neodymium magnet.
a 2.5:1 ratio. drops were using a neodymium magnet. Fabrication of light-responsive emulsions. Hexane dyed with Sudan Red 7b and perfluorohexane in equal volumes were used as the inner phase, and a mixture of 0.1% light-sensitive surfactant and 0.1% Zonyl FS-300 in an 8:2 ratio was used as surfactants in the aqueous phase. Slight adjustments to the surfactant concentra- tions were made during imaging to tune the droplet morphological transitions to achieve the desired outcome; for example, more Zonyl was needed to generate drop- lets that transitioned from a Janus droplet to a hexane-perfluorohexane-water double emulsion. A mercury lamp was used as the intense light source, and DAPI and FITC filters were used to selectively allow ultraviolet (A = 365 nm) and blue (4 = 470 * 20 nm) light to reach the sample while imaged on an inverted microscope. Fabrication of reversibly responsive pH-sensitive emulsions. Hexane and perfluo- rohexane in equal volumes were emulsified in a solution of 2 mM N-dodecylpropane- 1,3-diamine and 1.2% Zonyl in 0.2 M NaCl. Salt solution was used to control for effects of ionic strength on the emulsion morphology. The pH was adjusted to above and below pH 4.7 by addition of HCl and NaOH.
Synthesis of the fluorinated crosslinker. To synthesize the fluorinated crosslinker 2,2,3,3,4,4,5,5,6,6,7,7,8,8,9,9-hexadecafluorodecane-1,10-diyl bis(2-methylacrylate), a mixture of 2,2,3,3,4,4,5,5,6,6,7,7,8,8,9,9-hexadecafluorodecane-1,10-diol (10 g, 22 mmol, | equiv.), methacrylic acid (20 ml, 240 mmol, 11 equiv.), 2,6-di-tert-butyl- 4-methylphenol (200 mg, 0.9 mmol, 0.04 equiv.) and sulphuric acid (0.1 ml, 18 M) in toluene (250 ml) was heated to reflux with azeotropic removal of water (Dean-Stark trap). After three days, the mixture was cooled to room temperature and washed with saturated aqueous sodium bicarbonate (3 X 300 ml). The toluene was dried with MgSO, and evaporated. The remaining residue/product was dissolved in perfluorohexanes (150 ml) and filtered. The filtrate was evaporated to yield 7.81 g of colourless liquid (13.1 mmol, 61% yield). 'H-NMR (400 MHz, CDCİ;): 6 6.22 (p, J= 1.1 Hz, 2H), 5.70 (p, J= 1.5 Hz, 2H), 4.65 (t, J= 13.3 Hz, AH), 1.98 (t, J = 1.3 Hz, 6H). C-NMR (101 MHz, CDC): 5 165.8, 135.0, 127.9, 117.5-107.5 (m, CE), 60.1 (t, J= 27.3 Hz), 18.1. '°F-NMR (376 MHz, CDCls): 6 —119.3 (p, J = 13.3 Hz, 4F), —121.8-—122.0 (m, 8F), — 123.3 (bs, 4F). LRMS (ED): calculated for CigH,4F\¢O, İM) *, 598; found, 598. NMR spectra were obtained on a Bruker Avance 400 MHz spectrometer. LRMS was acquired on an Agilent 5973N GCMS. Please see Extended Data Fig. 3 and Extended Data Fig. 4 for reaction scheme and NMR spectra.
General fabrication of complex emulsions. The hydrocarbon and fluorocarbon liquids of choice were heated until miscible and emulsified. The temperature required varied depending on the solutions. Solutions were emulsified either in bulk by shak- ing or by coaxial glass capillary microfluidics and cooled to induce phase separation. For hexane-perfluorohexane emulsions, the emulsions were chilled on ice before imaging and often imaged while immersed in a cool water bath to maintain a temperature below 20 °C. For microfluidics, Harvard Apparatus PHD Ultra syringe pumps were used to inject the outer phase and inner phase using a glass capillary microfluidic device made from an outer square capillary (outer diameter, 1.5 mm; inner diameter, 1.05 mm; AIT Glass) and inner cylindrical capillary (outer diameter, 1mm; World Precision Instruments) pulled to a 30 jm tip using a P-1000 Micro- pipette Puller (Sutter Instrument Company). The microfluidic set-up was heated above the 7; of the inner phase solution using a heat lamp. Emulsions were then cooled below T, to induce phase separation. Emulsions were observed to be stable during the time periods used (of the order of days). Longer-term stability experi- ments were not conducted.
Fabrication of emulsions using an acid-cleavable surfactant. Hexane and per- fluorohexane in equal volumes were emulsified in a solution of 0.3% sodium 2,2- bis(hexyloxy)propyl sulphate and 0.4% Zonyl in 0.1 M NaCl. The pH was adjusted to 3 using HCl and the solution was allowed to sit undisturbed on the inverted microscope stage while images were periodically made over an hour.
Fabrication of hemispherical particles. 1,6-hexanediol diacrylate with 4% Darocur 1173 photo-initiator was heated with an equal volume of methoxyperfluorobutane above T. and emulsified. 1% SDS and 1% Zonyl in a 3:2 ratio yielded Janus droplets which were then polymerized under a Dymax Blue Wave 200 ultraviolet lamp while kept cold on ice.
Fabrication of fluorous-hydrocarbon Janus particles. Fluorinated acrylate oli- gomer, fluorinated crosslinker, 1,4-butanediol diacrylate and trimethylolpropane ethoxylate triacrylate were used in a volume ratio of 15:3:14:10. 5% Darocur 1173 was used as the photo-initiator. The inner phase mixture was heated above T., emul- sified in 1% SDS and polymerized with ultraviolet light over ice.
Fabrication of four-phase emulsions. Light mineral oil with 20 wt% octadecane (used to reduce 7, in a mixture with the other liquids), silicone oil and ethyl nona- fluorobutyl ether were used as the inner phases in a volume ratio of 6:7:13. The mineral oil and ethyl nonafluorobutyl ether both partition into the silicone oil such that on phase separation the silicone oil phase is enriched with some quantity of the two other phases. Aqueous mixtures of varying ratios of 1% Zonyl and 1% SDS were used as the outer phase, and emulsions were formed in bulk by shaking.
Estimation of equilibrium interfacial tensions in the presence of surfactants. Experimentally, it proved difficult to measure the interfacial tensions with the pendant-drop method accurately for very long periods of time while maintaining a stable drop volume because surfactant continued to adsorb at the droplet inter- face, further reducing the already low interfacial tensions. Therefore, it is likely that the final interfacial tensions, 7, recorded for the hexane-water interface and, respec- tively, the perfluorohexane-water interface for different surfactant compositions, İsos, as illustrated in Extended Data Fig. 1a, are not the equilibrium interfacial ten- sions, Yegi» for that system. In the absence of additional experimental data for long time scales, we used theories of dynamic interfacial tension to estimate 7egb-
Rosen suggested” that the dynamic interfacial tension of surfactants could be accurately modelled using an empirical model of the form
Measurement of interfacial tensions. Interfacial tension measurements were made using the pendant-drop method (ramé-hart Model 500 Advanced Gonio- meter). Measurements on a drop were taken every 5 s until the interfacial tension appeared to be nearing equilibrium or the droplet became unstable. The hexane- water interfacial tension was measured to be 50 mN m” ' and the perfluorohexane- water interfacial tension was measured to be 55mN m |,
Microscopy. Lateral confocal cross-sections of the droplets were imaged using a Nikon 1AR ultrafast spectral scanning confocal microscope. Scanning electron microscopy was conducted on gold-sputtered samples with a JEOL 6010LA scan- ning electron microscope. Fluorescence and bright-field images were taken with a Zeiss Axiovert 200 inverted microscope equipped with a Zeiss AxioCam HRc camera. Droplets typically orient themselves with the denser, fluorous phase downward. To take side-view images of the drops, emulsions were shaken to induce the drops to roll around while images were made with a 1 ms exposure.
NE) Yep + Gon (5)
where 7p = 7(t = 0) and f* and n are positive, empirically determined constants. The value of n depends on the type of surfactant and the interface considered, as well as on the concentration of the surfactant, and must therefore be independently determined for every set of experimental conditions. The constant ¢* is the half-life of the interfacial tension decay process—it is the time taken for the difference, Ay = 7(t) — eqn to decrease to halfits initial value. The three unknown model param- eters (/eqh, t* and n) can be determined from a least-squares fit of equation (5) to the experimental dynamic interfacial tension data. The measure of the ‘goodness- of-fit’ of equation (5) to the experimental data is quantified by the coefficient of determination, or R’, value”, The closer this value is to 1, the better the fit.
Fabrication of magnetic Janus droplets. Magnetite nanoparticles were made as follows: 25 ml of concentrated NH,OH was added to an acidified solution of 1.6 g of FeCl; and 1 g of FeCl,*4H,0 in 50 ml of water at 80 °C. The magnetite nanopar- ticle precipitate was collected with a magnet, washed with water and redispersed. One gram of sodium oleate in 10 ml of water was added while stirring at room tem- perature. The oily black precipitate was extracted with hexanes. The solid was collected by evaporation of solvent, and was subsequently redispersed in dichlorobenzene. Janus droplets were obtained by heating the nanoparticle-dichlorobenzene solution
An example of the fit of the model to the dynamic interfacial tension data is shown in red in Extended Data Fig. la. The parameter results of using equation (5) to fit the interfacial tension data for all the surfactant systems considered are pre- sented in Extended Data Table 1. The smallest value of R? obtained for the experi- mental data across all the systems studied was 0.909, indicating very good agreement between the predictions made using equation (5) and the experimental data gen- erated using the goniometer.
The resulting extracted values of 7egp have been used to plot the variation in the interfacial tensions 74 and 7; as a function of the 0.1% SDS fraction, fops, in Fig. 2
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
and Extended Data Fig. 1b. We note that some droplets fall just outside the desig- nated Janus region. Discrepancies may be due to our inability to measure and esti- mate the equilibrium interfacial tensions with the 0.1 mN m ' accuracy required, or there may be slight mixing of hexane and perfluorohexane in the droplets, which would alter the interfacial tensions relative to those of the pure liquids.
Estimation of yı from the analysis of Janus droplet images. Bulk mixtures of hexane and perfluorohexane have an upper consolute temperature, T., of approxi- mately 22 °C, with a critical density of 1.14 gem” * anda volume ratio very close to 1:1 (ref. 18). The interfacial tension of the hexane-perfluorohexane interface at temperatures close to (but below) this upper consolute temperature is close to zero, and, consequently, it is difficult to measure accurately using conventional labora- tory techniques such as pendant-drop or Du Nuoy ring tensiometry. The best esti- mate in the physical chemistry literature comes from the predictions of a model for capillary fluctuations at an interface fitted to X-ray reflectivity measurements”, In ref, 23 it is proposed that the variation in the interfacial tension 7p; (in units of mN m”) with temperature 7 (in units of K) is given by
126 (1) =232 2) € (6)
D allows us to solve equation (8) for the radius of curvature of the F-H interface, Rpy. Equation (7) can then be used to solve for the missing interfacial tension, 74}.
The two radii of curvature, Ry; and Rp, and the diameter, D, of the circle of contact can be computed by analysing the images of the Janus droplets. A schem- atic of the analysis process is shown in Extended Data Fig. 2c-e. A raw image of a Janus droplet (Extended Data Fig. 2c) is first subjected to the Canny edge-detection algorithm” based on the non-normalized contrast, with a Gaussian kernel radius of 2.0 and low and high thresholds of 2.0 and 7.0, respectively. This procedure results in the detection of the outer (water-facing) interfaces of the Janus droplet as shown in Extended Data Fig. 2d. Arcs corresponding to the H-W and F-W inter- faces are then selected by visual inspection and fitted to circles using Taubin’s algorithm”. These arcs and their fitted circles are depicted as blue and red circles (with green dots for centres) in Extended Data Fig. 2e. The radii of these two circles correspond to the radii of curvature, Rj; and Ry, of the H-W and F-W interfaces, respectively. The diameter, D, of the circle of contact between the hexane and per- fluorohexane phases is calculated as the minimum separation between the points lying between the arcs chosen to represent the H-W and F-W interfaces. This line of minimum separation is shown in Extended Data Fig. 2e. Image preprocessing and edge detection were performed in Image] v1.48, and the circle-fitting and interfacial tension calculation was performed in MATLAB.
Equation (6) predicts that at a temperature of 283.15 K (10°C), the hexane- perfluorohexane interfacial tension should be 0.4 mN m” !, We were unable to find other experimental data in the literature to support this estimate. However, given the eguilibrium interfacial tension values and images of the resulting Janus drop- lets, it was possible to also estimate 71; for our system.
The relationships between the shapes of Janus droplets and the ratio of the vol- umes of the two constituent phases, k, and the relative magnitudes of the interfacial tensions operating at the various interfaces of the droplet have been examined in ref. 21. The theoretical treatment there can be used to relate the radii of curvature of the F-H, H-W and F-W interfaces to the volume ratio and the interfacial ten- sions 7/4; ’p and Yeşi
Yu _ Ve Yer (7) Ru Re Rew
Given the interfacial tension values at the H-W and F-W interfaces (7, and 7p, respectively), equation (7) can be used to determine 7'py if we can independently calculate the three radii of curvature Ry, Rp and Rey.
This droplet analysis was performed for six Janus droplets, three from each of the two conditions fgps = 0.4 and fgps = 0.6. The interfacial tensions obtained are tabulated in Extended Data Table 2. The values of yp}; do not vary significantly between the two sets of surfactant systems, which is consistent with our expecta- tion that 7p is at best only weakly affected by the composition of the surfactant. The average value of }p4; deduced from this analysis is 1.07 + 0.1 mNm
We note that accurately determining the interfacial tensions through the radii of curvature of the interfaces of the droplet is feasible only in the absence of fluid flow around the complex emulsion droplets. The presence of such a flow, even at low shear rates, leads to a deformation of the surface area of the droplet, causing local gradients in surfactant concentration and inducing Marangoni stresses. The inter- play between these effects ultimately determines the interfacial tensions at any point ona complex emulsion droplet. Higher shear rates can dramatically disrupt droplet stability, leading to the fragmentation of the complex emulsion and the release of the encapsulated phases—a phenomenon whose onset has been examined in detail by theoretical” and experimental studies.
The shape ofa typical hexane-perfluorohexane Janus droplet in water is illustrated in Extended Data Fig. 2a. The droplet's interfaces are spherical arcs with different radii of curvature (denoted by Ayı, Rp and Rrı). Furthermore, the line of contact between the two phases on the outer (water-facing) surface of the droplet isa circle whose diameter, D, is indicated by the dotted line in Extended Data Fig. 2a, b. The droplet in Extended Data Fig. 2a can be deconstructed into three spherical caps of the type shown in Extended Data Fig. 2b, each with the same base—a circle of diameter D—and respective radii Ry, Rr and Ry}. The volumes of these three spher- ical caps are denoted by Veapl Arş D), Veap(Rr» D) and Veap(Rr» D), respectively.
- 32. Haiying,L. & Zhongfan, L. A convenient synthesis of novel mercapto-ended azobenzene derivatives. Synth. Commun. 28, 3779-3785 (1998).
- 33. Hua, X. Y. & Rosen, M. J. Dynamic surface tension of aqueous surfactant solutions: 1. Basic parameters. J. Colloid Interface Sci. 124, 652-659 (1988).
- 34. Glantz, S. A. & Slinker, B. K. Primer of Applied Regression and Analysis of Variance 2nd edn, 248 (McGraw-Hill, 1990).
- 35. Harris, J. W. & Stocker, H. Handbook of Mathematics and Computational Science 107 (Springer, 1998).
- Canny, J. A computational approach to edge detection. IEEE Trans. Patt. Anal. Mach, Interf. PAMI-8, 679-698 (1986).
36.
From the definition of the volume ratio, k, between the hexane and perfluoro- hexane phases, it follows that:
- Taubin,G. Estimation of planar curves, surfaces, and nonplanar space curves defined by implicit equations with applications to edge and range image segmentation. [EEE Trans. Patt. Anal. Mach. Interf. 13, 1115-1138 (1991).
37.
Ve Veap(Re;D) — Veap(Ren-D) nS ee Vit ~ Veap(RiD)+ Veap(Ren,D) ©
- Fischer, P. & Erni, P. Emulsion drops in external flow fields — the role of liquid interfaces. Curr. Opin. Colloid Interface Sci. 12, 196-205 (2007).
38.
- 39. Stone, H.A.& Leal, L.G. Breakup of concentric double emulsion droplets in linear flows. J. Fluid Mech. 211, 123-156 (1990).
The volume of a spherical cap, V.aplr, 4), can be calculated using the principles of elementary three-dimensional geometry”. Knowledge of the values of k, Ry, Rp and
40.
- Muguet,V.etal W/O/W multiple emulsions submitted to a linear shear flow: correlation between fragmentation and release. J. Colloid Interface Sci. 218, 335-337 (1999).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
b
y(t) (mN m) o 50 100 150 200 t(s) 250 300 350 b 20 15 4 E z E > 10 | — 5, 400 0.0 0.2 0.4 0.6 08 1.0 0.1% Zonyl fos 0.1% SDS
Extended Data Figure 1 | Dynamic interfacial tension data was used to estimate the equilibrium interfacial tensions for the hexane-water and perfluorohexane-water interfaces. a, Dynamic interfacial tension data (in blue) was obtained from the pendant-drop method; the representative data shown here was measured for the hexane-water interface at fgps = 0.9 (such that the aqueous solution contained 0.1% SDS and 0.1% Zonyl in a 9:1 ratio). The data was fitted to an empirical model (in red) to estimate the
equilibrium value of the interfacial tension yp = (4 — ©). Such fitting was performed for all measured interfacial tensions and the fitted parameter results are tabulated in Extended Data Table 1. b, The estimated equilibrium interfacial tension values were used to plot the hexane-water (squares) and perfluorohexane-water (circles) interfacial tensions as a function of the fraction of 0.1% SDS, fons, where the other fraction is 0.1% Zonyl. See discussion in Methods for more details.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER


Extended Data Figure 2 | The geometry of a Janus droplet can be used to estimate the interfacial tension between hydrocarbon and fluorocarbon internal phases. a, Sketch of a Janus droplet consisting of hydrocarbon (grey) and fluorocarbon (white) phases within an aqueous outer phase. The radii of curvature of the H-W (Rj;), F-W (Re) and F-H (R>;) interfaces are related. to their respective interfacial tensions through the Young-Laplace equation. The diameter of the circle of contact between the two phases (dashed line) is denoted as D. b, The Janus droplet is composed of three spherical caps, and the volume, Va», of each constituent spherical cap is a function of the radius of curvature of the spherical surface and the base diameter D, Here we show the cap at the intersection of the hydrocarbon and fluorocarbon phases in
|
# O
|
which V.., is a function of Rey; and D. c, An exemplary image of a hexane- perfluorohexane Janus droplet obtained at fgps = 0.6, which was used to estimate Ry}; and, in turn, pp. d, The droplet pictured in c is subjected to edge detection to determine the H-W and F-W interfaces. e, The resulting edges are fitted to circles (red lines with green centres). The diameter of the circle of contact is then computed (green line). Given the ratio of the volumes of the two phases, we then determined the radius of curvature, Rrşı, of the hexane- perfluorohexane interface, which was subsequently used to estimate jp. See discussion in Methods for more details and Extended Data Table 2 for estimated values of j¢4.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
9 FRR PR FFF Yen 9 EFFEFEFEF a Ho e Ye ay PEF:FEFEE HzSO,, BHT FFF FF FFF fo)
b ANNE Reee 2882888 g 833 S6ocosnuas Yeyeyess Sue AZ Reger TITY Ww T T T a ö 5 x © 8 & 8 3 a 2 T T T T 7 T T + 7 + T + + T + — 60 55 50 45 40 35 30 25 20 15 10 05 00 T 80 7.5 70 65
# f1 (ppm)
Extended Data Figure 3 | Reaction scheme and 'H-NMR of the fluorinated crosslinker. a, Reaction scheme for the synthesis of the fluorinated crosslinker. BHT, 3,5-di-tert-butyl-4-hydroxytoluene. b, "H-NMR spectrum of the fluorinated crosslinker.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# a
N 3 338 | a
5 |

210 200 190 180 170 160 150 140 130 120 119, {00 oo 80 70 60 50 40 30 20 10 O -10 ppm
—ği nrc el £8 © 3.74 4.00-
65 -70 -75 -80 -85 -90 -95 -100 -105 -110 ais ap -125 -130 -135 -140 -145 -150 -155 -160 -165 ppm:
Extended Data Figure 4 | *C-NMR and '°F-NMR of the fluorinated crosslinker. a, '*C-NMR spectrum of the fluorinated crosslinker. b, '*F-NMR of the fluorinated crosslinker.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Extended Data Table 1 | Parameter values obtained by fitting an empirical model to interfacial tension measurements obtained using pendant-drop goniometry
0.1% SDS Hexane-water interface Perfluorohexane-water interface rai Ye, (MN m) t*(s) fb | Yaş, (MN m”) t*(s) R? 0.00 7.33 + 0.12 156 + 24 0.925 442 + 0.14 159 + 42 0.980 0.20 7.15 + 0.03 337 + 15 0.989 5.80 + 0.04 106+ 5 0.979 0.40 7.21 + 0.14 315 + 53 0.956 6.49 + 0.05 233 + 22 0.978 0.60 6.33 + 0.22 293 + 113 0.947 645+ 0.12 301 + 54 0.982 0.90 6.39 + 0.02 3521 0.992 7.66 + 0.03 57 +1 0.994 0.95 5.78 + 0.06 4141 0.993 7.73 + 0.06 85 + 2 0.991 1.00 14.04 + 0.15 2143 0.909 19.56 + 03 702 + 50 0.967
Dynamic interfacial tension data were fitted to a model (Methods) to estimate equilibrium interfacial tension values. The parameters obtained from the fitting are presented here. The larger the value of the characteristic time, t*, the slower the decay of the interfacial tension to its equilibrium value, 74. The value of R is a measure of how well the model fits the experimental data. For all the data used in this study, R? was above 0.9, signifying a good fit.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Extended Data Table 2 | Values of yr were estimated from geometrical analysis of images of Janus droplets and are independent of the composition of the surfactant added to the system, fsps
fogs = 0.4 1.06 Yeu (MN m”) 1.17 0.99
The values of yy presented here were obtained by computing the radii of curvature of the hexane-perfluorohexane, hexane-water and perfluorohexane-water interfaces, and subsequently using the Young- Laplace equation (Methods). Six droplets in total, three each under the conditions feos = 0.4 and feos = 0.6, were used to estimate a value of yr = 1.07 + 0.1 mNm
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# METHODS
Image availability. Amateur images. Available at the following databases: Associ- ation of Lunar and Planetary Observers-Japan, http://alpo-j.asahikawa-med.ac.jp/ Latest/Mars.htm (2012), and Société Astronomique de France (Commission sur- faces Planetaires), http://www.astrosurf.com/pellier/mars (2012). Hubble Space Telescope images. Available at NASA Planetary Data System (PDS) or at ESA HST archive, http://archives.esac.esa.int/hst/#Q-+R-S+D-1 (2012). MRO MARCI Weather Reports. Malin Space Science Systems. Captioned Image Release No, MSSS-216 — 21 March 2012 http://www.msss.com/msss_images/2012/ 03/21/ and Captioned Image Release No. MSSS-217 — 28 March 2012 http://www. msss.com/msss_images/2012/03/28/.
c, and c, being respectively the y and z coordinates of the cloud top, we have (e +2 } ia Ry +2z(B). In terms of 8 and the real cloud altitude H, we get
Di — (RH) cos, cosp @) cz =(Rv +H)sing,
The minimum height of the plume Hin to protrude from the limb (2(/5) = 0) is
Ru _ (1— cos? g,sin?p}"/? Arnin = Rui (3)
Real plume altitude derived from its projected length. When f < 0, by expression (2), 2(B) must satisfy
MOLA maps. Available at http://mola.gsfc.nasa.gov/images/topo_labeled.jpg. Image analysis and measurement. Most images were obtained with broadband filters covering the blue (B, 4, = 450 + 50 nm), green (G, 4 = 520 + 50 nm) and red (R, Ap > 625 * 75 nm) spectral bands. Each image was constructed by sum- ming, aligning and re-centring sequences of frames captured in video mode, and application of the ‘lucky imaging’ method” and the ‘wavelet’ as the main pro- cessing technigue”. The Airy disk fora telescope of 356 mm in diameter (Fig. 1a,b, d-g and photometric analysis) at a wavelength of 450 nm is 0.317 arcsec and the plate scale employed was 0.045 arcsec per pixel equivalent to an effective pixel size of 25 km on Mars, typically used in high resolution planetary images where record- ing features depends, for excellent seeing conditions, on resolution and features contrast”. The total spatial coverage of the projected plume is ~850 pixels (21 March, Fig. 1g). Image navigation, that is, the determination of the planetary limb and ter- minator was performed using two well-tested software packages (LAIA, WinJupos; ref. 33 and http://www.grischa-hahn.homepage.t-online.de/astro/index.htm). The bright Martian limb was used as a reference but not the terminator whose defini- tion is affected by the presence of the dark and bright features. Navigation was accu- rately controlled by measuring the position of well-known surface features (Olympus Mons and Tharsis volcanoes). The projected plume top altitude on the sky plane (defined as the border between the feature and the dark background, contrast forced) was simply determined from its distance to the Mars disk centre and subtraction of the measured Martian radius (all in pixel units). Navigation and image measure- ments were performed independently by four of us. From these multiple measure- ments, we derived the mean values of the Mars radius and top altitude for each image. The uncertainties represent the root mean square (r.m.s.) values of the altitude determinations. We estimate that the positioning error is +2 pixels and that, for the above plate-scale, this corresponds to +50 km. Uncertainties in plume mea- surements result from brightness diffusion at the top and sides of the plumes and from the determination of the base location of the plume relative to the navigated its
2(B) = (Ru +H) {cos? g, cos? p+sin?g.}'? —Ryy (4)
and the actual plume altitude is:
2(B)+R = — e —Ru (5) (1— cos? p,sin B}
If B > 0, the whole plume is visible (we see its projected total height). The cloud height H is now given by:
y=-——2)_ (1— cos? g, sin? 6)” (6)
Correction when Mars has a declination Dg as observed from Earth. By taking the declination angle + Dy when the north pole is tilted towards Earth and — Dy when the planet's south pole is visible, the transformation matrix due to a rotation around the y axis by an angle * Dp is
sek cosDe 0 sinDg | [x yim 0 1 0 y (7) zg —sinDy O cosD, | | z
Therefore, expressions (5) and (6) must be corrected taking into account that:
{ yay @) z= —xsinDg +z 0s Dy
In the rotated coordinate system, the projected plume altitude is 7'(6)— ti aa ©) vi — Ry, with 4 and c, being the transformed coordinates of the plume top, with the non-transformed coordinates of the plume top given by:
terminator, including
# brightness irregularities.
Asa further step in navigation control, we used the image series of 20 March by WJ. to compare the results of the top altitude measurements by two different pro- cedures: (1) through measurements on each image of the planet and plume top radius to calculate the plume maximum altitude; (2) through determination of the mean radius for the whole image series (and its uncertainty) and then using this mean value for each image to determine the top altitude. Both approaches yield consis- tent results within the uncertainties indicated in the text.
cy =(Ru +H) cos p, sin B = —(Rw +H) cos ¢, cos p (9)
Cz =(Ru + H)sing,
Expressing c, and c_ in terms of c,, c, and c, when the plume is behind the limb gives its real height
The measured top altitude values (H) and the terminator west longitudes (L, defined as the central meridian longitude for each observing time + 70°) for 20 and 21 March, fitted to a second order polynomial H = a, + a,L + aşl?, gave the peak values at Linax = —4)/2a, and then the corresponding Hjyax. The retrieved coefficients are, for 20 March, a, = —15,233.8, a, = 144.7, a, = —0.3395, and for 21 March, ay = —43,164.8, aş = 401.8, a, = —0.929.
2'(B)+Rm 7 {cos? p, cos? B + (sing, cos Dy — cos p, sin sinDg) } e —Ru (10)
H=
Owing to the planet's tilt, the plume will be on the limb not for 8 = 0 but for a different value fii. Therefore, for B = Prim, z’(f) = 0 if H = 0. Applying this con- dition to expression (10) and after some algebraic manipulations, we obtain:
Plume altitude model. Angular distance between the plume’s base and the central meridian. Our coordinate system is placed at the centre of Mars: the x axis will be on the observer's visual and the y-z plane is coincident with the sky plane. Coordinate system geometry is depicted in Extended Data Fig. 4, with the cloud rising near the west morning limb and the sub-Earth point on the equator. The longitude of the Central Meridian (CM) is 4cm, and the latitude and longitude of the plume are respectively gy. and 4... The longitudinal angular distance of the plume to the CM is related to the angular distance /} between the plume and the limb taking as a reference the negative y axis, by:
(11) Bim = — sin” tan(g,) x tan(De)
When De = 0 we recover yim, = 0. Note that at the equator /j,,, is always 0 regard- less the value of Dp. Expression (10) is valid for 4 < Ay, (the plume is behind the planet’s limb). When f > fi,,, the plume is before the limb and is not blocked by the planet and we have:
# H
i z'(B) (cos? p, cos? + (sing, cos De — cos p, sin sine)? } 2 (12)
90° —B —İcm (1)
Mars’ shadow projection on the plume. Given the coordinates of the subsolar point on Mars (A,, g,) and the plume coordinates (4.., ©.) we have both points separated by an angle:
where f is positive if the limb is behind the cloud, and negative if the cloud is located behind the limb.
y= cos”! [cos g, cos g, cos(A, — 4.) +sing,sing,] (13)
Limit altitude for plume visibility. Let z(B) be the visible projected plume altitude above Mars’ limb not blocked by the planet’s disk when 4 < 0. When the plume is behind the limb it protrudes from the planet’s limb only if its total projected length on the y-z plane from the planet centre is larger than the planet’s radius Ry. With
This expression is also valid when /, and 4, are the respective distances to the CM. In this case, 4. = (7/2) — B when we are close to the morning limb and:
y= cos '|cos 9, cos g,sin(/, +f) +sing, sing,] (14)
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Looking at Extended Data Fig. 5, the angle « planet’s shadow projected on the plume is: / — (8/2) and the height of the
(15)
For 4 < 0, using equations (5) and (15), the visible projected length of the planet's shadow projected on the protruding plume is:
2. R S(B) = cou (cos? yp, cos? B + sin’. } Ru (16) os
For 4 > 0 the planet's shadow projected on the plume is completely visible, since the planetary disk does not block it. The real height of the shadow will be Sç again, and according to equation (6) the projected shadow’s height will be:
0.514 + 0.02, and is 0.404 + 0.03 for the farthest point of the plume from the terminator. The phase angle (Sun-Mars-Earth) was 35.6°. Correspondingly, for the 2012 event, the prescribed azimuth angle and the cosine of the polar angle are 0° and 0.014 + 0.03, and the phase angle (Sun-Mars-Earth) was 7.15”. Solar photons enter the atmosphere following grazing trajectories, and penetrate half way through the limb before being scattered towards the observer at the terminator. To compare with the box-averaged I/Fs from observations, the model-calculated I/Fs were also averaged over the cloud geometrical thickness D from the observer's vantage point. A number of photon realizations between 10” and 10° ensured accuracies typically better than 5%, which is much less than the measured uncertainties. The atmo- spheric inputs to the model include the cloud particles’ optical properties and the cloud (vertical) optical thickness ty, in addition to the geometrical thickness of the cloud D.
S(B) =Sc{cos* 9, cos” B-4siny,)'? (17) 1 2 2 ia =R ~1| {cos? 9, cos? B+sin29, MM [core | {co 9. cos B+sin’e, }
In the prescription of particle sizes, we used a power law distribution described by the two moments reg (effective radius) and Ver (effective variance)'*. Particle candidates are CO,-ice, HzO-ice and dust, and we considered their wavelength- dependent refractive indices'*"’,
Again, if the planet’s rotation axis is tilted by + Dg degrees, we obtain
R 2 21 S(B) = cou (cos? p, cos? B + (sing. cos Dy — cos p, sinf sinDe)? } Ri Ry (18)
for B < Bims and
S(B) =Sc{ cos’ p, cos? B+ (sing, cos Dy — cos g, sin sinDp)” } ye (19)
for B= Prim.
The model predictions are shown in Fig, 2. A 7? calculation between the mea- sured top altitudes considering the individual error and the predictions from the geometric model for a uniform plume shows that for 20 March 7? <4 for top alti- tudes 190-195 km (best fit), but for 21 March 7°(min) ~ 20 for top altitudes 240- 250 km. The large deviations between the model and the observations for 21 March data rule out a uniform structure for the plume.
Altitude of the 1997 HST feature. The 17 May 1997 event was captured by HST ona single image at each specific wavelength, and thus they do not show the plume in rotation at the limb/terminator. Therefore we can only constrain the maximum and minimum altitudes for the feature as seen projected on the terminator. The feature protrudes on the terminator by 10° (angle in Extended Data Fig. 4). For solar zenith angle SZA = 90” (grazing illumination) the minimum altitude is given by equation (15): Hinin = Sc = 53 km. On the other hand, the maximum altitude Hynax Will occur for a vertical feature whose projection L as seen on the terminator from Earth (phase angle ¢ = 35.5°) is given by L= \/ (Rw +S¢)° — Ru? and then Hinax = Ltanp = 480 km.
Photometric calibration. (1) Ground-based observations (20-21 March 2012). Stacked and aligned unprocessed images were used for reflectivity measurements. Because of their higher resolution, only the series B, G, R filtered images from W.J. and D. Parker were used for that purpose. The photometry from the brightest part of the cloud was compared to different bright and dark regions of Mars whose abso- lute (I/F) reflectivity is known”, The resulting I/F can be seen as an average for the cloud at B, Gand R, and from independent measurements by the team members of the same images, we estimate an uncertainty of 20%.
(2) HST calibration (17 May 1997). Hubble Space Telescope observations acquired in filters from the ultraviolet (F255W) to the near-infrared (F1042M) (Extended Data Fig. 5) were photometrically calibrated following the WFPC2 handbook instructions. Radiances were converted into absolute reflectivity I/F using the solar spectrum“, The resulting /Fas a function of planetary geographical coordinates was confirmed against values given by other authors for selected locations of the planet™ and with global albedo values of the planet”. The intensity calibration method for ground-based images was validated with a similar procedure to that used in HST images and comparison with the above procedure.
Radiative transfer model. We conducted radiative transfer calculations of the reflectivity I/F with a backward Monte Carlo model designed for spherical-shell atmospheres**. The model produces as output the full Stokes vector of radiances, but only the first vector element (intensity) was considered for comparison with the observations. The model naturally accounts for curvature effects in the (nearly) grazing illumination/viewing geometries of both the 1997 and 2012 events. In its specific implementation here, the model assumes a uniform cloud of geometrical thickness D that enshrouds the entire planet. The observer looks upon the planet terminator with a line of sight that sequentially probes the full range of altitudes where the cloud is present. For the 1997 event, plane çı, (formed by the local ver- tical direction at the planet’s terminator and the Sun illumination direction), and plane 7,45 (formed by the same local vertical direction and the observer viewing direction) are offset by an azimuth angle of 15°. The cosine of the polar angle between the observer's viewing direction and the local vertical at the terminator is
The particles’ optical properties required in the radiative transfer calculations are , p(0) and o, which stand for the particles’ single scattering albedo, scattering phase function and extinction cross-section, respectively. Each of these properties is specified at the appropriate radiation wavelength 2. For the determination of the optical properties we used Mie theory, which applies to scattering by spherical par- ticles. Ifthe particles are non-spherical, Mie theory may bias the properties of inferred particles that fit best the measured //F reflectivities. The differences in optical properties between spherical and non-spherical particles depend on the particles’ shape and size distribution, as well as on their composition and on the radiation wavelength. Numerical investigations have explored these differences over a limited range of parameter space'*, Without additional information on the cloud particles, it is difficult to assess the consistency in the treatment with Mie theory of the radi- ative transfer problem. We can nevertheless tentatively quantify it by comparison against specific solutions from the literature. For req = 0.1 um and 2 = 0.55 pm, the effective size parameter Xep = 27req/A ~ 1. Focusing on the scattering phase function, which is particularly sensitive to the particles’ shape, a few studies'* sug- gest that for x. 1 the differences between spherical and non-spherical particles are not usually more than a few tens of per cent. These differences are less than for larger values of Xejr.
Overall, each radiative transfer calculation requires user-inputted values for /, particle composition (and therefore wavelength-dependent refractive indices), reg, Vege and ty (also dependent on wavelength). The problem of comparing reflectiv- ities I/F from model calculations and observations is therefore multi-parametric. In the exploration of the space of parameters, we built a grid of solutions to the radi- ative transfer problem for D (10, 50, 100, 200 km), reg-(0.01 to 2 jum; 20 values with varying steps between 0.01 and 0.1 pm), ver (0.1, 0.2, 0.5, 1 and 2; range consistent with values adopted in prior Mars investigations''***) and Tp so2nm (XT, at 502 nm; from 16 to 10“; 31 values in steps following a power law). The total number of radiative transfer calculations for each of the 1997 and 2012 events exceeds 10°.
For each possible spectrum within the grid of calculations with identical particle composition, rej Ver AN Tp,s02nm> We Calculated the average quadratic deviation of the model from the observed data weighted by the observation errors (77). A subset of the free parameter space with best-fitting values is shown in Extended Data Fig, 6 for the 1997 event and Extended Data Fig. 7 for the 2012 event. In the former case, dust particles do not fit the data with 7? < 1.0. Whilea number of sub-optimal solu- tions for different combinations of the free parameters can be found, the highest density of low 7? solutions can be interpreted as the most likely combination of the free parameters. This happens for particles with effective radii of the order of 0.1 ım, depending on the particles’ effective variance and composition. In this framework, there is no way to discriminate between the variances and this also impedes making a choice between HO and CO, as the most likely particle type. Similarly, the 7’ results for the 2012 observations show a broad range of particle sizes and optical thicknesses that provide 7? < 1.0. They include dust particles that, in any case, result in fits that are worse than for the HO and CO, composi- tions. Extended Data Fig. 8 shows how, given the narrow spectral range covered during the 2012 event, it is possible to retrieve acceptable fits for particle sizes rang- ing from 0.1 to 1.0 jm, provided that the rest of the free parameters are adequately tuned. Observations at shorter/longer wavelengths might potentially break this degeneracy.
All in all, the radiative transfer calculations for both the 1997 and 2012 events constrain to some extent the effective particle radius r,,, and the vertical extension of the cloud D, and give a lower limit for the normal optical thickness Ta 502nm- Unfor- tunately, additional data are required to constrain other parameters, such as Vçgror the cloud particle composition. The optical thickness T,, 502nm retrieved from the model fit translates into particle density after dividing the optical thickness by the particle cross-section at 502 nm and the cloud geometrical thickness.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Code availability. All radiative transport calculations were conducted with a novel pre-conditioned backward Monte Carlo (PBMC) model**. A plane-parallel version of the model is freely available through: http://dx.doi.org/10.1051/0004- 6361/201424042. This version contains instructions to run the model, test cases and model outputs that can be compared to calculations tabulated in ref. 38. In the analysis of the Mars plume, we used a version of the PBMC suitable for spherical shell atmospheres. The spherical-shell model is available on request from A.G.M. (tonhingm@gmail.com).
molecular weight of CO). For the retrieved cloud particle density of 0.01 cm” *, cloud evaporation will produce ~ 10° particles per cm’, On the other hand, from the above atmospheric conditions, and vmr CO, = 0.09, the number density of molecules in the atmosphere is N(CO> atm.) = (peo, X Navo/ 1) x vmrco, ~10” particlesem —. Therefore, CO, ice evaporation will produce a 5% enrichment of CO).
- 29. Law, N. M., Mackay, C. D. & Baldwin, J. E, Lucky imaging: high angular resolution imaging in the visible from the ground. Astron. Astrophys. 446, 739-745 (2006).
Mars General Circulation Model. As supplied in the Mars Climate Database at http://www-mars.İmd.jussieu.fr/.
Cloud evaporation. H,O-cloud. Taking water-ice density as 0.93 g cm” * (ref. 40), the mass of a spherical particle with radius 0.1 um is m = 4 X 10” "İ g; the corres- ponding number of molecules resulting from evaporation of this particle is mN vo! 410, taking Nayo = 6.023 X 10” particles per mol and the molecular weight of water is > 18 gmol” '. For the retrieved cloud density 0.01 cm” ?, evaporation of the cloud will produce ~10° particles per cm’, On the other hand, the atmospheric density is p = 5 X 10 '° gem? atanaltitude of 200 km, at latitude —40°, L, = 90°, 210° W longitude and local time 6h (ref. 18). The mean molecular weight of the atmosphere at this altitude is now ji = 17 gmol ' because of oxygen dominance with a volume mixing ratio vmr = 0.6 and a 13% contribution from N2 and CO. The vmr of H,0 at this altitude is 4 X 10°. The number density of molecules in the atmosphere is N(H2O,atm.) = (pj3,0 * Navo/t) vmry,0~ 10*particlesem ~*. Therefore, evaporation of the water ice will produce an enrichment of water by a factor ~1,000 over normal conditions.
- 30. Berrevoets, C. et al. RegiStax6. http://www.astronomie.be/registax/ (2011).
- 31. Kraaikamp, E. AustoStakkert!2. http://www.autostakkert.com/ (2012).
- 32. Peach, D. A. The 2010 perihelic opposition of Jupiter observed from Barbados. J. Br. Astron. Assoc. 121, 337-340 (2011).
- 33. Cano, J. A. LAIA: Laboratorio de Anâlisis de Imagenes Astronömicas (Grup d'Estudis Astronomics, Barcelona, 1998).
- 34, Bell, J. F. Ill etal Mars surface mineralogy from Hubble Space Telescope imaging during 1994-1995: observations, calibration, and initial results. J. Geophys. Res. 102, 9109-9123 (1997).
- 35. de Pater, |, et al, A multi-wavelength study of the 2009 impact on Jupiter: comparison of high-resolution images from Gemini, Keck and HST. Icarus 210, 722-741 (2010).
- 36. Colina, L., Bohlin, R.C. & Castelli, F. The 0.12-2.5 micron absolute flux distribution of the Sun for comparison with solar analog stars. Astron. J. 112, 307-315 (1996).
- 37. Mallama, A. The magnitude and albedo of Mars. Icarus 192, 404-416 (2007).
- 38. Garcia Mufioz, A. & Mills, F. P. Pre-conditioned backward Monte Carlo solutions radiative transport in planetary atmospheres. Fundamentals: sampling of propagation directions in polarising media. Astron. Astrophys. 573, A72 (2015).
CO;-cloud, Taking COy-ice density as 1.5 gem”, the mass of a spherical particle with radius 0.1 jum will be m = 6 X 107 '° gand the corresponding number of molec- ules resulting from evaporation is #1NAyo//4' ~ 10° where 4” = 44 gmol ' is the
- 39. Santer, R., Deschamps, M., Ksanfomaliti, L. V. & Dollfus, A. Photopolarimetry of Martian aerosols. II, Limb and terminator measurements. Astron. Astrophys. 158, 247-258 (1986).
- 40. de Pater, |. & Lissauer, J. J. Planetary Sciences 2nd edn (Cambridge Univ. Press, 2010).
# ©2015 Macmillan Publishers Limited. All rights reserved
# to
LETTER

Extended Data Figure 1 | Images of the 2012 plume event (ringed) on 12-20 March. Dates in March (and authors) are as follows: a, 12 (M.D.); b, 15 (D. Peach); c-f, 20, plume in rotation (W.J.). Time indicated at top left each panel is in UTC.
# of
# ©2015 Macmillan Publishers Limited. All rights reserved
b,
LETTER

Extended Data Figure 2 | Images of the 2012 plume events (ringed) on 22 March and 13 April. a, First event on 22 March, 04:12 UTC (image by W.J); second event on 13 April, 20:03 UTC (image by D. Peach).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER

Extended Data Figure 3 | Images of the 2012 plume event (ringed) at different wavelengths on 21 March. a-c, Images by J.P., d-£, images by D. Parker, with filters indicated: a, d, B (blue); b, e, G (green); c, f, R (red). Time indicated is in UTC.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Earth To the X observer
Extended Data Figure 4 | Martian viewing geometry. a, Angle definitions with the simulated protrusion of altitude H located at point c and out of the illuminated part of the disk near the limb. b, Top view, taking as a reference the planet's terminator and definition of y and x angles when the cloud is on the equator (but the latitude of the subsolar point is not zero). Green arrows represent the projected cloud altitude as seen from Earth in the extreme
situations when the cloud is on the terminator and follows the grazing sunlight, and when it follows the planet’s radius. To simplify the figure, and without loss of generality, the sub-Earth point is placed on the arc linking the subsolar point and the cloud base. c, General side view of the geometry of the planet's projected shadow.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER


Identification HST number Filter Wavelength (nm) | FWHM (nm) a u3gi7701m.fits F255W 257.3 42.6 b u3gi7703m.fits F410M 408.8 18.2 c u3gi7704m. fits F502N 5012.2 3.6 e u3gi7705m.fits F673N 673.2 6.3 f u3gi7707m.fits | FI042M 1045.3 89.7 d u3gi7708m.fits F588N 589.3 6.4
Extended Data Figure 5 | Hubble Space Telescope images of the event on May 17 1997. Wavelengths and times in UTC were: a, 255 nm (17:27); b, 410 nm (17:35); c, 502 nm (17:38); d, 588 nm (17:50); e, 673 nm (17:41);
_f, 1,042 nm (17:47); g, Colour composite. Plume ringed in a-f, arrowed in g. Table at bottom identifies each image and its HST number, and also shows filters used, giving their central wavelength and bandwidth (FWHM).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
0.08 a a a 0.07} 0.06} & 0.05} 0.04} 0.03; N , N 2 03 04 05 06 07 08 09 1 1 12 Wavelength (um) 0.08 Z d >
Extended Data Figure 6 | Radiative transfer model fit for the 2012 event. This is an example of the degeneracy of the model solution due to the narrow wavelength range covered in the 2012 event. Model fit as follows: solid lines, CO>; dot-dashed lines, H,O particles; blue, data for an effective radius
_
Teg = 0.1 jum; red, data for rag — 1.0 um; stars, wavelengths used in the calculations. As in Fig. 3, open black triangles show the observed reflectivity of the 2012 cloud. The error bars represent the average quadratic deviation of the measured reflectivity in the integration box.
# ©2015 Macmillan Publishers Limited. All rights reserved
# LETTER
a CO2-V .4=0.50 1 10 08 06 5 10 04 02 10 o 5 19 t N (502nm) d DST-V .4=0.50 1 08 06 04 0.2 0 5 19 15 ty (502nm| H20 -V 4=0.50 15 5 1 8 t, (502nm)
b CO2-V 1.00 1 08 06 04 0.2 o 5 1 ty (S02nm)
c CO2-v 4=2.00 1 08 06 04 02 0 1 08 06 04 02 o 1 08 06 04 0.2 o 15 5 18 t,, (502nm)
Extended Data Figure 7 | Assessment of the radiative transfer model fit for the 1997 event. a-i, Colours show values of 7? (for measured I/F versus model calculation, colour scale at right) for the effective radius (reg in ım) versus optical depth (Tw at 502 nm), and for different particle types and values of the indicated particle variance (vg shown in parentheses) as follows. a, CO, (0.5);
# o
(|
b, CO, (1.0); c, CO, (2.0); d, DST (dust, 0.5); e, DST (dust, 1.0); f, DST (dust, 2.0); g, HO (0.5); h, HO (1.0); and i, HO (2.0). The calculations are for a vertical extension of the cloud with D = 100 km, and they provide the best-fitting values of the whole free parameter space.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
e e e» e e = NN B ö & e e e e e = NN 2 ö ö& e e » e e = NN 5 6 & 15 5 © t,, (502nm) c 10° ş ” “1 10 107 dt: 10° 5 1 i 10” i 10° ii i 107
a CO2-v , 0.50 10 10 -2 10 "eff 5 u a 3 ri e e e» e e = NN B ö & 5 ty (502nm) d DST V 0.50 5 b eff 8 3 3 b 0 ° wa z ı 5 e e e e e = NN 2 ö ö& t,, (502nm) H20—v _.=0.50 r ise) 5 5 8 a 2 Gi ni e e » e e = NN 5 6 & 15 ty (650201)
CO2-v ,=2.00
19 a (502nm, 47200 DsT-v
# v5 (so2nm
H20—v 2.00
15
# ie
# hy (so2nnd
%
Extended Data Figure 8 | Assessment of the radiative transfer model fit for the 2012 event. As Extended Data Fig. 7 in terms of variables plotted, particle types and particle variance, but for the 2012 event.
# ©2015 Macmillan Publishers Limited. All rights reserved
1
08
06
04
02
# o
# 1 ai
06
04
02
# o
1
08
06
04
02
# o
LETTER
# METHODS
Preprocessing. Working directly with raw Atari 2600 frames, which are 210 X 160 pixel images with a 128-colour palette, can be demanding in terms of computation and memory requirements. We apply a basic preprocessing step aimed at reducing the input dimensionality and dealing with some artefacts of the Atari 2600 emu- lator. First, to encode a single frame we take the maximum value for each pixel colour value over the frame being encoded and the previous frame. This was necessary to remove flickering that is present in games where some objects appear only in even frames while other objects appear only in odd frames, an artefact caused by the limited number of sprites Atari 2600 can display at once. Second, we then extract the Y channel, also known as luminance, from the RGB frame and rescale it to 84 X 84. The function ¢ from algorithm 1 described below applies this preprocess- ing to the m most recent frames and stacks them to produce the input to the Q-function, in which m = 4, although the algorithm is robust to different values of m (for example, 3 or 5).
Code availability. The source code can be accessed at https://sites.google.com/a/ deepmind.com/dgn for non-commercial uses only.
Our experimental setup amounts to using the following minimal prior know- ledge: that the input data consisted of visual images (motivating our use of a con- volutional deep network), the game-specific score (with no modification), number of actions, although not their correspondences (for example, specification of the up button”) and the life count.
Evaluation procedure. The trained agents were evaluated by playing each game 30 times for up to 5 min each time with different initial random conditions (‘no- op’; see Extended Data Table 1) and an g-greedy policy with & = 0.05. This pro- cedure is adopted to minimize the possibility of overfitting during evaluation. The random agent served as a baseline comparison and chose a random action at 10 Hz which is every sixth frame, repeating its last action on intervening frames. 10 Hz is about the fastest that a human player can select the ‘fire’ button, and setting the random agent to this frequency avoids spurious baseline scores in a handful of the games. We did also assess the performance ofa random agent that selected an action at 60 Hz (that is, every frame). This had a minimal effect: changing the normalized DQN performance by more than 5% in only six games (Boxing, Breakout, Crazy Climber, Demon Attack, Krull and Robotank), and in all these games DQN out- performed the expert human by a considerable margin.
Model architecture. There are several possible ways of parameterizing Q using a neural network. Because Q maps history-action pairs to scalar estimates of their Q-value, the history and the action have been used as inputs to the neural network by some previous approaches?"“, The main drawback of this type of architecture is that a separate forward pass is required to compute the Q-value of each action, resulting in a cost that scales linearly with the number of actions. We instead use an architecture in which there is a separate output unit for each possible action, and only the state representation is an input to the neural network. The outputs cor- respond to the predicted Q-values of the individual actions for the input state. The main advantage of this type of architecture is the ability to compute Q-values for all possible actions ina given state with only a single forward pass through the network.
The exact architecture, shown schematically in Fig. 1, is as follows. The input to the neural network consists of an 84 X 84 X 4 image produced by the preprocess- ing map p. The first hidden layer convolves 32 filters of 8 X 8 with stride 4 with the input image and applies a rectifier nonlinearity” >. The second hidden layer con- volves 64 filters of 4 X 4 with stride 2, again followed by a rectifier nonlinearity. This is followed by a third convolutional layer that convolves 64 filters of 3 X 3 with stride 1 followed by a rectifier. The final hidden layer is fully-connected and con- sists of 512 rectifier units. The output layer is a fully-connected linear layer with a single output for each valid action. The number of valid actions varied between 4 and 18 on the games we considered.
Training details. We performed experiments on 49 Atari 2600 games where results were available for all other comparable methods'*”’, A different network was trained on each game: the same network architecture, learning algorithm and hyperpara- meter settings (see Extended Data Table 1) were used across all games, showing that our approach is robust enough to work on a variety of games while incorporating only minimal prior knowledge (see below). While we evaluated our agents on unmodi- fied games, we made one change to the reward structure of the games during training only. As the scale of scores varies greatly from game to game, we clipped all posi- tive rewards at 1 and all negative rewards at —1, leaving 0 rewards unchanged. Clipping the rewards in this manner limits the scale of the error derivatives and makes it easier to use the same learning rate across multiple games. At the same time, it could affect the performance of our agent since it cannot differentiate between rewards of different magnitude. For games where there is a life counter, the Atari 2600 emulator also sends the number of lives left in the game, which is then used to mark the end of an episode during training.
In these experiments, we used the RMSProp (see http://www.cs.toronto.edu/ ~tijmen/csc321/slides/lecture_slides_lec6.pdf) algorithm with minibatches of size 32. The behaviour policy during training was ¢-greedy with ¢ annealed linearly from 1.0 to 0.1 over the first million frames, and fixed at 0.1 thereafter. We trained fora total of 50 million frames (that is, around 38 days of game experience in total) and used a replay memory of 1 million most recent frames.
Following previous approaches to playing Atari 2600 games, we also use a simple frame-skipping technigue'*, More precisely, the agent sees and selects actions on every kth frame instead of every frame, and its last action is repeated on skipped frames. Because running the emulator forward for one step requires much less computation than having the agent select an action, this technique allows the agent to play roughly k times more games without significantly increasing the runtime. We use k = 4 for all games.
The values of all the hyperparameters and optimization parameters were selected by performing an informal search on the games Pong, Breakout, Seaquest, Space Invaders and Beam Rider. We did not perform a systematic grid search owing to the high computational cost. These parameters were then held fixed across all other games. The values and descriptions of all hyperparameters are provided in Extended Data Table 1.
The professional human tester used the same emulator engine as the agents, and played under controlled conditions. The human tester was not allowed to pause, save or reload games. As in the original Atari 2600 environment, the emulator was run at 60 Hz and the audio output was disabled: as such, the sensory input was equated between human player and agents. The human performance is the average reward achieved from around 20 episodes of each game lasting a maximum of 5 min each, following around 2h of practice playing each game.
Algorithm. We consider tasks in which an agent interacts with an environment, in this case the Atari emulator, in a sequence of actions, observations and rewards. At each time-step the agent selects an action a, from the set of legal game actions, A= {1,...,K}. The action is passed to the emulator and modifies its internal state and the game score. In general the environment may be stochastic. The emulator’s internal state is not observed by the agent; instead the agent observes an image x,elR* from the emulator, which is a vector of pixel values representing the current screen. In addition it receives a reward r, representing the change in game score. Note that in general the game score may depend on the whole previous sequence of actions and observations; feedback about an action may only be received after many thousands of time-steps have elapsed.
Because the agent only observes the current screen, the task is partially observed and many emulator states are perceptually aliased (that is, it is impossible to fully understand the current situation from only the current screen x;). Therefore, sequences of actions and observations, s;= x; ,@).X2,..-,4;—1,X+, are input to the algorithm, which then learns game strategies depending upon these sequences. All sequences in the emulator are assumed to terminate in a finite number of time- steps. This formalism gives rise toa large but finite Markov decision process (MDP) in which each sequence is a distinct state. As a result, we can apply standard rein- forcement learning methods for MDPs, simply by using the complete sequence s; as the state representation at time f.
The goal of the agent is to interact with the emulator by selecting actions in a way that maximizes future rewards. We make the standard assumption that future rewards are discounted by a factor of y per time-step ( was set to 0.99 throughout), and
define the future discounted return at time tas R, = yf “re, in which T is the
time-step at which the game terminates. We define the optimal action-value function Q*(s,a) as the maximum expected return achievable by following any policy, after seeing some sequence s and then taking some action a, Q"(s,a)= max, BR;ls, —s,a, = a,7] in which z is a policy mapping sequences to actions (or distributions over actions).
The optimal action-value function obeys an important identity known as the Bellman equation. This is based on the following intuition: if the optimal value O'(s',a') of the sequence s' at the next time-step was known for all possible actions a’, then the optimal strategy is to select the action a’ maximizing the expected value of r+yQ"(s,a'):
Olsa) =Ky [r+ ymax Q*(s',a’)\s,a
The basic idea behind many reinforcement learning algorithms is to estimate the action-value function by using the Bellman equation as an iterative update, Q;,;(s,a)=Ey[r+y max Q;(s’,a’)|s,a). Such value iteration algorithms converge to the optimal action-value function, O, » O* as i «0. In practice, this basic approach is impractical, because the action-value function is estimated separately for each sequence, without any generalization. Instead, it is common to use a function approx- imator to estimate the action-value function, Q(s,a; 0) -O' (s,a). In the reinforce- ment learning community this is typically a linear function approximator, but
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
sometimes a nonlinear function approximator is used instead, such as a neural network. We refer to a neural network function approximator with weights 0 as a Q-network. A Q-network can be trained by adjusting the parameters 0; at iteration i to reduce the mean-squared error in the Bellman equation, where the optimal target values r --7 max, Q*(s',a’) are substituted with approximate target values y=rt+y max, O(s',a'; 07 ), using parameters 0; from some previous iteration. This leads to a sequence of loss functions L,(0;) that changes at each iteration i,
L4(0)) Esa, (Es [yls,a] — Q(s,a3 0)))”] Ear (9 Ossa; 0:))7] + EsarlVe yi.
Note that the targets depend on the network weights; this is in contrast with the targets used for supervised learning, which are fixed before learning begins. At each stage of optimization, we hold the parameters from the previous iteration 0; fixed when optimizing the ith loss function Z,(0)), resulting in a sequence of well- defined optimization problems. The final term is the variance of the targets, which does not depend on the parameters 0; that we are currently optimizing, and may therefore be ignored. Differentiating the loss function with respect to the weights we arrive at the following gradient:
Vo,L(0;) =Esars e max O(a‘; 0; ) — Olsa; 0) Vo, O(s,a; 0; | N
replay the behaviour distribution is averaged over many of its previous states, smoothing out learning and avoiding oscillations or divergence in the parameters. Note that when learning by experience replay, it is necessary to learn off-policy (because our current parameters are different to those used to generate the sam- ple), which motivates the choice of Q-learning.
In practice, our algorithm only stores the last N experience tuples in the replay memory, and samples uniformly at random from D when performing updates. This approach is in some respects limited because the memory buffer does not differ- entiate important transitions and always overwrites with recent transitions owing to the finite memory size N. Similarly, the uniform sampling gives equal impor- tance to all transitions in the replay memory. A more sophisticated sampling strat- egy might emphasize transitions from which we can learn the most, similar to prioritized sweeping”.
The second modification to online Q-learning aimed at further improving the stability of our method with neural networks is to use a separate network for gen- erating the targets y, in the Q-learning update. More precisely, every C updates we clone the network O to obtain a target network Q and use Q for generating the Q-learning targets y; for the following C updates to Q. This modification makes the algorithm more stable compared to standard online Q-learning, where an update that increases Q(s,,a,) often also increases Q(s; ,,4) for all a and hence also increases the target y,, possibly leading to oscillations or divergence of the policy. Generating the targets using an older set of parameters adds a delay between the time an update to O is made and the time the update affects the targets y, making divergence or oscillations much more unlikely.
Rather than computing the full expectations in the above gradient, it is often computationally expedient to optimize the loss function by stochastic gradient descent. The familiar Q-learning algorithm’? can be recovered in this framework by updating the weights after every time step, replacing the expectations using single samples, and setting 0; =0;_).
Note that this algorithm is model-free: it solves the reinforcement learning task directly using samples from the emulator, without explicitly estimating the reward and transition dynamics P(r,s’|s,a). It is also off-policy: it learns about the greedy policy a=argmax,, O(s,a'; 0), while following a behaviour distribution that ensures adequate exploration of the state space. In practice, the behaviour distribution is often selected by an «-greedy policy that follows the greedy policy with probability 1 — and selects a random action with probability &.
We also found it helpful to clip the error term from the update r+y max, O (s.a; 0; ) — Q(s,a; 0;) to be between —1 and 1, Because the absolute value loss function |x| has a derivative of —1 for all negative values of x and a derivative of 1 for all positive values of x, clipping the squared error to be between — 1 and 1 cor- responds to using an absolute value loss function for errors outside of the (—1,1) interval. This form of error clipping further improved the stability of the algorithm. Algorithm 1: deep Q-learning with experience replay.
Initialize replay memory D to capacity
# N
Initialize action-value function Q with random weights 0
Initialize target action-value function Q with weights 0” = 0
Training algorithm for deep Q-networks. The full algorithm for training deep Q-networks is presented in Algorithm 1. The agent selects and executes actions according to an ¢-greedy policy based on O. Because using histories of arbitrary length as inputs to a neural network can be difficult, our Q-function instead works ona fixed length representation of histories produced by the function ¢ described above. The algorithm modifies standard online Q-learning in two ways to make it suitable for training large neural networks without diverging.
First, we use a technique known as experience replay” in which we store the agent's experiences at each time-step, €, = (5, 4p Tp S¢ + 1), ina data set D, = {e),....e4, pooled over many episodes (where the end of an episode occurs when a termi- nal state is reached) into a replay memory. During the inner loop of the algorithm, we apply Q-learning updates, or minibatch updates, to samples of experience, (s,a, r,s’) ~ U(D), drawn at random from the pool of stored samples. This approach has several advantages over standard online Q-learning. First, each step of experience is potentially used in many weight updates, which allows for greater data efficiency. Second, learning directly from consecutive samples is inefficient, owing to the strong correlations between the samples; randomizing the samples breaks these correla- tions and therefore reduces the variance of the updates. Third, when learning on- policy the current parameters determine the next data sample that the parameters aretrained on. For example, if the maximizing action is to move left then the train- ing samples will be dominated by samples from the left-hand side; if the maximiz- ing action then switches to the right then the training distribution will also switch. It is easy to see how unwanted feedback loops may arise and the parameters could get stuck ina poor local minimum, or even diverge catastrophically”, By using experience
For episode = 1, M do
Initialize sequence $, = {x, } and preprocessed sequence #, = (sı ) For t= 1,T do
With probability e select a random action a,
otherwise select a; =argmax, O(¢(s;),a; 0)
Execute action a, in emulator and observe reward r, and image x; Set 544 1 =S1.4¢.X¢41 and preprocess ¢, , , =@(Sp41) Store transition ($,.4;,7:.9,4.) in D
+
Sample random minibatch of transitions (Bayri 1) from D
4 a= { 1/47 max 04,050“) Perform a gradient descent step on 6 — (4.45 0) ) with respect to the network parameters 0 Ö- if episode terminates at step j+1 otherwise
Every C steps reset Ö- O
# End For
# End For
- 31. Jarrett,K. Kavukcuoglu,K.,Ranzato,M.A.&LeCun,Y.Whatisthe best multi-stage architecture for object recognition? Proc. IEEE. Int. Conf. Comput. Vis. 2146-2153 (2009).
- 32. Nair, V. & Hinton, G. E. Rectified linear units improve restricted Boltzmann machines. Proc. Int. Conf. Mach. Learn. 807-814 (2010).
- 33. Kaelbling, L. P., Littman, M. L. & Cassandra, A. R. Planning and acting in partially observable stochastic domains. Artificial Intelligence 101, 99-134 (1994).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER

Extended Data Figure 1 | Two-dimensional t-SNE embedding of the representations in the last hidden layer assigned by DQN to game states experienced during a combination of human and agent play in Space Invaders. The plot was generated by running the t-SNE algorithm” on the last hidden layer representation assigned by DQN to game states experienced during a combination of human (30 min) and agent (2 h) play. The fact that there is similar structure in the two-dimensional embeddings corresponding to the DQN representation of states experienced during human play (orange
points) and DQN play (blue points) suggests that the representations learned by DQN do indeed generalize to data generated from policies other than its own. The presence in the t-SNE embedding of overlapping clusters of points corresponding to the network representation of states experienced during human and agent play shows that the DQN agent also follows sequences of states similar to those found in human play. Screenshots corresponding to selected states are shown (human: orange border; DQN: blue border).
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Value (V) 8 20 25 30 35 4 45 50 55 60 65 70 75 80 8 90 95 100 105 110 115 120 Frame # -0.5 Action-Values (0) -1 Öz sors Öz sees Öz eu Öz er ya
# b
Extended Data Figure 2 | Visualization of learned value functions on two games, Breakout and Pong. a, A visualization of the learned value function on the game Breakout. At time points 1 and 2, the state value is predicted to be ~17 and the agent is clearing the bricks at the lowest level. Each of the peaks in the value function curve corresponds to a reward obtained by clearing a brick. Attime point 3, the agent is about to break through to the top level of bricks and the value increases to ~21 in anticipation of breaking out and clearing a large set of bricks. At point 4, the value is above 23 and the agent has broken through. After this point, the ball will bounce at the upper part of the bricks clearing many of them by itself. b, A visualization of the learned action-value function on the game Pong. At time point 1, the ball is moving towards the paddle controlled by the agent on the right side of the screen and the values of
all actions are around 0.7, reflecting the expected value of this state based on previous experience. At time point 2, the agent starts moving the paddle towards the ball and the value of the ‘up’ action stays high while the value of the ‘down’ action falls to — 0.9, This reflects the fact that pressing ‘down’ would lead to the agent losing the ball and incurring a reward of —1. At time point 3, the agent hits the ball by pressing ‘up’ and the expected reward keeps increasing until time point 4, when the ball reaches the left edge of the screen and the value of all actions reflects that the agent is about to receive a reward of 1. Note, the dashed line shows the past trajectory of the ball purely for illustrative purposes (that is, not shown during the game). With permission from Atari Interactive, Inc.
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
# Extended Data Table 1 | List of hyperparameters and their values
Hyperparameter Value Description iü i Number of training cases over which each stochastic gradient descent (SGD) update minibatch size 32 is computed. replay memory size 1000000 SGD updates are sampled from this number of most recent frames. i, The number of most recent frames experienced by the agent that are given as input to agent history length 4 the Q network. The frequency (measured in the number of parameter updates) with which the target target network update frequency 10000 network is updated (this corresponds to the parameter C from Algorithm 1). discount factor 0.99 Discount factor gamma used in the O-learning update. i Repeat each action selected by the agent this many times. Using a value of 4 results action repeat 4 in the agent seeing only every 4th input frame. The number of actions selected by the agent between successive SGD updates. update frequency 4 Using a value of 4 results in the agent selecting 4 actions between each pair of successive updates. learning rate 0.00025 The learning rate used by RMSProp. gradient momentum 0.95 Gradient momentum used by RMSProp. squared gradient momentum 0.95 Squared gradient (denominator) momentum used by RMSProp. min squared gradient 0.01 Constant added to the squared gradient in the denominator of the RMSProp update. initial exploration 1 Initial value of € in €-greedy exploration. final exploration 0.1 Final value of £ in €-greedy exploration. final exploration frame 4000000 bi umber of frames over which the initial value of € is linearly annealed to its final value. ” A uniform random policy is run for this number of frames before learning starts and the ieplay startsize 80000, resulting experience is used to populate the replay memory. no-op max 30 Maximum number of “do nothing” actions to be performed by the agent at the start of an episode.
The values of all the hyperparameters were selected by performing an informal search on the games Pong, Breakout, Seaquest, Space Invaders and Beam Rider. We did not perform a systematic grid search to the high computational cost, although it is conceivable that even better results could be obtained by systematically tuning the hyperparameter values.
# ©2015 Macmillan Publishers Limited. All rights reserved
owing
LETTER
Extended Data Table 2 | Comparison of games scores obtained by DON agents with methods from the literature12'15 and a professional human games tester
Random Best Linear Contingency Normalized Game Play Learner (SARSA) Human DON Ge ete) (% Human) Alien 227.8 939.2 103.2 6875 3069 (+1093) 42.7% Amidar 5.8 103.4 183.6 1676 739.5 (+3024) 43.9% Assault 222.4 628 537 1496 3359(+775) 246.2% Asterix 210 987.3 1332 8503 6012 (+1744) 70.0% Asteroids 719.1 907.3 89 13157 1629 (4542) 7.3% Atlantis 12850 62687 852.9 29028 85641(+17600) 449.9% Bank Heist 14.2 190.8 67.4 734.4 429.7 (+650) 57.7% Battle Zone 2360 15820 16.2 37800 26300 (+7725) 67.6% Beam Rider 363.9 929.4 1743 5775 6846 (+1619) 119.8% Bowling 23.1 43.9 36.4 154.8 42.4 (+88) 14.7% Boxing 0.1 44 9.8 43 71.8 (48.4) 1707.96 Breakout 17 5.2 6.1 31.8 401.2 (426.9) 1327.26 Centipede 2091 8803 4647 11963 8309(+5237) 63.0% Chopper Command 811 1582 16.9 9882 6687 (+2916) 64.8% Crazy Climber 10781 23411 149.8 35411 114103 (422797) 419.5% Demon Attack 152.1 520.5 o 3401 9711 (+2406) 294.2% Double Dunk -18.6 -13.1 -16 -15.5 -18.1 (42.6) 17.1% Enduro o 129.1 159.4 309.6 301.8 (424.6) 97.5% Fishing Derby -91.7 -89.5 -85.1 5.5 -0.8 (419.0) 93.5% Freeway 0 19.1 19.7 29.6 30.3 (40.7) 102.4% Frostbite 65.2 216.9 180.9 4335 328.3 (250.5) 6.2% Gopher 257.6 1288 2368 2321 8520 (+3279) 400.4% Gravitar 173 387.7 429 2672 306.7 (+223.9) 5.3% H.E.R.O. 1027 6459 7295 25763 19950 (+158) 76.5% Ice Hockey 411.2 -9.5 -3.2 0.9 -1.6 (42.5) 79.3% James Bond 29 202.8 354.1 406.7 576.7 (+175.5) 145.0% Kangaroo 52 1622 8.8 3035 6740 (+2959) 224.2% Krull 1598 3372 3341 2395 3805 (+1033) 277.0% Kung-Fu Master 258.5 19544 29151 22736 23270 (+5955) 102.4% Montezuma's Revenge o 10.7 259 4367 0 (40) 0.0% Ms. Pacman 307.3 1692 1227 15693 2311 (4525) 13.0% Name This Game 2292 2500 2247 4076 7257 (+547) 278.3% Pong -20.7 -19 17.4 93 18.9 (41.3) 132.0% Private Eye 24.9 684.3 86 69571 1788 (+5473) 2.5% Q*Bert 163.9 613.5 960.3 13455 10596 (+3294) 78.5% River Raid 1339 1904 2650 13513 8316 (+1049) 57.3% Road Runner 11.5 67.7 89.1 7845 18257 (+4268) 232.9% Robotank 2.2 28.7 12.4 11.9 51.6 (+4.7) 509.0% Seaquest 68.4 664.8 675.5 20182 5286(+1310) 25.9% Space Invaders 148 250.1 267.9 1652 1976 (+893) 121.5% Star Gunner 664 1070 9.4 10250 57997 (+3152) 598.1%
# DQN
# Tennis Time Pilot
# Tutankham Up and Down Venture Video Pinball
- 23.8
3568 11.4 533.4 o
16257
- 0.1 3741
114.3 3533 66 16871
# o
24.9
98.2 2449
0.6
19761
- 8.9 5925
167.6 9082 1188
17298
- 2.5 (41.9)
5947 (+1600)
186.7 (441.9)
8456 (+3162)
3800 (+238.6)
42684 (+16287)
143.2%
100.9%
112.2%
92.7%
32.0%
2539.4%
# Wizard of Wor
# Zaxxon
563.5 32.5
1981
3365
36.9 21.4
4757
9173
3393 (+2019)
4977 (+1235)
67.5%
54.1%
Best Linear Learner is the best result obtained by a linear function approximator on different types of hand designed features", Contingency (SARSA) agent figures are the results obtained in ref. 15, Note the figures in the last column indicate the performance of DON relative to the human games tester, expressed as a percentage, that is, 100 x (DON score ~ random play score)/(human score ~ random play score)
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
Extended Data Table 3 | The effects of replay and separating the target Q-network
Game With replay, With replay, Without replay, Without replay, with target Q without target Q with target Q without target Q Breakout 316.8 240.7 10.2 3.2 Enduro 1006.3 831.4 141.9 29.1 River Raid 7446.6 4102.8 2867.7 1453.0 Seaquest 2894.4 822.6 1003.0 275.8 Space Invaders 1088.9 826.3 373.2 302.0
DON agents were trained for 10 million frames using standard hyperparameters for all possible combinations of turning replay on or off, using or not using a separate target Q-network, and three different learning rates. Each agent was evaluated every 250,000 training frames for 135,000 validation frames and the highest average episode score is reported. Note that these evaluation episodes were not truncated at 5 min leading to higher scores on Enduro than the ones reported in Extended Data Table 2, Note also that the number of training frames was shorter (10 million frames) as compared to the main results presented in Extended Data Table 2 (50 million frames).
# ©2015 Macmillan Publishers Limited. All rights reserved
Extended Data Table 4 | Comparison of DQN performance with lin- ear function approximator
Game DQN Linear Breakout 316.8 3.00 Enduro 1006.3 62.0 River Raid 7446.6 2346.9 Seaquest 2894.4 656.9 Space Invaders 1088.9 301.3
The performance of the DON agent is compared with the performance of a linear function approximator on the 5 validation games (that is, where a single linear layer was used instead of the convolutional network, in combination with replay and separate target network), Agents were trained for 10 million frames using standard hyperparameters, and three different learning rates. Each agent was evaluated every 250,000 training frames for 135,000 validation frames and the highest average episode score is reported. Note that these evaluation episodes were not truncated at 5 min leading to higher scores on Enduro than the ones reported in Extended Data Table 2, Note also that the number of training frames was shorter (10 million frames) as compared to the main results presented in Extended Data Table 2 (0 million frames)
# ©2015 Macmillan Publishers Limited. All rights reserved
LETTER
